<!DOCTYPE HTML>
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (11.0.8) on Wed Sep 23 10:46:35 CEST 2020 -->
<title>C-Index</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2020-09-23">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery/jquery-ui.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../jquery/jszip/dist/jszip.min.js"></script>
<script type="text/javascript" src="../jquery/jszip-utils/dist/jszip-utils.min.js"></script>
<!--[if IE]>
<script type="text/javascript" src="../jquery/jszip-utils/dist/jszip-utils-ie.min.js"></script>
<![endif]-->
<script type="text/javascript" src="../jquery/jquery-3.3.1.js"></script>
<script type="text/javascript" src="../jquery/jquery-migrate-3.0.1.js"></script>
<script type="text/javascript" src="../jquery/jquery-ui.js"></script>
</head>
<body>
<script type="text/javascript"><!--
    try {
        if (location.href.indexOf('is-external=true') == -1) {
            parent.document.title="C-Index";
        }
    }
    catch(err) {
    }
//-->
var pathtoroot = "../";
var useModuleDirectories = true;
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<header role="banner">
<nav role="navigation">
<div class="fixedNav">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a id="navbar.top">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.top" title="Skip navigation links">Skip navigation links</a></div>
<a id="navbar.top.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../allclasses.html">All&nbsp;Classes</a></li>
</ul>
<ul class="navListSearch">
<li><label for="search">SEARCH:</label>
<input type="text" id="search" value="search" disabled="disabled">
<input type="reset" id="reset" value="reset" disabled="disabled">
</li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
</div>
<a id="skip.navbar.top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
</div>
<div class="navPadding">&nbsp;</div>
<script type="text/javascript"><!--
$('.navPadding').css('padding-top', $('.fixedNav').css("height"));
//-->
</script>
</nav>
</header>
<main role="main">
<div class="contentContainer"><a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Y</a>&nbsp;<a href="index-26.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes</a>&nbsp;<a href="../allpackages-index.html">All&nbsp;Packages</a><a id="I:C">
<!--   -->
</a>
<h2 class="title">C</h2>
<dl>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C">C</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C10">C10</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C11">C11</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C12">C12</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C14">C14</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C15">C15</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C16">C16</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C17">C17</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C1Pr">C1Pr</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C1St">C1St</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C2">C2</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C26">C26</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C27">C27</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C28">C28</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C29">C29</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C2Pr">C2Pr</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C2St">C2St</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C32">C32</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C38">C38</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C39">C39</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C3Pr">C3Pr</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C3St">C3St</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C4">C4</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C4Pr">C4Pr</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C4St">C4St</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C5">C5</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C53">C53</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C54">C54</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C55">C55</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C56">C56</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C5M">C5M</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C5Pr">C5Pr</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C5St">C5St</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C6">C6</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C7">C7</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#C8">C8</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CA">CA</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/kineticlaws/AbstractKineticLaw.html#cachedFeature">cachedFeature</a></span> - Variable in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.kineticlaws.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/kineticlaws/AbstractKineticLaw.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.kineticlaws">AbstractKineticLaw</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#CADMIUM">CADMIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#CALCIUM">CALCIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/geometry/SphereVolumeEstimaton.html#calculate()">calculate()</a></span> - Method in class bio.singa.mathematics.algorithms.geometry.<a href="../bio/singa/mathematics/algorithms/geometry/SphereVolumeEstimaton.html" title="class in bio.singa.mathematics.algorithms.geometry">SphereVolumeEstimaton</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/PixelDiffusivity.html#calculate(javax.measure.Quantity)">calculate(Quantity&lt;Length&gt;)</a></span> - Static method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/PixelDiffusivity.html" title="class in bio.singa.features.quantities">PixelDiffusivity</a></dt>
<dd>
<div class="block">The diffusivity can be calculated according to the Stokes–Einstein equation:
 D = (k_B * T) / (6 * pi * nu * radius)
 k_B is the <a href="../bio/singa/features/quantities/NaturalConstants.html#BOLTZMANN_CONSTANT"><code>NaturalConstants.BOLTZMANN_CONSTANT</code></a> (in (N * m) / K),
 T is the Temperature (in K),
 nu is the dynamic viscosity (in (N * s) / m^2 ) and,</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/layouts/force/UnaryAttractiveForce.html#calculateAcceleration(NodeType)">calculateAcceleration(NodeType)</a></span> - Method in class bio.singa.javafx.renderer.layouts.force.<a href="../bio/singa/javafx/renderer/layouts/force/UnaryAttractiveForce.html" title="class in bio.singa.javafx.renderer.layouts.force">UnaryAttractiveForce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/layouts/force/BinaryAttractiveForce.html#calculateAcceleration(NodeType,NodeType)">calculateAcceleration(NodeType, NodeType)</a></span> - Method in class bio.singa.javafx.renderer.layouts.force.<a href="../bio/singa/javafx/renderer/layouts/force/BinaryAttractiveForce.html" title="class in bio.singa.javafx.renderer.layouts.force">BinaryAttractiveForce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/layouts/force/BinaryRepulsiveForce.html#calculateAcceleration(NodeType,NodeType)">calculateAcceleration(NodeType, NodeType)</a></span> - Method in class bio.singa.javafx.renderer.layouts.force.<a href="../bio/singa/javafx/renderer/layouts/force/BinaryRepulsiveForce.html" title="class in bio.singa.javafx.renderer.layouts.force">BinaryRepulsiveForce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/Geometry.html#calculateArea(javax.measure.Quantity)">calculateArea(Quantity&lt;Length&gt;)</a></span> - Static method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/Geometry.html" title="class in bio.singa.features.quantities">Geometry</a></dt>
<dd>
<div class="block">The area of a sphere is calculated by
 V = 4/3 * pi * radius * radius * radius</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateAtomDistanceMatrix(bio.singa.structure.model.interfaces.AtomContainer)">calculateAtomDistanceMatrix(AtomContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the distance matrix of the given <a href="../bio/singa/structure/model/interfaces/Atom.html" title="interface in bio.singa.structure.model.interfaces"><code>Atom</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateAtomDistanceMatrix(java.util.List)">calculateAtomDistanceMatrix(List&lt;Atom&gt;)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the distance matrix of the given <a href="../bio/singa/structure/model/interfaces/Atom.html" title="interface in bio.singa.structure.model.interfaces"><code>Atom</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/bodies/Spheres.html#calculateCenteredSphereTriangleSurface(bio.singa.mathematics.geometry.bodies.Sphere,bio.singa.mathematics.vectors.Vector3D,bio.singa.mathematics.vectors.Vector3D,bio.singa.mathematics.vectors.Vector3D)">calculateCenteredSphereTriangleSurface(Sphere, Vector3D, Vector3D, Vector3D)</a></span> - Static method in class bio.singa.mathematics.geometry.bodies.<a href="../bio/singa/mathematics/geometry/bodies/Spheres.html" title="class in bio.singa.mathematics.geometry.bodies">Spheres</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateClosestDistance(java.util.List,SubType)">calculateClosestDistance(List&lt;SubType&gt;, SubType)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>
<div class="block">Returns the closest element in the given target list to the reference vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/Matrices.html#calculateCovarianceMatrix(bio.singa.mathematics.matrices.Matrix,bio.singa.mathematics.matrices.Matrix)">calculateCovarianceMatrix(Matrix, Matrix)</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/Matrices.html" title="class in bio.singa.mathematics.matrices">Matrices</a></dt>
<dd>
<div class="block">calculates the covariance matrix between to matrices in respect to matrix A
 cov(A) = B'*A</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/EndocytosisActinBoost.html#calculateDisplacement(bio.singa.simulation.model.agents.pointlike.Vesicle)">calculateDisplacement(Vesicle)</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/EndocytosisActinBoost.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">EndocytosisActinBoost</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleConfinedDiffusion.html#calculateDisplacement(bio.singa.simulation.model.agents.pointlike.Vesicle)">calculateDisplacement(Vesicle)</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleConfinedDiffusion.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">VesicleConfinedDiffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleCytoplasmDiffusion.html#calculateDisplacement(bio.singa.simulation.model.agents.pointlike.Vesicle)">calculateDisplacement(Vesicle)</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleCytoplasmDiffusion.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">VesicleCytoplasmDiffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleTransport.html#calculateDisplacement(bio.singa.simulation.model.agents.pointlike.Vesicle)">calculateDisplacement(Vesicle)</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleTransport.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">VesicleTransport</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/ShortestPathMetric.html#calculateDistance(bio.singa.mathematics.graphs.model.Node,bio.singa.mathematics.graphs.model.Node)">calculateDistance(Node&lt;?, ?, ?&gt;, Node&lt;?, ?, ?&gt;)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/ShortestPathMetric.html" title="class in bio.singa.mathematics.metrics.implementations">ShortestPathMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/OverlapDistance.html#calculateDistance(bio.singa.mathematics.intervals.IntegerInterval,bio.singa.mathematics.intervals.IntegerInterval)">calculateDistance(IntegerInterval, IntegerInterval)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/OverlapDistance.html" title="class in bio.singa.mathematics.metrics.implementations">OverlapDistance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/TanimotoCoefficient.html#calculateDistance(BitVectorType,BitVectorType)">calculateDistance(BitVectorType, BitVectorType)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/TanimotoCoefficient.html" title="class in bio.singa.mathematics.metrics.implementations">TanimotoCoefficient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/JaccardMetric.html#calculateDistance(java.util.Collection,java.util.Collection)">calculateDistance(Collection&lt;Type&gt;, Collection&lt;Type&gt;)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/JaccardMetric.html" title="class in bio.singa.mathematics.metrics.implementations">JaccardMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateDistance(MetrizableType,MetrizableType)">calculateDistance(MetrizableType, MetrizableType)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>
<div class="block">Calculates the distance between both Objects.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/AngularDistance.html#calculateDistance(VectorDimension,VectorDimension)">calculateDistance(VectorDimension, VectorDimension)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/AngularDistance.html" title="class in bio.singa.mathematics.metrics.implementations">AngularDistance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html#calculateDistance(VectorDimension,VectorDimension)">calculateDistance(VectorDimension, VectorDimension)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html" title="class in bio.singa.mathematics.metrics.implementations">CosineSimilarity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/EuclideanDistance.html#calculateDistance(VectorType,VectorType)">calculateDistance(VectorType, VectorType)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/EuclideanDistance.html" title="class in bio.singa.mathematics.metrics.implementations">EuclideanDistance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/MinkowskiMetric.html#calculateDistance(VectorType,VectorType)">calculateDistance(VectorType, VectorType)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/MinkowskiMetric.html" title="class in bio.singa.mathematics.metrics.implementations">MinkowskiMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/SquaredEuclideanDistance.html#calculateDistance(VectorType,VectorType)">calculateDistance(VectorType, VectorType)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/SquaredEuclideanDistance.html" title="class in bio.singa.mathematics.metrics.implementations">SquaredEuclideanDistance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateDistanceMatrix(bio.singa.structure.model.interfaces.LeafSubstructureContainer)">calculateDistanceMatrix(LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the distance matrix of the given <a href="../bio/singa/structure/model/interfaces/LeafSubstructureContainer.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructureContainer</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateDistancesPairwise(java.util.List,java.util.function.Function)">calculateDistancesPairwise(List&lt;LabelType&gt;, Function&lt;LabelType, SubType&gt;)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateDistancesPairwise(java.util.List)">calculateDistancesPairwise(List&lt;SubType&gt;)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>
<div class="block">Calculates the pairwise distance of all elements in the given list.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateDistancesPairwise(java.util.List,java.util.List)">calculateDistancesPairwise(List&lt;SubType&gt;, List&lt;SubType&gt;)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>
<div class="block">Computes pairwise distances for two given lists, resulting in a rectangular, non-symmetric matric.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateDistancesToReference(java.util.List,SubType)">calculateDistancesToReference(List&lt;SubType&gt;, SubType)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>
<div class="block">Calculates the distance for each Vector in the given target list to the reference vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateExtent(bio.singa.structure.model.interfaces.LeafSubstructureContainer)">calculateExtent(LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the maximal extent of the given <a href="../bio/singa/structure/model/interfaces/LeafSubstructureContainer.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructureContainer</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/geometry/ConvexHull.html#calculateHullFor(java.util.Collection)">calculateHullFor(Collection&lt;Vector2D&gt;)</a></span> - Static method in class bio.singa.mathematics.algorithms.geometry.<a href="../bio/singa/mathematics/algorithms/geometry/ConvexHull.html" title="class in bio.singa.mathematics.algorithms.geometry">ConvexHull</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition(bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.model.interfaces.LeafSubstructureContainer)">calculateIdealSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition(bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme)">calculateIdealSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, RepresentationScheme)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition(bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.model.interfaces.LeafSubstructureContainer,java.util.function.Predicate)">calculateIdealSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, Predicate&lt;Atom&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition(java.util.List,java.util.List)">calculateIdealSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition(java.util.List,java.util.List,bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme)">calculateIdealSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, RepresentationScheme)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition(java.util.List,java.util.List,java.util.function.Predicate)">calculateIdealSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, Predicate&lt;Atom&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#calculateIdealVectorSuperimposition(java.util.List,java.util.List)">calculateIdealVectorSuperimposition(List&lt;Vector&gt;, List&lt;Vector&gt;)</a></span> - Static method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition(bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme,bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix,boolean)">calculateKuhnMunkresSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, RepresentationScheme, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition(bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix,boolean)">calculateKuhnMunkresSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition(bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.model.interfaces.LeafSubstructureContainer,java.util.function.Predicate,bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix,boolean)">calculateKuhnMunkresSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, Predicate&lt;Atom&gt;, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition(java.util.List,java.util.List,bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme,bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix,boolean)">calculateKuhnMunkresSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, RepresentationScheme, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition(java.util.List,java.util.List,bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix,boolean)">calculateKuhnMunkresSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition(java.util.List,java.util.List,java.util.function.Predicate,bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix,boolean)">calculateKuhnMunkresSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, Predicate&lt;Atom&gt;, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#calculateKuhnMunkresSuperimposition(java.util.List,java.util.List)">calculateKuhnMunkresSuperimposition(List&lt;Vector&gt;, List&lt;Vector&gt;)</a></span> - Static method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>
<div class="block">Finds the superimposition with the minimal distances between pairs of <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors"><code>Vector</code></a>s by using <a href="../bio/singa/mathematics/algorithms/optimization/KuhnMunkres.html" title="class in bio.singa.mathematics.algorithms.optimization"><code>KuhnMunkres</code></a> optimization.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/FofanovEstimation.html#calculatePvalues(java.util.List)">calculatePvalues(List&lt;Fit3DMatch&gt;)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.statistics.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/FofanovEstimation.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.statistics">FofanovEstimation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/StarkEstimation.html#calculatePvalues(java.util.List)">calculatePvalues(List&lt;Fit3DMatch&gt;)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.statistics.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/StarkEstimation.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.statistics">StarkEstimation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/StatisticalModel.html#calculatePvalues(java.util.List)">calculatePvalues(List&lt;Fit3DMatch&gt;)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.fit3d.statistics.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/StatisticalModel.html" title="interface in bio.singa.structure.algorithms.superimposition.fit3d.statistics">StatisticalModel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/matrix/QRDecomposition.html#calculateQRDecomposition(bio.singa.mathematics.matrices.Matrix)">calculateQRDecomposition(Matrix)</a></span> - Static method in class bio.singa.mathematics.algorithms.matrix.<a href="../bio/singa/mathematics/algorithms/matrix/QRDecomposition.html" title="class in bio.singa.mathematics.algorithms.matrix">QRDecomposition</a></dt>
<dd>
<div class="block">Calculates the QR decomposition of the given matrix, using the modified Graham-Schmidt algorithm.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vectors3D.html#calculateReflection(bio.singa.mathematics.vectors.Vector3D,bio.singa.mathematics.vectors.Vector3D)">calculateReflection(Vector3D, Vector3D)</a></span> - Static method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vectors3D.html" title="class in bio.singa.mathematics.vectors">Vectors3D</a></dt>
<dd>
<div class="block">Calculate the reflection of a vector given a mirror vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotifs.html#calculateRmsdMatrix(java.util.List,boolean)">calculateRmsdMatrix(List&lt;StructuralMotif&gt;, boolean)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotifs.html" title="class in bio.singa.structure.model.oak">StructuralMotifs</a></dt>
<dd>
<div class="block">Performs a superimposition of given <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a>s using arbitrary atoms and returns the RMSD distance
 matrix between all elements.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotifs.html#calculateRmsdMatrix(java.util.List,java.util.function.Predicate,boolean)">calculateRmsdMatrix(List&lt;StructuralMotif&gt;, Predicate&lt;Atom&gt;, boolean)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotifs.html" title="class in bio.singa.structure.model.oak">StructuralMotifs</a></dt>
<dd>
<div class="block">Performs a superimposition of given <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a>s using the specified <a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak"><code>StructuralEntityFilter.AtomFilter</code></a> and returns the RMSD distance matrix between all elements.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vectors3D.html#calculateRotation(bio.singa.mathematics.vectors.Vector3D,bio.singa.mathematics.vectors.Vector3D)">calculateRotation(Vector3D, Vector3D)</a></span> - Static method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vectors3D.html" title="class in bio.singa.mathematics.vectors">Vectors3D</a></dt>
<dd>
<div class="block">Calculates the rotation necessary to rotate the first vector onto the second, no scaling is done.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/matrix/LUDecomposition.html#calculateRowEchelonMatrix(bio.singa.mathematics.matrices.Matrix)">calculateRowEchelonMatrix(Matrix)</a></span> - Static method in class bio.singa.mathematics.algorithms.matrix.<a href="../bio/singa/mathematics/algorithms/matrix/LUDecomposition.html" title="class in bio.singa.mathematics.algorithms.matrix">LUDecomposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/Line.html#calculateSlope(bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D)">calculateSlope(Vector2D, Vector2D)</a></span> - Static method in class bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/Line.html" title="class in bio.singa.mathematics.geometry.edges">Line</a></dt>
<dd>
<div class="block">Gets the slope or gradient of the line, calculated by</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/bodies/Spheres.html#calculateSphereSlice(bio.singa.mathematics.vectors.Vector2D,double,bio.singa.mathematics.geometry.edges.LineSegment)">calculateSphereSlice(Vector2D, double, LineSegment)</a></span> - Static method in class bio.singa.mathematics.geometry.bodies.<a href="../bio/singa/mathematics/geometry/bodies/Spheres.html" title="class in bio.singa.mathematics.geometry.bodies">Spheres</a></dt>
<dd>
<div class="block">Very specific method to slice sphere with an edge that represents a flat surface.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/bodies/Spheres.html#calculateSphereSlice(bio.singa.mathematics.vectors.Vector2D,double,bio.singa.mathematics.vectors.Vector2D)">calculateSphereSlice(Vector2D, double, Vector2D)</a></span> - Static method in class bio.singa.mathematics.geometry.bodies.<a href="../bio/singa/mathematics/geometry/bodies/Spheres.html" title="class in bio.singa.mathematics.geometry.bodies">Spheres</a></dt>
<dd>
<div class="block">Vey specific method to calculate the surface sliced by a horizontal beam from a sphere with z coordinate 0.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/bodies/Spheres.html#calculateSphereTriangleSurface(bio.singa.mathematics.geometry.bodies.Sphere,bio.singa.mathematics.vectors.Vector3D,bio.singa.mathematics.vectors.Vector3D,bio.singa.mathematics.vectors.Vector3D)">calculateSphereTriangleSurface(Sphere, Vector3D, Vector3D, Vector3D)</a></span> - Static method in class bio.singa.mathematics.geometry.bodies.<a href="../bio/singa/mathematics/geometry/bodies/Spheres.html" title="class in bio.singa.mathematics.geometry.bodies">Spheres</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateSquaredDistanceMatrix(bio.singa.structure.model.interfaces.LeafSubstructureContainer)">calculateSquaredDistanceMatrix(LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the squared distance matrix of the given <a href="../bio/singa/structure/model/interfaces/LeafSubstructureContainer.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructureContainer</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateSquaredExtent(bio.singa.structure.model.interfaces.LeafSubstructureContainer)">calculateSquaredExtent(LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the maximal squared extent of the given <a href="../bio/singa/structure/model/interfaces/LeafSubstructureContainer.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructureContainer</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition(bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.model.interfaces.LeafSubstructureContainer)">calculateSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition(bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme)">calculateSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, RepresentationScheme)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition(bio.singa.structure.model.interfaces.LeafSubstructureContainer,bio.singa.structure.model.interfaces.LeafSubstructureContainer,java.util.function.Predicate)">calculateSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, Predicate&lt;Atom&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/FragmentSubstructureSuperimposer.html#calculateSubstructureSuperimposition(java.util.List,java.util.List)">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/FragmentSubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">FragmentSubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/MaximumCommonSubgraphSuperimposer.html#calculateSubstructureSuperimposition(java.util.List,java.util.List)">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/MaximumCommonSubgraphSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">MaximumCommonSubgraphSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition(java.util.List,java.util.List)">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition(java.util.List,java.util.List,bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme)">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, RepresentationScheme)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/FragmentSubstructureSuperimposer.html#calculateSubstructureSuperimposition(java.util.List,java.util.List,java.util.function.BiFunction,java.util.function.BiFunction)">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, BiFunction&lt;MoleculeAtom, MoleculeAtom, Boolean&gt;, BiFunction&lt;MoleculeBond, MoleculeBond, Boolean&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/FragmentSubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">FragmentSubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/MaximumCommonSubgraphSuperimposer.html#calculateSubstructureSuperimposition(java.util.List,java.util.List,java.util.function.BiFunction,java.util.function.BiFunction)">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, BiFunction&lt;MoleculeAtom, MoleculeAtom, Boolean&gt;, BiFunction&lt;MoleculeBond, MoleculeBond, Boolean&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/MaximumCommonSubgraphSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">MaximumCommonSubgraphSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition(java.util.List,java.util.List,java.util.function.Predicate)">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, Predicate&lt;Atom&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html#calculateSuperimposition()">calculateSuperimposition()</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorQuaternionSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#calculateSuperimposition()">calculateSuperimposition()</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSuperimposition()">calculateSuperimposition()</a></span> - Method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>
<div class="block">Finds the superimposition for a list of <a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructure</code></a> according to their input order</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/bodies/Spheres.html#calculateSurface(double)">calculateSurface(double)</a></span> - Static method in class bio.singa.mathematics.geometry.bodies.<a href="../bio/singa/mathematics/geometry/bodies/Spheres.html" title="class in bio.singa.mathematics.geometry.bodies">Spheres</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateTorsionAngle(bio.singa.structure.model.interfaces.Atom,bio.singa.structure.model.interfaces.Atom,bio.singa.structure.model.interfaces.Atom,bio.singa.structure.model.interfaces.Atom)">calculateTorsionAngle(Atom, Atom, Atom, Atom)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Calculated the torsion angle between the given <a href="../bio/singa/structure/model/interfaces/Atom.html" title="interface in bio.singa.structure.model.interfaces"><code>Atom</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html#calculateTotalDisplacement()">calculateTotalDisplacement()</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html" title="class in bio.singa.simulation.model.agents.pointlike">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/DisplacementDeltaManager.html#calculateTotalDisplacement()">calculateTotalDisplacement()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/DisplacementDeltaManager.html" title="class in bio.singa.simulation.model.modules.displacement">DisplacementDeltaManager</a></dt>
<dd>
<div class="block">Calculates the total displacement resulting from the potential deltas and sets the result as the next position.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html#calculateUpdates()">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.<a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html" title="class in bio.singa.simulation.model.modules">AbstractUpdateModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html#calculateUpdates()">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationBasedModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/DisplacementBasedModule.html#calculateUpdates()">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/DisplacementBasedModule.html" title="class in bio.singa.simulation.model.modules.displacement">DisplacementBasedModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/ClathrinMediatedEndocytosis.html#calculateUpdates()">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/ClathrinMediatedEndocytosis.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">ClathrinMediatedEndocytosis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/ConcentrationApplier.html#calculateUpdates()">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/ConcentrationApplier.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">ConcentrationApplier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/ConcentrationStateChange.html#calculateUpdates()">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/ConcentrationStateChange.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">ConcentrationStateChange</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/DiffusivityScaling.html#calculateUpdates()">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/DiffusivityScaling.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">DiffusivityScaling</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/LineLikeAgentAttachment.html#calculateUpdates()">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/LineLikeAgentAttachment.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">LineLikeAgentAttachment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/VesicleFusion.html#calculateUpdates()">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/VesicleFusion.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">VesicleFusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/VolumeLikeAgentContainment.html#calculateUpdates()">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/VolumeLikeAgentContainment.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">VolumeLikeAgentContainment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#calculateVectorSuperimposition(java.util.List,java.util.List)">calculateVectorSuperimposition(List&lt;Vector&gt;, List&lt;Vector&gt;)</a></span> - Static method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html#calculateVectorSuperimposition(java.util.List,java.util.List)">calculateVectorSuperimposition(List&lt;Vector3D&gt;, List&lt;Vector3D&gt;)</a></span> - Static method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorQuaternionSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/Geometry.html#calculateVolume(javax.measure.Quantity)">calculateVolume(Quantity&lt;Length&gt;)</a></span> - Static method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/Geometry.html" title="class in bio.singa.features.quantities">Geometry</a></dt>
<dd>
<div class="block">The volume of a sphere is calculated by
 V = 4/3 * pi * radius * radius * radius</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/Line.html#calculateYIntercept(bio.singa.mathematics.vectors.Vector2D,double)">calculateYIntercept(Vector2D, double)</a></span> - Static method in class bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/Line.html" title="class in bio.singa.mathematics.geometry.edges">Line</a></dt>
<dd>
<div class="block">Gets the y-intercept of the line, calculated by</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#CALIFORNIUM">CALIFORNIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/Featureable.html#canBeFeaturedWith(java.lang.Class)">canBeFeaturedWith(Class&lt;FeatureType&gt;)</a></span> - Method in interface bio.singa.features.model.<a href="../bio/singa/features/model/Featureable.html" title="interface in bio.singa.features.model">Featureable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#candidate">candidate</a></span> - Variable in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/reactions/conditions/CandidateCondition.html" title="interface in bio.singa.simulation.reactions.conditions"><span class="typeNameLink">CandidateCondition</span></a> - Interface in <a href="../bio/singa/simulation/reactions/conditions/package-summary.html">bio.singa.simulation.reactions.conditions</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/reactions/conditions/CandidateConditionBuilder.html" title="class in bio.singa.simulation.reactions.conditions"><span class="typeNameLink">CandidateConditionBuilder</span></a> - Class in <a href="../bio/singa/simulation/reactions/conditions/package-summary.html">bio.singa.simulation.reactions.conditions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/CandidateConditionBuilder.html#%3Cinit%3E()">CandidateConditionBuilder()</a></span> - Constructor for class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/CandidateConditionBuilder.html" title="class in bio.singa.simulation.reactions.conditions">CandidateConditionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilterType.html#CARBON">CARBON</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilterType.html" title="enum in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilterType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#CARBON">CARBON</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/MaterialProvider.html#CARBON">CARBON</a></span> - Static variable in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/MaterialProvider.html" title="class in bio.singa.javafx.viewer">MaterialProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/MolSvgWriter.html#CARBON_COLOR">CARBON_COLOR</a></span> - Static variable in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/MolSvgWriter.html" title="class in bio.singa.javafx.viewer">MolSvgWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/MembraneDiffusion.CargoStep.html#cargo(bio.singa.simulation.entities.ChemicalEntity)">cargo(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/MembraneDiffusion.CargoStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">MembraneDiffusion.CargoStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/MembraneDiffusion.MembraneDiffusionBuilder.html#cargo(bio.singa.simulation.entities.ChemicalEntity)">cargo(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/MembraneDiffusion.MembraneDiffusionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">MembraneDiffusion.MembraneDiffusionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/SingleFileChannelMembraneTransport.CargoStep.html#cargo(bio.singa.simulation.entities.ChemicalEntity)">cargo(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/SingleFileChannelMembraneTransport.CargoStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">SingleFileChannelMembraneTransport.CargoStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/SingleFileChannelMembraneTransport.SingleFileChannelMembraneTransportBuilder.html#cargo(bio.singa.simulation.entities.ChemicalEntity)">cargo(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/SingleFileChannelMembraneTransport.SingleFileChannelMembraneTransportBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">SingleFileChannelMembraneTransport.SingleFileChannelMembraneTransportBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.FeatureStep.html#cargo(bio.singa.simulation.features.Cargo)">cargo(Cargo)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.FeatureStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">PorousDiffusion.FeatureStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.PorousDiffusionBuilder.html#cargo(bio.singa.simulation.features.Cargo)">cargo(Cargo)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.PorousDiffusionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">PorousDiffusion.PorousDiffusionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html#cargo(bio.singa.simulation.features.Cargoes)">cargo(Cargoes)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EntityStep.html#cargo(bio.singa.simulation.features.Cargoes)">cargo(Cargoes)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EntityStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.EntityStep</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/Cargo.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">Cargo</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/Cargo.html#%3Cinit%3E(bio.singa.simulation.entities.ChemicalEntity)">Cargo(ChemicalEntity)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/Cargo.html" title="class in bio.singa.simulation.features">Cargo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/Cargo.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">Cargo.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html#cargoAdditionRate(bio.singa.simulation.features.CargoAdditionRate)">cargoAdditionRate(CargoAdditionRate)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.RateStep.html#cargoAdditionRate(bio.singa.simulation.features.CargoAdditionRate)">cargoAdditionRate(CargoAdditionRate)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.RateStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.RateStep</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/CargoAdditionRate.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">CargoAdditionRate</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/CargoAdditionRate.html#%3Cinit%3E(javax.measure.Quantity)">CargoAdditionRate(Quantity&lt;FirstOrderRate&gt;)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/CargoAdditionRate.html" title="class in bio.singa.simulation.features">CargoAdditionRate</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/CargoAdditionRate.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">CargoAdditionRate.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/Cargoes.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">Cargoes</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/Cargoes.html#%3Cinit%3E(java.util.List)">Cargoes(List&lt;ChemicalEntity&gt;)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/Cargoes.html" title="class in bio.singa.simulation.features">Cargoes</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/Cargoes.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">Cargoes.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/ReactantRole.html#CATALYTIC">CATALYTIC</a></span> - bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/ReactantRole.html" title="enum in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">ReactantRole</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CB">CB</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CD">CD</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CD1">CD1</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CD2">CD2</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CE">CE</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CE1">CE1</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CE2">CE2</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CE3">CE3</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/organelles/OrganelleTypes.html#CELL">CELL</a></span> - bio.singa.simulation.model.agents.organelles.<a href="../bio/singa/simulation/model/agents/organelles/OrganelleTypes.html" title="enum in bio.singa.simulation.model.agents.organelles">OrganelleTypes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegions.html#CELL_CORTEX">CELL_CORTEX</a></span> - Static variable in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegions.html" title="class in bio.singa.simulation.model.sections">CellRegions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegions.html#CELL_INNER_MEMBRANE_REGION">CELL_INNER_MEMBRANE_REGION</a></span> - Static variable in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegions.html" title="class in bio.singa.simulation.model.sections">CellRegions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellSubsections.html#CELL_OUTER_MEMBRANE">CELL_OUTER_MEMBRANE</a></span> - Static variable in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellSubsections.html" title="class in bio.singa.simulation.model.sections">CellSubsections</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegions.html#CELL_OUTER_MEMBRANE_REGION">CELL_OUTER_MEMBRANE_REGION</a></span> - Static variable in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegions.html" title="class in bio.singa.simulation.model.sections">CellRegions</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/sections/CellRegion.html" title="class in bio.singa.simulation.model.sections"><span class="typeNameLink">CellRegion</span></a> - Class in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegion.html#%3Cinit%3E(java.lang.String)">CellRegion(String)</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegion.html" title="class in bio.singa.simulation.model.sections">CellRegion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegion.html#%3Cinit%3E(java.lang.String,bio.singa.features.identifiers.GoTerm)">CellRegion(String, GoTerm)</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegion.html" title="class in bio.singa.simulation.model.sections">CellRegion</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/sections/CellRegions.html" title="class in bio.singa.simulation.model.sections"><span class="typeNameLink">CellRegions</span></a> - Class in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegions.html#%3Cinit%3E()">CellRegions()</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegions.html" title="class in bio.singa.simulation.model.sections">CellRegions</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections"><span class="typeNameLink">CellSubsection</span></a> - Class in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>
<div class="block">A cell subsection organizes the contents of a <a href="../bio/singa/simulation/model/simulation/Updatable.html" title="interface in bio.singa.simulation.model.simulation"><code>Updatable</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellSubsection.html#%3Cinit%3E(java.lang.String)">CellSubsection(String)</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections">CellSubsection</a></dt>
<dd>
<div class="block">Creates a new cell subsection with the given identifier and dynamic preferred concentration unit.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellSubsection.html#%3Cinit%3E(java.lang.String,bio.singa.features.identifiers.GoTerm)">CellSubsection(String, GoTerm)</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections">CellSubsection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellSubsection.html#%3Cinit%3E(java.lang.String,bio.singa.features.identifiers.GoTerm,boolean)">CellSubsection(String, GoTerm, boolean)</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections">CellSubsection</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/sections/CellSubsections.html" title="class in bio.singa.simulation.model.sections"><span class="typeNameLink">CellSubsections</span></a> - Class in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellSubsections.html#%3Cinit%3E()">CellSubsections()</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellSubsections.html" title="class in bio.singa.simulation.model.sections">CellSubsections</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/sections/CellTopology.html" title="enum in bio.singa.simulation.model.sections"><span class="typeNameLink">CellTopology</span></a> - Enum in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>
<div class="block">The cell topology defines how the subsections are spatially organized.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html#center()">center()</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorQuaternionSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#center()">center()</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphAligner.html#centerGraph()">centerGraph()</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphAligner.html" title="class in bio.singa.javafx.renderer.graphs">GraphAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html#centerGraph(GraphType)">centerGraph(GraphType)</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html" title="class in bio.singa.javafx.renderer.graphs">GraphRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html#centerGraph(GraphType,java.util.function.Predicate)">centerGraph(GraphType, Predicate&lt;NodeType&gt;)</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html" title="class in bio.singa.javafx.renderer.graphs">GraphRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html#centerOnce(GraphType)">centerOnce(GraphType)</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html" title="class in bio.singa.javafx.renderer.graphs">GraphRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html#centerOnce(GraphType,java.util.function.Predicate)">centerOnce(GraphType, Predicate&lt;NodeType&gt;)</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html" title="class in bio.singa.javafx.renderer.graphs">GraphRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html#CENTIMETRE">CENTIMETRE</a></span> - Static variable in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html" title="class in bio.singa.chemistry.features.permeability">MembranePermeability</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html#CENTIMETRE_PER_SECOND">CENTIMETRE_PER_SECOND</a></span> - Static variable in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html" title="class in bio.singa.chemistry.features.permeability">MembranePermeability</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Circles.html#centralAngle(bio.singa.mathematics.geometry.faces.Circle,bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D)">centralAngle(Circle, Vector2D, Vector2D)</a></span> - Static method in class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Circles.html" title="class in bio.singa.mathematics.geometry.faces">Circles</a></dt>
<dd>
<div class="block">Returns the angle between two vectors that are projected on the circumference of the circle.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationSchemeType.html#CENTROID">CENTROID</a></span> - bio.singa.structure.algorithms.superimposition.fit3d.representations.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationSchemeType.html" title="enum in bio.singa.structure.algorithms.superimposition.fit3d.representations">RepresentationSchemeType</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/CentroidRepresentationScheme.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.representations"><span class="typeNameLink">CentroidRepresentationScheme</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/package-summary.html">bio.singa.structure.algorithms.superimposition.fit3d.representations</a></dt>
<dd>
<div class="block">An implementation to represent a given <a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructure</code></a> by its centroid <b>EXCLUDING</b> hydrogen.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/CentroidRepresentationScheme.html#%3Cinit%3E()">CentroidRepresentationScheme()</a></span> - Constructor for class bio.singa.structure.algorithms.superimposition.fit3d.representations.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/CentroidRepresentationScheme.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.representations">CentroidRepresentationScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#CERIUM">CERIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#CESIUM">CESIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CG">CG</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CG1">CG1</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CG2">CG2</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CH2">CH2</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.MultiChainStep.html#chain(java.lang.String)">chain(String)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.MultiChainStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.MultiChainStep</a></dt>
<dd>
<div class="block">If only a single chain should be parsed, choose it here.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.ChainStep.html#chain(java.lang.String)">chain(String)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.ChainStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureSelector.ChainStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.Selector.html#chain(java.lang.String)">chain(String)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.Selector.html" title="class in bio.singa.structure.parser.pdb.structures">StructureSelector.Selector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/model/interfaces/Chain.html" title="interface in bio.singa.structure.model.interfaces"><span class="typeNameLink">Chain</span></a> - Interface in <a href="../bio/singa/structure/model/interfaces/package-summary.html">bio.singa.structure.model.interfaces</a></dt>
<dd>
<div class="block"><a href="../bio/singa/structure/model/interfaces/Chain.html" title="interface in bio.singa.structure.model.interfaces"><code>Chain</code></a>s that represent one continuous macro molecule (most often the biggest macro molecules and its ligands
 are collected in one model).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html#CHAIN_IDENTIFIER">CHAIN_IDENTIFIER</a></span> - bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">AtomToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/ChainTerminatorToken.html#CHAIN_IDENTIFIER">CHAIN_IDENTIFIER</a></span> - bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/ChainTerminatorToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">ChainTerminatorToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/tokens/PfamToken.PDBToken.html#CHAIN_IDENTIFIER">CHAIN_IDENTIFIER</a></span> - bio.singa.structure.parser.pfam.tokens.<a href="../bio/singa/structure/parser/pfam/tokens/PfamToken.PDBToken.html" title="enum in bio.singa.structure.parser.pfam.tokens">PfamToken.PDBToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/TerminatorTokens.html#CHAIN_TERMINATOR">CHAIN_TERMINATOR</a></span> - Static variable in class bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/TerminatorTokens.html" title="class in bio.singa.structure.parser.pdb.structures.tokens">TerminatorTokens</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/model/interfaces/ChainContainer.html" title="interface in bio.singa.structure.model.interfaces"><span class="typeNameLink">ChainContainer</span></a> - Interface in <a href="../bio/singa/structure/model/interfaces/package-summary.html">bio.singa.structure.model.interfaces</a></dt>
<dd>
<div class="block">Everything that contains <a href="../bio/singa/structure/model/interfaces/Chain.html" title="interface in bio.singa.structure.model.interfaces"><code>Chain</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.ChainFilter.html#%3Cinit%3E()">ChainFilter()</a></span> - Constructor for class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.ChainFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.ChainFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.SingleChainStep.html#chainIdentifier(java.lang.String)">chainIdentifier(String)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.SingleChainStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.SingleChainStep</a></dt>
<dd>
<div class="block">If only a single chain should be parsed, choose it here.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.IdentifierStep.html#chainList(java.nio.file.Path)">chainList(Path)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.IdentifierStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.IdentifierStep</a></dt>
<dd>
<div class="block">Reads the provided chainIdentifier list from a file.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html#chainList(java.nio.file.Path)">chainList(Path)</a></span> - Method in class bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html" title="class in bio.singa.structure.parser.pfam">PfamParser.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.ChainStep.html#chainList(java.nio.file.Path)">chainList(Path)</a></span> - Method in interface bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.ChainStep.html" title="interface in bio.singa.structure.parser.pfam">PfamParser.ChainStep</a></dt>
<dd>
<div class="block">Parse all chains annotated in Pfam and listed in the given chain list.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.IdentifierStep.html#chainList(java.nio.file.Path,java.lang.String)">chainList(Path, String)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.IdentifierStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.IdentifierStep</a></dt>
<dd>
<div class="block">Reads the provided chainIdentifier list from a file.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html#chainList(java.nio.file.Path,java.lang.String)">chainList(Path, String)</a></span> - Method in class bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html" title="class in bio.singa.structure.parser.pfam">PfamParser.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.ChainStep.html#chainList(java.nio.file.Path,java.lang.String)">chainList(Path, String)</a></span> - Method in interface bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.ChainStep.html" title="interface in bio.singa.structure.parser.pfam">PfamParser.ChainStep</a></dt>
<dd>
<div class="block">Parse all chains annotated in Pfam and listed in the given chain list with the custom separator.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html#chains()">chains()</a></span> - Method in class bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html" title="class in bio.singa.structure.parser.pfam">PfamParser.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.DomainStep.html#chains()">chains()</a></span> - Method in interface bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.DomainStep.html" title="interface in bio.singa.structure.parser.pfam">PfamParser.DomainStep</a></dt>
<dd>
<div class="block">Parse all chains carrying one Pfam domain with the desired Pfam-ID.</div>
</dd>
<dt><a href="../bio/singa/structure/parser/pdb/structures/tokens/ChainTerminatorToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens"><span class="typeNameLink">ChainTerminatorToken</span></a> - Enum in <a href="../bio/singa/structure/parser/pdb/structures/tokens/package-summary.html">bio.singa.structure.parser.pdb.structures.tokens</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html#characters(char%5B%5D,int,int)">characters(char[], int, int)</a></span> - Method in class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/uniprot/UniProtContentHandler.html#characters(char%5B%5D,int,int)">characters(char[], int, int)</a></span> - Method in class bio.singa.chemistry.features.databases.uniprot.<a href="../bio/singa/chemistry/features/databases/uniprot/UniProtContentHandler.html" title="class in bio.singa.chemistry.features.databases.uniprot">UniProtContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/parser/GraphMLContentHandler.html#characters(char%5B%5D,int,int)">characters(char[], int, int)</a></span> - Method in class bio.singa.mathematics.graphs.parser.<a href="../bio/singa/mathematics/graphs/parser/GraphMLContentHandler.html" title="class in bio.singa.mathematics.graphs.parser">GraphMLContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/parser/ena/ENAContentHandler.html#characters(char%5B%5D,int,int)">characters(char[], int, int)</a></span> - Method in class bio.singa.sequence.parser.ena.<a href="../bio/singa/sequence/parser/ena/ENAContentHandler.html" title="class in bio.singa.sequence.parser.ena">ENAContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/PlipContentHandler.html#characters(char%5B%5D,int,int)">characters(char[], int, int)</a></span> - Method in class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/PlipContentHandler.html" title="class in bio.singa.structure.parser.plip">PlipContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/sifts/ResidueMapContentHandler.html#characters(char%5B%5D,int,int)">characters(char[], int, int)</a></span> - Method in class bio.singa.structure.parser.sifts.<a href="../bio/singa/structure/parser/sifts/ResidueMapContentHandler.html" title="class in bio.singa.structure.parser.sifts">ResidueMapContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html" title="class in bio.singa.chemistry.features.databases.chebi"><span class="typeNameLink">ChEBIContentHandler</span></a> - Class in <a href="../bio/singa/chemistry/features/databases/chebi/package-summary.html">bio.singa.chemistry.features.databases.chebi</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html#%3Cinit%3E()">ChEBIContentHandler()</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html#%3Cinit%3E(java.lang.String)">ChEBIContentHandler(String)</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIDatabase.html" title="class in bio.singa.chemistry.features.databases.chebi"><span class="typeNameLink">ChEBIDatabase</span></a> - Class in <a href="../bio/singa/chemistry/features/databases/chebi/package-summary.html">bio.singa.chemistry.features.databases.chebi</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIDatabase.html#%3Cinit%3E()">ChEBIDatabase()</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIDatabase.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIDatabase</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/identifiers/ChEBIIdentifier.html" title="class in bio.singa.features.identifiers"><span class="typeNameLink">ChEBIIdentifier</span></a> - Class in <a href="../bio/singa/features/identifiers/package-summary.html">bio.singa.features.identifiers</a></dt>
<dd>
<div class="block">This identifier is used by the <a href="https://www.ebi.ac.uk/chebi/">ChEBI Database</a> to identify small molecules.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/ChEBIIdentifier.html#%3Cinit%3E(java.lang.String)">ChEBIIdentifier(String)</a></span> - Constructor for class bio.singa.features.identifiers.<a href="../bio/singa/features/identifiers/ChEBIIdentifier.html" title="class in bio.singa.features.identifiers">ChEBIIdentifier</a></dt>
<dd>
<div class="block">Creates a new identifier.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/ChEBIIdentifier.html#%3Cinit%3E(java.lang.String,bio.singa.features.model.Evidence)">ChEBIIdentifier(String, Evidence)</a></span> - Constructor for class bio.singa.features.identifiers.<a href="../bio/singa/features/identifiers/ChEBIIdentifier.html" title="class in bio.singa.features.identifiers">ChEBIIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/ChEBIIdentifier.html#%3Cinit%3E(java.lang.String,bio.singa.features.model.Evidence...)">ChEBIIdentifier(String, Evidence...)</a></span> - Constructor for class bio.singa.features.identifiers.<a href="../bio/singa/features/identifiers/ChEBIIdentifier.html" title="class in bio.singa.features.identifiers">ChEBIIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIImageService.html" title="class in bio.singa.chemistry.features.databases.chebi"><span class="typeNameLink">ChEBIImageService</span></a> - Class in <a href="../bio/singa/chemistry/features/databases/chebi/package-summary.html">bio.singa.chemistry.features.databases.chebi</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIImageService.html#%3Cinit%3E(java.lang.String)">ChEBIImageService(String)</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIImageService.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIImageService</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIParserService.html" title="class in bio.singa.chemistry.features.databases.chebi"><span class="typeNameLink">ChEBIParserService</span></a> - Class in <a href="../bio/singa/chemistry/features/databases/chebi/package-summary.html">bio.singa.chemistry.features.databases.chebi</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIParserService.html#%3Cinit%3E(bio.singa.features.identifiers.ChEBIIdentifier)">ChEBIParserService(ChEBIIdentifier)</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIParserService.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIParserService</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIParserService.html#%3Cinit%3E(bio.singa.features.identifiers.ChEBIIdentifier,java.lang.String)">ChEBIParserService(ChEBIIdentifier, String)</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIParserService.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIParserService</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/identifiers/ChEBIToInChIKey.html" title="class in bio.singa.chemistry.features.identifiers"><span class="typeNameLink">ChEBIToInChIKey</span></a> - Class in <a href="../bio/singa/chemistry/features/identifiers/package-summary.html">bio.singa.chemistry.features.identifiers</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/identifiers/ChEBIToInChIKey.html#%3Cinit%3E()">ChEBIToInChIKey()</a></span> - Constructor for class bio.singa.chemistry.features.identifiers.<a href="../bio/singa/chemistry/features/identifiers/ChEBIToInChIKey.html" title="class in bio.singa.chemistry.features.identifiers">ChEBIToInChIKey</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/VectorMetricProvider.html#CHEBYCHEV_METRIC">CHEBYCHEV_METRIC</a></span> - Static variable in class bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/VectorMetricProvider.html" title="class in bio.singa.mathematics.metrics.model">VectorMetricProvider</a></dt>
<dd>
<div class="block">The Chebyshev metric (also known as Tchebychev metric or maximum metric) is a metric defined on a vector space
 where the distance between two vectors is the greatest of their differences along any coordinate dimension.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/model/IdentifierPatternRegistry.html#check(java.lang.Class,bio.singa.features.identifiers.model.Identifier)">check(Class&lt;IdentifierType&gt;, Identifier)</a></span> - Static method in class bio.singa.features.identifiers.model.<a href="../bio/singa/features/identifiers/model/IdentifierPatternRegistry.html" title="class in bio.singa.features.identifiers.model">IdentifierPatternRegistry</a></dt>
<dd>
<div class="block">Returns true, if the identifier is valid.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/InteractionContainer.html#checkAddedInteractions">checkAddedInteractions</a></span> - Static variable in class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/InteractionContainer.html" title="class in bio.singa.structure.parser.plip">InteractionContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html#checkFeatures()">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.<a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html" title="class in bio.singa.simulation.model.modules">AbstractUpdateModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.html#checkFeatures()">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">Diffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/LateralMembraneDiffusion.html#checkFeatures()">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/LateralMembraneDiffusion.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">LateralMembraneDiffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleConfinedDiffusion.html#checkFeatures()">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleConfinedDiffusion.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">VesicleConfinedDiffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleCytoplasmDiffusion.html#checkFeatures()">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleCytoplasmDiffusion.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">VesicleCytoplasmDiffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/UpdateModule.html#checkFeatures()">checkFeatures()</a></span> - Method in interface bio.singa.simulation.model.modules.<a href="../bio/singa/simulation/model/modules/UpdateModule.html" title="interface in bio.singa.simulation.model.modules">UpdateModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/primes/Primes.html#checkPrime(int)">checkPrime(int)</a></span> - Static method in class bio.singa.mathematics.algorithms.primes.<a href="../bio/singa/mathematics/algorithms/primes/Primes.html" title="class in bio.singa.mathematics.algorithms.primes">Primes</a></dt>
<dd>
<div class="block">Checks for the given number if it is a prime number.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/error/GlobalNumericalErrorManager.html#checkSkipping()">checkSkipping()</a></span> - Method in class bio.singa.simulation.model.simulation.error.<a href="../bio/singa/simulation/model/simulation/error/GlobalNumericalErrorManager.html" title="class in bio.singa.simulation.model.simulation.error">GlobalNumericalErrorManager</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/entities/ChemicalEntities.html" title="class in bio.singa.simulation.entities"><span class="typeNameLink">ChemicalEntities</span></a> - Class in <a href="../bio/singa/simulation/entities/package-summary.html">bio.singa.simulation.entities</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/entities/ChemicalEntity.html" title="interface in bio.singa.simulation.entities"><span class="typeNameLink">ChemicalEntity</span></a> - Interface in <a href="../bio/singa/simulation/entities/package-summary.html">bio.singa.simulation.entities</a></dt>
<dd>
<div class="block">Chemical Entity is an abstract class that provides the common features of all chemical substances on a descriptive
 level.</div>
</dd>
<dt><a href="../bio/singa/chemistry/features/ChemistryFeatureContainer.html" title="class in bio.singa.chemistry.features"><span class="typeNameLink">ChemistryFeatureContainer</span></a> - Class in <a href="../bio/singa/chemistry/features/package-summary.html">bio.singa.chemistry.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/ChemistryFeatureContainer.html#%3Cinit%3E()">ChemistryFeatureContainer()</a></span> - Constructor for class bio.singa.chemistry.features.<a href="../bio/singa/chemistry/features/ChemistryFeatureContainer.html" title="class in bio.singa.chemistry.features">ChemistryFeatureContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#CHLORINE">CHLORINE</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#CHROMIUM">CHROMIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/parser/pdb/ligands/CifFileParser.html" title="class in bio.singa.structure.parser.pdb.ligands"><span class="typeNameLink">CifFileParser</span></a> - Class in <a href="../bio/singa/structure/parser/pdb/ligands/package-summary.html">bio.singa.structure.parser.pdb.ligands</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/geometry/faces/Circle.html" title="class in bio.singa.mathematics.geometry.faces"><span class="typeNameLink">Circle</span></a> - Class in <a href="../bio/singa/mathematics/geometry/faces/package-summary.html">bio.singa.mathematics.geometry.faces</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Circle.html#%3Cinit%3E(bio.singa.mathematics.vectors.Vector2D,double)">Circle(Vector2D, double)</a></span> - Constructor for class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Circle.html" title="class in bio.singa.mathematics.geometry.faces">Circle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Circle.html#%3Cinit%3E(bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D)">Circle(Vector2D, Vector2D, Vector2D)</a></span> - Constructor for class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Circle.html" title="class in bio.singa.mathematics.geometry.faces">Circle</a></dt>
<dd>
<div class="block">Circumscribed circle from 3 points (e.g.</div>
</dd>
<dt><a href="../bio/singa/mathematics/algorithms/voronoi/model/CircleEvent.html" title="class in bio.singa.mathematics.algorithms.voronoi.model"><span class="typeNameLink">CircleEvent</span></a> - Class in <a href="../bio/singa/mathematics/algorithms/voronoi/model/package-summary.html">bio.singa.mathematics.algorithms.voronoi.model</a></dt>
<dd>
<div class="block">A circle event happens when a parabolic arc shrinks to a point and disappears from the beach line.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/CircleEvent.html#%3Cinit%3E()">CircleEvent()</a></span> - Constructor for class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/CircleEvent.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">CircleEvent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonGraphs.html#circleRegion(bio.singa.simulation.model.graphs.AutomatonGraph,bio.singa.simulation.model.sections.CellRegion,bio.singa.mathematics.topology.grids.rectangular.RectangularCoordinate,int)">circleRegion(AutomatonGraph, CellRegion, RectangularCoordinate, int)</a></span> - Static method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonGraphs.html" title="class in bio.singa.simulation.model.graphs">AutomatonGraphs</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/geometry/faces/Circles.html" title="class in bio.singa.mathematics.geometry.faces"><span class="typeNameLink">Circles</span></a> - Class in <a href="../bio/singa/mathematics/geometry/faces/package-summary.html">bio.singa.mathematics.geometry.faces</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Circles.html#circumference(double)">circumference(double)</a></span> - Static method in class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Circles.html" title="class in bio.singa.mathematics.geometry.faces">Circles</a></dt>
<dd>
<div class="block">Returns the circumference of a circle with the given radius.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/HeaderToken.html#CLASSIFICATION">CLASSIFICATION</a></span> - bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/HeaderToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">HeaderToken</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/qualitative/implementations/ClathrinMediatedEndocytosis.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations"><span class="typeNameLink">ClathrinMediatedEndocytosis</span></a> - Class in <a href="../bio/singa/simulation/model/modules/qualitative/implementations/package-summary.html">bio.singa.simulation.model.modules.qualitative.implementations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/ClathrinMediatedEndocytosis.html#%3Cinit%3E()">ClathrinMediatedEndocytosis()</a></span> - Constructor for class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/ClathrinMediatedEndocytosis.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">ClathrinMediatedEndocytosis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.AreaMapping.html#clear()">clear()</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.AreaMapping.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode.AreaMapping</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/parameters/ParameterStorage.html#clear()">clear()</a></span> - Static method in class bio.singa.simulation.model.parameters.<a href="../bio/singa/simulation/model/parameters/ParameterStorage.html" title="class in bio.singa.simulation.model.parameters">ParameterStorage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/AbstractComplexEntityModification.html#clear()">clear()</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/AbstractComplexEntityModification.html" title="class in bio.singa.simulation.reactions.modifications">AbstractComplexEntityModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModification.html#clear()">clear()</a></span> - Method in interface bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModification.html" title="interface in bio.singa.simulation.reactions.modifications">ComplexEntityModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/AbstractGraphComplexReactor.html#clear()">clear()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/AbstractGraphComplexReactor.html" title="class in bio.singa.simulation.reactions.reactors">AbstractGraphComplexReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ComplexReactor.html#clear()">clear()</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ComplexReactor.html" title="interface in bio.singa.simulation.reactions.reactors">ComplexReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/OneToTwoReactor.html#clear()">clear()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/OneToTwoReactor.html" title="class in bio.singa.simulation.reactions.reactors">OneToTwoReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/TwoToOneReactor.html#clear()">clear()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/TwoToOneReactor.html" title="class in bio.singa.simulation.reactions.reactors">TwoToOneReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/errors/DebugRecorder.html#clear()">clear()</a></span> - Method in class bio.singa.simulation.trajectories.errors.<a href="../bio/singa/simulation/trajectories/errors/DebugRecorder.html" title="class in bio.singa.simulation.trajectories.errors">DebugRecorder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html#clearAssociatedNodes()">clearAssociatedNodes()</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html" title="class in bio.singa.simulation.model.agents.pointlike">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html#clearAttachmentInformation()">clearAttachmentInformation()</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html" title="class in bio.singa.simulation.model.agents.pointlike">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#clearCaches()">clearCaches()</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/FieldSupplier.html#clearDeltas()">clearDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/FieldSupplier.html" title="class in bio.singa.simulation.model.modules.concentration">FieldSupplier</a></dt>
<dd>
<div class="block">Clears all full end half deltas.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html#clearMisguidedFilamens()">clearMisguidedFilamens()</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html" title="class in bio.singa.simulation.model.agents.linelike">LineLikeAgentLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html#clearPotentialDeltas()">clearPotentialDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaManager</a></dt>
<dd>
<div class="block">Clears the list of potential deltas.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/scope/DependentUpdate.html#clearPotentialDeltas()">clearPotentialDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.scope.<a href="../bio/singa/simulation/model/modules/concentration/scope/DependentUpdate.html" title="class in bio.singa.simulation.model.modules.concentration.scope">DependentUpdate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/scope/IndependentUpdate.html#clearPotentialDeltas()">clearPotentialDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.scope.<a href="../bio/singa/simulation/model/modules/concentration/scope/IndependentUpdate.html" title="class in bio.singa.simulation.model.modules.concentration.scope">IndependentUpdate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/scope/SemiDependentUpdate.html#clearPotentialDeltas()">clearPotentialDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.scope.<a href="../bio/singa/simulation/model/modules/concentration/scope/SemiDependentUpdate.html" title="class in bio.singa.simulation.model.modules.concentration.scope">SemiDependentUpdate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/scope/UpdateScope.html#clearPotentialDeltas()">clearPotentialDeltas()</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.scope.<a href="../bio/singa/simulation/model/modules/concentration/scope/UpdateScope.html" title="interface in bio.singa.simulation.model.modules.concentration.scope">UpdateScope</a></dt>
<dd>
<div class="block">Clears all concentration deltas that may be assigned by this module</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html#clearPotentialDisplacementDeltas()">clearPotentialDisplacementDeltas()</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html" title="class in bio.singa.simulation.model.agents.pointlike">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/DisplacementDeltaManager.html#clearPotentialDisplacementDeltas()">clearPotentialDisplacementDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/DisplacementDeltaManager.html" title="class in bio.singa.simulation.model.modules.displacement">DisplacementDeltaManager</a></dt>
<dd>
<div class="block">Clears all potential displacement deltas.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Simulation.html#clearPreviouslyObservedDeltas()">clearPreviouslyObservedDeltas()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Simulation.html" title="class in bio.singa.simulation.model.simulation">Simulation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/VesicleLayer.html#clearUpdates()">clearUpdates()</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/VesicleLayer.html" title="class in bio.singa.simulation.model.agents.pointlike">VesicleLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/geometry/SutherandHodgmanClipping.html#clip(bio.singa.mathematics.geometry.model.Polygon,bio.singa.mathematics.geometry.model.Polygon)">clip(Polygon, Polygon)</a></span> - Static method in class bio.singa.mathematics.algorithms.geometry.<a href="../bio/singa/mathematics/algorithms/geometry/SutherandHodgmanClipping.html" title="class in bio.singa.mathematics.algorithms.geometry">SutherandHodgmanClipping</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html#clipEdges()">clipEdges()</a></span> - Method in class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">VoronoiDiagram</a></dt>
<dd>
<div class="block">Clips all edges sticking out of the bounding box.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/GifWriter.html#close()">close()</a></span> - Method in class bio.singa.core.utility.<a href="../bio/singa/core/utility/GifWriter.html" title="class in bio.singa.core.utility">GifWriter</a></dt>
<dd>
<div class="block">Close this GifSequenceWriter.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/Token.html#CLOSE_BRACKET">CLOSE_BRACKET</a></span> - Static variable in class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/Token.html" title="class in uk.co.cogitolearning.cogpar">Token</a></dt>
<dd>
<div class="block">Token id for closing brackets</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html#closeCells()">closeCells()</a></span> - Method in class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">VoronoiDiagram</a></dt>
<dd>
<div class="block">Closes all cells that have not been closed until now.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.html#closeWriters()">closeWriters()</a></span> - Method in class bio.singa.simulation.trajectories.flat.<a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.html" title="class in bio.singa.simulation.trajectories.flat">FlatUpdateRecorder</a></dt>
<dd>
<div class="block">Close all associated writers.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html#clusterCutoff(double)">clusterCutoff(double)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.ParameterStep.html#clusterCutoff(double)">clusterCutoff(double)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.ParameterStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.ParameterStep</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/algorithms/clustering/Clustering.html" title="interface in bio.singa.mathematics.algorithms.clustering"><span class="typeNameLink">Clustering</span></a>&lt;<a href="../bio/singa/mathematics/algorithms/clustering/Clustering.html" title="type parameter in Clustering">DataType</a>&gt; - Interface in <a href="../bio/singa/mathematics/algorithms/clustering/package-summary.html">bio.singa.mathematics.algorithms.clustering</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CO">CO</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#COBALT">COBALT</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/mmtf/MmtfSecondaryStructure.html#COIL">COIL</a></span> - bio.singa.structure.model.mmtf.<a href="../bio/singa/structure/model/mmtf/MmtfSecondaryStructure.html" title="enum in bio.singa.structure.model.mmtf">MmtfSecondaryStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ComplexReactor.html#collectCandidates(java.util.List)">collectCandidates(List&lt;ComplexEntity&gt;)</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ComplexReactor.html" title="interface in bio.singa.simulation.reactions.reactors">ComplexReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/OneToOneReactor.html#collectCandidates(java.util.List)">collectCandidates(List&lt;ComplexEntity&gt;)</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/OneToOneReactor.html" title="class in bio.singa.simulation.reactions.reactors">OneToOneReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/OneToTwoReactor.html#collectCandidates(java.util.List)">collectCandidates(List&lt;ComplexEntity&gt;)</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/OneToTwoReactor.html" title="class in bio.singa.simulation.reactions.reactors">OneToTwoReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/TwoToOneReactor.html#collectCandidates(java.util.List)">collectCandidates(List&lt;ComplexEntity&gt;)</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/TwoToOneReactor.html" title="class in bio.singa.simulation.reactions.reactors">TwoToOneReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/DeltaBehavior.html#collectCatalysts()">collectCatalysts()</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/DeltaBehavior.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas">DeltaBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionEvent.html#collectDeltas(bio.singa.simulation.model.simulation.Updatable)">collectDeltas(Updatable)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionEvent.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionEvent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html#collectLeavesPreOrder(bio.singa.mathematics.graphs.trees.BinaryTreeNode,java.util.List)">collectLeavesPreOrder(BinaryTreeNode&lt;T&gt;, List&lt;BinaryTreeNode&lt;T&gt;&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTree</a></dt>
<dd>
<div class="block">Traverses the tree in pre order and collects leave nodes.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html#collectNodesPreOrder(bio.singa.mathematics.graphs.trees.BinaryTreeNode,java.util.List)">collectNodesPreOrder(BinaryTreeNode&lt;T&gt;, List&lt;BinaryTreeNode&lt;T&gt;&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTree</a></dt>
<dd>
<div class="block">Traverses the tree in pre order and collects nodes.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/DeltaBehavior.html#collectProducts()">collectProducts()</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/DeltaBehavior.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas">DeltaBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/DeltaBehavior.html#collectReactants(java.util.Collection)">collectReactants(Collection&lt;Reactant&gt;)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/DeltaBehavior.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas">DeltaBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/MembraneRestrictedBehavior.html#collectReactants(java.util.Collection)">collectReactants(Collection&lt;Reactant&gt;)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/MembraneRestrictedBehavior.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas">MembraneRestrictedBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/NodeBehavior.html#collectReactants(java.util.Collection)">collectReactants(Collection&lt;Reactant&gt;)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/NodeBehavior.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas">NodeBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/DeltaBehavior.html#collectSubstrates()">collectSubstrates()</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/DeltaBehavior.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas">DeltaBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Simulation.html#collectUpdatables()">collectUpdatables()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Simulation.html" title="class in bio.singa.simulation.model.simulation">Simulation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#colorize(java.lang.Object,bio.singa.core.utility.AnsiColor...)">colorize(Object, AnsiColor...)</a></span> - Static method in enum bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/javafx/renderer/colors/ColorScale.html" title="class in bio.singa.javafx.renderer.colors"><span class="typeNameLink">ColorScale</span></a> - Class in <a href="../bio/singa/javafx/renderer/colors/package-summary.html">bio.singa.javafx.renderer.colors</a></dt>
<dd>
<div class="block">This object provides a color scale between the given minimal and maximal
 values.</div>
</dd>
<dt><a href="../bio/singa/javafx/renderer/colors/ColorScale.Builder.html" title="class in bio.singa.javafx.renderer.colors"><span class="typeNameLink">ColorScale.Builder</span></a> - Class in <a href="../bio/singa/javafx/renderer/colors/package-summary.html">bio.singa.javafx.renderer.colors</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/StructureViewer.html#colorScheme">colorScheme</a></span> - Static variable in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/StructureViewer.html" title="class in bio.singa.javafx.viewer">StructureViewer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/javafx/viewer/ColorScheme.html" title="enum in bio.singa.javafx.viewer"><span class="typeNameLink">ColorScheme</span></a> - Enum in <a href="../bio/singa/javafx/viewer/package-summary.html">bio.singa.javafx.viewer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/TeXFormat.html#COLUMN_END_BREAKING">COLUMN_END_BREAKING</a></span> - Static variable in class bio.singa.simulation.export.<a href="../bio/singa/simulation/export/TeXFormat.html" title="class in bio.singa.simulation.export">TeXFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/TeXFormat.html#COLUMN_END_NON_BREAKING">COLUMN_END_NON_BREAKING</a></span> - Static variable in class bio.singa.simulation.export.<a href="../bio/singa/simulation/export/TeXFormat.html" title="class in bio.singa.simulation.export">TeXFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/topology/grids/rectangular/RectangularCoordinate.html#COLUMN_FIRST">COLUMN_FIRST</a></span> - Static variable in class bio.singa.mathematics.topology.grids.rectangular.<a href="../bio/singa/mathematics/topology/grids/rectangular/RectangularCoordinate.html" title="class in bio.singa.mathematics.topology.grids.rectangular">RectangularCoordinate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/TeXFormat.html#COLUMN_SEPERATOR">COLUMN_SEPERATOR</a></span> - Static variable in class bio.singa.simulation.export.<a href="../bio/singa/simulation/export/TeXFormat.html" title="class in bio.singa.simulation.export">TeXFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/TeXFormat.html#COLUMN_SEPERATOR_SPACED">COLUMN_SEPERATOR_SPACED</a></span> - Static variable in class bio.singa.simulation.export.<a href="../bio/singa/simulation/export/TeXFormat.html" title="class in bio.singa.simulation.export">TeXFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/combinatorics/StreamCombinations.html#combinations(int,java.util.List)">combinations(int, List&lt;T&gt;)</a></span> - Static method in class bio.singa.mathematics.combinatorics.<a href="../bio/singa/mathematics/combinatorics/StreamCombinations.html" title="class in bio.singa.mathematics.combinatorics">StreamCombinations</a></dt>
<dd>
<div class="block">Generates all combinations of the requested size from the given list of objects.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.CombineStep.html#combine(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.entities.ChemicalEntity)">combine(ChemicalEntity, ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntityBuilder.CombineStep.html" title="interface in bio.singa.simulation.entities">ComplexEntityBuilder.CombineStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.ComplexEntityCombiner.html#combine(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.entities.ChemicalEntity)">combine(ChemicalEntity, ChemicalEntity)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntityBuilder.ComplexEntityCombiner.html" title="class in bio.singa.simulation.entities">ComplexEntityBuilder.ComplexEntityCombiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.CombineStep.html#combine(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.entities.BindingSite)">combine(ChemicalEntity, ChemicalEntity, BindingSite)</a></span> - Method in interface bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntityBuilder.CombineStep.html" title="interface in bio.singa.simulation.entities">ComplexEntityBuilder.CombineStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.ComplexEntityCombiner.html#combine(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.entities.BindingSite)">combine(ChemicalEntity, ChemicalEntity, BindingSite)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntityBuilder.ComplexEntityCombiner.html" title="class in bio.singa.simulation.entities">ComplexEntityBuilder.ComplexEntityCombiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#combine(bio.singa.simulation.entities.GraphComplexNode,bio.singa.simulation.entities.GraphComplexNode,bio.singa.simulation.entities.ComplexEntity,bio.singa.simulation.entities.BindingSite)">combine(GraphComplexNode, GraphComplexNode, ComplexEntity, BindingSite)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/reactions/RateConstant.BuilderStep.html#comment(java.lang.String)">comment(String)</a></span> - Method in interface bio.singa.chemistry.features.reactions.<a href="../bio/singa/chemistry/features/reactions/RateConstant.BuilderStep.html" title="interface in bio.singa.chemistry.features.reactions">RateConstant.BuilderStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/reactions/RateConstant.RateBuilder.html#comment(java.lang.String)">comment(String)</a></span> - Method in class bio.singa.chemistry.features.reactions.<a href="../bio/singa/chemistry/features/reactions/RateConstant.RateBuilder.html" title="class in bio.singa.chemistry.features.reactions">RateConstant.RateBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractFeature.Builder.html#comment(java.lang.String)">comment(String)</a></span> - Method in class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractFeature.Builder.html" title="class in bio.singa.features.model">AbstractFeature.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/core/utility/CommutablePair.html" title="class in bio.singa.core.utility"><span class="typeNameLink">CommutablePair</span></a>&lt;<a href="../bio/singa/core/utility/CommutablePair.html" title="type parameter in CommutablePair">ValueType</a>&gt; - Class in <a href="../bio/singa/core/utility/package-summary.html">bio.singa.core.utility</a></dt>
<dd>
<div class="block">This subclass of <a href="../bio/singa/core/utility/Pair.html" title="class in bio.singa.core.utility"><code>Pair</code></a> only implements different equals and hashcode methods.<br> The values are exchangeable.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/CommutablePair.html#%3Cinit%3E(ValueType,ValueType)">CommutablePair(ValueType, ValueType)</a></span> - Constructor for class bio.singa.core.utility.<a href="../bio/singa/core/utility/CommutablePair.html" title="class in bio.singa.core.utility">CommutablePair</a></dt>
<dd>
<div class="block">Creates a new <a href="../bio/singa/core/utility/Pair.html" title="class in bio.singa.core.utility"><code>Pair</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html#compactToSymmetricMatrix(double%5B%5D%5B%5D)">compactToSymmetricMatrix(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html" title="class in bio.singa.mathematics.matrices">SymmetricMatrix</a></dt>
<dd>
<div class="block">Compacts the values of a symmetric matrix into a jagged array, that represents the lower triangular part of a
 symmetric matrix.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/VertexPolygon.html#compare(bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D)">compare(Vector2D, Vector2D)</a></span> - Method in class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/VertexPolygon.html" title="class in bio.singa.mathematics.geometry.faces">VertexPolygon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/LeafIdentifier.html#compareTo(bio.singa.features.identifiers.LeafIdentifier)">compareTo(LeafIdentifier)</a></span> - Method in class bio.singa.features.identifiers.<a href="../bio/singa/features/identifiers/LeafIdentifier.html" title="class in bio.singa.features.identifiers">LeafIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/UniqueAtomIdentifer.html#compareTo(bio.singa.features.identifiers.UniqueAtomIdentifer)">compareTo(UniqueAtomIdentifer)</a></span> - Method in class bio.singa.features.identifiers.<a href="../bio/singa/features/identifiers/UniqueAtomIdentifer.html" title="class in bio.singa.features.identifiers">UniqueAtomIdentifer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DMatch.html#compareTo(bio.singa.structure.algorithms.superimposition.fit3d.Fit3DMatch)">compareTo(Fit3DMatch)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DMatch.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DMatch</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/LigandFamily.html#compareTo(bio.singa.structure.model.families.LigandFamily)">compareTo(LigandFamily)</a></span> - Method in class bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/LigandFamily.html" title="class in bio.singa.structure.model.families">LigandFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.KineticStep.html#complexBuilding()">complexBuilding()</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.KineticStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.KineticStep</a></dt>
<dd>
<div class="block">Complex building reactions classically require two substrates that react to build one complex.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.RuleBasedBuilder.html#complexBuilding()">complexBuilding()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.RuleBasedBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.RuleBasedBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#complexBuilding()">complexBuilding()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities"><span class="typeNameLink">ComplexEntity</span></a> - Class in <a href="../bio/singa/simulation/entities/package-summary.html">bio.singa.simulation.entities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#%3Cinit%3E()">ComplexEntity()</a></span> - Constructor for class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.html" title="class in bio.singa.simulation.entities"><span class="typeNameLink">ComplexEntityBuilder</span></a> - Class in <a href="../bio/singa/simulation/entities/package-summary.html">bio.singa.simulation.entities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.html#%3Cinit%3E()">ComplexEntityBuilder()</a></span> - Constructor for class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntityBuilder.html" title="class in bio.singa.simulation.entities">ComplexEntityBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.CombineStep.html" title="interface in bio.singa.simulation.entities"><span class="typeNameLink">ComplexEntityBuilder.CombineStep</span></a> - Interface in <a href="../bio/singa/simulation/entities/package-summary.html">bio.singa.simulation.entities</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.ComplexEntityCombiner.html" title="class in bio.singa.simulation.entities"><span class="typeNameLink">ComplexEntityBuilder.ComplexEntityCombiner</span></a> - Class in <a href="../bio/singa/simulation/entities/package-summary.html">bio.singa.simulation.entities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.ComplexEntityCombiner.html#%3Cinit%3E()">ComplexEntityCombiner()</a></span> - Constructor for class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntityBuilder.ComplexEntityCombiner.html" title="class in bio.singa.simulation.entities">ComplexEntityBuilder.ComplexEntityCombiner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModification.html" title="interface in bio.singa.simulation.reactions.modifications"><span class="typeNameLink">ComplexEntityModification</span></a> - Interface in <a href="../bio/singa/simulation/reactions/modifications/package-summary.html">bio.singa.simulation.reactions.modifications</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModificationBuilder.html" title="class in bio.singa.simulation.reactions.modifications"><span class="typeNameLink">ComplexEntityModificationBuilder</span></a> - Class in <a href="../bio/singa/simulation/reactions/modifications/package-summary.html">bio.singa.simulation.reactions.modifications</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModificationBuilder.html#%3Cinit%3E()">ComplexEntityModificationBuilder()</a></span> - Constructor for class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModificationBuilder.html" title="class in bio.singa.simulation.reactions.modifications">ComplexEntityModificationBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/geometry/faces/ComplexPolygon.html" title="class in bio.singa.mathematics.geometry.faces"><span class="typeNameLink">ComplexPolygon</span></a> - Class in <a href="../bio/singa/mathematics/geometry/faces/package-summary.html">bio.singa.mathematics.geometry.faces</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/ComplexPolygon.html#%3Cinit%3E(bio.singa.mathematics.vectors.Vector2D...)">ComplexPolygon(Vector2D...)</a></span> - Constructor for class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/ComplexPolygon.html" title="class in bio.singa.mathematics.geometry.faces">ComplexPolygon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/ComplexPolygon.html#%3Cinit%3E(java.util.List)">ComplexPolygon(List&lt;Vector2D&gt;)</a></span> - Constructor for class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/ComplexPolygon.html" title="class in bio.singa.mathematics.geometry.faces">ComplexPolygon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/reactions/reactors/ComplexReactor.html" title="interface in bio.singa.simulation.reactions.reactors"><span class="typeNameLink">ComplexReactor</span></a> - Interface in <a href="../bio/singa/simulation/reactions/reactors/package-summary.html">bio.singa.simulation.reactions.reactors</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html#composePdbRepresentation(bio.singa.structure.model.interfaces.Structure)">composePdbRepresentation(Structure)</a></span> - Static method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html" title="class in bio.singa.structure.parser.pdb.structures">StructureRepresentation</a></dt>
<dd>
<div class="block">Creates a pdb representation of the given structure.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html#composePdbRepresentation(bio.singa.structure.model.oak.OakModel)">composePdbRepresentation(OakModel)</a></span> - Static method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html" title="class in bio.singa.structure.parser.pdb.structures">StructureRepresentation</a></dt>
<dd>
<div class="block">Creates a representation of the given model.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html#composePdbRepresentation(java.util.List)">composePdbRepresentation(List&lt;LeafSubstructure&lt;?&gt;&gt;)</a></span> - Static method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html" title="class in bio.singa.structure.parser.pdb.structures">StructureRepresentation</a></dt>
<dd>
<div class="block">Creates a pdb representation of the given structure.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/BitVector.html#concat(bio.singa.mathematics.vectors.BitVector)">concat(BitVector)</a></span> - Method in interface bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/BitVector.html" title="interface in bio.singa.mathematics.vectors">BitVector</a></dt>
<dd>
<div class="block">Concatenates the elements of this <a href="../bio/singa/mathematics/vectors/BitVector.html" title="interface in bio.singa.mathematics.vectors"><code>BitVector</code></a> of length m with another of length n.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/units/UnitRegistry.html#concentration(double)">concentration(double)</a></span> - Static method in class bio.singa.features.units.<a href="../bio/singa/features/units/UnitRegistry.html" title="class in bio.singa.features.units">UnitRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/units/UnitRegistry.html#concentration(double,javax.measure.Unit)">concentration(double, Unit&lt;MolarConcentration&gt;)</a></span> - Static method in class bio.singa.features.units.<a href="../bio/singa/features/units/UnitRegistry.html" title="class in bio.singa.features.units">UnitRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html#concentration(javax.measure.Quantity)">concentration(Quantity&lt;MolarConcentration&gt;)</a></span> - Method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html" title="class in bio.singa.simulation.model.concentrations">ConcentrationBuilder.ConcentrationBuilderImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationStep.html#concentration(javax.measure.Quantity)">concentration(Quantity&lt;MolarConcentration&gt;)</a></span> - Method in interface bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationStep.html" title="interface in bio.singa.simulation.model.concentrations">ConcentrationBuilder.ConcentrationStep</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/qualitative/implementations/ConcentrationApplier.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations"><span class="typeNameLink">ConcentrationApplier</span></a> - Class in <a href="../bio/singa/simulation/model/modules/qualitative/implementations/package-summary.html">bio.singa.simulation.model.modules.qualitative.implementations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/ConcentrationApplier.html#%3Cinit%3E()">ConcentrationApplier()</a></span> - Constructor for class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/ConcentrationApplier.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">ConcentrationApplier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration"><span class="typeNameLink">ConcentrationBasedModule</span></a>&lt;<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="type parameter in ConcentrationBasedModule">DeltaFunctionType</a> extends <a href="../bio/singa/simulation/model/modules/concentration/functions/AbstractDeltaFunction.html" title="class in bio.singa.simulation.model.modules.concentration.functions">AbstractDeltaFunction</a>&gt; - Class in <a href="../bio/singa/simulation/model/modules/concentration/package-summary.html">bio.singa.simulation.model.modules.concentration</a></dt>
<dd>
<div class="block">Concentration based modules handle updates that are applied to the concentrations in <a href="../bio/singa/simulation/model/simulation/Updatable.html" title="interface in bio.singa.simulation.model.simulation"><code>Updatable</code></a>s during a
 simulation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html#%3Cinit%3E()">ConcentrationBasedModule()</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationBasedModule</a></dt>
<dd>
<div class="block">Creates a new concentration based module.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.html" title="class in bio.singa.simulation.model.concentrations"><span class="typeNameLink">ConcentrationBuilder</span></a> - Class in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.html#%3Cinit%3E()">ConcentrationBuilder()</a></span> - Constructor for class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.html" title="class in bio.singa.simulation.model.concentrations">ConcentrationBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.AdditionalConditionsStep.html" title="interface in bio.singa.simulation.model.concentrations"><span class="typeNameLink">ConcentrationBuilder.AdditionalConditionsStep</span></a> - Interface in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.BuildStep.html" title="interface in bio.singa.simulation.model.concentrations"><span class="typeNameLink">ConcentrationBuilder.BuildStep</span></a> - Interface in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html" title="class in bio.singa.simulation.model.concentrations"><span class="typeNameLink">ConcentrationBuilder.ConcentrationBuilderImpl</span></a> - Class in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationStep.html" title="interface in bio.singa.simulation.model.concentrations"><span class="typeNameLink">ConcentrationBuilder.ConcentrationStep</span></a> - Interface in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationUnitStep.html" title="interface in bio.singa.simulation.model.concentrations"><span class="typeNameLink">ConcentrationBuilder.ConcentrationUnitStep</span></a> - Interface in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.EntityStep.html" title="interface in bio.singa.simulation.model.concentrations"><span class="typeNameLink">ConcentrationBuilder.EntityStep</span></a> - Interface in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.PoolStep.html" title="interface in bio.singa.simulation.model.concentrations"><span class="typeNameLink">ConcentrationBuilder.PoolStep</span></a> - Interface in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.TimedUnitStep.html" title="interface in bio.singa.simulation.model.concentrations"><span class="typeNameLink">ConcentrationBuilder.TimedUnitStep</span></a> - Interface in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html#%3Cinit%3E()">ConcentrationBuilderImpl()</a></span> - Constructor for class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html" title="class in bio.singa.simulation.model.concentrations">ConcentrationBuilder.ConcentrationBuilderImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html#%3Cinit%3E(bio.singa.simulation.model.simulation.Simulation)">ConcentrationBuilderImpl(Simulation)</a></span> - Constructor for class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html" title="class in bio.singa.simulation.model.concentrations">ConcentrationBuilder.ConcentrationBuilderImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/ConcentrationCondition.html" title="interface in bio.singa.simulation.model.concentrations"><span class="typeNameLink">ConcentrationCondition</span></a> - Interface in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html" title="class in bio.singa.simulation.model.sections"><span class="typeNameLink">ConcentrationContainer</span></a> - Class in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>
<div class="block">The concentration container manages the concentrations of one updatable.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html#%3Cinit%3E()">ConcentrationContainer()</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html" title="class in bio.singa.simulation.model.sections">ConcentrationContainer</a></dt>
<dd>
<div class="block">Creates a new concentration container.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDelta.html" title="class in bio.singa.simulation.model.modules.concentration"><span class="typeNameLink">ConcentrationDelta</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/package-summary.html">bio.singa.simulation.model.modules.concentration</a></dt>
<dd>
<div class="block">The delta object manages the change that will be applied to the concentration of a specific <a href="../bio/singa/simulation/entities/ChemicalEntity.html" title="interface in bio.singa.simulation.entities"><code>ChemicalEntity</code></a> in
 a <a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections"><code>CellSubsection</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDelta.html#%3Cinit%3E(bio.singa.simulation.model.modules.UpdateModule,bio.singa.simulation.model.sections.CellSubsection,bio.singa.simulation.entities.ChemicalEntity,double)">ConcentrationDelta(UpdateModule, CellSubsection, ChemicalEntity, double)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDelta.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDelta</a></dt>
<dd>
<div class="block">Creates a new concentration delta.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaIdentifier.html" title="class in bio.singa.simulation.model.modules.concentration"><span class="typeNameLink">ConcentrationDeltaIdentifier</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/package-summary.html">bio.singa.simulation.model.modules.concentration</a></dt>
<dd>
<div class="block">Used to identify changes to concentrations (<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDelta.html" title="class in bio.singa.simulation.model.modules.concentration"><code>ConcentrationDelta</code></a>s) (mostly in maps).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaIdentifier.html#%3Cinit%3E(bio.singa.simulation.model.simulation.Updatable,bio.singa.simulation.model.sections.CellSubsection,bio.singa.simulation.entities.ChemicalEntity)">ConcentrationDeltaIdentifier(Updatable, CellSubsection, ChemicalEntity)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaIdentifier.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaIdentifier</a></dt>
<dd>
<div class="block">Creates a new DeltaIdentifier.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaIdentifier.html#%3Cinit%3E(bio.singa.simulation.model.simulation.Updatable,bio.singa.simulation.model.sections.CellSubsection,java.lang.String,bio.singa.simulation.entities.ChemicalEntity)">ConcentrationDeltaIdentifier(Updatable, CellSubsection, String, ChemicalEntity)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaIdentifier.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration"><span class="typeNameLink">ConcentrationDeltaManager</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/package-summary.html">bio.singa.simulation.model.modules.concentration</a></dt>
<dd>
<div class="block">The ConcentrationDeltaManager handles current concentrations of an updatable and the updates to those
 concentrations that should be applied during simulation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html#%3Cinit%3E(bio.singa.simulation.model.sections.ConcentrationContainer)">ConcentrationDeltaManager(ConcentrationContainer)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaManager</a></dt>
<dd>
<div class="block">Creates a new Concentration Delta Manager.</div>
</dd>
<dt><a href="../bio/singa/features/quantities/ConcentrationDiffusivity.html" title="class in bio.singa.features.quantities"><span class="typeNameLink">ConcentrationDiffusivity</span></a> - Class in <a href="../bio/singa/features/quantities/package-summary.html">bio.singa.features.quantities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/ConcentrationDiffusivity.html#%3Cinit%3E(javax.measure.Quantity)">ConcentrationDiffusivity(Quantity&lt;Diffusivity&gt;)</a></span> - Constructor for class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/ConcentrationDiffusivity.html" title="class in bio.singa.features.quantities">ConcentrationDiffusivity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/quantities/ConcentrationDiffusivity.Builder.html" title="class in bio.singa.features.quantities"><span class="typeNameLink">ConcentrationDiffusivity.Builder</span></a> - Class in <a href="../bio/singa/features/quantities/package-summary.html">bio.singa.features.quantities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.ConcentrationUnitStep.html#concentrationFormat(bio.singa.features.formatter.GeneralConcentrationFormatter)">concentrationFormat(GeneralConcentrationFormatter)</a></span> - Method in interface bio.singa.simulation.trajectories.flat.<a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.ConcentrationUnitStep.html" title="interface in bio.singa.simulation.trajectories.flat">FlatUpdateRecorder.ConcentrationUnitStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.EpochUpdateWriterBuilder.html#concentrationFormat(bio.singa.features.formatter.GeneralConcentrationFormatter)">concentrationFormat(GeneralConcentrationFormatter)</a></span> - Method in class bio.singa.simulation.trajectories.flat.<a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.EpochUpdateWriterBuilder.html" title="class in bio.singa.simulation.trajectories.flat">FlatUpdateRecorder.EpochUpdateWriterBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/formatter/ConcentrationFormatter.html" title="class in bio.singa.features.formatter"><span class="typeNameLink">ConcentrationFormatter</span></a> - Class in <a href="../bio/singa/features/formatter/package-summary.html">bio.singa.features.formatter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/formatter/ConcentrationFormatter.html#%3Cinit%3E(javax.measure.Unit)">ConcentrationFormatter(Unit&lt;MolarConcentration&gt;)</a></span> - Constructor for class bio.singa.features.formatter.<a href="../bio/singa/features/formatter/ConcentrationFormatter.html" title="class in bio.singa.features.formatter">ConcentrationFormatter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html#concentrationIsAtCap()">concentrationIsAtCap()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaManager</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/sections/ConcentrationPool.html" title="class in bio.singa.simulation.model.sections"><span class="typeNameLink">ConcentrationPool</span></a> - Class in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>
<div class="block">The concentration pool manages the <a href="../bio/singa/features/quantities/MolarConcentration.html" title="class in bio.singa.features.quantities"><code>MolarConcentration</code></a> of <a href="../bio/singa/simulation/entities/ChemicalEntity.html" title="interface in bio.singa.simulation.entities"><code>ChemicalEntity</code></a>s in <a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections"><code>CellSubsection</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/ConcentrationPool.html#%3Cinit%3E()">ConcentrationPool()</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/ConcentrationPool.html" title="class in bio.singa.simulation.model.sections">ConcentrationPool</a></dt>
<dd>
<div class="block">Creates a new, empty concentration pool.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/modules/qualitative/implementations/ConcentrationStateChange.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations"><span class="typeNameLink">ConcentrationStateChange</span></a> - Class in <a href="../bio/singa/simulation/model/modules/qualitative/implementations/package-summary.html">bio.singa.simulation.model.modules.qualitative.implementations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/ConcentrationStateChange.html#%3Cinit%3E()">ConcentrationStateChange()</a></span> - Constructor for class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/ConcentrationStateChange.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">ConcentrationStateChange</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarConcentration.html#concentrationToMolecules(double)">concentrationToMolecules(double)</a></span> - Static method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarConcentration.html" title="class in bio.singa.features.quantities">MolarConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarConcentration.html#concentrationToMoles(javax.measure.Quantity,javax.measure.Quantity)">concentrationToMoles(Quantity&lt;MolarConcentration&gt;, Quantity&lt;Volume&gt;)</a></span> - Static method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarConcentration.html" title="class in bio.singa.features.quantities">MolarConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/reactions/RateConstant.ConcentrationStep.html#concentrationUnit(javax.measure.Unit)">concentrationUnit(Unit&lt;MolarConcentration&gt;)</a></span> - Method in interface bio.singa.chemistry.features.reactions.<a href="../bio/singa/chemistry/features/reactions/RateConstant.ConcentrationStep.html" title="interface in bio.singa.chemistry.features.reactions">RateConstant.ConcentrationStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/reactions/RateConstant.RateBuilder.html#concentrationUnit(javax.measure.Unit)">concentrationUnit(Unit&lt;MolarConcentration&gt;)</a></span> - Method in class bio.singa.chemistry.features.reactions.<a href="../bio/singa/chemistry/features/reactions/RateConstant.RateBuilder.html" title="class in bio.singa.chemistry.features.reactions">RateConstant.RateBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.ConcentrationUnitStep.html#concentrationUnit(javax.measure.Unit)">concentrationUnit(Unit&lt;MolarConcentration&gt;)</a></span> - Method in interface bio.singa.simulation.trajectories.flat.<a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.ConcentrationUnitStep.html" title="interface in bio.singa.simulation.trajectories.flat">FlatUpdateRecorder.ConcentrationUnitStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.EpochUpdateWriterBuilder.html#concentrationUnit(javax.measure.Unit)">concentrationUnit(Unit&lt;MolarConcentration&gt;)</a></span> - Method in class bio.singa.simulation.trajectories.flat.<a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.EpochUpdateWriterBuilder.html" title="class in bio.singa.simulation.trajectories.flat">FlatUpdateRecorder.EpochUpdateWriterBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html#concentrationValue(double)">concentrationValue(double)</a></span> - Method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html" title="class in bio.singa.simulation.model.concentrations">ConcentrationBuilder.ConcentrationBuilderImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationStep.html#concentrationValue(double)">concentrationValue(double)</a></span> - Method in interface bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationStep.html" title="interface in bio.singa.simulation.model.concentrations">ConcentrationBuilder.ConcentrationStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/CandidateCondition.html#concerns(bio.singa.simulation.entities.BindingSite)">concerns(BindingSite)</a></span> - Method in interface bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/CandidateCondition.html" title="interface in bio.singa.simulation.reactions.conditions">CandidateCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/HasAnyNumberOfEntities.html#concerns(bio.singa.simulation.entities.BindingSite)">concerns(BindingSite)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/HasAnyNumberOfEntities.html" title="class in bio.singa.simulation.reactions.conditions">HasAnyNumberOfEntities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/HasNoMoreThanNumberOfPartners.html#concerns(bio.singa.simulation.entities.BindingSite)">concerns(BindingSite)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/HasNoMoreThanNumberOfPartners.html" title="class in bio.singa.simulation.reactions.conditions">HasNoMoreThanNumberOfPartners</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/HasNumberOfEntities.html#concerns(bio.singa.simulation.entities.BindingSite)">concerns(BindingSite)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/HasNumberOfEntities.html" title="class in bio.singa.simulation.reactions.conditions">HasNumberOfEntities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/HasOccupiedBindingSite.html#concerns(bio.singa.simulation.entities.BindingSite)">concerns(BindingSite)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/HasOccupiedBindingSite.html" title="class in bio.singa.simulation.reactions.conditions">HasOccupiedBindingSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/HasUnoccupiedBindingSite.html#concerns(bio.singa.simulation.entities.BindingSite)">concerns(BindingSite)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/HasUnoccupiedBindingSite.html" title="class in bio.singa.simulation.reactions.conditions">HasUnoccupiedBindingSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/IsOnlyBoundAt.html#concerns(bio.singa.simulation.entities.BindingSite)">concerns(BindingSite)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/IsOnlyBoundAt.html" title="class in bio.singa.simulation.reactions.conditions">IsOnlyBoundAt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/CandidateCondition.html#concerns(bio.singa.simulation.entities.ChemicalEntity)">concerns(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/CandidateCondition.html" title="interface in bio.singa.simulation.reactions.conditions">CandidateCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/HasAnyNumberOfEntities.html#concerns(bio.singa.simulation.entities.ChemicalEntity)">concerns(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/HasAnyNumberOfEntities.html" title="class in bio.singa.simulation.reactions.conditions">HasAnyNumberOfEntities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/HasNoMoreThanNumberOfPartners.html#concerns(bio.singa.simulation.entities.ChemicalEntity)">concerns(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/HasNoMoreThanNumberOfPartners.html" title="class in bio.singa.simulation.reactions.conditions">HasNoMoreThanNumberOfPartners</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/HasNumberOfEntities.html#concerns(bio.singa.simulation.entities.ChemicalEntity)">concerns(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/HasNumberOfEntities.html" title="class in bio.singa.simulation.reactions.conditions">HasNumberOfEntities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/HasOccupiedBindingSite.html#concerns(bio.singa.simulation.entities.ChemicalEntity)">concerns(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/HasOccupiedBindingSite.html" title="class in bio.singa.simulation.reactions.conditions">HasOccupiedBindingSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/HasUnoccupiedBindingSite.html#concerns(bio.singa.simulation.entities.ChemicalEntity)">concerns(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/HasUnoccupiedBindingSite.html" title="class in bio.singa.simulation.reactions.conditions">HasUnoccupiedBindingSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/IsOnlyBoundAt.html#concerns(bio.singa.simulation.entities.ChemicalEntity)">concerns(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/IsOnlyBoundAt.html" title="class in bio.singa.simulation.reactions.conditions">IsOnlyBoundAt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.FinalStep.html#condition(bio.singa.simulation.model.concentrations.ConcentrationCondition)">condition(ConcentrationCondition)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.FinalStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.FinalStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.GeneralReactionBuilder.html#condition(bio.singa.simulation.model.concentrations.ConcentrationCondition)">condition(ConcentrationCondition)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.GeneralReactionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.GeneralReactionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AddReactionChainBuilder.html#condition(bio.singa.simulation.reactions.conditions.CandidateCondition)">condition(CandidateCondition)</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AddReactionChainBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AddReactionChainBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AddReactorPrimaryConditionStep.html#condition(bio.singa.simulation.reactions.conditions.CandidateCondition)">condition(CandidateCondition)</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AddReactorPrimaryConditionStep.html" title="interface in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AddReactorPrimaryConditionStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ReleaseReactionChainBuilder.html#condition(bio.singa.simulation.reactions.conditions.CandidateCondition)">condition(CandidateCondition)</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ReleaseReactionChainBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.ReleaseReactionChainBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ReleaseReactorPrimaryConditionStep.html#condition(bio.singa.simulation.reactions.conditions.CandidateCondition)">condition(CandidateCondition)</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ReleaseReactorPrimaryConditionStep.html" title="interface in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.ReleaseReactorPrimaryConditionStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.RemoveReactionChainBuilder.html#condition(bio.singa.simulation.reactions.conditions.CandidateCondition)">condition(CandidateCondition)</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.RemoveReactionChainBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.RemoveReactionChainBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.RemoveReactorPrimaryConditionStep.html#condition(bio.singa.simulation.reactions.conditions.CandidateCondition)">condition(CandidateCondition)</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.RemoveReactorPrimaryConditionStep.html" title="interface in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.RemoveReactorPrimaryConditionStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakChain.html#connectChainBackbone()">connectChainBackbone()</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakChain.html" title="class in bio.singa.structure.model.oak">OakChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#connectHydrogens(boolean)">connectHydrogens(boolean)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option to <code>true</code> will try to connect hydrogen atom tho their respective partners.<br><br>
 Default value: <code>false</code></div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakChain.html#connectNucleotideBonds(bio.singa.structure.model.oak.OakNucleotide,bio.singa.structure.model.oak.OakNucleotide)">connectNucleotideBonds(OakNucleotide, OakNucleotide)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakChain.html" title="class in bio.singa.structure.model.oak">OakChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakChain.html#connectPeptideBonds(bio.singa.structure.model.oak.OakAminoAcid,bio.singa.structure.model.oak.OakAminoAcid)">connectPeptideBonds(OakAminoAcid, OakAminoAcid)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakChain.html" title="class in bio.singa.structure.model.oak">OakChain</a></dt>
<dd>
<div class="block">Connects two residues, using the Backbone Carbon (C) of the source residue and the Backbone Nitrogen (N) of the
 target residue.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/Renderer.html#connectPoints(java.util.Collection)">connectPoints(Collection&lt;Vector2D&gt;)</a></span> - Method in interface bio.singa.javafx.renderer.<a href="../bio/singa/javafx/renderer/Renderer.html" title="interface in bio.singa.javafx.renderer">Renderer</a></dt>
<dd>
<div class="block">Connects the points given in the List in order of their appearance with a line.<br>
 
  The line width of the point is determined by the LineWidth (set by
 <code>GraphicsContext.setLineWidth(double)</code>).
  The color is determined by the StrokeColor (set by <code>GraphicsContext.setStroke(Paint)</code>).
 </div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/SwingRenderer.html#connectPoints(java.util.Collection)">connectPoints(Collection&lt;Vector2D&gt;)</a></span> - Method in interface bio.singa.javafx.renderer.<a href="../bio/singa/javafx/renderer/SwingRenderer.html" title="interface in bio.singa.javafx.renderer">SwingRenderer</a></dt>
<dd>
<div class="block">Connects the points given in the List in order of their appearance with a line.<br>
 
  The line width of the point is determined by the LineWidth (set by
 <code>GraphicsContext.setLineWidth(double)</code>).
  The color is determined by the StrokeColor (set by <code>GraphicsContext.setStroke(Paint)</code>).
 </div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakBond.html#connectsAtom(bio.singa.structure.model.oak.OakAtom)">connectsAtom(OakAtom)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakBond.html" title="class in bio.singa.structure.model.oak">OakBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vectors2D.html#connectToSegments(java.util.List)">connectToSegments(List&lt;Vector2D&gt;)</a></span> - Static method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vectors2D.html" title="class in bio.singa.mathematics.vectors">Vectors2D</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusAlignment.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusAlignment</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>
<div class="block">A consensus alignment of same-sized <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a>s can be used to cluster them according their geometric
 similarity in a multi-structure alignment manner as described in:</div>
</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusBuilder</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>
<div class="block">A builder that guides through the creation of a <a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusAlignment.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><code>ConsensusAlignment</code></a>.</div>
</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.AtomStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusBuilder.AtomStep</span></a> - Interface in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusBuilder.Builder</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.InputStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusBuilder.InputStep</span></a> - Interface in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.ParameterStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusBuilder.ParameterStep</span></a> - Interface in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusContainer.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusContainer</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>
<div class="block">A container encapsulating a <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a> with a <a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees"><code>BinaryTree</code></a> that represents its associated
 consensus tree.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusContainer.html#%3Cinit%3E(bio.singa.structure.model.oak.StructuralMotif,boolean)">ConsensusContainer(StructuralMotif, boolean)</a></span> - Constructor for class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusContainer.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusContainer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusException.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusException</span></a> - Exception in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusException.html#%3Cinit%3E(java.lang.String)">ConsensusException(String)</a></span> - Constructor for exception bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusException.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html#considerInversion">considerInversion</a></span> - Variable in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AbstractReactorBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html#considerInversion()">considerInversion()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AbstractReactorBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ConnectorStep.html#considerInversion()">considerInversion()</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ConnectorStep.html" title="interface in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.ConnectorStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/ExpressionNode.html#CONSTANT_NODE">CONSTANT_NODE</a></span> - Static variable in interface uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/ExpressionNode.html" title="interface in uk.co.cogitolearning.cogpar">ExpressionNode</a></dt>
<dd>
<div class="block">Node id for constant nodes</div>
</dd>
<dt><a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html" title="class in uk.co.cogitolearning.cogpar"><span class="typeNameLink">ConstantExpressionNode</span></a> - Class in <a href="../uk/co/cogitolearning/cogpar/package-summary.html">uk.co.cogitolearning.cogpar</a></dt>
<dd>
<div class="block">An ExpressionNode that stores a constant value</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html#%3Cinit%3E(double)">ConstantExpressionNode(double)</a></span> - Constructor for class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">ConstantExpressionNode</a></dt>
<dd>
<div class="block">Construct with the fixed value.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html#%3Cinit%3E(java.lang.String)">ConstantExpressionNode(String)</a></span> - Constructor for class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">ConstantExpressionNode</a></dt>
<dd>
<div class="block">Convenience constructor that takes a string and converts it to a double
 before storing the value.</div>
</dd>
<dt><a href="../bio/singa/simulation/features/ContainmentRegion.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">ContainmentRegion</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/ContainmentRegion.html#%3Cinit%3E(bio.singa.simulation.model.sections.CellRegion)">ContainmentRegion(CellRegion)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/ContainmentRegion.html" title="class in bio.singa.simulation.features">ContainmentRegion</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/ContainmentRegion.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">ContainmentRegion.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/EntityRegistry.html#contains(java.lang.String)">contains(String)</a></span> - Static method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/EntityRegistry.html" title="class in bio.singa.simulation.entities">EntityRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html#containsAtomWithName(java.lang.String)">containsAtomWithName(String)</a></span> - Method in interface bio.singa.structure.model.interfaces.<a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html" title="interface in bio.singa.structure.model.interfaces">LeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/mmtf/MmtfLeafSubstructure.html#containsAtomWithName(java.lang.String)">containsAtomWithName(String)</a></span> - Method in class bio.singa.structure.model.mmtf.<a href="../bio/singa/structure/model/mmtf/MmtfLeafSubstructure.html" title="class in bio.singa.structure.model.mmtf">MmtfLeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html#containsAtomWithName(java.lang.String)">containsAtomWithName(String)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html" title="class in bio.singa.structure.model.oak">OakLeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html#containsEdge(java.lang.Object)">containsEdge(Object)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="class in bio.singa.mathematics.graphs.grid">AbstractGridGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#containsEdge(java.lang.Object)">containsEdge(Object)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graph.html#containsEdge(java.lang.Object)">containsEdge(Object)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graph.html" title="interface in bio.singa.mathematics.graphs.model">Graph</a></dt>
<dd>
<div class="block">Returns true, if the graph contains the edge and false otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#containsEdge(java.util.function.Predicate)">containsEdge(Predicate&lt;EdgeType&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#containsEntity(bio.singa.simulation.entities.ChemicalEntity)">containsEntity(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html#containsEntity(bio.singa.simulation.model.sections.CellSubsection,bio.singa.simulation.entities.ChemicalEntity)">containsEntity(CellSubsection, ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html" title="class in bio.singa.simulation.model.sections">ConcentrationContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html#containsEntity(bio.singa.simulation.model.sections.CellTopology,bio.singa.simulation.entities.ChemicalEntity)">containsEntity(CellTopology, ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html" title="class in bio.singa.simulation.model.sections">ConcentrationContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/AminoAcidFamily.html#containsExpectedAtoms(java.util.List,bio.singa.structure.model.families.AminoAcidFamily)">containsExpectedAtoms(List&lt;Atom&gt;, AminoAcidFamily)</a></span> - Method in enum bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families">AminoAcidFamily</a></dt>
<dd>
<div class="block">Returns true if the set of AtomFilter contains only Atom names, that can occur in the given residue type.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html#containsHiddenEntity(bio.singa.simulation.model.sections.CellTopology,bio.singa.simulation.entities.ChemicalEntity)">containsHiddenEntity(CellTopology, ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html" title="class in bio.singa.simulation.model.sections">ConcentrationContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html#containsNode(java.lang.Object)">containsNode(Object)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="class in bio.singa.mathematics.graphs.grid">AbstractGridGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#containsNode(java.lang.Object)">containsNode(Object)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graph.html#containsNode(java.lang.Object)">containsNode(Object)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graph.html" title="interface in bio.singa.mathematics.graphs.model">Graph</a></dt>
<dd>
<div class="block">Returns true, if the graph contains the node and false otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#containsNode(java.util.function.Predicate)">containsNode(Predicate&lt;NodeType&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>
<div class="block">Returns true if the graph contains any node that matches the predicate.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html#containsNode(NodeType)">containsNode(NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html" title="class in bio.singa.mathematics.graphs.model">AbstractEdge</a></dt>
<dd>
<div class="block">Returns true only if the node is source or target of this edge.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Edge.html#containsNode(NodeType)">containsNode(NodeType)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Edge.html" title="interface in bio.singa.mathematics.graphs.model">Edge</a></dt>
<dd>
<div class="block">Returns true if a node with the given identifier is either source or target of the node.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html#containsNode(T)">containsNode(T)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTree</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/GenericGraph.html#containsNodeWithContent(ContentType)">containsNodeWithContent(ContentType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/GenericGraph.html" title="class in bio.singa.mathematics.graphs.model">GenericGraph</a></dt>
<dd>
<div class="block">Returns true if a node with the given content is already present.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/DeltaBehavior.html#containsSubstrates(bio.singa.simulation.model.sections.ConcentrationContainer)">containsSubstrates(ConcentrationContainer)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/deltas/DeltaBehavior.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.deltas">DeltaBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/topology/grids/hexagonal/HexagonalGrid.html#containsValue(java.lang.Object)">containsValue(Object)</a></span> - Method in class bio.singa.mathematics.topology.grids.hexagonal.<a href="../bio/singa/mathematics/topology/grids/hexagonal/HexagonalGrid.html" title="class in bio.singa.mathematics.topology.grids.hexagonal">HexagonalGrid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/topology/grids/rectangular/RectangularGrid.html#containsValue(java.lang.Object)">containsValue(Object)</a></span> - Method in class bio.singa.mathematics.topology.grids.rectangular.<a href="../bio/singa/mathematics/topology/grids/rectangular/RectangularGrid.html" title="class in bio.singa.mathematics.topology.grids.rectangular">RectangularGrid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Polygons.html#containsVector(bio.singa.mathematics.geometry.model.Polygon,bio.singa.mathematics.vectors.Vector2D)">containsVector(Polygon, Vector2D)</a></span> - Static method in class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Polygons.html" title="class in bio.singa.mathematics.geometry.faces">Polygons</a></dt>
<dd>
<div class="block">Returns true, if the vector is inside the polygon, or false if not.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/model/Polygon.html#containsVector(bio.singa.mathematics.vectors.Vector2D)">containsVector(Vector2D)</a></span> - Method in interface bio.singa.mathematics.geometry.model.<a href="../bio/singa/mathematics/geometry/model/Polygon.html" title="interface in bio.singa.mathematics.geometry.model">Polygon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/ContentConverter.html" title="interface in bio.singa.structure.parser.pdb.structures.iterators.converters"><span class="typeNameLink">ContentConverter</span></a>&lt;<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/ContentConverter.html" title="type parameter in ContentConverter">ContentType</a>,&#8203;<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/ContentConverter.html" title="type parameter in ContentConverter">ConversionType</a>&gt; - Interface in <a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/package-summary.html">bio.singa.structure.parser.pdb.structures.iterators.converters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/TitleToken.html#CONTINUATION">CONTINUATION</a></span> - bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/TitleToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">TitleToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/GridCoordinateConverter.html#convert(int)">convert(int)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/GridCoordinateConverter.html" title="class in bio.singa.mathematics.graphs.grid">GridCoordinateConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/GridCoordinateConverter.html#convert(bio.singa.mathematics.vectors.Vector2D)">convert(Vector2D)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/GridCoordinateConverter.html" title="class in bio.singa.mathematics.graphs.grid">GridCoordinateConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/modules/ModuleTableConverter.html#convert(bio.singa.simulation.model.simulation.Simulation)">convert(Simulation)</a></span> - Static method in class bio.singa.simulation.export.modules.<a href="../bio/singa/simulation/export/modules/ModuleTableConverter.html" title="class in bio.singa.simulation.export.modules">ModuleTableConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/reactiongraph/ReactionGraphConverter.html#convert(bio.singa.simulation.model.simulation.Simulation)">convert(Simulation)</a></span> - Static method in class bio.singa.simulation.export.reactiongraph.<a href="../bio/singa/simulation/export/reactiongraph/ReactionGraphConverter.html" title="class in bio.singa.simulation.export.reactiongraph">ReactionGraphConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/ContentConverter.html#convert(ContentType)">convert(ContentType)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/ContentConverter.html" title="interface in bio.singa.structure.parser.pdb.structures.iterators.converters">ContentConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/FileToPathConverter.html#convert(java.io.File)">convert(File)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/FileToPathConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters">FileToPathConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/FileLocationToPathConverter.html#convert(java.lang.String)">convert(String)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/FileLocationToPathConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters">FileLocationToPathConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/IdentifierToMmtfBytesConverter.html#convert(java.lang.String)">convert(String)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/IdentifierToMmtfBytesConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters">IdentifierToMmtfBytesConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/LocalPDBToPathConverter.html#convert(java.lang.String)">convert(String)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/LocalPDBToPathConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters">LocalPDBToPathConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/PDBIdentifierToURLConverter.html#convert(java.lang.String)">convert(String)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/PDBIdentifierToURLConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters">PDBIdentifierToURLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/URLToPDBLinesConverter.html#convert(java.net.URL)">convert(URL)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/URLToPDBLinesConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters">URLToPDBLinesConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/PathToMmtfBytesConverter.html#convert(java.nio.file.Path)">convert(Path)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/PathToMmtfBytesConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters">PathToMmtfBytesConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/PathToObjectConverter.html#convert(java.nio.file.Path)">convert(Path)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/PathToObjectConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters">PathToObjectConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/PathToPDBLinesConverter.html#convert(java.nio.file.Path)">convert(Path)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/PathToPDBLinesConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters">PathToPDBLinesConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/units/UnitRegistry.html#convert(javax.measure.Quantity)">convert(Quantity&lt;QuantityType&gt;)</a></span> - Static method in class bio.singa.features.units.<a href="../bio/singa/features/units/UnitRegistry.html" title="class in bio.singa.features.units">UnitRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/IdentityConverter.html#convert(Type)">convert(Type)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.converters.<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/IdentityConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters">IdentityConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/parameters/Environment.html#convertSimulationToSystemScale(double)">convertSimulationToSystemScale(double)</a></span> - Static method in class bio.singa.features.parameters.<a href="../bio/singa/features/parameters/Environment.html" title="class in bio.singa.features.parameters">Environment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/parameters/Environment.html#convertSystemToSimulationScale(javax.measure.Quantity)">convertSystemToSimulationScale(Quantity&lt;Length&gt;)</a></span> - Static method in class bio.singa.features.parameters.<a href="../bio/singa/features/parameters/Environment.html" title="class in bio.singa.features.parameters">Environment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#convertToSpheres(bio.singa.structure.model.interfaces.AtomContainer)">convertToSpheres(AtomContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graphs.html#convertTreeToGraph(bio.singa.mathematics.graphs.trees.BinaryTree)">convertTreeToGraph(BinaryTree&lt;ContentType&gt;)</a></span> - Static method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graphs.html" title="class in bio.singa.mathematics.graphs.model">Graphs</a></dt>
<dd>
<div class="block">Converts a <a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees"><code>BinaryTree</code></a> to a <a href="../bio/singa/mathematics/graphs/model/GenericGraph.html" title="class in bio.singa.mathematics.graphs.model"><code>GenericGraph</code></a>.</div>
</dd>
<dt><a href="../bio/singa/mathematics/algorithms/geometry/ConvexHull.html" title="class in bio.singa.mathematics.algorithms.geometry"><span class="typeNameLink">ConvexHull</span></a> - Class in <a href="../bio/singa/mathematics/algorithms/geometry/package-summary.html">bio.singa.mathematics.algorithms.geometry</a></dt>
<dd>
<div class="block">Based on Graham Scan</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/geometry/ConvexHull.html#%3Cinit%3E(java.util.Collection)">ConvexHull(Collection&lt;Vector2D&gt;)</a></span> - Constructor for class bio.singa.mathematics.algorithms.geometry.<a href="../bio/singa/mathematics/algorithms/geometry/ConvexHull.html" title="class in bio.singa.mathematics.algorithms.geometry">ConvexHull</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#COPPER">COPPER</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html#copy()">copy()</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTreeNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graphs.html#copy(GraphType)">copy(GraphType)</a></span> - Static method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graphs.html" title="class in bio.singa.mathematics.graphs.model">Graphs</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/model/Correlation.html" title="interface in bio.singa.features.model"><span class="typeNameLink">Correlation</span></a>&lt;<a href="../bio/singa/features/model/Correlation.html" title="type parameter in Correlation">FeatureType</a> extends <a href="../bio/singa/features/model/Feature.html" title="interface in bio.singa.features.model">Feature</a>&lt;?&gt;&gt; - Interface in <a href="../bio/singa/features/model/package-summary.html">bio.singa.features.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/VectorMetricProvider.html#COSINE_SIMILARITY">COSINE_SIMILARITY</a></span> - Static variable in class bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/VectorMetricProvider.html" title="class in bio.singa.mathematics.metrics.model">VectorMetricProvider</a></dt>
<dd>
<div class="block">The cosine similarity is a similarity measure not a distance measure.</div>
</dd>
<dt><a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html" title="class in bio.singa.mathematics.metrics.implementations"><span class="typeNameLink">CosineSimilarity</span></a>&lt;<a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html" title="type parameter in CosineSimilarity">VectorDimension</a> extends <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a>&gt; - Class in <a href="../bio/singa/mathematics/metrics/implementations/package-summary.html">bio.singa.mathematics.metrics.implementations</a></dt>
<dd>
<div class="block">The resulting similarity ranges from −1 meaning exactly opposite, to 1
 meaning exactly the same, with 0 indicating orthogonality (decorrelation),
 and in-between values indicating intermediate similarity or dissimilarity.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html#%3Cinit%3E()">CosineSimilarity()</a></span> - Constructor for class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html" title="class in bio.singa.mathematics.metrics.implementations">CosineSimilarity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/BitVectors.html#count(java.util.List)">count(List&lt;BitVector&gt;)</a></span> - Static method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/BitVectors.html" title="class in bio.singa.mathematics.vectors">BitVectors</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html#countAtomsOfElement(bio.singa.chemistry.model.MoleculeGraph,bio.singa.chemistry.model.elements.Element)">countAtomsOfElement(MoleculeGraph, Element)</a></span> - Static method in class bio.singa.structure.model.molecules.<a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html" title="class in bio.singa.structure.model.molecules">MoleculeGraphs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/AbstractStructureIterator.html#counter">counter</a></span> - Variable in class bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/AbstractStructureIterator.html" title="class in bio.singa.structure.parser.pdb.structures.iterators">AbstractStructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#countParts(bio.singa.simulation.entities.ChemicalEntity)">countParts(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/model/CovalentBondType.html" title="enum in bio.singa.chemistry.model"><span class="typeNameLink">CovalentBondType</span></a> - Enum in <a href="../bio/singa/chemistry/model/package-summary.html">bio.singa.chemistry.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/MaterialProvider.html#crateMaterialFromColor(javafx.scene.paint.Color)">crateMaterialFromColor(Color)</a></span> - Static method in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/MaterialProvider.html" title="class in bio.singa.javafx.viewer">MaterialProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.html#create()">create()</a></span> - Static method in class bio.singa.mathematics.algorithms.clustering.<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.html" title="class in bio.singa.mathematics.algorithms.clustering">AffinityPropagation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.html#create()">create()</a></span> - Static method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntityBuilder.html" title="class in bio.singa.simulation.entities">ComplexEntityBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/organelles/OrganelleTypes.html#create()">create()</a></span> - Method in enum bio.singa.simulation.model.agents.organelles.<a href="../bio/singa/simulation/model/agents/organelles/OrganelleTypes.html" title="enum in bio.singa.simulation.model.agents.organelles">OrganelleTypes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.html#create()">create()</a></span> - Static method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.html" title="class in bio.singa.simulation.model.concentrations">ConcentrationBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.html#create()">create()</a></span> - Static method in class bio.singa.simulation.trajectories.flat.<a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.html" title="class in bio.singa.simulation.trajectories.flat">FlatUpdateRecorder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.html#create()">create()</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.affinity.<a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.html" title="class in bio.singa.structure.algorithms.superimposition.affinity">AffinityAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.html#create()">create()</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder</a></dt>
<dd>
<div class="block">Creates a new instance of the actual <a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><code>ConsensusBuilder.Builder</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.html#create()">create()</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DBuilder</a></dt>
<dd>
<div class="block">Creates a new instance of the actual <a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d"><code>Fit3DBuilder.Builder</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.html#create()">create()</a></span> - Static method in class bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.html" title="class in bio.singa.structure.parser.pfam">PfamParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/reactions/RateConstant.html#create(double)">create(double)</a></span> - Static method in class bio.singa.chemistry.features.reactions.<a href="../bio/singa/chemistry/features/reactions/RateConstant.html" title="class in bio.singa.chemistry.features.reactions">RateConstant</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.html#create(bio.singa.simulation.model.simulation.Simulation)">create(Simulation)</a></span> - Static method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.html" title="class in bio.singa.simulation.model.concentrations">ConcentrationBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/Protein.html#create(java.lang.String)">create(String)</a></span> - Static method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/Protein.html" title="class in bio.singa.chemistry.model">Protein</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/SmallMolecule.html#create(java.lang.String)">create(String)</a></span> - Static method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/SmallMolecule.html" title="class in bio.singa.chemistry.model">SmallMolecule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/SimpleEntity.html#create(java.lang.String)">create(String)</a></span> - Static method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/SimpleEntity.html" title="class in bio.singa.simulation.entities">SimpleEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/formatter/ConcentrationFormatter.html#create(javax.measure.Unit)">create(Unit&lt;MolarConcentration&gt;)</a></span> - Static method in class bio.singa.features.formatter.<a href="../bio/singa/features/formatter/ConcentrationFormatter.html" title="class in bio.singa.features.formatter">ConcentrationFormatter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.Setting.html#CREATE_EDGES">CREATE_EDGES</a></span> - bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.Setting.html" title="enum in bio.singa.structure.parser.pdb.structures">StructureParserOptions.Setting</a></dt>
<dd>
<div class="block">Create Edges in between leaf substructures (default).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html#createAminoAcidFromAtoms(bio.singa.features.identifiers.LeafIdentifier,bio.singa.structure.model.families.AminoAcidFamily,java.util.Map,bio.singa.structure.parser.pdb.structures.StructureParserOptions)">createAminoAcidFromAtoms(LeafIdentifier, AminoAcidFamily, Map&lt;String, OakAtom&gt;, StructureParserOptions)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html" title="class in bio.singa.structure.model.oak">LeafSubstructureFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html#createAminoAcidFromAtoms(bio.singa.features.identifiers.LeafIdentifier,bio.singa.structure.model.families.AminoAcidFamily,java.util.Set,bio.singa.structure.parser.pdb.structures.StructureParserOptions)">createAminoAcidFromAtoms(LeafIdentifier, AminoAcidFamily, Set&lt;OakAtom&gt;, StructureParserOptions)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html" title="class in bio.singa.structure.model.oak">LeafSubstructureFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html#createCell(int,bio.singa.mathematics.algorithms.voronoi.model.SiteEvent)">createCell(int, SiteEvent)</a></span> - Method in class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">VoronoiDiagram</a></dt>
<dd>
<div class="block">Creates a new cell, adds it to the diagram and returns it.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/volumelike/VolumeLikeAgentFactory.html#createCellCortex(bio.singa.simulation.model.agents.surfacelike.Membrane,bio.singa.mathematics.topology.grids.rectangular.MooreRectangularDirection,javax.measure.Quantity,javax.measure.Quantity)">createCellCortex(Membrane, MooreRectangularDirection, Quantity&lt;Length&gt;, Quantity&lt;Length&gt;)</a></span> - Static method in class bio.singa.simulation.model.agents.volumelike.<a href="../bio/singa/simulation/model/agents/volumelike/VolumeLikeAgentFactory.html" title="class in bio.singa.simulation.model.agents.volumelike">VolumeLikeAgentFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/Recorders.html#createDirectories(java.nio.file.Path)">createDirectories(Path)</a></span> - Static method in class bio.singa.simulation.trajectories.<a href="../bio/singa/simulation/trajectories/Recorders.html" title="class in bio.singa.simulation.trajectories">Recorders</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#createEdges(boolean)">createEdges(boolean)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option true will connect the molecule with bonds.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/Recorders.html#createFile(java.nio.file.Path,java.lang.String)">createFile(Path, String)</a></span> - Static method in class bio.singa.simulation.trajectories.<a href="../bio/singa/simulation/trajectories/Recorders.html" title="class in bio.singa.simulation.trajectories">Recorders</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html#createFromChainList(java.nio.file.Path,java.lang.String,bio.singa.structure.parser.pdb.structures.LocalPDBRepository)">createFromChainList(Path, String, LocalPDBRepository)</a></span> - Static method in interface bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html" title="interface in bio.singa.structure.parser.pdb.structures.iterators">StructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html#createFromChainList(java.nio.file.Path,java.lang.String,bio.singa.structure.parser.pdb.structures.SourceLocation)">createFromChainList(Path, String, SourceLocation)</a></span> - Static method in interface bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html" title="interface in bio.singa.structure.parser.pdb.structures.iterators">StructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html#createFromFiles(java.util.List,bio.singa.structure.parser.pdb.structures.SourceLocation)">createFromFiles(List&lt;File&gt;, SourceLocation)</a></span> - Static method in interface bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html" title="interface in bio.singa.structure.parser.pdb.structures.iterators">StructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html#createFromIdentifiers(java.util.List,bio.singa.structure.parser.pdb.structures.SourceLocation,bio.singa.structure.parser.pdb.structures.LocalPDBRepository)">createFromIdentifiers(List&lt;String&gt;, SourceLocation, LocalPDBRepository)</a></span> - Static method in interface bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html" title="interface in bio.singa.structure.parser.pdb.structures.iterators">StructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html#createFromLocalPdb(bio.singa.structure.parser.pdb.structures.LocalPDBRepository)">createFromLocalPdb(LocalPDBRepository)</a></span> - Static method in interface bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html" title="interface in bio.singa.structure.parser.pdb.structures.iterators">StructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html#createFromLocalPdb(bio.singa.structure.parser.pdb.structures.LocalPDBRepository,int)">createFromLocalPdb(LocalPDBRepository, int)</a></span> - Static method in interface bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html" title="interface in bio.singa.structure.parser.pdb.structures.iterators">StructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html#createFromLocations(java.util.List)">createFromLocations(List&lt;String&gt;)</a></span> - Static method in interface bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html" title="interface in bio.singa.structure.parser.pdb.structures.iterators">StructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html#createFromPaths(java.util.List,bio.singa.structure.parser.pdb.structures.SourceLocation)">createFromPaths(List&lt;Path&gt;, SourceLocation)</a></span> - Static method in interface bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureIterator.html" title="interface in bio.singa.structure.parser.pdb.structures.iterators">StructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/ReactionNetworkGenerator.html#createInitialEntities(java.util.Map)">createInitialEntities(Map&lt;ChemicalEntity, Set&lt;BindingSite&gt;&gt;)</a></span> - Method in class bio.singa.simulation.reactions.<a href="../bio/singa/simulation/reactions/ReactionNetworkGenerator.html" title="class in bio.singa.simulation.reactions">ReactionNetworkGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html#createLeafSubstructure(bio.singa.features.identifiers.LeafIdentifier,bio.singa.structure.model.families.StructuralFamily)">createLeafSubstructure(LeafIdentifier, StructuralFamily)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html" title="class in bio.singa.structure.model.oak">LeafSubstructureFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html#createLeafSubstructure(bio.singa.features.identifiers.LeafIdentifier,bio.singa.structure.model.families.StructuralFamily,java.util.Set)">createLeafSubstructure(LeafIdentifier, StructuralFamily, Set&lt;OakAtom&gt;)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html" title="class in bio.singa.structure.model.oak">LeafSubstructureFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.GeneralReactionBuilder.html#createModule(bio.singa.simulation.model.simulation.Simulation)">createModule(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.GeneralReactionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.GeneralReactionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.DiffusionBuilder.html#createModule(bio.singa.simulation.model.simulation.Simulation)">createModule(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.DiffusionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">Diffusion.DiffusionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html#createModule(bio.singa.simulation.model.simulation.Simulation)">createModule(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/LateralMembraneDiffusion.LateralMembraneDiffusionBuilder.html#createModule(bio.singa.simulation.model.simulation.Simulation)">createModule(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/LateralMembraneDiffusion.LateralMembraneDiffusionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">LateralMembraneDiffusion.LateralMembraneDiffusionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/MembraneDiffusion.MembraneDiffusionBuilder.html#createModule(bio.singa.simulation.model.simulation.Simulation)">createModule(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/MembraneDiffusion.MembraneDiffusionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">MembraneDiffusion.MembraneDiffusionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.PorousDiffusionBuilder.html#createModule(bio.singa.simulation.model.simulation.Simulation)">createModule(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.PorousDiffusionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">PorousDiffusion.PorousDiffusionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/SingleFileChannelMembraneTransport.SingleFileChannelMembraneTransportBuilder.html#createModule(bio.singa.simulation.model.simulation.Simulation)">createModule(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/SingleFileChannelMembraneTransport.SingleFileChannelMembraneTransportBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">SingleFileChannelMembraneTransport.SingleFileChannelMembraneTransportBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ModuleBuilder.html#createModule(bio.singa.simulation.model.simulation.Simulation)">createModule(Simulation)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ModuleBuilder.html" title="interface in bio.singa.simulation.model.modules.concentration">ModuleBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html#createMoleculeGraphFromStructure(bio.singa.structure.model.oak.OakLeafSubstructure)">createMoleculeGraphFromStructure(OakLeafSubstructure&lt;?&gt;)</a></span> - Static method in class bio.singa.structure.model.molecules.<a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html" title="class in bio.singa.structure.model.molecules">MoleculeGraphs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/BindingSite.html#createNamed(java.lang.String)">createNamed(String)</a></span> - Static method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/BindingSite.html" title="class in bio.singa.simulation.entities">BindingSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/RegularVector.html#createNewVector(double%5B%5D,java.lang.Class)">createNewVector(double[], Class&lt;VectorDimension&gt;)</a></span> - Static method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/RegularVector.html" title="class in bio.singa.mathematics.vectors">RegularVector</a></dt>
<dd>
<div class="block">A factory method for the creation of a new specific vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html#createNucleotideFromAtoms(bio.singa.features.identifiers.LeafIdentifier,bio.singa.structure.model.families.NucleotideFamily,java.util.Map,bio.singa.structure.parser.pdb.structures.StructureParserOptions)">createNucleotideFromAtoms(LeafIdentifier, NucleotideFamily, Map&lt;String, OakAtom&gt;, StructureParserOptions)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html" title="class in bio.singa.structure.model.oak">LeafSubstructureFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AttachedMotor.Builder.html#createObject(bio.singa.simulation.entities.ChemicalEntity)">createObject(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AttachedMotor.Builder.html" title="class in bio.singa.simulation.features">AttachedMotor.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/BoostMediatingEntity.Builder.html#createObject(bio.singa.simulation.entities.ChemicalEntity)">createObject(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/BoostMediatingEntity.Builder.html" title="class in bio.singa.simulation.features">BoostMediatingEntity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/Cargo.Builder.html#createObject(bio.singa.simulation.entities.ChemicalEntity)">createObject(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/Cargo.Builder.html" title="class in bio.singa.simulation.features">Cargo.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/Transporter.Builder.html#createObject(bio.singa.simulation.entities.ChemicalEntity)">createObject(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/Transporter.Builder.html" title="class in bio.singa.simulation.features">Transporter.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AffectedRegion.Builder.html#createObject(bio.singa.simulation.model.sections.CellRegion)">createObject(CellRegion)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AffectedRegion.Builder.html" title="class in bio.singa.simulation.features">AffectedRegion.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/ContainmentRegion.Builder.html#createObject(bio.singa.simulation.model.sections.CellRegion)">createObject(CellRegion)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/ContainmentRegion.Builder.html" title="class in bio.singa.simulation.features">ContainmentRegion.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AffectedSection.Builder.html#createObject(bio.singa.simulation.model.sections.CellSubsection)">createObject(CellSubsection)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AffectedSection.Builder.html" title="class in bio.singa.simulation.features">AffectedSection.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractFeature.Builder.html#createObject(FeatureContent)">createObject(FeatureContent)</a></span> - Method in class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractFeature.Builder.html" title="class in bio.singa.features.model">AbstractFeature.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html#createObject(java.lang.String)">createObject(String)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/Protein.Builder.html#createObject(java.lang.String)">createObject(String)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/Protein.Builder.html" title="class in bio.singa.chemistry.model">Protein.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/SmallMolecule.Builder.html#createObject(java.lang.String)">createObject(String)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/SmallMolecule.Builder.html" title="class in bio.singa.chemistry.model">SmallMolecule.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AppliedVesicleState.Builder.html#createObject(java.lang.String)">createObject(String)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AppliedVesicleState.Builder.html" title="class in bio.singa.simulation.features">AppliedVesicleState.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AttachedFilament.Builder.html#createObject(java.lang.String)">createObject(String)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AttachedFilament.Builder.html" title="class in bio.singa.simulation.features">AttachedFilament.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/MotorPullDirection.Builder.html#createObject(java.lang.String)">createObject(String)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/MotorPullDirection.Builder.html" title="class in bio.singa.simulation.features">MotorPullDirection.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/RequiredVesicleState.Builder.html#createObject(java.lang.String)">createObject(String)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/RequiredVesicleState.Builder.html" title="class in bio.singa.simulation.features">RequiredVesicleState.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/Cargoes.Builder.html#createObject(java.util.List)">createObject(List&lt;ChemicalEntity&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/Cargoes.Builder.html" title="class in bio.singa.simulation.features">Cargoes.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/MatchingQSnares.Builder.html#createObject(java.util.List)">createObject(List&lt;ChemicalEntity&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/MatchingQSnares.Builder.html" title="class in bio.singa.simulation.features">MatchingQSnares.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/MatchingRSnares.Builder.html#createObject(java.util.List)">createObject(List&lt;ChemicalEntity&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/MatchingRSnares.Builder.html" title="class in bio.singa.simulation.features">MatchingRSnares.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/PrimaryCargoes.Builder.html#createObject(java.util.List)">createObject(List&lt;ChemicalEntity&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/PrimaryCargoes.Builder.html" title="class in bio.singa.simulation.features">PrimaryCargoes.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/ScalingEntities.Builder.html#createObject(java.util.List)">createObject(List&lt;ChemicalEntity&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/ScalingEntities.Builder.html" title="class in bio.singa.simulation.features">ScalingEntities.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/Solutes.Builder.html#createObject(java.util.List)">createObject(List&lt;ChemicalEntity&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/Solutes.Builder.html" title="class in bio.singa.simulation.features">Solutes.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/InitialConcentrations.Builder.html#createObject(java.util.List)">createObject(List&lt;InitialConcentration&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/InitialConcentrations.Builder.html" title="class in bio.singa.simulation.features">InitialConcentrations.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/BlackListVesicleStates.Builder.html#createObject(java.util.List)">createObject(List&lt;String&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/BlackListVesicleStates.Builder.html" title="class in bio.singa.simulation.features">BlackListVesicleStates.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/WhiteListVesicleStates.Builder.html#createObject(java.util.List)">createObject(List&lt;String&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/WhiteListVesicleStates.Builder.html" title="class in bio.singa.simulation.features">WhiteListVesicleStates.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/CargoAdditionRate.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;FirstOrderRate&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/CargoAdditionRate.Builder.html" title="class in bio.singa.simulation.features">CargoAdditionRate.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/diffusivity/ModifiedDiffusivity.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Diffusivity&gt;)</a></span> - Method in class bio.singa.chemistry.features.diffusivity.<a href="../bio/singa/chemistry/features/diffusivity/ModifiedDiffusivity.Builder.html" title="class in bio.singa.chemistry.features.diffusivity">ModifiedDiffusivity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/diffusivity/OriginalDiffusivity.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Diffusivity&gt;)</a></span> - Method in class bio.singa.chemistry.features.diffusivity.<a href="../bio/singa/chemistry/features/diffusivity/OriginalDiffusivity.Builder.html" title="class in bio.singa.chemistry.features.diffusivity">OriginalDiffusivity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/ConcentrationDiffusivity.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Diffusivity&gt;)</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/ConcentrationDiffusivity.Builder.html" title="class in bio.singa.features.quantities">ConcentrationDiffusivity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MembraneDiffusivity.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Diffusivity&gt;)</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MembraneDiffusivity.Builder.html" title="class in bio.singa.features.quantities">MembraneDiffusivity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/PixelDiffusivity.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Diffusivity&gt;)</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/PixelDiffusivity.Builder.html" title="class in bio.singa.features.quantities">PixelDiffusivity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/EndocytosisCheckpointConcentration.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;MolarConcentration&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/EndocytosisCheckpointConcentration.Builder.html" title="class in bio.singa.simulation.features">EndocytosisCheckpointConcentration.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarMass.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;MolarMass&gt;)</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarMass.Builder.html" title="class in bio.singa.features.quantities">MolarMass.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/PitFormationRate.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;SpawnRate&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/PitFormationRate.Builder.html" title="class in bio.singa.simulation.features">PitFormationRate.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/Ratio.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Dimensionless&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/Ratio.Builder.html" title="class in bio.singa.simulation.features">Ratio.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/SNAREFusionPairs.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Dimensionless&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/SNAREFusionPairs.Builder.html" title="class in bio.singa.simulation.features">SNAREFusionPairs.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/radius/Radius.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Length&gt;)</a></span> - Method in class bio.singa.chemistry.features.radius.<a href="../bio/singa/chemistry/features/radius/Radius.Builder.html" title="class in bio.singa.chemistry.features.radius">Radius.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AttachmentDistance.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Length&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AttachmentDistance.Builder.html" title="class in bio.singa.simulation.features">AttachmentDistance.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/MembraneTickness.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Length&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/MembraneTickness.Builder.html" title="class in bio.singa.simulation.features">MembraneTickness.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/VesicleRadius.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Length&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/VesicleRadius.Builder.html" title="class in bio.singa.simulation.features">VesicleRadius.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/ActinBoostVelocity.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Speed&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/ActinBoostVelocity.Builder.html" title="class in bio.singa.simulation.features">ActinBoostVelocity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/MotorMovementVelocity.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Speed&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/MotorMovementVelocity.Builder.html" title="class in bio.singa.simulation.features">MotorMovementVelocity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/EndocytosisCheckpointTime.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Time&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/EndocytosisCheckpointTime.Builder.html" title="class in bio.singa.simulation.features">EndocytosisCheckpointTime.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/FusionTime.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Time&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/FusionTime.Builder.html" title="class in bio.singa.simulation.features">FusionTime.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/MaturationTime.Builder.html#createObject(javax.measure.Quantity)">createObject(Quantity&lt;Time&gt;)</a></span> - Method in class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/MaturationTime.Builder.html" title="class in bio.singa.simulation.features">MaturationTime.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionElement.html#createOneToOne(bio.singa.simulation.entities.ComplexEntity,bio.singa.simulation.entities.ComplexEntity)">createOneToOne(ComplexEntity, ComplexEntity)</a></span> - Static method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionElement.html" title="class in bio.singa.simulation.reactions.reactors">ReactionElement</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionElement.html#createOneToTwo(bio.singa.simulation.entities.ComplexEntity,bio.singa.simulation.entities.ComplexEntity,bio.singa.simulation.entities.ComplexEntity)">createOneToTwo(ComplexEntity, ComplexEntity, ComplexEntity)</a></span> - Static method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionElement.html" title="class in bio.singa.simulation.reactions.reactors">ReactionElement</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Nodes.html#createRandomlyPlacedNode(int)">createRandomlyPlacedNode(int)</a></span> - Static method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Nodes.html" title="class in bio.singa.mathematics.graphs.model">Nodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Nodes.html#createRandomlyPlacedNode(int,bio.singa.mathematics.geometry.faces.Rectangle)">createRandomlyPlacedNode(int, Rectangle)</a></span> - Static method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Nodes.html" title="class in bio.singa.mathematics.graphs.model">Nodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonGraphs.html#createRectangularAutomatonGraph(int,int)">createRectangularAutomatonGraph(int, int)</a></span> - Static method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonGraphs.html" title="class in bio.singa.simulation.model.graphs">AutomatonGraphs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/FastMatrices.html#createRegularMatrix(double%5B%5D%5B%5D)">createRegularMatrix(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/FastMatrices.html" title="class in bio.singa.mathematics.matrices">FastMatrices</a></dt>
<dd>
<div class="block">Creates a regular matrix without checking if the values are well formed or symmetric.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationSchemeFactory.html#createRepresentationScheme(bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationSchemeType)">createRepresentationScheme(RepresentationSchemeType)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.fit3d.representations.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationSchemeFactory.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.representations">RepresentationSchemeFactory</a></dt>
<dd>
<div class="block">Creates a <a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationScheme.html" title="interface in bio.singa.structure.algorithms.superimposition.fit3d.representations"><code>RepresentationScheme</code></a> for the given <a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationSchemeType.html" title="enum in bio.singa.structure.algorithms.superimposition.fit3d.representations"><code>RepresentationSchemeType</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/FastMatrices.html#createSquareMatrix(double%5B%5D%5B%5D)">createSquareMatrix(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/FastMatrices.html" title="class in bio.singa.mathematics.matrices">FastMatrices</a></dt>
<dd>
<div class="block">Creates a symmetric matrix without checking if the array has the right format or has to be compacted.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/FastMatrices.html#createSymmetricMatrix(double%5B%5D%5B%5D)">createSymmetricMatrix(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/FastMatrices.html" title="class in bio.singa.mathematics.matrices">FastMatrices</a></dt>
<dd>
<div class="block">Creates a symmetric matrix without checking if the array has the right format or has to be compacted.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/surfacelike/GridMembraneBuilder.html#createTopology()">createTopology()</a></span> - Method in class bio.singa.simulation.model.agents.surfacelike.<a href="../bio/singa/simulation/model/agents/surfacelike/GridMembraneBuilder.html" title="class in bio.singa.simulation.model.agents.surfacelike">GridMembraneBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionElement.html#createTwoToOne(bio.singa.simulation.entities.ComplexEntity,bio.singa.simulation.entities.ComplexEntity,bio.singa.simulation.entities.ComplexEntity)">createTwoToOne(ComplexEntity, ComplexEntity, ComplexEntity)</a></span> - Static method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionElement.html" title="class in bio.singa.simulation.reactions.reactors">ReactionElement</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AminoAcids.html#createVirtualCBAtom(bio.singa.structure.model.interfaces.AminoAcid)">createVirtualCBAtom(AminoAcid)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AminoAcids.html" title="class in bio.singa.structure.model.oak">AminoAcids</a></dt>
<dd>
<div class="block">This creates a virtual beta carbon for glycine by superimposing alanine.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#crossProduct(bio.singa.mathematics.vectors.Vector3D)">crossProduct(Vector3D)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>
<div class="block">Given two linearly independent vectors, the cross product, is a vector
 that is perpendicular to both and therefore normal to the plane
 containing them</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DMatch.html#CSV_HEADER">CSV_HEADER</a></span> - Static variable in class bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DMatch.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DMatch</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/geometry/bodies/Cube.html" title="class in bio.singa.mathematics.geometry.bodies"><span class="typeNameLink">Cube</span></a> - Class in <a href="../bio/singa/mathematics/geometry/bodies/package-summary.html">bio.singa.mathematics.geometry.bodies</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/bodies/Cube.html#%3Cinit%3E(bio.singa.mathematics.vectors.Vector3D,double)">Cube(Vector3D, double)</a></span> - Constructor for class bio.singa.mathematics.geometry.bodies.<a href="../bio/singa/mathematics/geometry/bodies/Cube.html" title="class in bio.singa.mathematics.geometry.bodies">Cube</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/topology/grids/cube/CubeCoordinate.html" title="class in bio.singa.mathematics.topology.grids.cube"><span class="typeNameLink">CubeCoordinate</span></a> - Class in <a href="../bio/singa/mathematics/topology/grids/cube/package-summary.html">bio.singa.mathematics.topology.grids.cube</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/topology/grids/cube/CubeCoordinate.html#%3Cinit%3E(int,int,int)">CubeCoordinate(int, int, int)</a></span> - Constructor for class bio.singa.mathematics.topology.grids.cube.<a href="../bio/singa/mathematics/topology/grids/cube/CubeCoordinate.html" title="class in bio.singa.mathematics.topology.grids.cube">CubeCoordinate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/StructureViewer.html#cubes">cubes</a></span> - Static variable in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/StructureViewer.html" title="class in bio.singa.javafx.viewer">StructureViewer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarVolume.html#CUBIC_ANGSTROEM_PER_MOLE">CUBIC_ANGSTROEM_PER_MOLE</a></span> - Static variable in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarVolume.html" title="class in bio.singa.features.quantities">MolarVolume</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/OsmoticPermeability.html#CUBIC_CENTIMETRE_PER_SECOND">CUBIC_CENTIMETRE_PER_SECOND</a></span> - Static variable in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/OsmoticPermeability.html" title="class in bio.singa.chemistry.features.permeability">OsmoticPermeability</a></dt>
<dd>
<div class="block">Unit most commonly used to describe osmotic permeability.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarVolume.html#CUBIC_METRE_PER_MOLE">CUBIC_METRE_PER_MOLE</a></span> - Static variable in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarVolume.html" title="class in bio.singa.features.quantities">MolarVolume</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#CURIUM">CURIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/AbstractStructureIterator.html#currentSource">currentSource</a></span> - Variable in class bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/AbstractStructureIterator.html" title="class in bio.singa.structure.parser.pdb.structures.iterators">AbstractStructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.Builder.html#cutoffScore(double)">cutoffScore(double)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DBuilder.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.SiteParameterConfigurationStep.html#cutoffScore(double)">cutoffScore(double)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.SiteParameterConfigurationStep.html" title="interface in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DBuilder.SiteParameterConfigurationStep</a></dt>
<dd>
<div class="block">The cutoff score that should be used when extending the site alignment.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/AminoAcidFamily.html#CYSTEINE">CYSTEINE</a></span> - bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families">AminoAcidFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CYSTEINE_ATOM_NAMES">CYSTEINE_ATOM_NAMES</a></span> - Static variable in enum bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/NucleotideFamily.html#CYTIDINE">CYTIDINE</a></span> - bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/NucleotideFamily.html" title="enum in bio.singa.structure.model.families">NucleotideFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellSubsections.html#CYTOPLASM">CYTOPLASM</a></span> - Static variable in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellSubsections.html" title="class in bio.singa.simulation.model.sections">CellSubsections</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegions.html#CYTOPLASM_REGION">CYTOPLASM_REGION</a></span> - Static variable in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegions.html" title="class in bio.singa.simulation.model.sections">CellRegions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CZ">CZ</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CZ2">CZ2</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CZ3">CZ3</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Y</a>&nbsp;<a href="index-26.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes</a>&nbsp;<a href="../allpackages-index.html">All&nbsp;Packages</a></div>
</main>
<footer role="contentinfo">
<nav role="navigation">
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a id="navbar.bottom">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.bottom" title="Skip navigation links">Skip navigation links</a></div>
<a id="navbar.bottom.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../allclasses.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
</div>
<a id="skip.navbar.bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</nav>
</footer>
</body>
</html>
