<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (1.8.0_181) on Tue Jul 24 14:29:07 CEST 2018 -->
<title>C-Index</title>
<meta name="date" content="2018-07-24">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
</head>
<body>
<script type="text/javascript"><!--
    try {
        if (location.href.indexOf('is-external=true') == -1) {
            parent.document.title="C-Index";
        }
    }
    catch(err) {
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar.top">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.top" title="Skip navigation links">Skip navigation links</a></div>
<a name="navbar.top.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="index-2.html">Prev Letter</a></li>
<li><a href="index-4.html">Next Letter</a></li>
</ul>
<ul class="navList">
<li><a href="../index.html?index-files/index-3.html" target="_top">Frames</a></li>
<li><a href="index-3.html" target="_top">No&nbsp;Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../allclasses-noframe.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip.navbar.top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="contentContainer"><a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Y</a>&nbsp;<a href="index-26.html">Z</a>&nbsp;<a name="I:C">
<!--   -->
</a>
<h2 class="title">C</h2>
<dl>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#CADMIUM">CADMIUM</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#CALCIUM">CALCIUM</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/diffusivity/Diffusivity.html#calculate-javax.measure.Quantity-">calculate(Quantity&lt;Length&gt;)</a></span> - Static method in class bio.singa.chemistry.features.diffusivity.<a href="../bio/singa/chemistry/features/diffusivity/Diffusivity.html" title="class in bio.singa.chemistry.features.diffusivity">Diffusivity</a></dt>
<dd>
<div class="block">The diffusivity can be calculated according to the Stokesâ€“Einstein equation:
 D = (k_B * T) / (6 * pi * nu * radius)
 k_B is the <a href="../bio/singa/features/quantities/NaturalConstants.html#BOLTZMANN_CONSTANT"><code>NaturalConstants.BOLTZMANN_CONSTANT</code></a> (in (N * m) / K),
 T is the Temperature (in K),
 nu is the dynamic viscosity (in (N * s) / m^2 ) and,</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/geometry/SphereVolumeEstimaton.html#calculate--">calculate()</a></span> - Method in class bio.singa.mathematics.algorithms.geometry.<a href="../bio/singa/mathematics/algorithms/geometry/SphereVolumeEstimaton.html" title="class in bio.singa.mathematics.algorithms.geometry">SphereVolumeEstimaton</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/forces/Force.html#calculateAcceleration-bio.singa.mathematics.vectors.Vector2D-bio.singa.mathematics.vectors.Vector2D-">calculateAcceleration(Vector2D, Vector2D)</a></span> - Method in interface bio.singa.mathematics.forces.<a href="../bio/singa/mathematics/forces/Force.html" title="interface in bio.singa.mathematics.forces">Force</a></dt>
<dd>
<div class="block">The default method to calculate the acceleration of a node, depending on
 the distance between the given vectors.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateAtomDistanceMatrix-bio.singa.structure.model.interfaces.AtomContainer-">calculateAtomDistanceMatrix(AtomContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the distance matrix of the given <a href="../bio/singa/structure/model/interfaces/Atom.html" title="interface in bio.singa.structure.model.interfaces"><code>Atom</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateAtomDistanceMatrix-java.util.List-">calculateAtomDistanceMatrix(List&lt;Atom&gt;)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the distance matrix of the given <a href="../bio/singa/structure/model/interfaces/Atom.html" title="interface in bio.singa.structure.model.interfaces"><code>Atom</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateClosestDistance-java.util.List-SubType-">calculateClosestDistance(List&lt;SubType&gt;, SubType)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>
<div class="block">Returns the closest element in the given target list to the reference vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/Matrices.html#calculateCovarianceMatrix-bio.singa.mathematics.matrices.Matrix-bio.singa.mathematics.matrices.Matrix-">calculateCovarianceMatrix(Matrix, Matrix)</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/Matrices.html" title="class in bio.singa.mathematics.matrices">Matrices</a></dt>
<dd>
<div class="block">calculates the covariance matrix between to matrices in respect to matrix A
 cov(A) = B'*A</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/Reaction.html#calculateDeltas-bio.singa.simulation.model.sections.ConcentrationContainer-">calculateDeltas(ConcentrationContainer)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/Reaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">Reaction</a></dt>
<dd>
<div class="block">Calculates all deltas for all reactants for the reaction.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/EndocytosisActinBoost.html#calculateDisplacement-bio.singa.simulation.model.modules.displacement.Vesicle-">calculateDisplacement(Vesicle)</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/EndocytosisActinBoost.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">EndocytosisActinBoost</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleDiffusion.html#calculateDisplacement-bio.singa.simulation.model.modules.displacement.Vesicle-">calculateDisplacement(Vesicle)</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleDiffusion.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">VesicleDiffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleTransport.html#calculateDisplacement-bio.singa.simulation.model.modules.displacement.Vesicle-">calculateDisplacement(Vesicle)</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleTransport.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">VesicleTransport</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/AngularDistance.html#calculateDistance-VectorDimension-VectorDimension-">calculateDistance(VectorDimension, VectorDimension)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/AngularDistance.html" title="class in bio.singa.mathematics.metrics.implementations">AngularDistance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html#calculateDistance-VectorDimension-VectorDimension-">calculateDistance(VectorDimension, VectorDimension)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html" title="class in bio.singa.mathematics.metrics.implementations">CosineSimilarity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/EuclideanDistance.html#calculateDistance-VectorType-VectorType-">calculateDistance(VectorType, VectorType)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/EuclideanDistance.html" title="class in bio.singa.mathematics.metrics.implementations">EuclideanDistance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/JaccardMetric.html#calculateDistance-java.util.Collection-java.util.Collection-">calculateDistance(Collection&lt;Type&gt;, Collection&lt;Type&gt;)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/JaccardMetric.html" title="class in bio.singa.mathematics.metrics.implementations">JaccardMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/MinkowskiMetric.html#calculateDistance-VectorType-VectorType-">calculateDistance(VectorType, VectorType)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/MinkowskiMetric.html" title="class in bio.singa.mathematics.metrics.implementations">MinkowskiMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/ShortestPathMetric.html#calculateDistance-bio.singa.mathematics.graphs.model.Node-bio.singa.mathematics.graphs.model.Node-">calculateDistance(Node&lt;?, ?, ?&gt;, Node&lt;?, ?, ?&gt;)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/ShortestPathMetric.html" title="class in bio.singa.mathematics.metrics.implementations">ShortestPathMetric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/SquaredEuclideanDistance.html#calculateDistance-VectorType-VectorType-">calculateDistance(VectorType, VectorType)</a></span> - Method in class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/SquaredEuclideanDistance.html" title="class in bio.singa.mathematics.metrics.implementations">SquaredEuclideanDistance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateDistance-MetrizableType-MetrizableType-">calculateDistance(MetrizableType, MetrizableType)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>
<div class="block">Calculates the distance between both Objects.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateDistanceMatrix-bio.singa.structure.model.interfaces.LeafSubstructureContainer-">calculateDistanceMatrix(LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the distance matrix of the given <a href="../bio/singa/structure/model/interfaces/LeafSubstructureContainer.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructureContainer</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateDistancesPairwise-java.util.List-">calculateDistancesPairwise(List&lt;SubType&gt;)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>
<div class="block">Calculates the pairwise distance of all elements in the given list.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateDistancesPairwise-java.util.List-java.util.function.Function-">calculateDistancesPairwise(List&lt;LabelType&gt;, Function&lt;LabelType, SubType&gt;)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/Metric.html#calculateDistancesToReference-java.util.List-SubType-">calculateDistancesToReference(List&lt;SubType&gt;, SubType)</a></span> - Method in interface bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/Metric.html" title="interface in bio.singa.mathematics.metrics.model">Metric</a></dt>
<dd>
<div class="block">Calculates the distance for each Vector in the given target list to the reference vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateExtent-bio.singa.structure.model.interfaces.LeafSubstructureContainer-">calculateExtent(LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the maximal extent of the given <a href="../bio/singa/structure/model/interfaces/LeafSubstructureContainer.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructureContainer</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/forces/AttractiveForce.html#calculateForce-double-">calculateForce(double)</a></span> - Method in class bio.singa.mathematics.forces.<a href="../bio/singa/mathematics/forces/AttractiveForce.html" title="class in bio.singa.mathematics.forces">AttractiveForce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/forces/Force.html#calculateForce-double-">calculateForce(double)</a></span> - Method in interface bio.singa.mathematics.forces.<a href="../bio/singa/mathematics/forces/Force.html" title="interface in bio.singa.mathematics.forces">Force</a></dt>
<dd>
<div class="block">Calculates the strength of the applied force.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/forces/RepulsiveForce.html#calculateForce-double-">calculateForce(double)</a></span> - Method in class bio.singa.mathematics.forces.<a href="../bio/singa/mathematics/forces/RepulsiveForce.html" title="class in bio.singa.mathematics.forces">RepulsiveForce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/geometry/ConvexHull.html#calculateHullFor-java.util.Collection-">calculateHullFor(Collection&lt;Vector2D&gt;)</a></span> - Static method in class bio.singa.mathematics.algorithms.geometry.<a href="../bio/singa/mathematics/algorithms/geometry/ConvexHull.html" title="class in bio.singa.mathematics.algorithms.geometry">ConvexHull</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition-java.util.List-java.util.List-">calculateIdealSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition-java.util.List-java.util.List-java.util.function.Predicate-">calculateIdealSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, Predicate&lt;Atom&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition-java.util.List-java.util.List-bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme-">calculateIdealSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, RepresentationScheme)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.model.interfaces.LeafSubstructureContainer-">calculateIdealSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.model.interfaces.LeafSubstructureContainer-java.util.function.Predicate-">calculateIdealSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, Predicate&lt;Atom&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateIdealSubstructureSuperimposition-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme-">calculateIdealSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, RepresentationScheme)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#calculateIdealVectorSuperimposition-java.util.List-java.util.List-">calculateIdealVectorSuperimposition(List&lt;Vector&gt;, List&lt;Vector&gt;)</a></span> - Static method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition-java.util.List-java.util.List-bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix-boolean-">calculateKuhnMunkresSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition-java.util.List-java.util.List-java.util.function.Predicate-bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix-boolean-">calculateKuhnMunkresSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, Predicate&lt;Atom&gt;, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition-java.util.List-java.util.List-bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme-bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix-boolean-">calculateKuhnMunkresSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, RepresentationScheme, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix-boolean-">calculateKuhnMunkresSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.model.interfaces.LeafSubstructureContainer-java.util.function.Predicate-bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix-boolean-">calculateKuhnMunkresSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, Predicate&lt;Atom&gt;, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateKuhnMunkresSubstructureSuperimposition-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme-bio.singa.structure.algorithms.superimposition.scores.SubstitutionMatrix-boolean-">calculateKuhnMunkresSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, RepresentationScheme, SubstitutionMatrix, boolean)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#calculateKuhnMunkresSuperimposition-java.util.List-java.util.List-">calculateKuhnMunkresSuperimposition(List&lt;Vector&gt;, List&lt;Vector&gt;)</a></span> - Static method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>
<div class="block">Finds the superimposition with the minimal distances between pairs of <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors"><code>Vector</code></a>s by using <a href="../bio/singa/mathematics/algorithms/optimization/KuhnMunkres.html" title="class in bio.singa.mathematics.algorithms.optimization"><code>KuhnMunkres</code></a> optimization.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/estimations/OctanolWaterPartition.html#calculateOctanolWaterPartitionCoefficient-bio.singa.structure.model.molecules.MoleculeGraph-bio.singa.chemistry.estimations.OctanolWaterPartition.Method-">calculateOctanolWaterPartitionCoefficient(MoleculeGraph, OctanolWaterPartition.Method)</a></span> - Static method in class bio.singa.chemistry.estimations.<a href="../bio/singa/chemistry/estimations/OctanolWaterPartition.html" title="class in bio.singa.chemistry.estimations">OctanolWaterPartition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/FofanovEstimation.html#calculatePvalues-java.util.List-">calculatePvalues(List&lt;Fit3DMatch&gt;)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.statistics.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/FofanovEstimation.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.statistics">FofanovEstimation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/StarkEstimation.html#calculatePvalues-java.util.List-">calculatePvalues(List&lt;Fit3DMatch&gt;)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.statistics.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/StarkEstimation.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.statistics">StarkEstimation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/StatisticalModel.html#calculatePvalues-java.util.List-">calculatePvalues(List&lt;Fit3DMatch&gt;)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.fit3d.statistics.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/StatisticalModel.html" title="interface in bio.singa.structure.algorithms.superimposition.fit3d.statistics">StatisticalModel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/matrix/QRDecomposition.html#calculateQRDecomposition-bio.singa.mathematics.matrices.Matrix-">calculateQRDecomposition(Matrix)</a></span> - Static method in class bio.singa.mathematics.algorithms.matrix.<a href="../bio/singa/mathematics/algorithms/matrix/QRDecomposition.html" title="class in bio.singa.mathematics.algorithms.matrix">QRDecomposition</a></dt>
<dd>
<div class="block">Calculates the QR decomposition of the given matrix, using the modified Graham-Schmidt algorithm.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotifs.html#calculateRmsdMatrix-java.util.List-boolean-">calculateRmsdMatrix(List&lt;StructuralMotif&gt;, boolean)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotifs.html" title="class in bio.singa.structure.model.oak">StructuralMotifs</a></dt>
<dd>
<div class="block">Performs a superimposition of given <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a>s using arbitrary atoms and returns the RMSD distance
 matrix between all elements.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotifs.html#calculateRmsdMatrix-java.util.List-java.util.function.Predicate-boolean-">calculateRmsdMatrix(List&lt;StructuralMotif&gt;, Predicate&lt;Atom&gt;, boolean)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotifs.html" title="class in bio.singa.structure.model.oak">StructuralMotifs</a></dt>
<dd>
<div class="block">Performs a superimposition of given <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a>s using the specified <a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak"><code>StructuralEntityFilter.AtomFilter</code></a> and returns the RMSD distance matrix between all elements.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/matrix/LUDecomposition.html#calculateRowEchelonMatrix-bio.singa.mathematics.matrices.Matrix-">calculateRowEchelonMatrix(Matrix)</a></span> - Static method in class bio.singa.mathematics.algorithms.matrix.<a href="../bio/singa/mathematics/algorithms/matrix/LUDecomposition.html" title="class in bio.singa.mathematics.algorithms.matrix">LUDecomposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/Line.html#calculateSlope-bio.singa.mathematics.vectors.Vector2D-bio.singa.mathematics.vectors.Vector2D-">calculateSlope(Vector2D, Vector2D)</a></span> - Static method in class bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/Line.html" title="class in bio.singa.mathematics.geometry.edges">Line</a></dt>
<dd>
<div class="block">Gets the slope or gradient of the line, calculated by</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateSquaredDistanceMatrix-bio.singa.structure.model.interfaces.LeafSubstructureContainer-">calculateSquaredDistanceMatrix(LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the squared distance matrix of the given <a href="../bio/singa/structure/model/interfaces/LeafSubstructureContainer.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructureContainer</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#calculateSquaredExtent-bio.singa.structure.model.interfaces.LeafSubstructureContainer-">calculateSquaredExtent(LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns the maximal squared extent of the given <a href="../bio/singa/structure/model/interfaces/LeafSubstructureContainer.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructureContainer</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/FragmentSubstructureSuperimposer.html#calculateSubstructureSuperimposition-java.util.List-java.util.List-">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/FragmentSubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">FragmentSubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/FragmentSubstructureSuperimposer.html#calculateSubstructureSuperimposition-java.util.List-java.util.List-java.util.function.BiFunction-java.util.function.BiFunction-">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, BiFunction&lt;MoleculeAtom, MoleculeAtom, Boolean&gt;, BiFunction&lt;MoleculeBond, MoleculeBond, Boolean&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/FragmentSubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">FragmentSubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition-java.util.List-java.util.List-">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition-java.util.List-java.util.List-java.util.function.Predicate-">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, Predicate&lt;Atom&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition-java.util.List-java.util.List-bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme-">calculateSubstructureSuperimposition(List&lt;LeafSubstructure&lt;?&gt;&gt;, List&lt;LeafSubstructure&lt;?&gt;&gt;, RepresentationScheme)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.model.interfaces.LeafSubstructureContainer-">calculateSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.model.interfaces.LeafSubstructureContainer-java.util.function.Predicate-">calculateSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, Predicate&lt;Atom&gt;)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSubstructureSuperimposition-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.model.interfaces.LeafSubstructureContainer-bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationScheme-">calculateSubstructureSuperimposition(LeafSubstructureContainer, LeafSubstructureContainer, RepresentationScheme)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html#calculateSuperimposition--">calculateSuperimposition()</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorQuaternionSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#calculateSuperimposition--">calculateSuperimposition()</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#calculateSuperimposition--">calculateSuperimposition()</a></span> - Method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>
<div class="block">Finds the superimposition for a list of <a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructure</code></a> according to their input order</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/DisplacementDeltaManager.html#calculateTotalDisplacement--">calculateTotalDisplacement()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/DisplacementDeltaManager.html" title="class in bio.singa.simulation.model.modules.displacement">DisplacementDeltaManager</a></dt>
<dd>
<div class="block">Calculates the total displacement resulting from the potential deltas and sets the result as the next position.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/Vesicle.html#calculateTotalDisplacement--">calculateTotalDisplacement()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/Vesicle.html" title="class in bio.singa.simulation.model.modules.displacement">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html#calculateUpdates--">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationBasedModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/DisplacementBasedModule.html#calculateUpdates--">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/DisplacementBasedModule.html" title="class in bio.singa.simulation.model.modules.displacement">DisplacementBasedModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/ClathrinMediatedEndocytosis.html#calculateUpdates--">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/ClathrinMediatedEndocytosis.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">ClathrinMediatedEndocytosis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/EndocytosisActinBoost.html#calculateUpdates--">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/EndocytosisActinBoost.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">EndocytosisActinBoost</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleAttachment.html#calculateUpdates--">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleAttachment.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">VesicleAttachment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleDiffusion.html#calculateUpdates--">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleDiffusion.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">VesicleDiffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleFusion.html#calculateUpdates--">calculateUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/VesicleFusion.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">VesicleFusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/UpdateModule.html#calculateUpdates--">calculateUpdates()</a></span> - Method in interface bio.singa.simulation.model.modules.<a href="../bio/singa/simulation/model/modules/UpdateModule.html" title="interface in bio.singa.simulation.model.modules">UpdateModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html#calculateVectorSuperimposition-java.util.List-java.util.List-">calculateVectorSuperimposition(List&lt;Vector3D&gt;, List&lt;Vector3D&gt;)</a></span> - Static method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorQuaternionSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#calculateVectorSuperimposition-java.util.List-java.util.List-">calculateVectorSuperimposition(List&lt;Vector&gt;, List&lt;Vector&gt;)</a></span> - Static method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/DynamicReaction.html#calculateVelocity-bio.singa.simulation.model.sections.ConcentrationContainer-">calculateVelocity(ConcentrationContainer)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/DynamicReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">DynamicReaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/MichaelisMentenReaction.html#calculateVelocity-bio.singa.simulation.model.sections.ConcentrationContainer-">calculateVelocity(ConcentrationContainer)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/MichaelisMentenReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">MichaelisMentenReaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/NthOrderReaction.html#calculateVelocity-bio.singa.simulation.model.sections.ConcentrationContainer-">calculateVelocity(ConcentrationContainer)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/NthOrderReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">NthOrderReaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/Reaction.html#calculateVelocity-bio.singa.simulation.model.sections.ConcentrationContainer-">calculateVelocity(ConcentrationContainer)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/Reaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">Reaction</a></dt>
<dd>
<div class="block">Calculates the reaction velocity, depending on the kinetic law.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ReversibleReaction.html#calculateVelocity-bio.singa.simulation.model.sections.ConcentrationContainer-">calculateVelocity(ConcentrationContainer)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/ReversibleReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">ReversibleReaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/reactants/KineticLaw.html#calculateVelocity-bio.singa.simulation.model.sections.ConcentrationContainer-">calculateVelocity(ConcentrationContainer)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.reactants.<a href="../bio/singa/simulation/model/modules/concentration/reactants/KineticLaw.html" title="class in bio.singa.simulation.model.modules.concentration.reactants">KineticLaw</a></dt>
<dd>
<div class="block">Calculates the velocity of the reaction based on the entities in the concentration container.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/Line.html#calculateYIntercept-bio.singa.mathematics.vectors.Vector2D-double-">calculateYIntercept(Vector2D, double)</a></span> - Static method in class bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/Line.html" title="class in bio.singa.mathematics.geometry.edges">Line</a></dt>
<dd>
<div class="block">Gets the y-intercept of the line, calculated by</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#CALIFORNIUM">CALIFORNIUM</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationManager.html#call--">call()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationManager.html" title="class in bio.singa.simulation.model.simulation">SimulationManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/Featureable.html#canBeFeaturedWith-java.lang.Class-">canBeFeaturedWith(Class&lt;FeatureType&gt;)</a></span> - Method in interface bio.singa.features.model.<a href="../bio/singa/features/model/Featureable.html" title="interface in bio.singa.features.model">Featureable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#canBePlacedIn-bio.singa.mathematics.geometry.faces.Rectangle-">canBePlacedIn(Rectangle)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>
<div class="block">Validates, if this vector can be placed in the given <a href="../bio/singa/mathematics/geometry/faces/Rectangle.html" title="class in bio.singa.mathematics.geometry.faces"><code>Rectangle</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html#candidate">candidate</a></span> - Variable in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposer.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/MaterialProvider.html#CARBON">CARBON</a></span> - Static variable in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/MaterialProvider.html" title="class in bio.singa.javafx.viewer">MaterialProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#CARBON">CARBON</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/MembraneDiffusion.CargoStep.html#cargo-bio.singa.chemistry.entities.ChemicalEntity-">cargo(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/MembraneDiffusion.CargoStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations">MembraneDiffusion.CargoStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/MembraneDiffusion.MembraneDiffusionBuilder.html#cargo-bio.singa.chemistry.entities.ChemicalEntity-">cargo(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/MembraneDiffusion.MembraneDiffusionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">MembraneDiffusion.MembraneDiffusionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/SingleFileChannelMembraneTransport.CargoStep.html#cargo-bio.singa.chemistry.entities.ChemicalEntity-">cargo(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/SingleFileChannelMembraneTransport.CargoStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations">SingleFileChannelMembraneTransport.CargoStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/SingleFileChannelMembraneTransport.SingleFileChannelMembraneTransportBuilder.html#cargo-bio.singa.chemistry.entities.ChemicalEntity-">cargo(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/SingleFileChannelMembraneTransport.SingleFileChannelMembraneTransportBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">SingleFileChannelMembraneTransport.SingleFileChannelMembraneTransportBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/reactants/CatalyticReactant.html" title="class in bio.singa.simulation.model.modules.concentration.reactants"><span class="typeNameLink">CatalyticReactant</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/reactants/package-summary.html">bio.singa.simulation.model.modules.concentration.reactants</a></dt>
<dd>
<div class="block"><code>CatalyticReactant</code>s are <a href="../bio/singa/simulation/model/modules/concentration/reactants/Reactant.html" title="class in bio.singa.simulation.model.modules.concentration.reactants"><code>Reactant</code></a>s that are not consumed or produced during Reactions, but
 influence its velocity (reaction rate).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/reactants/CatalyticReactant.html#CatalyticReactant-bio.singa.chemistry.entities.ChemicalEntity-bio.singa.simulation.model.modules.concentration.reactants.ReactantRole-">CatalyticReactant(ChemicalEntity, ReactantRole)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.reactants.<a href="../bio/singa/simulation/model/modules/concentration/reactants/CatalyticReactant.html" title="class in bio.singa.simulation.model.modules.concentration.reactants">CatalyticReactant</a></dt>
<dd>
<div class="block">Creates a new catalytic reactant.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/sections/CellRegion.html" title="class in bio.singa.simulation.model.sections"><span class="typeNameLink">CellRegion</span></a> - Class in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegion.html#CellRegion-java.lang.String-">CellRegion(String)</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegion.html" title="class in bio.singa.simulation.model.sections">CellRegion</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections"><span class="typeNameLink">CellSubsection</span></a> - Class in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>
<div class="block">A cell subsection organizes the contents of a <a href="../bio/singa/simulation/model/simulation/Updatable.html" title="interface in bio.singa.simulation.model.simulation"><code>Updatable</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellSubsection.html#CellSubsection-java.lang.String-">CellSubsection(String)</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections">CellSubsection</a></dt>
<dd>
<div class="block">Creates a new cell subsection with the given identifier and dynamic preferred concentration unit.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellSubsection.html#CellSubsection-java.lang.String-boolean-">CellSubsection(String, boolean)</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections">CellSubsection</a></dt>
<dd>
<div class="block">Creates a new cell subsection.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/sections/CellTopology.html" title="enum in bio.singa.simulation.model.sections"><span class="typeNameLink">CellTopology</span></a> - Enum in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>
<div class="block">The cell topology defines how the subsections are spatially organized.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html#center--">center()</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorQuaternionSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#center--">center()</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html#CENTIMETRE_PER_SECOND">CENTIMETRE_PER_SECOND</a></span> - Static variable in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html" title="class in bio.singa.chemistry.features.permeability">MembranePermeability</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/CentroidRepresentationScheme.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.representations"><span class="typeNameLink">CentroidRepresentationScheme</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/package-summary.html">bio.singa.structure.algorithms.superimposition.fit3d.representations</a></dt>
<dd>
<div class="block">An implementation to represent a given <a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructure</code></a> by its centroid <b>EXCLUDING</b> hydrogen.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/CentroidRepresentationScheme.html#CentroidRepresentationScheme--">CentroidRepresentationScheme()</a></span> - Constructor for class bio.singa.structure.algorithms.superimposition.fit3d.representations.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/CentroidRepresentationScheme.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.representations">CentroidRepresentationScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#CERIUM">CERIUM</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#CESIUM">CESIUM</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/model/interfaces/Chain.html" title="interface in bio.singa.structure.model.interfaces"><span class="typeNameLink">Chain</span></a> - Interface in <a href="../bio/singa/structure/model/interfaces/package-summary.html">bio.singa.structure.model.interfaces</a></dt>
<dd>
<div class="block"><a href="../bio/singa/structure/model/interfaces/Chain.html" title="interface in bio.singa.structure.model.interfaces"><code>Chain</code></a>s that represent one continuous macro molecule (most often the biggest macro molecules and its ligands
 are collected in one model).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.MultiChainStep.html#chain-java.lang.String-">chain(String)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.MultiChainStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.MultiChainStep</a></dt>
<dd>
<div class="block">If only a single chain should be parsed, choose it here.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.ChainStep.html#chain-java.lang.String-">chain(String)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.ChainStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureSelector.ChainStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.Selector.html#chain-java.lang.String-">chain(String)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.Selector.html" title="class in bio.singa.structure.parser.pdb.structures">StructureSelector.Selector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/TerminatorTokens.html#CHAIN_TERMINATOR">CHAIN_TERMINATOR</a></span> - Static variable in class bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/TerminatorTokens.html" title="class in bio.singa.structure.parser.pdb.structures.tokens">TerminatorTokens</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/model/interfaces/ChainContainer.html" title="interface in bio.singa.structure.model.interfaces"><span class="typeNameLink">ChainContainer</span></a> - Interface in <a href="../bio/singa/structure/model/interfaces/package-summary.html">bio.singa.structure.model.interfaces</a></dt>
<dd>
<div class="block">Everything that contains <a href="../bio/singa/structure/model/interfaces/Chain.html" title="interface in bio.singa.structure.model.interfaces"><code>Chain</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.ChainFilter.html#ChainFilter--">ChainFilter()</a></span> - Constructor for class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.ChainFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.ChainFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.SingleChainStep.html#chainIdentifier-java.lang.String-">chainIdentifier(String)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.SingleChainStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.SingleChainStep</a></dt>
<dd>
<div class="block">If only a single chain should be parsed, choose it here.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.AdditionalLocalSourceStep.html#chainList-java.nio.file.Path-">chainList(Path)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.AdditionalLocalSourceStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.AdditionalLocalSourceStep</a></dt>
<dd>
<div class="block">Reads the provided chainIdentifier list from a file.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.AdditionalLocalSourceStep.html#chainList-java.nio.file.Path-java.lang.String-">chainList(Path, String)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.AdditionalLocalSourceStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.AdditionalLocalSourceStep</a></dt>
<dd>
<div class="block">Reads the provided chainIdentifier list from a file.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html#chainList-java.nio.file.Path-">chainList(Path)</a></span> - Method in class bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html" title="class in bio.singa.structure.parser.pfam">PfamParser.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html#chainList-java.nio.file.Path-java.lang.String-">chainList(Path, String)</a></span> - Method in class bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html" title="class in bio.singa.structure.parser.pfam">PfamParser.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.ChainStep.html#chainList-java.nio.file.Path-">chainList(Path)</a></span> - Method in interface bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.ChainStep.html" title="interface in bio.singa.structure.parser.pfam">PfamParser.ChainStep</a></dt>
<dd>
<div class="block">Parse all chains annotated in Pfam and listed in the given chain list.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.ChainStep.html#chainList-java.nio.file.Path-java.lang.String-">chainList(Path, String)</a></span> - Method in interface bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.ChainStep.html" title="interface in bio.singa.structure.parser.pfam">PfamParser.ChainStep</a></dt>
<dd>
<div class="block">Parse all chains annotated in Pfam and listed in the given chain list with the custom separator.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html#chains--">chains()</a></span> - Method in class bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html" title="class in bio.singa.structure.parser.pfam">PfamParser.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.DomainStep.html#chains--">chains()</a></span> - Method in interface bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.DomainStep.html" title="interface in bio.singa.structure.parser.pfam">PfamParser.DomainStep</a></dt>
<dd>
<div class="block">Parse all chains carrying one Pfam domain with the desired Pfam-ID.</div>
</dd>
<dt><a href="../bio/singa/structure/parser/pdb/structures/tokens/ChainTerminatorToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens"><span class="typeNameLink">ChainTerminatorToken</span></a> - Enum in <a href="../bio/singa/structure/parser/pdb/structures/tokens/package-summary.html">bio.singa.structure.parser.pdb.structures.tokens</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html#characters-char:A-int-int-">characters(char[], int, int)</a></span> - Method in class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/uniprot/UniProtContentHandler.html#characters-char:A-int-int-">characters(char[], int, int)</a></span> - Method in class bio.singa.chemistry.features.databases.uniprot.<a href="../bio/singa/chemistry/features/databases/uniprot/UniProtContentHandler.html" title="class in bio.singa.chemistry.features.databases.uniprot">UniProtContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/parser/GraphMLContentHandler.html#characters-char:A-int-int-">characters(char[], int, int)</a></span> - Method in class bio.singa.mathematics.graphs.parser.<a href="../bio/singa/mathematics/graphs/parser/GraphMLContentHandler.html" title="class in bio.singa.mathematics.graphs.parser">GraphMLContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/parser/ena/ENAContentHandler.html#characters-char:A-int-int-">characters(char[], int, int)</a></span> - Method in class bio.singa.sequence.parser.ena.<a href="../bio/singa/sequence/parser/ena/ENAContentHandler.html" title="class in bio.singa.sequence.parser.ena">ENAContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/PlipContentHandler.html#characters-char:A-int-int-">characters(char[], int, int)</a></span> - Method in class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/PlipContentHandler.html" title="class in bio.singa.structure.parser.plip">PlipContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/sifts/ResidueMapContentHandler.html#characters-char:A-int-int-">characters(char[], int, int)</a></span> - Method in class bio.singa.structure.parser.sifts.<a href="../bio/singa/structure/parser/sifts/ResidueMapContentHandler.html" title="class in bio.singa.structure.parser.sifts">ResidueMapContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html" title="class in bio.singa.chemistry.features.databases.chebi"><span class="typeNameLink">ChEBIContentHandler</span></a> - Class in <a href="../bio/singa/chemistry/features/databases/chebi/package-summary.html">bio.singa.chemistry.features.databases.chebi</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html#ChEBIContentHandler--">ChEBIContentHandler()</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html#ChEBIContentHandler-java.lang.String-">ChEBIContentHandler(String)</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIDatabase.html" title="class in bio.singa.chemistry.features.databases.chebi"><span class="typeNameLink">ChEBIDatabase</span></a> - Class in <a href="../bio/singa/chemistry/features/databases/chebi/package-summary.html">bio.singa.chemistry.features.databases.chebi</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIDatabase.html#ChEBIDatabase--">ChEBIDatabase()</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIDatabase.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIDatabase</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/identifiers/ChEBIIdentifier.html" title="class in bio.singa.features.identifiers"><span class="typeNameLink">ChEBIIdentifier</span></a> - Class in <a href="../bio/singa/features/identifiers/package-summary.html">bio.singa.features.identifiers</a></dt>
<dd>
<div class="block">This identifier is used by the <a href="https://www.ebi.ac.uk/chebi/">ChEBI Database</a> to identify small molecules.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/ChEBIIdentifier.html#ChEBIIdentifier-java.lang.String-">ChEBIIdentifier(String)</a></span> - Constructor for class bio.singa.features.identifiers.<a href="../bio/singa/features/identifiers/ChEBIIdentifier.html" title="class in bio.singa.features.identifiers">ChEBIIdentifier</a></dt>
<dd>
<div class="block">Creates a new identifier.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/ChEBIIdentifier.html#ChEBIIdentifier-java.lang.String-bio.singa.features.model.FeatureOrigin-">ChEBIIdentifier(String, FeatureOrigin)</a></span> - Constructor for class bio.singa.features.identifiers.<a href="../bio/singa/features/identifiers/ChEBIIdentifier.html" title="class in bio.singa.features.identifiers">ChEBIIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIImageService.html" title="class in bio.singa.chemistry.features.databases.chebi"><span class="typeNameLink">ChEBIImageService</span></a> - Class in <a href="../bio/singa/chemistry/features/databases/chebi/package-summary.html">bio.singa.chemistry.features.databases.chebi</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIImageService.html#ChEBIImageService-java.lang.String-">ChEBIImageService(String)</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIImageService.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIImageService</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIParserService.html" title="class in bio.singa.chemistry.features.databases.chebi"><span class="typeNameLink">ChEBIParserService</span></a> - Class in <a href="../bio/singa/chemistry/features/databases/chebi/package-summary.html">bio.singa.chemistry.features.databases.chebi</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIParserService.html#ChEBIParserService-bio.singa.features.identifiers.ChEBIIdentifier-">ChEBIParserService(ChEBIIdentifier)</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIParserService.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIParserService</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIParserService.html#ChEBIParserService-bio.singa.features.identifiers.ChEBIIdentifier-java.lang.String-">ChEBIParserService(ChEBIIdentifier, String)</a></span> - Constructor for class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIParserService.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIParserService</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/identifiers/ChEBIToInChIKey.html" title="class in bio.singa.chemistry.features.identifiers"><span class="typeNameLink">ChEBIToInChIKey</span></a> - Class in <a href="../bio/singa/chemistry/features/identifiers/package-summary.html">bio.singa.chemistry.features.identifiers</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/identifiers/ChEBIToInChIKey.html#ChEBIToInChIKey--">ChEBIToInChIKey()</a></span> - Constructor for class bio.singa.chemistry.features.identifiers.<a href="../bio/singa/chemistry/features/identifiers/ChEBIToInChIKey.html" title="class in bio.singa.chemistry.features.identifiers">ChEBIToInChIKey</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/VectorMetricProvider.html#CHEBYCHEV_METRIC">CHEBYCHEV_METRIC</a></span> - Static variable in class bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/VectorMetricProvider.html" title="class in bio.singa.mathematics.metrics.model">VectorMetricProvider</a></dt>
<dd>
<div class="block">The Chebyshev metric (also known as Tchebychev metric or maximum metric) is a metric defined on a vector space
 where the distance between two vectors is the greatest of their differences along any coordinate dimension.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/model/IdentifierPatternRegistry.html#check-java.lang.Class-bio.singa.features.identifiers.model.Identifier-">check(Class&lt;IdentifierType&gt;, Identifier)</a></span> - Static method in class bio.singa.features.identifiers.model.<a href="../bio/singa/features/identifiers/model/IdentifierPatternRegistry.html" title="class in bio.singa.features.identifiers.model">IdentifierPatternRegistry</a></dt>
<dd>
<div class="block">Returns true, if the identifier is valid.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html#checkFeatures--">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationBasedModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.html#checkFeatures--">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">ComplexBuildingReaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/MichaelisMentenReaction.html#checkFeatures--">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/MichaelisMentenReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">MichaelisMentenReaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/NthOrderReaction.html#checkFeatures--">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/NthOrderReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">NthOrderReaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ReversibleReaction.html#checkFeatures--">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/ReversibleReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">ReversibleReaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/SingleFileChannelMembraneTransport.html#checkFeatures--">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/SingleFileChannelMembraneTransport.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">SingleFileChannelMembraneTransport</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/DisplacementBasedModule.html#checkFeatures--">checkFeatures()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/DisplacementBasedModule.html" title="class in bio.singa.simulation.model.modules.displacement">DisplacementBasedModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/UpdateModule.html#checkFeatures--">checkFeatures()</a></span> - Method in interface bio.singa.simulation.model.modules.<a href="../bio/singa/simulation/model/modules/UpdateModule.html" title="interface in bio.singa.simulation.model.modules">UpdateModule</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/entities/ChemicalEntity.html" title="class in bio.singa.chemistry.entities"><span class="typeNameLink">ChemicalEntity</span></a> - Class in <a href="../bio/singa/chemistry/entities/package-summary.html">bio.singa.chemistry.entities</a></dt>
<dd>
<div class="block">Chemical Entity is an abstract class that provides the common features of all chemical substances on a descriptive
 level.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/ChemicalEntity.html#ChemicalEntity-bio.singa.features.identifiers.SimpleStringIdentifier-">ChemicalEntity(SimpleStringIdentifier)</a></span> - Constructor for class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/ChemicalEntity.html" title="class in bio.singa.chemistry.entities">ChemicalEntity</a></dt>
<dd>
<div class="block">Creates a new Chemical Entity with the given identifier.</div>
</dd>
<dt><a href="../bio/singa/chemistry/entities/ChemicalEntity.Builder.html" title="class in bio.singa.chemistry.entities"><span class="typeNameLink">ChemicalEntity.Builder</span></a>&lt;<a href="../bio/singa/chemistry/entities/ChemicalEntity.Builder.html" title="type parameter in ChemicalEntity.Builder">TopLevelType</a> extends <a href="../bio/singa/chemistry/entities/ChemicalEntity.html" title="class in bio.singa.chemistry.entities">ChemicalEntity</a>,<a href="../bio/singa/chemistry/entities/ChemicalEntity.Builder.html" title="type parameter in ChemicalEntity.Builder">BuilderType</a> extends <a href="../bio/singa/chemistry/entities/ChemicalEntity.Builder.html" title="class in bio.singa.chemistry.entities">ChemicalEntity.Builder</a>&gt; - Class in <a href="../bio/singa/chemistry/entities/package-summary.html">bio.singa.chemistry.entities</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/ChemistryFeatureContainer.html" title="class in bio.singa.chemistry.features"><span class="typeNameLink">ChemistryFeatureContainer</span></a> - Class in <a href="../bio/singa/chemistry/features/package-summary.html">bio.singa.chemistry.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/ChemistryFeatureContainer.html#ChemistryFeatureContainer--">ChemistryFeatureContainer()</a></span> - Constructor for class bio.singa.chemistry.features.<a href="../bio/singa/chemistry/features/ChemistryFeatureContainer.html" title="class in bio.singa.chemistry.features">ChemistryFeatureContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#CHLORINE">CHLORINE</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#CHROMIUM">CHROMIUM</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/parser/pdb/ligands/CifFileParser.html" title="class in bio.singa.structure.parser.pdb.ligands"><span class="typeNameLink">CifFileParser</span></a> - Class in <a href="../bio/singa/structure/parser/pdb/ligands/package-summary.html">bio.singa.structure.parser.pdb.ligands</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/geometry/faces/Circle.html" title="class in bio.singa.mathematics.geometry.faces"><span class="typeNameLink">Circle</span></a> - Class in <a href="../bio/singa/mathematics/geometry/faces/package-summary.html">bio.singa.mathematics.geometry.faces</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Circle.html#Circle-bio.singa.mathematics.vectors.Vector2D-bio.singa.mathematics.vectors.Vector2D-bio.singa.mathematics.vectors.Vector2D-">Circle(Vector2D, Vector2D, Vector2D)</a></span> - Constructor for class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Circle.html" title="class in bio.singa.mathematics.geometry.faces">Circle</a></dt>
<dd>
<div class="block">Circumscribed circle from 3 points (e.g.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Circle.html#Circle-bio.singa.mathematics.vectors.Vector2D-double-">Circle(Vector2D, double)</a></span> - Constructor for class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Circle.html" title="class in bio.singa.mathematics.geometry.faces">Circle</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/algorithms/voronoi/model/CircleEvent.html" title="class in bio.singa.mathematics.algorithms.voronoi.model"><span class="typeNameLink">CircleEvent</span></a> - Class in <a href="../bio/singa/mathematics/algorithms/voronoi/model/package-summary.html">bio.singa.mathematics.algorithms.voronoi.model</a></dt>
<dd>
<div class="block">A circle event happens when a parabolic arc shrinks to a point and disappears from the beach line.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/CircleEvent.html#CircleEvent--">CircleEvent()</a></span> - Constructor for class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/CircleEvent.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">CircleEvent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonGraphs.html#circleRegion-bio.singa.simulation.model.graphs.AutomatonGraph-bio.singa.simulation.model.sections.CellRegion-bio.singa.mathematics.topology.grids.rectangular.RectangularCoordinate-int-">circleRegion(AutomatonGraph, CellRegion, RectangularCoordinate, int)</a></span> - Static method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonGraphs.html" title="class in bio.singa.simulation.model.graphs">AutomatonGraphs</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/displacement/implementations/ClathrinMediatedEndocytosis.html" title="class in bio.singa.simulation.model.modules.displacement.implementations"><span class="typeNameLink">ClathrinMediatedEndocytosis</span></a> - Class in <a href="../bio/singa/simulation/model/modules/displacement/implementations/package-summary.html">bio.singa.simulation.model.modules.displacement.implementations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/implementations/ClathrinMediatedEndocytosis.html#ClathrinMediatedEndocytosis--">ClathrinMediatedEndocytosis()</a></span> - Constructor for class bio.singa.simulation.model.modules.displacement.implementations.<a href="../bio/singa/simulation/model/modules/displacement/implementations/ClathrinMediatedEndocytosis.html" title="class in bio.singa.simulation.model.modules.displacement.implementations">ClathrinMediatedEndocytosis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/Vesicle.html#clearAssociatedNodes--">clearAssociatedNodes()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/Vesicle.html" title="class in bio.singa.simulation.model.modules.displacement">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/FieldSupplier.html#clearDeltas--">clearDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/FieldSupplier.html" title="class in bio.singa.simulation.model.modules.concentration">FieldSupplier</a></dt>
<dd>
<div class="block">Clears all full end half deltas.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#clearPotentialConcentrationDeltas--">clearPotentialConcentrationDeltas()</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>
<div class="block">Clears the list of potential deltas.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/Vesicle.html#clearPotentialConcentrationDeltas--">clearPotentialConcentrationDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/Vesicle.html" title="class in bio.singa.simulation.model.modules.displacement">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Updatable.html#clearPotentialConcentrationDeltas--">clearPotentialConcentrationDeltas()</a></span> - Method in interface bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Updatable.html" title="interface in bio.singa.simulation.model.simulation">Updatable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html#clearPotentialDeltas--">clearPotentialDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaManager</a></dt>
<dd>
<div class="block">Clears the list of potential deltas.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/scope/DependentUpdate.html#clearPotentialDeltas-bio.singa.simulation.model.simulation.Updatable-">clearPotentialDeltas(Updatable)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.scope.<a href="../bio/singa/simulation/model/modules/concentration/scope/DependentUpdate.html" title="class in bio.singa.simulation.model.modules.concentration.scope">DependentUpdate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/scope/IndependentUpdate.html#clearPotentialDeltas-bio.singa.simulation.model.simulation.Updatable-">clearPotentialDeltas(Updatable)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.scope.<a href="../bio/singa/simulation/model/modules/concentration/scope/IndependentUpdate.html" title="class in bio.singa.simulation.model.modules.concentration.scope">IndependentUpdate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/scope/SemiDependentUpdate.html#clearPotentialDeltas-bio.singa.simulation.model.simulation.Updatable-">clearPotentialDeltas(Updatable)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.scope.<a href="../bio/singa/simulation/model/modules/concentration/scope/SemiDependentUpdate.html" title="class in bio.singa.simulation.model.modules.concentration.scope">SemiDependentUpdate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/scope/UpdateScope.html#clearPotentialDeltas-bio.singa.simulation.model.simulation.Updatable-">clearPotentialDeltas(Updatable)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.scope.<a href="../bio/singa/simulation/model/modules/concentration/scope/UpdateScope.html" title="interface in bio.singa.simulation.model.modules.concentration.scope">UpdateScope</a></dt>
<dd>
<div class="block">Clears all concentration deltas that may be assigned to the given updatable.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#clearPotentialDeltasBut-bio.singa.simulation.model.modules.UpdateModule-">clearPotentialDeltasBut(UpdateModule)</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html#clearPotentialDeltasBut-bio.singa.simulation.model.modules.UpdateModule-">clearPotentialDeltasBut(UpdateModule)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaManager</a></dt>
<dd>
<div class="block">Clears the list of potential deltas retaining updates from a specific module.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/Vesicle.html#clearPotentialDeltasBut-bio.singa.simulation.model.modules.UpdateModule-">clearPotentialDeltasBut(UpdateModule)</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/Vesicle.html" title="class in bio.singa.simulation.model.modules.displacement">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Updatable.html#clearPotentialDeltasBut-bio.singa.simulation.model.modules.UpdateModule-">clearPotentialDeltasBut(UpdateModule)</a></span> - Method in interface bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Updatable.html" title="interface in bio.singa.simulation.model.simulation">Updatable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/DisplacementDeltaManager.html#clearPotentialDisplacementDeltas--">clearPotentialDisplacementDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/DisplacementDeltaManager.html" title="class in bio.singa.simulation.model.modules.displacement">DisplacementDeltaManager</a></dt>
<dd>
<div class="block">Clears all potential displacement deltas.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/Vesicle.html#clearPotentialDisplacementDeltas--">clearPotentialDisplacementDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/Vesicle.html" title="class in bio.singa.simulation.model.modules.displacement">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/VesicleLayer.html#clearUpdates--">clearUpdates()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/VesicleLayer.html" title="class in bio.singa.simulation.model.modules.displacement">VesicleLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html#clipEdges--">clipEdges()</a></span> - Method in class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">VoronoiDiagram</a></dt>
<dd>
<div class="block">Clips all edges sticking out of the bounding box.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/parameters/MixedParameterList.html#clone--">clone()</a></span> - Method in class bio.singa.core.parameters.<a href="../bio/singa/core/parameters/MixedParameterList.html" title="class in bio.singa.core.parameters">MixedParameterList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/parameters/UniqueParameterList.html#clone--">clone()</a></span> - Method in class bio.singa.core.parameters.<a href="../bio/singa/core/parameters/UniqueParameterList.html" title="class in bio.singa.core.parameters">UniqueParameterList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/GifWriter.html#close--">close()</a></span> - Method in class bio.singa.core.utility.<a href="../bio/singa/core/utility/GifWriter.html" title="class in bio.singa.core.utility">GifWriter</a></dt>
<dd>
<div class="block">Close this GifSequenceWriter.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/Token.html#CLOSE_BRACKET">CLOSE_BRACKET</a></span> - Static variable in class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/Token.html" title="class in uk.co.cogitolearning.cogpar">Token</a></dt>
<dd>
<div class="block">Token id for closing brackets</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html#closeCells--">closeCells()</a></span> - Method in class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">VoronoiDiagram</a></dt>
<dd>
<div class="block">Closes all cells that have not been closed until now.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/events/EpochUpdateWriter.html#closeWriters--">closeWriters()</a></span> - Method in class bio.singa.simulation.events.<a href="../bio/singa/simulation/events/EpochUpdateWriter.html" title="class in bio.singa.simulation.events">EpochUpdateWriter</a></dt>
<dd>
<div class="block">Close all associated writers.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html#clusterCutoff-double-">clusterCutoff(double)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.ParameterStep.html#clusterCutoff-double-">clusterCutoff(double)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.ParameterStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.ParameterStep</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/algorithms/clustering/Clustering.html" title="interface in bio.singa.mathematics.algorithms.clustering"><span class="typeNameLink">Clustering</span></a>&lt;<a href="../bio/singa/mathematics/algorithms/clustering/Clustering.html" title="type parameter in Clustering">DataType</a>&gt; - Interface in <a href="../bio/singa/mathematics/algorithms/clustering/package-summary.html">bio.singa.mathematics.algorithms.clustering</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#COBALT">COBALT</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html#collectLeavesPreOrder-bio.singa.mathematics.graphs.trees.BinaryTreeNode-java.util.List-">collectLeavesPreOrder(BinaryTreeNode&lt;T&gt;, List&lt;BinaryTreeNode&lt;T&gt;&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTree</a></dt>
<dd>
<div class="block">Traverses the tree in pre order and collects leave nodes.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html#collectNodesPreOrder-bio.singa.mathematics.graphs.trees.BinaryTreeNode-java.util.List-">collectNodesPreOrder(BinaryTreeNode&lt;T&gt;, List&lt;BinaryTreeNode&lt;T&gt;&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTree</a></dt>
<dd>
<div class="block">Traverses the tree in pre order and collects nodes.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/Reaction.html#collectProductsString--">collectProductsString()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/Reaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">Reaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/Reaction.html#collectSubstrateString--">collectSubstrateString()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/Reaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">Reaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Simulation.html#collectUpdatables--">collectUpdatables()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Simulation.html" title="class in bio.singa.simulation.model.simulation">Simulation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/javafx/renderer/colors/ColorScale.html" title="class in bio.singa.javafx.renderer.colors"><span class="typeNameLink">ColorScale</span></a> - Class in <a href="../bio/singa/javafx/renderer/colors/package-summary.html">bio.singa.javafx.renderer.colors</a></dt>
<dd>
<div class="block">This object provides a color scale between the given minimal and maximal
 values.</div>
</dd>
<dt><a href="../bio/singa/javafx/renderer/colors/ColorScale.Builder.html" title="class in bio.singa.javafx.renderer.colors"><span class="typeNameLink">ColorScale.Builder</span></a> - Class in <a href="../bio/singa/javafx/renderer/colors/package-summary.html">bio.singa.javafx.renderer.colors</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/javafx/viewer/ColorScheme.html" title="enum in bio.singa.javafx.viewer"><span class="typeNameLink">ColorScheme</span></a> - Enum in <a href="../bio/singa/javafx/viewer/package-summary.html">bio.singa.javafx.viewer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/StructureViewer.html#colorScheme">colorScheme</a></span> - Static variable in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/StructureViewer.html" title="class in bio.singa.javafx.viewer">StructureViewer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/combinatorics/StreamCombinations.html#combinations-int-java.util.List-">combinations(int, List&lt;T&gt;)</a></span> - Static method in class bio.singa.mathematics.combinatorics.<a href="../bio/singa/mathematics/combinatorics/StreamCombinations.html" title="class in bio.singa.mathematics.combinatorics">StreamCombinations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/events/GraphImageWriter.html#combineToGif--">combineToGif()</a></span> - Method in class bio.singa.simulation.events.<a href="../bio/singa/simulation/events/GraphImageWriter.html" title="class in bio.singa.simulation.events">GraphImageWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/core/utility/CommutablePair.html" title="class in bio.singa.core.utility"><span class="typeNameLink">CommutablePair</span></a>&lt;<a href="../bio/singa/core/utility/CommutablePair.html" title="type parameter in CommutablePair">ValueType</a>&gt; - Class in <a href="../bio/singa/core/utility/package-summary.html">bio.singa.core.utility</a></dt>
<dd>
<div class="block">This subclass of <a href="../bio/singa/core/utility/Pair.html" title="class in bio.singa.core.utility"><code>Pair</code></a> only implements different equals and hashcode methods.<br> The values are exchangeable.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/CommutablePair.html#CommutablePair-ValueType-ValueType-">CommutablePair(ValueType, ValueType)</a></span> - Constructor for class bio.singa.core.utility.<a href="../bio/singa/core/utility/CommutablePair.html" title="class in bio.singa.core.utility">CommutablePair</a></dt>
<dd>
<div class="block">Creates a new <a href="../bio/singa/core/utility/Pair.html" title="class in bio.singa.core.utility"><code>Pair</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html#compactToSymmetricMatrix-double:A:A-">compactToSymmetricMatrix(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html" title="class in bio.singa.mathematics.matrices">SymmetricMatrix</a></dt>
<dd>
<div class="block">Compacts the values of a symmetric matrix into a jagged array, that represents the lower triangular part of a
 symmetric matrix.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DMatch.html#compareTo-bio.singa.structure.algorithms.superimposition.fit3d.Fit3DMatch-">compareTo(Fit3DMatch)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DMatch.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DMatch</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/LigandFamily.html#compareTo-bio.singa.structure.model.families.LigandFamily-">compareTo(LigandFamily)</a></span> - Method in class bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/LigandFamily.html" title="class in bio.singa.structure.model.families">LigandFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/identifiers/LeafIdentifier.html#compareTo-bio.singa.structure.model.identifiers.LeafIdentifier-">compareTo(LeafIdentifier)</a></span> - Method in class bio.singa.structure.model.identifiers.<a href="../bio/singa/structure/model/identifiers/LeafIdentifier.html" title="class in bio.singa.structure.model.identifiers">LeafIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/identifiers/UniqueAtomIdentifer.html#compareTo-bio.singa.structure.model.identifiers.UniqueAtomIdentifer-">compareTo(UniqueAtomIdentifer)</a></span> - Method in class bio.singa.structure.model.identifiers.<a href="../bio/singa/structure/model/identifiers/UniqueAtomIdentifer.html" title="class in bio.singa.structure.model.identifiers">UniqueAtomIdentifer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/parser/sbml/converter/CompartmentHandler.html" title="class in bio.singa.simulation.parser.sbml.converter"><span class="typeNameLink">CompartmentHandler</span></a> - Class in <a href="../bio/singa/simulation/parser/sbml/converter/package-summary.html">bio.singa.simulation.parser.sbml.converter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/CompartmentHandler.html#CompartmentHandler--">CompartmentHandler()</a></span> - Constructor for class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/CompartmentHandler.html" title="class in bio.singa.simulation.parser.sbml.converter">CompartmentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations"><span class="typeNameLink">ComplexBuildingReaction</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/imlementations/package-summary.html">bio.singa.simulation.model.modules.concentration.imlementations</a></dt>
<dd>
<div class="block">Complex building or breaking reactions are a special kind of <a href="../bio/singa/simulation/model/modules/concentration/imlementations/ReversibleReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations"><code>ReversibleReaction</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.html#ComplexBuildingReaction--">ComplexBuildingReaction()</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">ComplexBuildingReaction</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.BindeeSectionSelection.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations"><span class="typeNameLink">ComplexBuildingReaction.BindeeSectionSelection</span></a> - Interface in <a href="../bio/singa/simulation/model/modules/concentration/imlementations/package-summary.html">bio.singa.simulation.model.modules.concentration.imlementations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.BindeeSelection.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations"><span class="typeNameLink">ComplexBuildingReaction.BindeeSelection</span></a> - Interface in <a href="../bio/singa/simulation/model/modules/concentration/imlementations/package-summary.html">bio.singa.simulation.model.modules.concentration.imlementations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.BinderSectionSelection.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations"><span class="typeNameLink">ComplexBuildingReaction.BinderSectionSelection</span></a> - Interface in <a href="../bio/singa/simulation/model/modules/concentration/imlementations/package-summary.html">bio.singa.simulation.model.modules.concentration.imlementations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.BinderSelection.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations"><span class="typeNameLink">ComplexBuildingReaction.BinderSelection</span></a> - Interface in <a href="../bio/singa/simulation/model/modules/concentration/imlementations/package-summary.html">bio.singa.simulation.model.modules.concentration.imlementations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.BindingBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations"><span class="typeNameLink">ComplexBuildingReaction.BindingBuilder</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/imlementations/package-summary.html">bio.singa.simulation.model.modules.concentration.imlementations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ComplexBuildingReaction.BuilderStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations"><span class="typeNameLink">ComplexBuildingReaction.BuilderStep</span></a> - Interface in <a href="../bio/singa/simulation/model/modules/concentration/imlementations/package-summary.html">bio.singa.simulation.model.modules.concentration.imlementations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/entities/ComplexedChemicalEntity.html" title="class in bio.singa.chemistry.entities"><span class="typeNameLink">ComplexedChemicalEntity</span></a> - Class in <a href="../bio/singa/chemistry/entities/package-summary.html">bio.singa.chemistry.entities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/ComplexedChemicalEntity.html#ComplexedChemicalEntity-bio.singa.features.identifiers.SimpleStringIdentifier-">ComplexedChemicalEntity(SimpleStringIdentifier)</a></span> - Constructor for class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/ComplexedChemicalEntity.html" title="class in bio.singa.chemistry.entities">ComplexedChemicalEntity</a></dt>
<dd>
<div class="block">Creates a new Chemical Entity with the given pdbIdentifier.</div>
</dd>
<dt><a href="../bio/singa/chemistry/entities/ComplexedChemicalEntity.Builder.html" title="class in bio.singa.chemistry.entities"><span class="typeNameLink">ComplexedChemicalEntity.Builder</span></a> - Class in <a href="../bio/singa/chemistry/entities/package-summary.html">bio.singa.chemistry.entities</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html#composePdbRepresentation-bio.singa.structure.model.interfaces.Structure-">composePdbRepresentation(Structure)</a></span> - Static method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html" title="class in bio.singa.structure.parser.pdb.structures">StructureRepresentation</a></dt>
<dd>
<div class="block">Creates a pdb representation of the given structure.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html#composePdbRepresentation-java.util.List-">composePdbRepresentation(List&lt;LeafSubstructure&lt;?&gt;&gt;)</a></span> - Static method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html" title="class in bio.singa.structure.parser.pdb.structures">StructureRepresentation</a></dt>
<dd>
<div class="block">Creates a pdb representation of the given structure.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html#composePdbRepresentation-bio.singa.structure.model.oak.OakModel-">composePdbRepresentation(OakModel)</a></span> - Static method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureRepresentation.html" title="class in bio.singa.structure.parser.pdb.structures">StructureRepresentation</a></dt>
<dd>
<div class="block">Creates a representation of the given model.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration"><span class="typeNameLink">ConcentrationBasedModule</span></a>&lt;<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="type parameter in ConcentrationBasedModule">DeltaFunctionType</a> extends <a href="../bio/singa/simulation/model/modules/concentration/functions/AbstractDeltaFunction.html" title="class in bio.singa.simulation.model.modules.concentration.functions">AbstractDeltaFunction</a>&gt; - Class in <a href="../bio/singa/simulation/model/modules/concentration/package-summary.html">bio.singa.simulation.model.modules.concentration</a></dt>
<dd>
<div class="block">Concentration based modules handle updates that are applied to the concentrations in <a href="../bio/singa/simulation/model/simulation/Updatable.html" title="interface in bio.singa.simulation.model.simulation"><code>Updatable</code></a>s during a
 simulation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html#ConcentrationBasedModule--">ConcentrationBasedModule()</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationBasedModule</a></dt>
<dd>
<div class="block">Creates a new concentration based module.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html" title="class in bio.singa.simulation.model.sections"><span class="typeNameLink">ConcentrationContainer</span></a> - Class in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>
<div class="block">The concentration container manages the concentrations of one updatable.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html#ConcentrationContainer--">ConcentrationContainer()</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html" title="class in bio.singa.simulation.model.sections">ConcentrationContainer</a></dt>
<dd>
<div class="block">Creates a new concentration container.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDelta.html" title="class in bio.singa.simulation.model.modules.concentration"><span class="typeNameLink">ConcentrationDelta</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/package-summary.html">bio.singa.simulation.model.modules.concentration</a></dt>
<dd>
<div class="block">The delta object manages the change that will be applied to the concentration of a specific <a href="../bio/singa/chemistry/entities/ChemicalEntity.html" title="class in bio.singa.chemistry.entities"><code>ChemicalEntity</code></a> in
 a <a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections"><code>CellSubsection</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDelta.html#ConcentrationDelta-bio.singa.simulation.model.modules.concentration.ConcentrationBasedModule-bio.singa.simulation.model.sections.CellSubsection-bio.singa.chemistry.entities.ChemicalEntity-javax.measure.Quantity-">ConcentrationDelta(ConcentrationBasedModule, CellSubsection, ChemicalEntity, Quantity&lt;MolarConcentration&gt;)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDelta.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDelta</a></dt>
<dd>
<div class="block">Creates a new concentration delta.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaIdentifier.html" title="class in bio.singa.simulation.model.modules.concentration"><span class="typeNameLink">ConcentrationDeltaIdentifier</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/package-summary.html">bio.singa.simulation.model.modules.concentration</a></dt>
<dd>
<div class="block">Used to identify changes to concentrations (<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDelta.html" title="class in bio.singa.simulation.model.modules.concentration"><code>ConcentrationDelta</code></a>s) (mostly in maps).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaIdentifier.html#ConcentrationDeltaIdentifier-bio.singa.simulation.model.simulation.Updatable-bio.singa.simulation.model.sections.CellSubsection-bio.singa.chemistry.entities.ChemicalEntity-">ConcentrationDeltaIdentifier(Updatable, CellSubsection, ChemicalEntity)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaIdentifier.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaIdentifier</a></dt>
<dd>
<div class="block">Creates a new DeltaIdentifier.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration"><span class="typeNameLink">ConcentrationDeltaManager</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/package-summary.html">bio.singa.simulation.model.modules.concentration</a></dt>
<dd>
<div class="block">The ConcentrationDeltaManager handles current concentrations of an updatable and the updates to those
 concentrations that should be applied during simulation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html#ConcentrationDeltaManager-bio.singa.simulation.model.sections.ConcentrationContainer-">ConcentrationDeltaManager(ConcentrationContainer)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaManager</a></dt>
<dd>
<div class="block">Creates a new Concentration Delta Manager.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/sections/ConcentrationPool.html" title="class in bio.singa.simulation.model.sections"><span class="typeNameLink">ConcentrationPool</span></a> - Class in <a href="../bio/singa/simulation/model/sections/package-summary.html">bio.singa.simulation.model.sections</a></dt>
<dd>
<div class="block">The concentration pool manages the <a href="../bio/singa/features/quantities/MolarConcentration.html" title="class in bio.singa.features.quantities"><code>MolarConcentration</code></a> of <a href="../bio/singa/chemistry/entities/ChemicalEntity.html" title="class in bio.singa.chemistry.entities"><code>ChemicalEntity</code></a>s in <a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections"><code>CellSubsection</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/ConcentrationPool.html#ConcentrationPool--">ConcentrationPool()</a></span> - Constructor for class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/ConcentrationPool.html" title="class in bio.singa.simulation.model.sections">ConcentrationPool</a></dt>
<dd>
<div class="block">Creates a new, empty concentration pool.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarConcentration.html#concentrationToMolecules-javax.measure.Quantity-javax.measure.Quantity-">concentrationToMolecules(Quantity&lt;MolarConcentration&gt;, Quantity&lt;Volume&gt;)</a></span> - Static method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarConcentration.html" title="class in bio.singa.features.quantities">MolarConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarConcentration.html#concentrationToMoles-javax.measure.Quantity-javax.measure.Quantity-">concentrationToMoles(Quantity&lt;MolarConcentration&gt;, Quantity&lt;Volume&gt;)</a></span> - Static method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarConcentration.html" title="class in bio.singa.features.quantities">MolarConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/reactions/RateConstant.ConcentrationStep.html#concentrationUnit-javax.measure.Unit-">concentrationUnit(Unit&lt;MolarConcentration&gt;)</a></span> - Method in interface bio.singa.chemistry.features.reactions.<a href="../bio/singa/chemistry/features/reactions/RateConstant.ConcentrationStep.html" title="interface in bio.singa.chemistry.features.reactions">RateConstant.ConcentrationStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/reactions/RateConstant.RateBuilder.html#concentrationUnit-javax.measure.Unit-">concentrationUnit(Unit&lt;MolarConcentration&gt;)</a></span> - Method in class bio.singa.chemistry.features.reactions.<a href="../bio/singa/chemistry/features/reactions/RateConstant.RateBuilder.html" title="class in bio.singa.chemistry.features.reactions">RateConstant.RateBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakChain.html#connectChainBackbone--">connectChainBackbone()</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakChain.html" title="class in bio.singa.structure.model.oak">OakChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#connectHydrogens-boolean-">connectHydrogens(boolean)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option to <code>true</code> will try to connect hydrogen atom tho their respective partners.<br><br>
 Default value: <code>false</code></div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakChain.html#connectNucleotideBonds-bio.singa.structure.model.oak.OakNucleotide-bio.singa.structure.model.oak.OakNucleotide-">connectNucleotideBonds(OakNucleotide, OakNucleotide)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakChain.html" title="class in bio.singa.structure.model.oak">OakChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakChain.html#connectPeptideBonds-bio.singa.structure.model.oak.OakAminoAcid-bio.singa.structure.model.oak.OakAminoAcid-">connectPeptideBonds(OakAminoAcid, OakAminoAcid)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakChain.html" title="class in bio.singa.structure.model.oak">OakChain</a></dt>
<dd>
<div class="block">Connects two residues, using the Backbone Carbon (C) of the source residue and the Backbone Nitrogen (N) of the
 target residue.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/Renderer.html#connectPoints-java.util.Collection-">connectPoints(Collection&lt;Vector2D&gt;)</a></span> - Method in interface bio.singa.javafx.renderer.<a href="../bio/singa/javafx/renderer/Renderer.html" title="interface in bio.singa.javafx.renderer">Renderer</a></dt>
<dd>
<div class="block">Connects the points given in the List in order of their appearance with a line.<br>
 
  The line width of the point is determined by the LineWidth (set by
 <code>GraphicsContext.setLineWidth(double)</code>).
  The color is determined by the StrokeColor (set by <code>GraphicsContext.setStroke(Paint)</code>).
 </div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakBond.html#connectsAtom-bio.singa.structure.model.oak.OakAtom-">connectsAtom(OakAtom)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakBond.html" title="class in bio.singa.structure.model.oak">OakBond</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusAlignment.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusAlignment</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>
<div class="block">A consensus alignment of same-sized <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a>s can be used to cluster them according their geometric
 similarity in a multi-structure alignment manner as described in:</div>
</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusBuilder</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>
<div class="block">A builder that guides through the creation of a <a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusAlignment.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><code>ConsensusAlignment</code></a>.</div>
</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.AtomStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusBuilder.AtomStep</span></a> - Interface in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusBuilder.Builder</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.InputStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusBuilder.InputStep</span></a> - Interface in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.ParameterStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusBuilder.ParameterStep</span></a> - Interface in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusContainer.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusContainer</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>
<div class="block">A container encapsulating a <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a> with a <a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees"><code>BinaryTree</code></a> that represents its associated
 consensus tree.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusContainer.html#ConsensusContainer-bio.singa.structure.model.oak.StructuralMotif-boolean-">ConsensusContainer(StructuralMotif, boolean)</a></span> - Constructor for class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusContainer.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusContainer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusException.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><span class="typeNameLink">ConsensusException</span></a> - Exception in <a href="../bio/singa/structure/algorithms/superimposition/consensus/package-summary.html">bio.singa.structure.algorithms.superimposition.consensus</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusException.html#ConsensusException-java.lang.String-">ConsensusException(String)</a></span> - Constructor for exception bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusException.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/ExpressionNode.html#CONSTANT_NODE">CONSTANT_NODE</a></span> - Static variable in interface uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/ExpressionNode.html" title="interface in uk.co.cogitolearning.cogpar">ExpressionNode</a></dt>
<dd>
<div class="block">Node id for constant nodes</div>
</dd>
<dt><a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html" title="class in uk.co.cogitolearning.cogpar"><span class="typeNameLink">ConstantExpressionNode</span></a> - Class in <a href="../uk/co/cogitolearning/cogpar/package-summary.html">uk.co.cogitolearning.cogpar</a></dt>
<dd>
<div class="block">An ExpressionNode that stores a constant value</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html#ConstantExpressionNode-double-">ConstantExpressionNode(double)</a></span> - Constructor for class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">ConstantExpressionNode</a></dt>
<dd>
<div class="block">Construct with the fixed value.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html#ConstantExpressionNode-java.lang.String-">ConstantExpressionNode(String)</a></span> - Constructor for class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">ConstantExpressionNode</a></dt>
<dd>
<div class="block">Convenience constructor that takes a string and converts it to a double
 before storing the value.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html#containsAtomWithName-java.lang.String-">containsAtomWithName(String)</a></span> - Method in interface bio.singa.structure.model.interfaces.<a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html" title="interface in bio.singa.structure.model.interfaces">LeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/mmtf/MmtfLeafSubstructure.html#containsAtomWithName-java.lang.String-">containsAtomWithName(String)</a></span> - Method in class bio.singa.structure.model.mmtf.<a href="../bio/singa/structure/model/mmtf/MmtfLeafSubstructure.html" title="class in bio.singa.structure.model.mmtf">MmtfLeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html#containsAtomWithName-java.lang.String-">containsAtomWithName(String)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html" title="class in bio.singa.structure.model.oak">OakLeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html#containsEdge-java.lang.Object-">containsEdge(Object)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="class in bio.singa.mathematics.graphs.grid">AbstractGridGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#containsEdge-java.lang.Object-">containsEdge(Object)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graph.html#containsEdge-java.lang.Object-">containsEdge(Object)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graph.html" title="interface in bio.singa.mathematics.graphs.model">Graph</a></dt>
<dd>
<div class="block">Returns true, if the graph contains the edge and false otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/AminoAcidFamily.html#containsExpectedAtoms-java.util.List-bio.singa.structure.model.families.AminoAcidFamily-">containsExpectedAtoms(List&lt;Atom&gt;, AminoAcidFamily)</a></span> - Method in enum bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families">AminoAcidFamily</a></dt>
<dd>
<div class="block">Returns true if the set of AtomFilter contains only Atom names, that can occur in the given residue type.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html#containsNode-java.lang.Object-">containsNode(Object)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="class in bio.singa.mathematics.graphs.grid">AbstractGridGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html#containsNode-NodeType-">containsNode(NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html" title="class in bio.singa.mathematics.graphs.model">AbstractEdge</a></dt>
<dd>
<div class="block">Returns true only if the node is source or target of this edge.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#containsNode-java.lang.Object-">containsNode(Object)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Edge.html#containsNode-NodeType-">containsNode(NodeType)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Edge.html" title="interface in bio.singa.mathematics.graphs.model">Edge</a></dt>
<dd>
<div class="block">Returns true if a node with the given identifier is either source or target of the node.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graph.html#containsNode-java.lang.Object-">containsNode(Object)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graph.html" title="interface in bio.singa.mathematics.graphs.model">Graph</a></dt>
<dd>
<div class="block">Returns true, if the graph contains the node and false otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html#containsNode-T-">containsNode(T)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTree</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/GenericGraph.html#containsNodeWithContent-ContentType-">containsNodeWithContent(ContentType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/GenericGraph.html" title="class in bio.singa.mathematics.graphs.model">GenericGraph</a></dt>
<dd>
<div class="block">Returns true if a node with the given content is already present.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/topology/grids/rectangular/RectangularGrid.html#containsValue-java.lang.Object-">containsValue(Object)</a></span> - Method in class bio.singa.mathematics.topology.grids.rectangular.<a href="../bio/singa/mathematics/topology/grids/rectangular/RectangularGrid.html" title="class in bio.singa.mathematics.topology.grids.rectangular">RectangularGrid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/GridCoordinateConverter.html#convert-int-">convert(int)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/GridCoordinateConverter.html" title="class in bio.singa.mathematics.graphs.grid">GridCoordinateConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/GridCoordinateConverter.html#convert-bio.singa.mathematics.vectors.Vector2D-">convert(Vector2D)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/GridCoordinateConverter.html" title="class in bio.singa.mathematics.graphs.grid">GridCoordinateConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLAssignmentRuleConverter.html#convertAssignmentRule-org.sbml.jsbml.AssignmentRule-">convertAssignmentRule(AssignmentRule)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLAssignmentRuleConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLAssignmentRuleConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLKineticLawConverter.html#convertKineticLaw-org.sbml.jsbml.KineticLaw-">convertKineticLaw(KineticLaw)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLKineticLawConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLKineticLawConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLParameterConverter.html#convertLocalParameter-org.sbml.jsbml.LocalParameter-">convertLocalParameter(LocalParameter)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLParameterConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLParameterConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLExpressionConverter.html#convertRawExpression-org.sbml.jsbml.ASTNode-org.sbml.jsbml.ListOf-javax.measure.Unit-">convertRawExpression(ASTNode, ListOf&lt;LocalParameter&gt;, Unit&lt;?&gt;)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLExpressionConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLExpressionConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLExpressionConverter.html#convertRawExpression-org.sbml.jsbml.ASTNode-javax.measure.Unit-">convertRawExpression(ASTNode, Unit&lt;?&gt;)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLExpressionConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLExpressionConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLReactionConverter.html#convertReaction-org.sbml.jsbml.Reaction-">convertReaction(Reaction)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLReactionConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLReactionConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLReactionConverter.html#convertReactions-org.sbml.jsbml.ListOf-">convertReactions(ListOf&lt;Reaction&gt;)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLReactionConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLReactionConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLParameterConverter.html#convertSimulationParameter-org.sbml.jsbml.Parameter-">convertSimulationParameter(Parameter)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLParameterConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLParameterConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLParameterConverter.html#convertSimulationParameters-org.sbml.jsbml.ListOf-">convertSimulationParameters(ListOf&lt;Parameter&gt;)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLParameterConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLParameterConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/parameters/Environment.html#convertSimulationToSystemScale-double-">convertSimulationToSystemScale(double)</a></span> - Static method in class bio.singa.features.parameters.<a href="../bio/singa/features/parameters/Environment.html" title="class in bio.singa.features.parameters">Environment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/parameters/Environment.html#convertSystemToSimulationScale-javax.measure.Quantity-">convertSystemToSimulationScale(Quantity&lt;Length&gt;)</a></span> - Static method in class bio.singa.features.parameters.<a href="../bio/singa/features/parameters/Environment.html" title="class in bio.singa.features.parameters">Environment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#convertToSpheres-bio.singa.structure.model.interfaces.AtomContainer-">convertToSpheres(AtomContainer)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graphs.html#convertTreeToGraph-bio.singa.mathematics.graphs.trees.BinaryTree-">convertTreeToGraph(BinaryTree&lt;ContentType&gt;)</a></span> - Static method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graphs.html" title="class in bio.singa.mathematics.graphs.model">Graphs</a></dt>
<dd>
<div class="block">Converts a <a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees"><code>BinaryTree</code></a> to a <a href="../bio/singa/mathematics/graphs/model/GenericGraph.html" title="class in bio.singa.mathematics.graphs.model"><code>GenericGraph</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLUnitConverter.html#convertUnit-org.sbml.jsbml.UnitDefinition-">convertUnit(UnitDefinition)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLUnitConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLUnitConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/parser/sbml/converter/SBMLUnitConverter.html#convertUnits-org.sbml.jsbml.ListOf-">convertUnits(ListOf&lt;UnitDefinition&gt;)</a></span> - Method in class bio.singa.simulation.parser.sbml.converter.<a href="../bio/singa/simulation/parser/sbml/converter/SBMLUnitConverter.html" title="class in bio.singa.simulation.parser.sbml.converter">SBMLUnitConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/algorithms/geometry/ConvexHull.html" title="class in bio.singa.mathematics.algorithms.geometry"><span class="typeNameLink">ConvexHull</span></a> - Class in <a href="../bio/singa/mathematics/algorithms/geometry/package-summary.html">bio.singa.mathematics.algorithms.geometry</a></dt>
<dd>
<div class="block">Based on Graham Scan</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/geometry/ConvexHull.html#ConvexHull-java.util.Collection-">ConvexHull(Collection&lt;Vector2D&gt;)</a></span> - Constructor for class bio.singa.mathematics.algorithms.geometry.<a href="../bio/singa/mathematics/algorithms/geometry/ConvexHull.html" title="class in bio.singa.mathematics.algorithms.geometry">ConvexHull</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#COPPER">COPPER</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/model/Correlation.html" title="interface in bio.singa.features.model"><span class="typeNameLink">Correlation</span></a>&lt;<a href="../bio/singa/features/model/Correlation.html" title="type parameter in Correlation">FeatureType</a> extends <a href="../bio/singa/features/model/Feature.html" title="interface in bio.singa.features.model">Feature</a>&lt;?&gt;&gt; - Interface in <a href="../bio/singa/features/model/package-summary.html">bio.singa.features.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/VectorMetricProvider.html#COSINE_SIMILARITY">COSINE_SIMILARITY</a></span> - Static variable in class bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/VectorMetricProvider.html" title="class in bio.singa.mathematics.metrics.model">VectorMetricProvider</a></dt>
<dd>
<div class="block">The cosine similarity is a similarity measure not a distance measure.</div>
</dd>
<dt><a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html" title="class in bio.singa.mathematics.metrics.implementations"><span class="typeNameLink">CosineSimilarity</span></a>&lt;<a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html" title="type parameter in CosineSimilarity">VectorDimension</a> extends <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a>&gt; - Class in <a href="../bio/singa/mathematics/metrics/implementations/package-summary.html">bio.singa.mathematics.metrics.implementations</a></dt>
<dd>
<div class="block">The resulting similarity ranges from âˆ’1 meaning exactly opposite, to 1
 meaning exactly the same, with 0 indicating orthogonality (decorrelation),
 and in-between values indicating intermediate similarity or dissimilarity.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html#CosineSimilarity--">CosineSimilarity()</a></span> - Constructor for class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html" title="class in bio.singa.mathematics.metrics.implementations">CosineSimilarity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html#countAtomsOfElement-bio.singa.structure.model.molecules.MoleculeGraph-bio.singa.structure.elements.Element-">countAtomsOfElement(MoleculeGraph, Element)</a></span> - Static method in class bio.singa.structure.model.molecules.<a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html" title="class in bio.singa.structure.model.molecules">MoleculeGraphs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/MaterialProvider.html#crateMaterialFromColor-javafx.scene.paint.Color-">crateMaterialFromColor(Color)</a></span> - Static method in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/MaterialProvider.html" title="class in bio.singa.javafx.viewer">MaterialProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/ComplexedChemicalEntity.html#create-java.lang.String-">create(String)</a></span> - Static method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/ComplexedChemicalEntity.html" title="class in bio.singa.chemistry.entities">ComplexedChemicalEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/ComplexedChemicalEntity.html#create-bio.singa.features.identifiers.SimpleStringIdentifier-">create(SimpleStringIdentifier)</a></span> - Static method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/ComplexedChemicalEntity.html" title="class in bio.singa.chemistry.entities">ComplexedChemicalEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/SmallMolecule.html#create-java.lang.String-">create(String)</a></span> - Static method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/SmallMolecule.html" title="class in bio.singa.chemistry.entities">SmallMolecule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/SmallMolecule.html#create-bio.singa.features.identifiers.SimpleStringIdentifier-">create(SimpleStringIdentifier)</a></span> - Static method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/SmallMolecule.html" title="class in bio.singa.chemistry.entities">SmallMolecule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/reactions/RateConstant.html#create-double-">create(double)</a></span> - Static method in class bio.singa.chemistry.features.reactions.<a href="../bio/singa/chemistry/features/reactions/RateConstant.html" title="class in bio.singa.chemistry.features.reactions">RateConstant</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.html#create--">create()</a></span> - Static method in class bio.singa.mathematics.algorithms.clustering.<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.html" title="class in bio.singa.mathematics.algorithms.clustering">AffinityPropagation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.html#create--">create()</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.affinity.<a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.html" title="class in bio.singa.structure.algorithms.superimposition.affinity">AffinityAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.html#create--">create()</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder</a></dt>
<dd>
<div class="block">Creates a new instance of the actual <a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><code>ConsensusBuilder.Builder</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.html#create--">create()</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DBuilder</a></dt>
<dd>
<div class="block">Creates a new instance of the actual <a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d"><code>Fit3DBuilder.Builder</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.html#create--">create()</a></span> - Static method in class bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.html" title="class in bio.singa.structure.parser.pfam">PfamParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html#createAminoAcidFromAtoms-bio.singa.structure.model.identifiers.LeafIdentifier-bio.singa.structure.model.families.AminoAcidFamily-java.util.Map-bio.singa.structure.parser.pdb.structures.StructureParserOptions-">createAminoAcidFromAtoms(LeafIdentifier, AminoAcidFamily, Map&lt;String, OakAtom&gt;, StructureParserOptions)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html" title="class in bio.singa.structure.model.oak">LeafSubstructureFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html#createCell-int-bio.singa.mathematics.algorithms.voronoi.model.SiteEvent-">createCell(int, SiteEvent)</a></span> - Method in class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiDiagram.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">VoronoiDiagram</a></dt>
<dd>
<div class="block">Creates a new cell, adds it to the diagram and returns it.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationExamples.html#createCompartmentTestEnvironment--">createCompartmentTestEnvironment()</a></span> - Static method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationExamples.html" title="class in bio.singa.simulation.model.simulation">SimulationExamples</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationExamples.html#createDecompositionReactionExample--">createDecompositionReactionExample()</a></span> - Static method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationExamples.html" title="class in bio.singa.simulation.model.simulation">SimulationExamples</a></dt>
<dd>
<div class="block">This simulation simulates the thermal decomposition of dinitrogen pentaoxide.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationExamples.html#createDiffusionModuleExample-int-javax.measure.Quantity-">createDiffusionModuleExample(int, Quantity&lt;Time&gt;)</a></span> - Static method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationExamples.html" title="class in bio.singa.simulation.model.simulation">SimulationExamples</a></dt>
<dd>
<div class="block">This simulation simulates a diffusion of small molecules in a rectangular gird graph.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#createEdges-boolean-">createEdges(boolean)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option true will connect the molecule with bonds.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationExamples.html#createEquilibriumReactionExample--">createEquilibriumReactionExample()</a></span> - Static method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationExamples.html" title="class in bio.singa.simulation.model.simulation">SimulationExamples</a></dt>
<dd>
<div class="block">This simulation simulates a equilibrium reaction.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationExamples.html#createIodineMultiReactionExample--">createIodineMultiReactionExample()</a></span> - Static method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationExamples.html" title="class in bio.singa.simulation.model.simulation">SimulationExamples</a></dt>
<dd>
<div class="block">This simulation simulates a multiple reactions involving iodine.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html#createLeafSubstructure-bio.singa.structure.model.identifiers.LeafIdentifier-bio.singa.structure.model.families.StructuralFamily-">createLeafSubstructure(LeafIdentifier, StructuralFamily)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html" title="class in bio.singa.structure.model.oak">LeafSubstructureFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationExamples.html#createMichaelisMentenReactionExample--">createMichaelisMentenReactionExample()</a></span> - Static method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationExamples.html" title="class in bio.singa.simulation.model.simulation">SimulationExamples</a></dt>
<dd>
<div class="block">This simulation simulates a <a href="../bio/singa/simulation/model/modules/concentration/imlementations/MichaelisMentenReaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations"><code>MichaelisMentenReaction</code></a>, where D-Fructose 1-phosphate is convertet to
 glycerone phosphate and D-glyceraldehyde using fructose bisphosphate aldolase.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html#createMoleculeGraphFromStructure-bio.singa.structure.model.oak.OakLeafSubstructure-">createMoleculeGraphFromStructure(OakLeafSubstructure&lt;?&gt;)</a></span> - Static method in class bio.singa.structure.model.molecules.<a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html" title="class in bio.singa.structure.model.molecules">MoleculeGraphs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/RegularVector.html#createNewVector-double:A-java.lang.Class-">createNewVector(double[], Class&lt;VectorDimension&gt;)</a></span> - Static method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/RegularVector.html" title="class in bio.singa.mathematics.vectors">RegularVector</a></dt>
<dd>
<div class="block">A factory method for the creation of a new specific vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html#createNucleotideFromAtoms-bio.singa.structure.model.identifiers.LeafIdentifier-bio.singa.structure.model.families.NucleotideFamily-java.util.Map-bio.singa.structure.parser.pdb.structures.StructureParserOptions-">createNucleotideFromAtoms(LeafIdentifier, NucleotideFamily, Map&lt;String, OakAtom&gt;, StructureParserOptions)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/LeafSubstructureFactory.html" title="class in bio.singa.structure.model.oak">LeafSubstructureFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/ChemicalEntity.Builder.html#createObject-bio.singa.features.identifiers.SimpleStringIdentifier-">createObject(SimpleStringIdentifier)</a></span> - Method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/ChemicalEntity.Builder.html" title="class in bio.singa.chemistry.entities">ChemicalEntity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/ComplexedChemicalEntity.Builder.html#createObject-bio.singa.features.identifiers.SimpleStringIdentifier-">createObject(SimpleStringIdentifier)</a></span> - Method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/ComplexedChemicalEntity.Builder.html" title="class in bio.singa.chemistry.entities">ComplexedChemicalEntity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/Enzyme.Builder.html#createObject-bio.singa.features.identifiers.SimpleStringIdentifier-">createObject(SimpleStringIdentifier)</a></span> - Method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/Enzyme.Builder.html" title="class in bio.singa.chemistry.entities">Enzyme.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/Protein.Builder.html#createObject-bio.singa.features.identifiers.SimpleStringIdentifier-">createObject(SimpleStringIdentifier)</a></span> - Method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/Protein.Builder.html" title="class in bio.singa.chemistry.entities">Protein.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/Receptor.Builder.html#createObject-bio.singa.features.identifiers.SimpleStringIdentifier-">createObject(SimpleStringIdentifier)</a></span> - Method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/Receptor.Builder.html" title="class in bio.singa.chemistry.entities">Receptor.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/SmallMolecule.Builder.html#createObject-bio.singa.features.identifiers.SimpleStringIdentifier-">createObject(SimpleStringIdentifier)</a></span> - Method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/SmallMolecule.Builder.html" title="class in bio.singa.chemistry.entities">SmallMolecule.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/entities/Transporter.Builder.html#createObject-bio.singa.features.identifiers.SimpleStringIdentifier-">createObject(SimpleStringIdentifier)</a></span> - Method in class bio.singa.chemistry.entities.<a href="../bio/singa/chemistry/entities/Transporter.Builder.html" title="class in bio.singa.chemistry.entities">Transporter.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/MichaelisMentenReaction.Builder.html#createObject-bio.singa.simulation.model.simulation.Simulation-">createObject(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/MichaelisMentenReaction.Builder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">MichaelisMentenReaction.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/NthOrderReaction.Builder.html#createObject-bio.singa.simulation.model.simulation.Simulation-">createObject(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/NthOrderReaction.Builder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">NthOrderReaction.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/Reaction.Builder.html#createObject-bio.singa.simulation.model.simulation.Simulation-">createObject(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/Reaction.Builder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">Reaction.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/ReversibleReaction.Builder.html#createObject-bio.singa.simulation.model.simulation.Simulation-">createObject(Simulation)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/ReversibleReaction.Builder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations">ReversibleReaction.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Nodes.html#createRandomlyPlacedNode-int-">createRandomlyPlacedNode(int)</a></span> - Static method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Nodes.html" title="class in bio.singa.mathematics.graphs.model">Nodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Nodes.html#createRandomlyPlacedNode-int-bio.singa.mathematics.geometry.faces.Rectangle-">createRandomlyPlacedNode(int, Rectangle)</a></span> - Static method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Nodes.html" title="class in bio.singa.mathematics.graphs.model">Nodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonGraphs.html#createRectangularAutomatonGraph-int-int-">createRectangularAutomatonGraph(int, int)</a></span> - Static method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonGraphs.html" title="class in bio.singa.simulation.model.graphs">AutomatonGraphs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/FastMatrices.html#createRegularMatrix-double:A:A-">createRegularMatrix(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/FastMatrices.html" title="class in bio.singa.mathematics.matrices">FastMatrices</a></dt>
<dd>
<div class="block">Creates a regular matrix without checking if the values are well formed or symmetric.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationSchemeFactory.html#createRepresentationScheme-bio.singa.structure.algorithms.superimposition.fit3d.representations.RepresentationSchemeType-">createRepresentationScheme(RepresentationSchemeType)</a></span> - Static method in class bio.singa.structure.algorithms.superimposition.fit3d.representations.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationSchemeFactory.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.representations">RepresentationSchemeFactory</a></dt>
<dd>
<div class="block">Creates a <a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationScheme.html" title="interface in bio.singa.structure.algorithms.superimposition.fit3d.representations"><code>RepresentationScheme</code></a> for the given <a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationSchemeType.html" title="enum in bio.singa.structure.algorithms.superimposition.fit3d.representations"><code>RepresentationSchemeType</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationExamples.html#createSimulationFromSBML--">createSimulationFromSBML()</a></span> - Static method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationExamples.html" title="class in bio.singa.simulation.model.simulation">SimulationExamples</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/FastMatrices.html#createSquareMatrix-double:A:A-">createSquareMatrix(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/FastMatrices.html" title="class in bio.singa.mathematics.matrices">FastMatrices</a></dt>
<dd>
<div class="block">Creates a symmetric matrix without checking if the array has the right format or has to be compacted.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/FastMatrices.html#createSymmetricMatrix-double:A:A-">createSymmetricMatrix(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/FastMatrices.html" title="class in bio.singa.mathematics.matrices">FastMatrices</a></dt>
<dd>
<div class="block">Creates a symmetric matrix without checking if the array has the right format or has to be compacted.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationExamples.html#createSynthesisReactionExample--">createSynthesisReactionExample()</a></span> - Static method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationExamples.html" title="class in bio.singa.simulation.model.simulation">SimulationExamples</a></dt>
<dd>
<div class="block">This simulation simulates the synthesis of 1,3,5-octatriene (C8H12) from Buta-1,3-diene (C4H6).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AminoAcids.html#createVirtualCBAtom-bio.singa.structure.model.interfaces.AminoAcid-">createVirtualCBAtom(AminoAcid)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AminoAcids.html" title="class in bio.singa.structure.model.oak">AminoAcids</a></dt>
<dd>
<div class="block">This creates a virtual beta carbon for glycine by superimposing alanine.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#crossProduct-bio.singa.mathematics.vectors.Vector3D-">crossProduct(Vector3D)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>
<div class="block">Given two linearly independent vectors, the cross product, is a vector
 that is perpendicular to both and therefore normal to the plane
 containing them</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DMatch.html#CSV_HEADER">CSV_HEADER</a></span> - Static variable in class bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DMatch.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DMatch</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/geometry/bodies/Cube.html" title="class in bio.singa.mathematics.geometry.bodies"><span class="typeNameLink">Cube</span></a> - Class in <a href="../bio/singa/mathematics/geometry/bodies/package-summary.html">bio.singa.mathematics.geometry.bodies</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/bodies/Cube.html#Cube-bio.singa.mathematics.vectors.Vector3D-double-">Cube(Vector3D, double)</a></span> - Constructor for class bio.singa.mathematics.geometry.bodies.<a href="../bio/singa/mathematics/geometry/bodies/Cube.html" title="class in bio.singa.mathematics.geometry.bodies">Cube</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/topology/grids/cube/CubeCoordinate.html" title="class in bio.singa.mathematics.topology.grids.cube"><span class="typeNameLink">CubeCoordinate</span></a> - Class in <a href="../bio/singa/mathematics/topology/grids/cube/package-summary.html">bio.singa.mathematics.topology.grids.cube</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/topology/grids/cube/CubeCoordinate.html#CubeCoordinate-int-int-int-">CubeCoordinate(int, int, int)</a></span> - Constructor for class bio.singa.mathematics.topology.grids.cube.<a href="../bio/singa/mathematics/topology/grids/cube/CubeCoordinate.html" title="class in bio.singa.mathematics.topology.grids.cube">CubeCoordinate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/StructureViewer.html#cubes">cubes</a></span> - Static variable in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/StructureViewer.html" title="class in bio.singa.javafx.viewer">StructureViewer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarVolume.html#CUBIC_ANGSTROEM_PER_MOLE">CUBIC_ANGSTROEM_PER_MOLE</a></span> - Static variable in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarVolume.html" title="class in bio.singa.features.quantities">MolarVolume</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/OsmoticPermeability.html#CUBIC_CENTIMETER_PER_SECOND">CUBIC_CENTIMETER_PER_SECOND</a></span> - Static variable in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/OsmoticPermeability.html" title="class in bio.singa.chemistry.features.permeability">OsmoticPermeability</a></dt>
<dd>
<div class="block">Unit most commonly used to describe osmotic permeability.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarVolume.html#CUBIC_METRE_PER_MOLE">CUBIC_METRE_PER_MOLE</a></span> - Static variable in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarVolume.html" title="class in bio.singa.features.quantities">MolarVolume</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/elements/ElementProvider.html#CURIUM">CURIUM</a></span> - Static variable in class bio.singa.structure.elements.<a href="../bio/singa/structure/elements/ElementProvider.html" title="class in bio.singa.structure.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.Builder.html#cutoffScore-double-">cutoffScore(double)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DBuilder.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.SiteParameterConfigurationStep.html#cutoffScore-double-">cutoffScore(double)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.SiteParameterConfigurationStep.html" title="interface in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DBuilder.SiteParameterConfigurationStep</a></dt>
<dd>
<div class="block">The cutoff score that should be used when extending the site alignment.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#CYSTEINE_ATOM_NAMES">CYSTEINE_ATOM_NAMES</a></span> - Static variable in enum bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegion.html#CYTOSOL_A">CYTOSOL_A</a></span> - Static variable in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegion.html" title="class in bio.singa.simulation.model.sections">CellRegion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegion.html#CYTOSOL_B">CYTOSOL_B</a></span> - Static variable in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegion.html" title="class in bio.singa.simulation.model.sections">CellRegion</a></dt>
<dd>&nbsp;</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Y</a>&nbsp;<a href="index-26.html">Z</a>&nbsp;</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar.bottom">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.bottom" title="Skip navigation links">Skip navigation links</a></div>
<a name="navbar.bottom.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="index-2.html">Prev Letter</a></li>
<li><a href="index-4.html">Next Letter</a></li>
</ul>
<ul class="navList">
<li><a href="../index.html?index-files/index-3.html" target="_top">Frames</a></li>
<li><a href="index-3.html" target="_top">No&nbsp;Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../allclasses-noframe.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip.navbar.bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
