<!DOCTYPE HTML>
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (11.0.8) on Wed Sep 23 10:46:35 CEST 2020 -->
<title>I-Index</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2020-09-23">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery/jquery-ui.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../jquery/jszip/dist/jszip.min.js"></script>
<script type="text/javascript" src="../jquery/jszip-utils/dist/jszip-utils.min.js"></script>
<!--[if IE]>
<script type="text/javascript" src="../jquery/jszip-utils/dist/jszip-utils-ie.min.js"></script>
<![endif]-->
<script type="text/javascript" src="../jquery/jquery-3.3.1.js"></script>
<script type="text/javascript" src="../jquery/jquery-migrate-3.0.1.js"></script>
<script type="text/javascript" src="../jquery/jquery-ui.js"></script>
</head>
<body>
<script type="text/javascript"><!--
    try {
        if (location.href.indexOf('is-external=true') == -1) {
            parent.document.title="I-Index";
        }
    }
    catch(err) {
    }
//-->
var pathtoroot = "../";
var useModuleDirectories = true;
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<header role="banner">
<nav role="navigation">
<div class="fixedNav">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a id="navbar.top">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.top" title="Skip navigation links">Skip navigation links</a></div>
<a id="navbar.top.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../allclasses.html">All&nbsp;Classes</a></li>
</ul>
<ul class="navListSearch">
<li><label for="search">SEARCH:</label>
<input type="text" id="search" value="search" disabled="disabled">
<input type="reset" id="reset" value="reset" disabled="disabled">
</li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
</div>
<a id="skip.navbar.top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
</div>
<div class="navPadding">&nbsp;</div>
<script type="text/javascript"><!--
$('.navPadding').css('padding-top', $('.fixedNav').css("height"));
//-->
</script>
</nav>
</header>
<main role="main">
<div class="contentContainer"><a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Y</a>&nbsp;<a href="index-26.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes</a>&nbsp;<a href="../allpackages-index.html">All&nbsp;Packages</a><a id="I:I">
<!--   -->
</a>
<h2 class="title">I</h2>
<dl>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/HeaderToken.html#ID_CODE">ID_CODE</a></span> - bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/HeaderToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">HeaderToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.Builder.html#idealSuperimposition(boolean)">idealSuperimposition(boolean)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.affinity.<a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.affinity">AffinityAlignment.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.ParameterStep.html#idealSuperimposition(boolean)">idealSuperimposition(boolean)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.affinity.<a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.ParameterStep.html" title="interface in bio.singa.structure.algorithms.superimposition.affinity">AffinityAlignment.ParameterStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html#idealSuperimposition(boolean)">idealSuperimposition(boolean)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.ParameterStep.html#idealSuperimposition(boolean)">idealSuperimposition(boolean)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.ParameterStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.ParameterStep</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/identifiers/model/Identifiable.html" title="interface in bio.singa.features.identifiers.model"><span class="typeNameLink">Identifiable</span></a>&lt;<a href="../bio/singa/features/identifiers/model/Identifiable.html" title="type parameter in Identifiable">IdentifierType</a> extends <a href="../bio/singa/features/identifiers/model/Identifier.html" title="interface in bio.singa.features.identifiers.model">Identifier</a>&gt; - Interface in <a href="../bio/singa/features/identifiers/model/package-summary.html">bio.singa.features.identifiers.model</a></dt>
<dd>
<div class="block">The Identifiable interface specifies that, Objects implementing this interface can be identified by a certain
 <a href="../bio/singa/features/identifiers/model/Identifier.html" title="interface in bio.singa.features.identifiers.model"><code>Identifier</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html#identifier">identifier</a></span> - Variable in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity</a></dt>
<dd>
<div class="block">The distinct <a href="../bio/singa/features/identifiers/model/Identifier.html" title="interface in bio.singa.features.identifiers.model"><code>Identifier</code></a> by which this entity is identified.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html#identifier">identifier</a></span> - Variable in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html" title="class in bio.singa.mathematics.graphs.model">AbstractEdge</a></dt>
<dd>
<div class="block">An unique identifier.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.FinalStep.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.FinalStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.FinalStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.GeneralReactionBuilder.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.GeneralReactionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.GeneralReactionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.DiffusionBuilder.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.DiffusionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">Diffusion.DiffusionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.EntityLimitationStep.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.EntityLimitationStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">Diffusion.EntityLimitationStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EntityStep.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EntityStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.EntityStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/LateralMembraneDiffusion.EntityLimitationStep.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/LateralMembraneDiffusion.EntityLimitationStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">LateralMembraneDiffusion.EntityLimitationStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/LateralMembraneDiffusion.LateralMembraneDiffusionBuilder.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/LateralMembraneDiffusion.LateralMembraneDiffusionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">LateralMembraneDiffusion.LateralMembraneDiffusionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.FeatureStep.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.FeatureStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">PorousDiffusion.FeatureStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.PorousDiffusionBuilder.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.PorousDiffusionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">PorousDiffusion.PorousDiffusionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AbstractReactorBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ConnectorStep.html#identifier(java.lang.String)">identifier(String)</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ConnectorStep.html" title="interface in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.ConnectorStep</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/identifiers/model/Identifier.html" title="interface in bio.singa.features.identifiers.model"><span class="typeNameLink">Identifier</span></a> - Interface in <a href="../bio/singa/features/identifiers/model/package-summary.html">bio.singa.features.identifiers.model</a></dt>
<dd>
<div class="block">The identifier is used to identify objects that have the same structure.</div>
</dd>
<dt><a href="../bio/singa/features/identifiers/model/IdentifierPatternRegistry.html" title="class in bio.singa.features.identifiers.model"><span class="typeNameLink">IdentifierPatternRegistry</span></a> - Class in <a href="../bio/singa/features/identifiers/model/package-summary.html">bio.singa.features.identifiers.model</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/graphs/model/IdentifierSupplier.html" title="class in bio.singa.mathematics.graphs.model"><span class="typeNameLink">IdentifierSupplier</span></a> - Class in <a href="../bio/singa/mathematics/graphs/model/package-summary.html">bio.singa.mathematics.graphs.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/IdentifierSupplier.html#%3Cinit%3E()">IdentifierSupplier()</a></span> - Constructor for class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/IdentifierSupplier.html" title="class in bio.singa.mathematics.graphs.model">IdentifierSupplier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/IdentifierToMmtfBytesConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters"><span class="typeNameLink">IdentifierToMmtfBytesConverter</span></a> - Class in <a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/package-summary.html">bio.singa.structure.parser.pdb.structures.iterators.converters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html#IDENTITY_100">IDENTITY_100</a></span> - bio.singa.structure.parser.pdb.rest.cluster.<a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html" title="enum in bio.singa.structure.parser.pdb.rest.cluster">PDBSequenceCluster.PDBSequenceClusterIdentity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html#IDENTITY_30">IDENTITY_30</a></span> - bio.singa.structure.parser.pdb.rest.cluster.<a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html" title="enum in bio.singa.structure.parser.pdb.rest.cluster">PDBSequenceCluster.PDBSequenceClusterIdentity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html#IDENTITY_40">IDENTITY_40</a></span> - bio.singa.structure.parser.pdb.rest.cluster.<a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html" title="enum in bio.singa.structure.parser.pdb.rest.cluster">PDBSequenceCluster.PDBSequenceClusterIdentity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html#IDENTITY_50">IDENTITY_50</a></span> - bio.singa.structure.parser.pdb.rest.cluster.<a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html" title="enum in bio.singa.structure.parser.pdb.rest.cluster">PDBSequenceCluster.PDBSequenceClusterIdentity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html#IDENTITY_70">IDENTITY_70</a></span> - bio.singa.structure.parser.pdb.rest.cluster.<a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html" title="enum in bio.singa.structure.parser.pdb.rest.cluster">PDBSequenceCluster.PDBSequenceClusterIdentity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html#IDENTITY_90">IDENTITY_90</a></span> - bio.singa.structure.parser.pdb.rest.cluster.<a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html" title="enum in bio.singa.structure.parser.pdb.rest.cluster">PDBSequenceCluster.PDBSequenceClusterIdentity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html#IDENTITY_95">IDENTITY_95</a></span> - bio.singa.structure.parser.pdb.rest.cluster.<a href="../bio/singa/structure/parser/pdb/rest/cluster/PDBSequenceCluster.PDBSequenceClusterIdentity.html" title="enum in bio.singa.structure.parser.pdb.rest.cluster">PDBSequenceCluster.PDBSequenceClusterIdentity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/IdentityConverter.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.converters"><span class="typeNameLink">IdentityConverter</span></a>&lt;<a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/IdentityConverter.html" title="type parameter in IdentityConverter">Type</a>&gt; - Class in <a href="../bio/singa/structure/parser/pdb/structures/iterators/converters/package-summary.html">bio.singa.structure.parser.pdb.structures.iterators.converters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html#ignorableWhitespace(char%5B%5D,int,int)">ignorableWhitespace(char[], int, int)</a></span> - Method in class bio.singa.chemistry.features.databases.chebi.<a href="../bio/singa/chemistry/features/databases/chebi/ChEBIContentHandler.html" title="class in bio.singa.chemistry.features.databases.chebi">ChEBIContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/uniprot/UniProtContentHandler.html#ignorableWhitespace(char%5B%5D,int,int)">ignorableWhitespace(char[], int, int)</a></span> - Method in class bio.singa.chemistry.features.databases.uniprot.<a href="../bio/singa/chemistry/features/databases/uniprot/UniProtContentHandler.html" title="class in bio.singa.chemistry.features.databases.uniprot">UniProtContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/parser/GraphMLContentHandler.html#ignorableWhitespace(char%5B%5D,int,int)">ignorableWhitespace(char[], int, int)</a></span> - Method in class bio.singa.mathematics.graphs.parser.<a href="../bio/singa/mathematics/graphs/parser/GraphMLContentHandler.html" title="class in bio.singa.mathematics.graphs.parser">GraphMLContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/parser/ena/ENAContentHandler.html#ignorableWhitespace(char%5B%5D,int,int)">ignorableWhitespace(char[], int, int)</a></span> - Method in class bio.singa.sequence.parser.ena.<a href="../bio/singa/sequence/parser/ena/ENAContentHandler.html" title="class in bio.singa.sequence.parser.ena">ENAContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/PlipContentHandler.html#ignorableWhitespace(char%5B%5D,int,int)">ignorableWhitespace(char[], int, int)</a></span> - Method in class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/PlipContentHandler.html" title="class in bio.singa.structure.parser.plip">PlipContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/sifts/ResidueMapContentHandler.html#ignorableWhitespace(char%5B%5D,int,int)">ignorableWhitespace(char[], int, int)</a></span> - Method in class bio.singa.structure.parser.sifts.<a href="../bio/singa/structure/parser/sifts/ResidueMapContentHandler.html" title="class in bio.singa.structure.parser.sifts">ResidueMapContentHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.Builder.html#ignoreSpecifiedExchanges()">ignoreSpecifiedExchanges()</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DBuilder.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.SiteConfigurationStep.html#ignoreSpecifiedExchanges()">ignoreSpecifiedExchanges()</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.SiteConfigurationStep.html" title="interface in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DBuilder.SiteConfigurationStep</a></dt>
<dd>
<div class="block">Ignores the specified exchanges of the input sites and allows alignment of any type against any type using a
 heuristic that does not necessarily yield the best alignment possible.</div>
</dd>
<dt><a href="../bio/singa/features/exceptions/IllegalFeatureRequestException.html" title="class in bio.singa.features.exceptions"><span class="typeNameLink">IllegalFeatureRequestException</span></a> - Exception in <a href="../bio/singa/features/exceptions/package-summary.html">bio.singa.features.exceptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/exceptions/IllegalFeatureRequestException.html#%3Cinit%3E(java.lang.Class,java.lang.InstantiationException)">IllegalFeatureRequestException(Class&lt;? extends Feature&gt;, InstantiationException)</a></span> - Constructor for exception bio.singa.features.exceptions.<a href="../bio/singa/features/exceptions/IllegalFeatureRequestException.html" title="class in bio.singa.features.exceptions">IllegalFeatureRequestException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/exceptions/IllegalFeatureRequestException.html#%3Cinit%3E(java.lang.Class,java.lang.NoSuchMethodException)">IllegalFeatureRequestException(Class&lt;? extends Feature&gt;, NoSuchMethodException)</a></span> - Constructor for exception bio.singa.features.exceptions.<a href="../bio/singa/features/exceptions/IllegalFeatureRequestException.html" title="class in bio.singa.features.exceptions">IllegalFeatureRequestException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/exceptions/IllegalFeatureRequestException.html#%3Cinit%3E(java.lang.Exception)">IllegalFeatureRequestException(Exception)</a></span> - Constructor for exception bio.singa.features.exceptions.<a href="../bio/singa/features/exceptions/IllegalFeatureRequestException.html" title="class in bio.singa.features.exceptions">IllegalFeatureRequestException</a></dt>
<dd>
<div class="block">Constructs an instance of this class.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/VesicleStateRegistry.html#IN_PERINUCLEAR_STORAGE">IN_PERINUCLEAR_STORAGE</a></span> - Static variable in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/VesicleStateRegistry.html" title="class in bio.singa.simulation.model.agents.pointlike">VesicleStateRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.AdditionalConditionsStep.html#inArea(bio.singa.mathematics.geometry.model.Polygon)">inArea(Polygon)</a></span> - Method in interface bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.AdditionalConditionsStep.html" title="interface in bio.singa.simulation.model.concentrations">ConcentrationBuilder.AdditionalConditionsStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html#inArea(bio.singa.mathematics.geometry.model.Polygon)">inArea(Polygon)</a></span> - Method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html" title="class in bio.singa.simulation.model.concentrations">ConcentrationBuilder.ConcentrationBuilderImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html#inBetweenHalfSteps()">inBetweenHalfSteps()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationBasedModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.html#inBetweenHalfSteps()">inBetweenHalfSteps()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">Diffusion</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/identifiers/InChIKey.html" title="class in bio.singa.features.identifiers"><span class="typeNameLink">InChIKey</span></a> - Class in <a href="../bio/singa/features/identifiers/package-summary.html">bio.singa.features.identifiers</a></dt>
<dd>
<div class="block">The IUPAC International Chemical Identifier is a textual identifier for chemical substances, designed to provide a
 standard way to encode molecular information and to facilitate the search for such information in databases and on
 the web.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/InChIKey.html#%3Cinit%3E(java.lang.String)">InChIKey(String)</a></span> - Constructor for class bio.singa.features.identifiers.<a href="../bio/singa/features/identifiers/InChIKey.html" title="class in bio.singa.features.identifiers">InChIKey</a></dt>
<dd>
<div class="block">Creates a new identifier.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/InChIKey.html#%3Cinit%3E(java.lang.String,bio.singa.features.model.Evidence)">InChIKey(String, Evidence)</a></span> - Constructor for class bio.singa.features.identifiers.<a href="../bio/singa/features/identifiers/InChIKey.html" title="class in bio.singa.features.identifiers">InChIKey</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/features/identifiers/InChIKeyProvider.html" title="class in bio.singa.chemistry.features.identifiers"><span class="typeNameLink">InChIKeyProvider</span></a> - Class in <a href="../bio/singa/chemistry/features/identifiers/package-summary.html">bio.singa.chemistry.features.identifiers</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/identifiers/InChIKeyProvider.html#%3Cinit%3E()">InChIKeyProvider()</a></span> - Constructor for class bio.singa.chemistry.features.identifiers.<a href="../bio/singa/chemistry/features/identifiers/InChIKeyProvider.html" title="class in bio.singa.chemistry.features.identifiers">InChIKeyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/exceptions/IncompatibleDimensionsException.html" title="class in bio.singa.mathematics.exceptions"><span class="typeNameLink">IncompatibleDimensionsException</span></a> - Exception in <a href="../bio/singa/mathematics/exceptions/package-summary.html">bio.singa.mathematics.exceptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/exceptions/IncompatibleDimensionsException.html#%3Cinit%3E(bio.singa.mathematics.concepts.MultiDimensional,bio.singa.mathematics.concepts.MultiDimensional)">IncompatibleDimensionsException(MultiDimensional&lt;?&gt;, MultiDimensional&lt;?&gt;)</a></span> - Constructor for exception bio.singa.mathematics.exceptions.<a href="../bio/singa/mathematics/exceptions/IncompatibleDimensionsException.html" title="class in bio.singa.mathematics.exceptions">IncompatibleDimensionsException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/error/ErrorManager.Reason.html#INCREASE">INCREASE</a></span> - bio.singa.simulation.model.simulation.error.<a href="../bio/singa/simulation/model/simulation/error/ErrorManager.Reason.html" title="enum in bio.singa.simulation.model.simulation.error">ErrorManager.Reason</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/error/TimeStepManager.html#increaseTimeStep()">increaseTimeStep()</a></span> - Static method in class bio.singa.simulation.model.simulation.error.<a href="../bio/singa/simulation/model/simulation/error/TimeStepManager.html" title="class in bio.singa.simulation.model.simulation.error">TimeStepManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/FofanovEstimation.html#incrementGs()">incrementGs()</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.statistics.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/FofanovEstimation.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.statistics">FofanovEstimation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/FofanovEstimation.html#incrementNs()">incrementNs()</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.statistics.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/statistics/FofanovEstimation.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.statistics">FofanovEstimation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/scope/IndependentUpdate.html" title="class in bio.singa.simulation.model.modules.concentration.scope"><span class="typeNameLink">IndependentUpdate</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/scope/package-summary.html">bio.singa.simulation.model.modules.concentration.scope</a></dt>
<dd>
<div class="block">Independent Updatable <a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration"><code>ConcentrationBasedModule</code></a>s require the integer state of the currently calculated
 <a href="../bio/singa/simulation/model/simulation/Updatable.html" title="interface in bio.singa.simulation.model.simulation"><code>Updatable</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/scope/IndependentUpdate.html#%3Cinit%3E(bio.singa.simulation.model.modules.concentration.ConcentrationBasedModule)">IndependentUpdate(ConcentrationBasedModule)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.scope.<a href="../bio/singa/simulation/model/modules/concentration/scope/IndependentUpdate.html" title="class in bio.singa.simulation.model.modules.concentration.scope">IndependentUpdate</a></dt>
<dd>
<div class="block">Initializes the update scope for the corresponding module.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#INDIUM">INDIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#inferIdentifierFromFileName(boolean)">inferIdentifierFromFileName(boolean)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option to <code>true</code> will try parse any pdb identifier from the file name.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#inferTitleFromFileName(boolean)">inferTitleFromFileName(boolean)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option to <code>true</code> will set the title of the structure to the file name.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html#inhibitingEntity(bio.singa.simulation.entities.ChemicalEntity)">inhibitingEntity(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.InhibitionStep.html#inhibitingEntity(bio.singa.simulation.entities.ChemicalEntity)">inhibitingEntity(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.InhibitionStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.InhibitionStep</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html" title="class in bio.singa.simulation.model.concentrations"><span class="typeNameLink">InitialConcentration</span></a> - Class in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html#%3Cinit%3E()">InitialConcentration()</a></span> - Constructor for class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html" title="class in bio.singa.simulation.model.concentrations">InitialConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html#%3Cinit%3E(javax.measure.Quantity)">InitialConcentration(Quantity&lt;MolarConcentration&gt;)</a></span> - Constructor for class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html" title="class in bio.singa.simulation.model.concentrations">InitialConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html#%3Cinit%3E(javax.measure.Quantity,bio.singa.features.model.Evidence)">InitialConcentration(Quantity&lt;MolarConcentration&gt;, Evidence)</a></span> - Constructor for class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html" title="class in bio.singa.simulation.model.concentrations">InitialConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html#%3Cinit%3E(javax.measure.Quantity,java.util.List)">InitialConcentration(Quantity&lt;MolarConcentration&gt;, List&lt;Evidence&gt;)</a></span> - Constructor for class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html" title="class in bio.singa.simulation.model.concentrations">InitialConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/InitialConcentrations.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">InitialConcentrations</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/InitialConcentrations.html#%3Cinit%3E(java.util.List)">InitialConcentrations(List&lt;InitialConcentration&gt;)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/InitialConcentrations.html" title="class in bio.singa.simulation.features">InitialConcentrations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/InitialConcentrations.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">InitialConcentrations.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html#initialize()">initialize()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationBasedModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/SingleFileChannelMembraneTransport.html#initialize()">initialize()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/SingleFileChannelMembraneTransport.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">SingleFileChannelMembraneTransport</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/DisplacementBasedModule.html#initialize()">initialize()</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/DisplacementBasedModule.html" title="class in bio.singa.simulation.model.modules.displacement">DisplacementBasedModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/ClathrinMediatedEndocytosis.html#initialize()">initialize()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/ClathrinMediatedEndocytosis.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">ClathrinMediatedEndocytosis</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/VesicleFusion.html#initialize()">initialize()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/VesicleFusion.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">VesicleFusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/QualitativeModule.html#initialize()">initialize()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.<a href="../bio/singa/simulation/model/modules/qualitative/QualitativeModule.html" title="class in bio.singa.simulation.model.modules.qualitative">QualitativeModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/UpdateModule.html#initialize()">initialize()</a></span> - Method in interface bio.singa.simulation.model.modules.<a href="../bio/singa/simulation/model/modules/UpdateModule.html" title="interface in bio.singa.simulation.model.modules">UpdateModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/error/ErrorManager.html#initialize()">initialize()</a></span> - Method in class bio.singa.simulation.model.simulation.error.<a href="../bio/singa/simulation/model/simulation/error/ErrorManager.html" title="class in bio.singa.simulation.model.simulation.error">ErrorManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/UpdateScheduler.html#initialize()">initialize()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/UpdateScheduler.html" title="class in bio.singa.simulation.model.simulation">UpdateScheduler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html#initialize(bio.singa.simulation.model.sections.CellSubsection,bio.singa.simulation.entities.ChemicalEntity,javax.measure.Quantity)">initialize(CellSubsection, ChemicalEntity, Quantity&lt;MolarConcentration&gt;)</a></span> - Method in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html" title="class in bio.singa.simulation.model.sections">ConcentrationContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html#initialize(bio.singa.simulation.model.sections.CellTopology,bio.singa.simulation.entities.ChemicalEntity,javax.measure.Quantity)">initialize(CellTopology, ChemicalEntity, Quantity&lt;MolarConcentration&gt;)</a></span> - Method in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html" title="class in bio.singa.simulation.model.sections">ConcentrationContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/error/TimeStepManager.html#initialize(bio.singa.simulation.model.simulation.UpdateScheduler)">initialize(UpdateScheduler)</a></span> - Static method in class bio.singa.simulation.model.simulation.error.<a href="../bio/singa/simulation/model/simulation/error/TimeStepManager.html" title="class in bio.singa.simulation.model.simulation.error">TimeStepManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/MicrotubuleOrganizingCentre.html#initializeActin(bio.singa.simulation.model.agents.linelike.LineLikeAgentLayer,bio.singa.simulation.model.sections.CellRegion,int)">initializeActin(LineLikeAgentLayer, CellRegion, int)</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/MicrotubuleOrganizingCentre.html" title="class in bio.singa.simulation.model.agents.linelike">MicrotubuleOrganizingCentre</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/NeighborhoodMappingManager.html#initializeDiffusiveReduction(bio.singa.simulation.model.graphs.AutomatonNode,bio.singa.mathematics.geometry.model.Polygon,bio.singa.simulation.features.Ratio)">initializeDiffusiveReduction(AutomatonNode, Polygon, Ratio)</a></span> - Static method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/NeighborhoodMappingManager.html" title="class in bio.singa.simulation.model.graphs">NeighborhoodMappingManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/organelles/OrganelleTemplate.html#initializeGroup(int,bio.singa.simulation.model.sections.CellRegion)">initializeGroup(int, CellRegion)</a></span> - Method in class bio.singa.simulation.model.agents.organelles.<a href="../bio/singa/simulation/model/agents/organelles/OrganelleTemplate.html" title="class in bio.singa.simulation.model.agents.organelles">OrganelleTemplate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/organelles/OrganelleTemplate.html#initializeGroup(int,java.lang.String,java.lang.String)">initializeGroup(int, String, String)</a></span> - Method in class bio.singa.simulation.model.agents.organelles.<a href="../bio/singa/simulation/model/agents/organelles/OrganelleTemplate.html" title="class in bio.singa.simulation.model.agents.organelles">OrganelleTemplate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/organelles/OrganelleTemplate.html#initializeGroup(bio.singa.simulation.model.sections.CellRegion)">initializeGroup(CellRegion)</a></span> - Method in class bio.singa.simulation.model.agents.organelles.<a href="../bio/singa/simulation/model/agents/organelles/OrganelleTemplate.html" title="class in bio.singa.simulation.model.agents.organelles">OrganelleTemplate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html#initializeMembraneToMembraneMicrotubules(bio.singa.simulation.model.agents.surfacelike.Membrane,bio.singa.simulation.model.agents.surfacelike.Membrane,int)">initializeMembraneToMembraneMicrotubules(Membrane, Membrane, int)</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html" title="class in bio.singa.simulation.model.agents.linelike">LineLikeAgentLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/MicrotubuleOrganizingCentre.html#initializeMicrotubules(bio.singa.simulation.model.agents.linelike.LineLikeAgentLayer,bio.singa.simulation.model.sections.CellRegion,int)">initializeMicrotubules(LineLikeAgentLayer, CellRegion, int)</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/MicrotubuleOrganizingCentre.html" title="class in bio.singa.simulation.model.agents.linelike">MicrotubuleOrganizingCentre</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/NeighborhoodMappingManager.html#initializeNeighborhoodForGraph(bio.singa.simulation.model.graphs.AutomatonGraph)">initializeNeighborhoodForGraph(AutomatonGraph)</a></span> - Static method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/NeighborhoodMappingManager.html" title="class in bio.singa.simulation.model.graphs">NeighborhoodMappingManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Simulation.html#initializeSpatialRepresentations()">initializeSpatialRepresentations()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Simulation.html" title="class in bio.singa.simulation.model.simulation">Simulation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html#initializeSubsection(bio.singa.simulation.model.sections.CellSubsection,bio.singa.simulation.model.sections.CellTopology)">initializeSubsection(CellSubsection, CellTopology)</a></span> - Method in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/ConcentrationContainer.html" title="class in bio.singa.simulation.model.sections">ConcentrationContainer</a></dt>
<dd>
<div class="block">Initializes a new concentration pool for the corresponding subsection and topology.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellTopology.html#INNER">INNER</a></span> - bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellTopology.html" title="enum in bio.singa.simulation.model.sections">CellTopology</a></dt>
<dd>
<div class="block">A membrane is enclosing this subsection.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html#inOrderTraversal(java.util.function.Consumer)">inOrderTraversal(Consumer&lt;BinaryTreeNode&lt;T&gt;&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTreeNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/AreaCondition.html#inPolygon(bio.singa.mathematics.geometry.model.Polygon)">inPolygon(Polygon)</a></span> - Static method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/AreaCondition.html" title="class in bio.singa.simulation.model.concentrations">AreaCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.LocalSourceStep.html#inputStream(java.io.InputStream)">inputStream(InputStream)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.LocalSourceStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.LocalSourceStep</a></dt>
<dd>
<div class="block">Parses a structure from an input stream of a pdb file.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.Builder.html#inputStructuralMotifs(java.util.List)">inputStructuralMotifs(List&lt;StructuralMotif&gt;)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.affinity.<a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.affinity">AffinityAlignment.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.InputStep.html#inputStructuralMotifs(java.util.List)">inputStructuralMotifs(List&lt;StructuralMotif&gt;)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.affinity.<a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.InputStep.html" title="interface in bio.singa.structure.algorithms.superimposition.affinity">AffinityAlignment.InputStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html#inputStructuralMotifs(java.util.List)">inputStructuralMotifs(List&lt;StructuralMotif&gt;)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.InputStep.html#inputStructuralMotifs(java.util.List)">inputStructuralMotifs(List&lt;StructuralMotif&gt;)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.InputStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.InputStep</a></dt>
<dd>
<div class="block">Defines the input <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a>s for the <a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusAlignment.html" title="class in bio.singa.structure.algorithms.superimposition.consensus"><code>ConsensusAlignment</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.AdditionalConditionsStep.html#inRegionArea(bio.singa.simulation.model.sections.CellRegion)">inRegionArea(CellRegion)</a></span> - Method in interface bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.AdditionalConditionsStep.html" title="interface in bio.singa.simulation.model.concentrations">ConcentrationBuilder.AdditionalConditionsStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html#inRegionArea(bio.singa.simulation.model.sections.CellRegion)">inRegionArea(CellRegion)</a></span> - Method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/ConcentrationBuilder.ConcentrationBuilderImpl.html" title="class in bio.singa.simulation.model.concentrations">ConcentrationBuilder.ConcentrationBuilderImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/RedBlackNode.html#insertSuccessor(NodeType,NodeType)">insertSuccessor(NodeType, NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/RedBlackNode.html" title="class in bio.singa.mathematics.graphs.trees">RedBlackNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Polygons.html#INSIDE">INSIDE</a></span> - Static variable in class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Polygons.html" title="class in bio.singa.mathematics.geometry.faces">Polygons</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.html#inSimulation(bio.singa.simulation.model.simulation.Simulation)">inSimulation(Simulation)</a></span> - Static method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/Diffusion.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">Diffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.html#inSimulation(bio.singa.simulation.model.simulation.Simulation)">inSimulation(Simulation)</a></span> - Static method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/LateralMembraneDiffusion.html#inSimulation(bio.singa.simulation.model.simulation.Simulation)">inSimulation(Simulation)</a></span> - Static method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/LateralMembraneDiffusion.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">LateralMembraneDiffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/MembraneDiffusion.html#inSimulation(bio.singa.simulation.model.simulation.Simulation)">inSimulation(Simulation)</a></span> - Static method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/MembraneDiffusion.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">MembraneDiffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.html#inSimulation(bio.singa.simulation.model.simulation.Simulation)">inSimulation(Simulation)</a></span> - Static method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/PorousDiffusion.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">PorousDiffusion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/SingleFileChannelMembraneTransport.html#inSimulation(bio.singa.simulation.model.simulation.Simulation)">inSimulation(Simulation)</a></span> - Static method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/SingleFileChannelMembraneTransport.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">SingleFileChannelMembraneTransport</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/model/IdentifierPatternRegistry.html#instantiate(java.lang.String)">instantiate(String)</a></span> - Static method in class bio.singa.features.identifiers.model.<a href="../bio/singa/features/identifiers/model/IdentifierPatternRegistry.html" title="class in bio.singa.features.identifiers.model">IdentifierPatternRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/intervals/IntegerInterval.html" title="class in bio.singa.mathematics.intervals"><span class="typeNameLink">IntegerInterval</span></a> - Class in <a href="../bio/singa/mathematics/intervals/package-summary.html">bio.singa.mathematics.intervals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/intervals/IntegerInterval.html#%3Cinit%3E(java.lang.Integer,java.lang.Integer)">IntegerInterval(Integer, Integer)</a></span> - Constructor for class bio.singa.mathematics.intervals.<a href="../bio/singa/mathematics/intervals/IntegerInterval.html" title="class in bio.singa.mathematics.intervals">IntegerInterval</a></dt>
<dd>
<div class="block">Creates a new interval with the given values.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotif.Type.html#INTER">INTER</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotif.Type.html" title="enum in bio.singa.structure.model.oak">StructuralMotif.Type</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/parser/plip/Interaction.html" title="class in bio.singa.structure.parser.plip"><span class="typeNameLink">Interaction</span></a> - Class in <a href="../bio/singa/structure/parser/plip/package-summary.html">bio.singa.structure.parser.plip</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/Interaction.html#%3Cinit%3E(int,bio.singa.structure.parser.plip.InteractionType)">Interaction(int, InteractionType)</a></span> - Constructor for class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/Interaction.html" title="class in bio.singa.structure.parser.plip">Interaction</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/parser/plip/InteractionContainer.html" title="class in bio.singa.structure.parser.plip"><span class="typeNameLink">InteractionContainer</span></a> - Class in <a href="../bio/singa/structure/parser/plip/package-summary.html">bio.singa.structure.parser.plip</a></dt>
<dd>
<div class="block">The interaction container takes all interactions in a PLIP xml-file that was generated using the inter-chain
 interaction annotation feature.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/InteractionContainer.html#%3Cinit%3E()">InteractionContainer()</a></span> - Constructor for class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/InteractionContainer.html" title="class in bio.singa.structure.parser.plip">InteractionContainer</a></dt>
<dd>
<div class="block">Creates a new empty interaction container.</div>
</dd>
<dt><a href="../bio/singa/structure/parser/plip/InteractionType.html" title="enum in bio.singa.structure.parser.plip"><span class="typeNameLink">InteractionType</span></a> - Enum in <a href="../bio/singa/structure/parser/plip/package-summary.html">bio.singa.structure.parser.plip</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/UpdateScheduler.html#interrupt()">interrupt()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/UpdateScheduler.html" title="class in bio.singa.simulation.model.simulation">UpdateScheduler</a></dt>
<dd>
<div class="block">Returns true if the calling module was the first to call the method, therefore being allowed to optimize the
 time step.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ModuleState.html#INTERRUPTED">INTERRUPTED</a></span> - bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ModuleState.html" title="enum in bio.singa.simulation.model.modules.concentration">ModuleState</a></dt>
<dd>
<div class="block">Interrupted by another calculation</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/bodies/Sphere.html#intersect(bio.singa.mathematics.geometry.bodies.Sphere)">intersect(Sphere)</a></span> - Method in class bio.singa.mathematics.geometry.bodies.<a href="../bio/singa/mathematics/geometry/bodies/Sphere.html" title="class in bio.singa.mathematics.geometry.bodies">Sphere</a></dt>
<dd>
<div class="block">Returns true if the given sphere intersects with this.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Circles.html#intersect(bio.singa.mathematics.geometry.faces.Circle,bio.singa.mathematics.geometry.edges.LineSegment)">intersect(Circle, LineSegment)</a></span> - Static method in class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Circles.html" title="class in bio.singa.mathematics.geometry.faces">Circles</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/intervals/IntegerInterval.html#intersection(bio.singa.mathematics.intervals.IntegerInterval)">intersection(IntegerInterval)</a></span> - Method in class bio.singa.mathematics.intervals.<a href="../bio/singa/mathematics/intervals/IntegerInterval.html" title="class in bio.singa.mathematics.intervals">IntegerInterval</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/model/Polygon.IntersectionFragment.html#%3Cinit%3E(bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D)">IntersectionFragment(Vector2D, Vector2D, Vector2D)</a></span> - Constructor for class bio.singa.mathematics.geometry.model.<a href="../bio/singa/mathematics/geometry/model/Polygon.IntersectionFragment.html" title="class in bio.singa.mathematics.geometry.model">Polygon.IntersectionFragment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/javafx/renderer/IntersectionRenderer.html" title="class in bio.singa.javafx.renderer"><span class="typeNameLink">IntersectionRenderer</span></a> - Class in <a href="../bio/singa/javafx/renderer/package-summary.html">bio.singa.javafx.renderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/IntersectionRenderer.html#%3Cinit%3E()">IntersectionRenderer()</a></span> - Constructor for class bio.singa.javafx.renderer.<a href="../bio/singa/javafx/renderer/IntersectionRenderer.html" title="class in bio.singa.javafx.renderer">IntersectionRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotif.Type.html#INTRA">INTRA</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotif.Type.html" title="enum in bio.singa.structure.model.oak">StructuralMotif.Type</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html#inverse()">inverse()</a></span> - Method in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html" title="class in bio.singa.chemistry.features.permeability">MembranePermeability</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/OsmoticPermeability.html#inverse()">inverse()</a></span> - Method in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/OsmoticPermeability.html" title="class in bio.singa.chemistry.features.permeability">OsmoticPermeability</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarConcentration.html#inverse()">inverse()</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarConcentration.html" title="class in bio.singa.features.quantities">MolarConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarMass.html#inverse()">inverse()</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarMass.html" title="class in bio.singa.features.quantities">MolarMass</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarVolume.html#inverse()">inverse()</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarVolume.html" title="class in bio.singa.features.quantities">MolarVolume</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/AddModification.html#invert()">invert()</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/AddModification.html" title="class in bio.singa.simulation.reactions.modifications">AddModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/BindModification.html#invert()">invert()</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/BindModification.html" title="class in bio.singa.simulation.reactions.modifications">BindModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModification.html#invert()">invert()</a></span> - Method in interface bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModification.html" title="interface in bio.singa.simulation.reactions.modifications">ComplexEntityModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/ReleaseModification.html#invert()">invert()</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/ReleaseModification.html" title="class in bio.singa.simulation.reactions.modifications">ReleaseModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/RemoveModification.html#invert()">invert()</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/RemoveModification.html" title="class in bio.singa.simulation.reactions.modifications">RemoveModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ComplexReactor.html#invert()">invert()</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ComplexReactor.html" title="interface in bio.singa.simulation.reactions.reactors">ComplexReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/OneToOneReactor.html#invert()">invert()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/OneToOneReactor.html" title="class in bio.singa.simulation.reactions.reactors">OneToOneReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/OneToTwoReactor.html#invert()">invert()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/OneToTwoReactor.html" title="class in bio.singa.simulation.reactions.reactors">OneToTwoReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/TwoToOneReactor.html#invert()">invert()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/TwoToOneReactor.html" title="class in bio.singa.simulation.reactions.reactors">TwoToOneReactor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/concepts/Invertible.html" title="interface in bio.singa.mathematics.concepts"><span class="typeNameLink">Invertible</span></a>&lt;<a href="../bio/singa/mathematics/concepts/Invertible.html" title="type parameter in Invertible">NumberConcept</a>&gt; - Interface in <a href="../bio/singa/mathematics/concepts/package-summary.html">bio.singa.mathematics.concepts</a></dt>
<dd>
<div class="block">In abstract algebra, the idea of an inverse element generalizes concepts of a
 negation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#invertX()">invertX()</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>
<div class="block">Returns a new vector where the x coordinate is additively inverted
 (negated).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#invertX()">invertX()</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>
<div class="block">Returns a new vector where the x coordinate is additively inverted
 (negated).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#invertY()">invertY()</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>
<div class="block">Returns a new vector where the y coordinate is additively inverted
 (negated).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#invertY()">invertY()</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>
<div class="block">Returns a new vector where the y coordinate is additively inverted
 (negated).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#invertZ()">invertZ()</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>
<div class="block">Returns a new vector where the z coordinate is additively inverted
 (negated).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#INVISIBLE_TEXT">INVISIBLE_TEXT</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#IODINE">IODINE</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/XForm.html#ip">ip</a></span> - Variable in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/XForm.html" title="class in bio.singa.javafx.viewer">XForm</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#IRIDIUM">IRIDIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#IRON">IRON</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.GeneralReactionBuilder.html#irreversible()">irreversible()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.GeneralReactionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.GeneralReactionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.KineticStep.html#irreversible()">irreversible()</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.KineticStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.KineticStep</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/kineticlaws/IrreversibleKineticLaw.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.kineticlaws"><span class="typeNameLink">IrreversibleKineticLaw</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/kineticlaws/package-summary.html">bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.kineticlaws</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/kineticlaws/IrreversibleKineticLaw.html#%3Cinit%3E(bio.singa.simulation.model.modules.concentration.imlementations.reactions.Reaction)">IrreversibleKineticLaw(Reaction)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.kineticlaws.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/kineticlaws/IrreversibleKineticLaw.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.kineticlaws">IrreversibleKineticLaw</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#isAbove(bio.singa.mathematics.vectors.Vector2D)">isAbove(Vector2D)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/BitVector.html#isAllFalse()">isAllFalse()</a></span> - Method in interface bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/BitVector.html" title="interface in bio.singa.mathematics.vectors">BitVector</a></dt>
<dd>
<div class="block">Returns true if all values evaluate to false.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/BitVector.html#isAllTrue()">isAllTrue()</a></span> - Method in interface bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/BitVector.html" title="interface in bio.singa.mathematics.vectors">BitVector</a></dt>
<dd>
<div class="block">Returns true if all values evaluate to true.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isAlphaCarbon()">isAlphaCarbon()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#isAlphaCarbonStructure(bio.singa.structure.model.interfaces.Structure)">isAlphaCarbonStructure(Structure)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns true iff the given <a href="../bio/singa/structure/model/interfaces/Structure.html" title="interface in bio.singa.structure.model.interfaces"><code>Structure</code></a> consists only of alpha carbon atoms (<b>this may include hydrogen
 atoms</b>).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilter.html#isAminoAcid()">isAminoAcid()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.LeafFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/Element.html#isAnion()">isAnion()</a></span> - Method in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/Element.html" title="class in bio.singa.chemistry.model.elements">Element</a></dt>
<dd>
<div class="block">Returns <code>true</code> if this Element is an anion (excess of electrons) and <code>false</code> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html#isAnnotatedAsHeteroAtom()">isAnnotatedAsHeteroAtom()</a></span> - Method in interface bio.singa.structure.model.interfaces.<a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html" title="interface in bio.singa.structure.model.interfaces">LeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/mmtf/MmtfAminoAcid.html#isAnnotatedAsHeteroAtom()">isAnnotatedAsHeteroAtom()</a></span> - Method in class bio.singa.structure.model.mmtf.<a href="../bio/singa/structure/model/mmtf/MmtfAminoAcid.html" title="class in bio.singa.structure.model.mmtf">MmtfAminoAcid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/mmtf/MmtfLigand.html#isAnnotatedAsHeteroAtom()">isAnnotatedAsHeteroAtom()</a></span> - Method in class bio.singa.structure.model.mmtf.<a href="../bio/singa/structure/model/mmtf/MmtfLigand.html" title="class in bio.singa.structure.model.mmtf">MmtfLigand</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/mmtf/MmtfNucleotide.html#isAnnotatedAsHeteroAtom()">isAnnotatedAsHeteroAtom()</a></span> - Method in class bio.singa.structure.model.mmtf.<a href="../bio/singa/structure/model/mmtf/MmtfNucleotide.html" title="class in bio.singa.structure.model.mmtf">MmtfNucleotide</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html#isAnnotatedAsHeteroAtom()">isAnnotatedAsHeteroAtom()</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html" title="class in bio.singa.structure.model.oak">OakLeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/molecules/MoleculeIsomorphismFinder.AtomConditions.html#isArbitrary()">isArbitrary()</a></span> - Static method in class bio.singa.structure.algorithms.molecules.<a href="../bio/singa/structure/algorithms/molecules/MoleculeIsomorphismFinder.AtomConditions.html" title="class in bio.singa.structure.algorithms.molecules">MoleculeIsomorphismFinder.AtomConditions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/molecules/MoleculeIsomorphismFinder.BondConditions.html#isArbitrary()">isArbitrary()</a></span> - Static method in class bio.singa.structure.algorithms.molecules.<a href="../bio/singa/structure/algorithms/molecules/MoleculeIsomorphismFinder.BondConditions.html" title="class in bio.singa.structure.algorithms.molecules">MoleculeIsomorphismFinder.BondConditions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isArbitrary()">isArbitrary()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isBackbone()">isBackbone()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isBackboneCarbon()">isBackboneCarbon()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isBackboneNitrogen()">isBackboneNitrogen()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isBackboneOxygen()">isBackboneOxygen()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#isBackboneStructure(bio.singa.structure.model.interfaces.Structure)">isBackboneStructure(Structure)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>
<div class="block">Returns true iff the given <a href="../bio/singa/structure/model/interfaces/Structure.html" title="interface in bio.singa.structure.model.interfaces"><code>Structure</code></a> consists only of backbone atoms (<b>this may include beta carbon and
 hydrogen atoms</b>).</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#isBelow(bio.singa.mathematics.vectors.Vector2D)">isBelow(Vector2D)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isBetaCarbon()">isBetaCarbon()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarConcentration.html#isBig()">isBig()</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarConcentration.html" title="class in bio.singa.features.quantities">MolarConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.AreaMapping.html#isCached()">isCached()</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.AreaMapping.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode.AreaMapping</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isCarbon()">isCarbon()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/Reactant.html#isCatalyst()">isCatalyst()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/Reactant.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">Reactant</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/Element.html#isCation()">isCation()</a></span> - Method in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/Element.html" title="class in bio.singa.chemistry.model.elements">Element</a></dt>
<dd>
<div class="block">Returns <code>true</code> if this Element is an cation (shortage of electrons) and <code>false</code> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiCell.html#isClosed()">isClosed()</a></span> - Method in class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiCell.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">VoronoiCell</a></dt>
<dd>
<div class="block">Returns true, if this cell is closed and false otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/EndocytoticPit.html#isCollecting()">isCollecting()</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/EndocytoticPit.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">EndocytoticPit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html#isCompact(double%5B%5D%5B%5D)">isCompact(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html" title="class in bio.singa.mathematics.matrices">SymmetricMatrix</a></dt>
<dd>
<div class="block">Returns <code>true</code> if the values are already arranged in an jagged array and <code>false</code> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/units/UnitRegistry.html#isConcentrationUnit(javax.measure.Unit)">isConcentrationUnit(Unit&lt;?&gt;)</a></span> - Static method in class bio.singa.features.units.<a href="../bio/singa/features/units/UnitRegistry.html" title="class in bio.singa.features.units">UnitRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/LineSegment.html#isCongruentTo(bio.singa.mathematics.geometry.edges.LineSegment)">isCongruentTo(LineSegment)</a></span> - Method in interface bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/LineSegment.html" title="interface in bio.singa.mathematics.geometry.edges">LineSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#isConnectingHydrogens()">isConnectingHydrogens()</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option to <code>true</code> will try to connect hydrogen atom tho their respective partners.<br><br>
 Default value: <code>false</code></div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusContainer.html#isConsensus()">isConsensus()</a></span> - Method in class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusContainer.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusContainer</a></dt>
<dd>
<div class="block">Returns true if the associated <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a> is an artificial consensus.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChain.html#isConsiderInversion()">isConsiderInversion()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChain.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#isCreatingEdges()">isCreatingEdges()</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option true will connect the molecule with bonds.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Simulation.html#isDebug()">isDebug()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Simulation.html" title="class in bio.singa.simulation.model.simulation">Simulation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphRenderOptions.html#isDisplayingEdges()">isDisplayingEdges()</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphRenderOptions.html" title="class in bio.singa.javafx.renderer.graphs">GraphRenderOptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphRenderOptions.html#isDisplayingNodes()">isDisplayingNodes()</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphRenderOptions.html" title="class in bio.singa.javafx.renderer.graphs">GraphRenderOptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphRenderOptions.html#isDisplayText()">isDisplayText()</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphRenderOptions.html" title="class in bio.singa.javafx.renderer.graphs">GraphRenderOptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.Builder.html#isDistance(boolean)">isDistance(boolean)</a></span> - Method in class bio.singa.mathematics.algorithms.clustering.<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.Builder.html" title="class in bio.singa.mathematics.algorithms.clustering">AffinityPropagation.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.DistanceStep.html#isDistance(boolean)">isDistance(boolean)</a></span> - Method in interface bio.singa.mathematics.algorithms.clustering.<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.DistanceStep.html" title="interface in bio.singa.mathematics.algorithms.clustering">AffinityPropagation.DistanceStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html#isElement(bio.singa.chemistry.model.elements.Element)">isElement(Element)</a></span> - Static method in class bio.singa.structure.model.molecules.<a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html" title="class in bio.singa.structure.model.molecules">MoleculeGraphs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#isEnforceAminoAcidAtomNames()">isEnforceAminoAcidAtomNames()</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/GraphComplexNode.html#isEntity(bio.singa.simulation.entities.ChemicalEntity)">isEntity(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/GraphComplexNode.html" title="class in bio.singa.simulation.entities">GraphComplexNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html#isFix()">isFix()</a></span> - Method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html" title="class in bio.singa.simulation.model.concentrations">InitialConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/organelles/OrganelleTemplate.html#isGrouped()">isGrouped()</a></span> - Method in class bio.singa.simulation.model.agents.organelles.<a href="../bio/singa/simulation/model/agents/organelles/OrganelleTemplate.html" title="class in bio.singa.simulation.model.agents.organelles">OrganelleTemplate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#isHeteroAtoms()">isHeteroAtoms()</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/Line.html#isHorizontal()">isHorizontal()</a></span> - Method in class bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/Line.html" title="class in bio.singa.mathematics.geometry.edges">Line</a></dt>
<dd>
<div class="block">Returns <code>true</code>, if this line is horizontal (i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/LineSegment.html#isHorizontal()">isHorizontal()</a></span> - Method in interface bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/LineSegment.html" title="interface in bio.singa.mathematics.geometry.edges">LineSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isHydrogen()">isHydrogen()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/AlignmentMethod.html#isIdealSuperimposition()">isIdealSuperimposition()</a></span> - Method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/AlignmentMethod.html" title="class in bio.singa.structure.algorithms.superimposition">AlignmentMethod</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.ChainFilter.html#isInChain(java.lang.String)">isInChain(String)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.ChainFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.ChainFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#isInferringIdentifierFromFileName()">isInferringIdentifierFromFileName()</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option to <code>true</code> will try parse any pdb identifier from the file name.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#isInferringTitleFromFileName()">isInferringTitleFromFileName()</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option to <code>true</code> will set the title of the structure to the file name.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/TopologyCondition.html#isInner()">isInner()</a></span> - Static method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/TopologyCondition.html" title="class in bio.singa.simulation.model.concentrations">TopologyCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/topology/grids/hexagonal/HexagonalGrid.html#isInRange(bio.singa.mathematics.topology.grids.hexagonal.HexagonalCoordinate)">isInRange(HexagonalCoordinate)</a></span> - Method in class bio.singa.mathematics.topology.grids.hexagonal.<a href="../bio/singa/mathematics/topology/grids/hexagonal/HexagonalGrid.html" title="class in bio.singa.mathematics.topology.grids.hexagonal">HexagonalGrid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/topology/grids/rectangular/RectangularGrid.html#isInRange(bio.singa.mathematics.topology.grids.rectangular.RectangularCoordinate)">isInRange(RectangularCoordinate)</a></span> - Method in class bio.singa.mathematics.topology.grids.rectangular.<a href="../bio/singa/mathematics/topology/grids/rectangular/RectangularGrid.html" title="class in bio.singa.mathematics.topology.grids.rectangular">RectangularGrid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/Bounded.html#isInRange(BoundType)">isInRange(BoundType)</a></span> - Method in interface bio.singa.core.utility.<a href="../bio/singa/core/utility/Bounded.html" title="interface in bio.singa.core.utility">Bounded</a></dt>
<dd>
<div class="block">Checks whether the given value is larger than the lower bound and smaller than the upper bound specified for this
 object.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/topology/model/DiscreteGrid.html#isInRange(CoordinateType)">isInRange(CoordinateType)</a></span> - Method in interface bio.singa.mathematics.topology.model.<a href="../bio/singa/mathematics/topology/model/DiscreteGrid.html" title="interface in bio.singa.mathematics.topology.model">DiscreteGrid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#isInside(bio.singa.mathematics.geometry.model.Polygon)">isInside(Polygon)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>
<div class="block">Validates if this vector can be placed in the given polygon.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionTrack.html#isInverse()">isInverse()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionTrack.html" title="class in bio.singa.simulation.reactions.reactors">ReactionTrack</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/units/UnitRegistry.html#isInverseTimeUnit(javax.measure.Unit)">isInverseTimeUnit(Unit&lt;?&gt;)</a></span> - Static method in class bio.singa.features.units.<a href="../bio/singa/features/units/UnitRegistry.html" title="class in bio.singa.features.units">UnitRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/concepts/Invertible.html#isInvertible()">isInvertible()</a></span> - Method in interface bio.singa.mathematics.concepts.<a href="../bio/singa/mathematics/concepts/Invertible.html" title="interface in bio.singa.mathematics.concepts">Invertible</a></dt>
<dd>
<div class="block">Determines whether this object has a corresponding inverse element.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/Element.html#isIon()">isIon()</a></span> - Method in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/Element.html" title="class in bio.singa.chemistry.model.elements">Element</a></dt>
<dd>
<div class="block">Returns <code>true</code> if this Element is an ion and <code>false</code> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/Element.html#isIsotope()">isIsotope()</a></span> - Method in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/Element.html" title="class in bio.singa.chemistry.model.elements">Element</a></dt>
<dd>
<div class="block">Returns <code>true</code> if this Element is an isotope and <code>false</code> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/error/NumericalError.html#isLargerThan(bio.singa.simulation.model.simulation.error.NumericalError)">isLargerThan(NumericalError)</a></span> - Method in class bio.singa.simulation.model.simulation.error.<a href="../bio/singa/simulation/model/simulation/error/NumericalError.html" title="class in bio.singa.simulation.model.simulation.error">NumericalError</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html#isLeaf()">isLeaf()</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTreeNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/GraphPredicates.html#isLeafNode(bio.singa.mathematics.graphs.model.Node)">isLeafNode(Node)</a></span> - Static method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/GraphPredicates.html" title="class in bio.singa.mathematics.graphs.model">GraphPredicates</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#isLeftOf(bio.singa.mathematics.vectors.Vector2D)">isLeftOf(Vector2D)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/units/UnitRegistry.html#isLengthUnit(javax.measure.Unit)">isLengthUnit(Unit&lt;?&gt;)</a></span> - Static method in class bio.singa.features.units.<a href="../bio/singa/features/units/UnitRegistry.html" title="class in bio.singa.features.units">UnitRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilter.html#isLigand()">isLigand()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.LeafFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/TopologyCondition.html#isMembrane()">isMembrane()</a></span> - Static method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/TopologyCondition.html" title="class in bio.singa.simulation.model.concentrations">TopologyCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellSubsection.html#isMembrane()">isMembrane()</a></span> - Method in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellSubsection.html" title="class in bio.singa.simulation.model.sections">CellSubsection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ChemicalEntity.html#isMembraneBound()">isMembraneBound()</a></span> - Method in interface bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ChemicalEntity.html" title="interface in bio.singa.simulation.entities">ChemicalEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#isMembraneBound()">isMembraneBound()</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/SimpleEntity.html#isMembraneBound()">isMembraneBound()</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/SimpleEntity.html" title="class in bio.singa.simulation.entities">SimpleEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/concepts/MultiplicativelyInvertible.html#isMultiplicativelyInvertible()">isMultiplicativelyInvertible()</a></span> - Method in interface bio.singa.mathematics.concepts.<a href="../bio/singa/mathematics/concepts/MultiplicativelyInvertible.html" title="interface in bio.singa.mathematics.concepts">MultiplicativelyInvertible</a></dt>
<dd>
<div class="block">Determines whether this object has a corresponding multiplicative inverse element.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#isNearVector(bio.singa.mathematics.vectors.Vector2D,double)">isNearVector(Vector2D, double)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>
<div class="block">Checks, if the given point is contained in a circle with the given radius
 around the position of this vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isNitrogen()">isNitrogen()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/NodeTypeCondition.html#isNode()">isNode()</a></span> - Static method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/NodeTypeCondition.html" title="class in bio.singa.simulation.model.concentrations">NodeTypeCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilter.html#isNucleotide()">isNucleotide()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.LeafFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html#isObserved()">isObserved()</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html" title="class in bio.singa.simulation.model.agents.pointlike">Vesicle</a></dt>
<dd>
<div class="block">Returns <code>true</code> if this node is observed.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#isObserved()">isObserved()</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>
<div class="block">Returns <code>true</code> if this node is observed.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html#isObserved()">isObserved()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaManager</a></dt>
<dd>
<div class="block">Returns true if the concentrations are observed.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/AminoAcidFamily.html#ISOLEUCINE">ISOLEUCINE</a></span> - bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families">AminoAcidFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#ISOLEUCINE_ATOM_NAMES">ISOLEUCINE_ATOM_NAMES</a></span> - Static variable in enum bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/CovalentBondType.html#ISOMERIC_BOND_DOWN">ISOMERIC_BOND_DOWN</a></span> - bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/CovalentBondType.html" title="enum in bio.singa.chemistry.model">CovalentBondType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/CovalentBondType.html#ISOMERIC_BOND_UP">ISOMERIC_BOND_UP</a></span> - bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/CovalentBondType.html" title="enum in bio.singa.chemistry.model">CovalentBondType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#isOmittingHydrogen()">isOmittingHydrogen()</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option to <code>true</code> will omit all hydrogen (and eventually deuterium) atoms in amino acids and
 nucleotides.</div>
</dd>
<dt><a href="../bio/singa/mathematics/algorithms/graphs/isomorphism/IsomorphismCondition.html" title="class in bio.singa.mathematics.algorithms.graphs.isomorphism"><span class="typeNameLink">IsomorphismCondition</span></a> - Class in <a href="../bio/singa/mathematics/algorithms/graphs/isomorphism/package-summary.html">bio.singa.mathematics.algorithms.graphs.isomorphism</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/graphs/isomorphism/IsomorphismCondition.html#%3Cinit%3E()">IsomorphismCondition()</a></span> - Constructor for class bio.singa.mathematics.algorithms.graphs.isomorphism.<a href="../bio/singa/mathematics/algorithms/graphs/isomorphism/IsomorphismCondition.html" title="class in bio.singa.mathematics.algorithms.graphs.isomorphism">IsomorphismCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html#isOneOfElements(java.util.Set)">isOneOfElements(Set&lt;Element&gt;)</a></span> - Static method in class bio.singa.structure.model.molecules.<a href="../bio/singa/structure/model/molecules/MoleculeGraphs.html" title="class in bio.singa.structure.model.molecules">MoleculeGraphs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/LineSegment.html#isOnLine(bio.singa.mathematics.vectors.Vector2D)">isOnLine(Vector2D)</a></span> - Method in interface bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/LineSegment.html" title="interface in bio.singa.mathematics.geometry.edges">LineSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/CandidateConditionBuilder.html#isOnlyBoundAt(bio.singa.simulation.entities.BindingSite)">isOnlyBoundAt(BindingSite)</a></span> - Static method in class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/CandidateConditionBuilder.html" title="class in bio.singa.simulation.reactions.conditions">CandidateConditionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/reactions/conditions/IsOnlyBoundAt.html" title="class in bio.singa.simulation.reactions.conditions"><span class="typeNameLink">IsOnlyBoundAt</span></a> - Class in <a href="../bio/singa/simulation/reactions/conditions/package-summary.html">bio.singa.simulation.reactions.conditions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/conditions/IsOnlyBoundAt.html#%3Cinit%3E(bio.singa.simulation.entities.BindingSite)">IsOnlyBoundAt(BindingSite)</a></span> - Constructor for class bio.singa.simulation.reactions.conditions.<a href="../bio/singa/simulation/reactions/conditions/IsOnlyBoundAt.html" title="class in bio.singa.simulation.reactions.conditions">IsOnlyBoundAt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiCell.html#isOpen()">isOpen()</a></span> - Method in class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/VoronoiCell.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">VoronoiCell</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/Parabola.html#isOpenTowardsXAxis()">isOpenTowardsXAxis()</a></span> - Method in class bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/Parabola.html" title="class in bio.singa.mathematics.geometry.edges">Parabola</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/TopologyCondition.html#isOuter()">isOuter()</a></span> - Static method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/TopologyCondition.html" title="class in bio.singa.simulation.model.concentrations">TopologyCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isOxygen()">isOxygen()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isPhosphorus()">isPhosphorus()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/Reactant.html#isProduct()">isProduct()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/Reactant.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">Reactant</a></dt>
<dd>
<div class="block">Returns true if this reactant is a product.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/PiCation.html#isProtcharged()">isProtcharged()</a></span> - Method in class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/PiCation.html" title="class in bio.singa.structure.parser.plip">PiCation</a></dt>
<dd>
<div class="block">Returns whether the protein provides the charge?</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/HydrogenBond.html#isProtIsDon()">isProtIsDon()</a></span> - Method in class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/HydrogenBond.html" title="class in bio.singa.structure.parser.plip">HydrogenBond</a></dt>
<dd>
<div class="block">Returns whether the protein is the donor.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/WaterBridge.html#isProtIsDon()">isProtIsDon()</a></span> - Method in class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/WaterBridge.html" title="class in bio.singa.structure.parser.plip">WaterBridge</a></dt>
<dd>
<div class="block">Returns whether the protein is the donor.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/SaltBridge.html#isProtIsPos()">isProtIsPos()</a></span> - Method in class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/SaltBridge.html" title="class in bio.singa.structure.parser.plip">SaltBridge</a></dt>
<dd>
<div class="block">Returns whether the protein carries the positive charge.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/RedBlackNode.html#isRed()">isRed()</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/RedBlackNode.html" title="class in bio.singa.mathematics.graphs.trees">RedBlackNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureReducer.html#isReducingChains()">isReducingChains()</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureReducer.html" title="class in bio.singa.structure.parser.pdb.structures.iterators">StructureReducer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureReducer.html#isReducingModels()">isReducingModels()</a></span> - Method in class bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/StructureReducer.html" title="class in bio.singa.structure.parser.pdb.structures.iterators">StructureReducer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphCanvas.html#isResizable()">isResizable()</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphCanvas.html" title="class in bio.singa.javafx.renderer.graphs">GraphCanvas</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#isRetrievingLigandInformation()">isRetrievingLigandInformation()</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>
<div class="block">Setting this option to <code>true</code> will retrieve additional information from cif files from the PDB.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#isRightOf(bio.singa.mathematics.vectors.Vector2D)">isRightOf(Vector2D)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/molecules/MoleculeIsomorphismFinder.AtomConditions.html#isSameElement()">isSameElement()</a></span> - Static method in class bio.singa.structure.algorithms.molecules.<a href="../bio/singa/structure/algorithms/molecules/MoleculeIsomorphismFinder.AtomConditions.html" title="class in bio.singa.structure.algorithms.molecules">MoleculeIsomorphismFinder.AtomConditions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/molecules/MoleculeIsomorphismFinder.BondConditions.html#isSameType()">isSameType()</a></span> - Static method in class bio.singa.structure.algorithms.molecules.<a href="../bio/singa/structure/algorithms/molecules/MoleculeIsomorphismFinder.BondConditions.html" title="class in bio.singa.structure.algorithms.molecules">MoleculeIsomorphismFinder.BondConditions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionTrack.html#isSealed()">isSealed()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionTrack.html" title="class in bio.singa.simulation.reactions.reactors">ReactionTrack</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/HydrogenBond.html#isSidechain()">isSidechain()</a></span> - Method in class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/HydrogenBond.html" title="class in bio.singa.structure.parser.plip">HydrogenBond</a></dt>
<dd>
<div class="block">Returns whether the hydrogen bond is formed with the sidechain of the protein.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isSideChain()">isSideChain()</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#isSimilarTo(bio.singa.mathematics.vectors.Vector2D,double)">isSimilarTo(Vector2D, double)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/UpdateScheduler.html#isSkipDisplacementChecks()">isSkipDisplacementChecks()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/UpdateScheduler.html" title="class in bio.singa.simulation.model.simulation">UpdateScheduler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ChemicalEntity.html#isSmall()">isSmall()</a></span> - Method in interface bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ChemicalEntity.html" title="interface in bio.singa.simulation.entities">ChemicalEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#isSmall()">isSmall()</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/SimpleEntity.html#isSmall()">isSmall()</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/SimpleEntity.html" title="class in bio.singa.simulation.entities">SimpleEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/error/NumericalError.html#isSmallerThan(bio.singa.simulation.model.simulation.error.NumericalError)">isSmallerThan(NumericalError)</a></span> - Method in class bio.singa.simulation.model.simulation.error.<a href="../bio/singa/simulation/model/simulation/error/NumericalError.html" title="class in bio.singa.simulation.model.simulation.error">NumericalError</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/SquareMatrix.html#isSquare(double%5B%5D%5B%5D)">isSquare(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/SquareMatrix.html" title="class in bio.singa.mathematics.matrices">SquareMatrix</a></dt>
<dd>
<div class="block">Returns <code>true</code> if all potential columns are of the same length as the potential rows and <code>false</code>
 otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/SquareMatrix.html#isSquare(bio.singa.mathematics.matrices.Matrix)">isSquare(Matrix)</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/SquareMatrix.html" title="class in bio.singa.mathematics.matrices">SquareMatrix</a></dt>
<dd>
<div class="block">Returns <code>true</code> if the given matrix has equal column and row dimension and <code>false</code> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/FieldSupplier.html#isStrutCalculation()">isStrutCalculation()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/FieldSupplier.html" title="class in bio.singa.simulation.model.modules.concentration">FieldSupplier</a></dt>
<dd>
<div class="block">Returns the current state of calculations.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/units/UnitRegistry.html#isSubstanceUnit(javax.measure.Unit)">isSubstanceUnit(Unit&lt;?&gt;)</a></span> - Static method in class bio.singa.features.units.<a href="../bio/singa/features/units/UnitRegistry.html" title="class in bio.singa.features.units">UnitRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/Reactant.html#isSubstrate()">isSubstrate()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/Reactant.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">Reactant</a></dt>
<dd>
<div class="block">Returns true if this reactant is a substrate.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html#isSymmetric(double%5B%5D%5B%5D)">isSymmetric(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html" title="class in bio.singa.mathematics.matrices">SymmetricMatrix</a></dt>
<dd>
<div class="block">Returns <code>true</code> if the potential values are square and symmetric (mirrored at the main diagonal) and <code>
 false</code> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html#isSymmetric(bio.singa.mathematics.matrices.Matrix)">isSymmetric(Matrix)</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html" title="class in bio.singa.mathematics.matrices">SymmetricMatrix</a></dt>
<dd>
<div class="block">Returns <code>true</code> if the given matrix is square and symmetric (mirrored at the main diagonal) and <code>
 false</code> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html#isTargetedGrowth()">isTargetedGrowth()</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html" title="class in bio.singa.simulation.model.agents.linelike">LineLikeAgentLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/error/TimeStepManager.html#isTimeStepRescaled()">isTimeStepRescaled()</a></span> - Static method in class bio.singa.simulation.model.simulation.error.<a href="../bio/singa/simulation/model/simulation/error/TimeStepManager.html" title="class in bio.singa.simulation.model.simulation.error">TimeStepManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/units/UnitRegistry.html#isTimeUnit(javax.measure.Unit)">isTimeUnit(Unit&lt;?&gt;)</a></span> - Static method in class bio.singa.features.units.<a href="../bio/singa/features/units/UnitRegistry.html" title="class in bio.singa.features.units">UnitRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/TopologyCondition.html#isTopology(bio.singa.simulation.model.sections.CellTopology)">isTopology(CellTopology)</a></span> - Static method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/TopologyCondition.html" title="class in bio.singa.simulation.model.concentrations">TopologyCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/features/FeatureTable.html#isUseContinuousIndex()">isUseContinuousIndex()</a></span> - Method in class bio.singa.simulation.export.features.<a href="../bio/singa/simulation/export/features/FeatureTable.html" title="class in bio.singa.simulation.export.features">FeatureTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#isVector2D(bio.singa.mathematics.vectors.Vector)">isVector2D(Vector)</a></span> - Static method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#isVector3D(bio.singa.mathematics.vectors.Vector)">isVector3D(Vector)</a></span> - Static method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/Line.html#isVertical()">isVertical()</a></span> - Method in class bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/Line.html" title="class in bio.singa.mathematics.geometry.edges">Line</a></dt>
<dd>
<div class="block">Returns <code>true</code>, if this line is vertical (i.e.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/LineSegment.html#isVertical()">isVertical()</a></span> - Method in interface bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/LineSegment.html" title="interface in bio.singa.mathematics.geometry.edges">LineSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/NodeTypeCondition.html#isVesicle()">isVesicle()</a></span> - Static method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/NodeTypeCondition.html" title="class in bio.singa.simulation.model.concentrations">NodeTypeCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/RegularMatrix.html#isWellFormed(double%5B%5D%5B%5D)">isWellFormed(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/RegularMatrix.html" title="class in bio.singa.mathematics.matrices">RegularMatrix</a></dt>
<dd>
<div class="block">Returns <code>true</code> if all rows are equal in length and <code>false</code> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#isWithinRange(int,int)">isWithinRange(int, int)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilter.html#isWithinRange(int,int)">isWithinRange(int, int)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.LeafFilter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationManager.html#isWritingAliveFile()">isWritingAliveFile()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationManager.html" title="class in bio.singa.simulation.model.simulation">SimulationManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector.html#isZero()">isZero()</a></span> - Method in interface bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a></dt>
<dd>
<div class="block">Checks if this vector contains only Zeros.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ITALIC">ITALIC</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Y</a>&nbsp;<a href="index-26.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes</a>&nbsp;<a href="../allpackages-index.html">All&nbsp;Packages</a></div>
</main>
<footer role="contentinfo">
<nav role="navigation">
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a id="navbar.bottom">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.bottom" title="Skip navigation links">Skip navigation links</a></div>
<a id="navbar.bottom.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../allclasses.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
</div>
<a id="skip.navbar.bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</nav>
</footer>
</body>
</html>
