<!DOCTYPE HTML>
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (11.0.8) on Wed Sep 23 10:46:35 CEST 2020 -->
<title>A-Index</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="dc.created" content="2020-09-23">
<link rel="stylesheet" type="text/css" href="../stylesheet.css" title="Style">
<link rel="stylesheet" type="text/css" href="../jquery/jquery-ui.css" title="Style">
<script type="text/javascript" src="../script.js"></script>
<script type="text/javascript" src="../jquery/jszip/dist/jszip.min.js"></script>
<script type="text/javascript" src="../jquery/jszip-utils/dist/jszip-utils.min.js"></script>
<!--[if IE]>
<script type="text/javascript" src="../jquery/jszip-utils/dist/jszip-utils-ie.min.js"></script>
<![endif]-->
<script type="text/javascript" src="../jquery/jquery-3.3.1.js"></script>
<script type="text/javascript" src="../jquery/jquery-migrate-3.0.1.js"></script>
<script type="text/javascript" src="../jquery/jquery-ui.js"></script>
</head>
<body>
<script type="text/javascript"><!--
    try {
        if (location.href.indexOf('is-external=true') == -1) {
            parent.document.title="A-Index";
        }
    }
    catch(err) {
    }
//-->
var pathtoroot = "../";
var useModuleDirectories = true;
loadScripts(document, 'script');</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<header role="banner">
<nav role="navigation">
<div class="fixedNav">
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a id="navbar.top">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.top" title="Skip navigation links">Skip navigation links</a></div>
<a id="navbar.top.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../allclasses.html">All&nbsp;Classes</a></li>
</ul>
<ul class="navListSearch">
<li><label for="search">SEARCH:</label>
<input type="text" id="search" value="search" disabled="disabled">
<input type="reset" id="reset" value="reset" disabled="disabled">
</li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
</div>
<a id="skip.navbar.top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
</div>
<div class="navPadding">&nbsp;</div>
<script type="text/javascript"><!--
$('.navPadding').css('padding-top', $('.fixedNav').css("height"));
//-->
</script>
</nav>
</header>
<main role="main">
<div class="contentContainer"><a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Y</a>&nbsp;<a href="index-26.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes</a>&nbsp;<a href="../allpackages-index.html">All&nbsp;Packages</a><a id="I:A">
<!--   -->
</a>
<h2 class="title">A</h2>
<dl>
<dt><a href="../bio/singa/javafx/geometry/AbacusVisualization.html" title="class in bio.singa.javafx.geometry"><span class="typeNameLink">AbacusVisualization</span></a> - Class in <a href="../bio/singa/javafx/geometry/package-summary.html">bio.singa.javafx.geometry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/geometry/AbacusVisualization.html#%3Cinit%3E(java.util.List,double,double,double,double)">AbacusVisualization(List&lt;BitPlane&gt;, double, double, double, double)</a></span> - Constructor for class bio.singa.javafx.geometry.<a href="../bio/singa/javafx/geometry/AbacusVisualization.html" title="class in bio.singa.javafx.geometry">AbacusVisualization</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html" title="class in bio.singa.chemistry.model"><span class="typeNameLink">AbstractChemicalEntity</span></a> - Class in <a href="../bio/singa/chemistry/model/package-summary.html">bio.singa.chemistry.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html#%3Cinit%3E(java.lang.String)">AbstractChemicalEntity(String)</a></span> - Constructor for class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity</a></dt>
<dd>
<div class="block">Creates a new chemical entity with the given identifier.</div>
</dd>
<dt><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html" title="class in bio.singa.chemistry.model"><span class="typeNameLink">AbstractChemicalEntity.Builder</span></a>&lt;<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html" title="type parameter in AbstractChemicalEntity.Builder">TopLevelType</a> extends <a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity</a>,&#8203;<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html" title="type parameter in AbstractChemicalEntity.Builder">BuilderType</a> extends <a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity.Builder</a>&gt; - Class in <a href="../bio/singa/chemistry/model/package-summary.html">bio.singa.chemistry.model</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/reactions/modifications/AbstractComplexEntityModification.html" title="class in bio.singa.simulation.reactions.modifications"><span class="typeNameLink">AbstractComplexEntityModification</span></a> - Class in <a href="../bio/singa/simulation/reactions/modifications/package-summary.html">bio.singa.simulation.reactions.modifications</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/AbstractComplexEntityModification.html#%3Cinit%3E(bio.singa.simulation.entities.BindingSite)">AbstractComplexEntityModification(BindingSite)</a></span> - Constructor for class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/AbstractComplexEntityModification.html" title="class in bio.singa.simulation.reactions.modifications">AbstractComplexEntityModification</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/AbstractConcentrationCondition.html" title="class in bio.singa.simulation.model.concentrations"><span class="typeNameLink">AbstractConcentrationCondition</span></a> - Class in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/AbstractConcentrationCondition.html#%3Cinit%3E(int)">AbstractConcentrationCondition(int)</a></span> - Constructor for class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/AbstractConcentrationCondition.html" title="class in bio.singa.simulation.model.concentrations">AbstractConcentrationCondition</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/functions/AbstractDeltaFunction.html" title="class in bio.singa.simulation.model.modules.concentration.functions"><span class="typeNameLink">AbstractDeltaFunction</span></a>&lt;<a href="../bio/singa/simulation/model/modules/concentration/functions/AbstractDeltaFunction.html" title="type parameter in AbstractDeltaFunction">FunctionReturnType</a>&gt; - Class in <a href="../bio/singa/simulation/model/modules/concentration/functions/package-summary.html">bio.singa.simulation.model.modules.concentration.functions</a></dt>
<dd>
<div class="block">Delta Functions are used to calculate updates for <a href="../bio/singa/simulation/model/simulation/Updatable.html" title="interface in bio.singa.simulation.model.simulation"><code>Updatable</code></a>s during simulation.</div>
</dd>
<dt><a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html" title="class in bio.singa.mathematics.graphs.model"><span class="typeNameLink">AbstractEdge</span></a>&lt;<a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html" title="type parameter in AbstractEdge">NodeType</a> extends <a href="../bio/singa/mathematics/graphs/model/Node.html" title="interface in bio.singa.mathematics.graphs.model">Node</a>&lt;<a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html" title="type parameter in AbstractEdge">NodeType</a>,&#8203;? extends <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a>,&#8203;?&gt;&gt; - Class in <a href="../bio/singa/mathematics/graphs/model/package-summary.html">bio.singa.mathematics.graphs.model</a></dt>
<dd>
<div class="block">A simple implementation of th edge interface.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html#%3Cinit%3E()">AbstractEdge()</a></span> - Constructor for class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html" title="class in bio.singa.mathematics.graphs.model">AbstractEdge</a></dt>
<dd>
<div class="block">Creates a new empty edge.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html#%3Cinit%3E(int)">AbstractEdge(int)</a></span> - Constructor for class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html" title="class in bio.singa.mathematics.graphs.model">AbstractEdge</a></dt>
<dd>
<div class="block">Creates a new edge with the given identifier.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html#%3Cinit%3E(int,NodeType,NodeType)">AbstractEdge(int, NodeType, NodeType)</a></span> - Constructor for class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html" title="class in bio.singa.mathematics.graphs.model">AbstractEdge</a></dt>
<dd>
<div class="block">Creates a new Edge connecting two <a href="../bio/singa/mathematics/graphs/model/Node.html" title="interface in bio.singa.mathematics.graphs.model"><code>Node</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html#%3Cinit%3E(bio.singa.mathematics.graphs.model.AbstractEdge)">AbstractEdge(AbstractEdge&lt;NodeType&gt;)</a></span> - Constructor for class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractEdge.html" title="class in bio.singa.mathematics.graphs.model">AbstractEdge</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/model/AbstractFeature.html" title="class in bio.singa.features.model"><span class="typeNameLink">AbstractFeature</span></a>&lt;<a href="../bio/singa/features/model/AbstractFeature.html" title="type parameter in AbstractFeature">FeatureContent</a>&gt; - Class in <a href="../bio/singa/features/model/package-summary.html">bio.singa.features.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractFeature.html#%3Cinit%3E()">AbstractFeature()</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractFeature.html" title="class in bio.singa.features.model">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractFeature.html#%3Cinit%3E(FeatureContent)">AbstractFeature(FeatureContent)</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractFeature.html" title="class in bio.singa.features.model">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractFeature.html#%3Cinit%3E(FeatureContent,bio.singa.features.model.Evidence)">AbstractFeature(FeatureContent, Evidence)</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractFeature.html" title="class in bio.singa.features.model">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractFeature.html#%3Cinit%3E(FeatureContent,java.util.List)">AbstractFeature(FeatureContent, List&lt;Evidence&gt;)</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractFeature.html" title="class in bio.singa.features.model">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/model/AbstractFeature.Builder.html" title="class in bio.singa.features.model"><span class="typeNameLink">AbstractFeature.Builder</span></a>&lt;<a href="../bio/singa/features/model/AbstractFeature.Builder.html" title="type parameter in AbstractFeature.Builder">FeatureContent</a>,&#8203;<a href="../bio/singa/features/model/AbstractFeature.Builder.html" title="type parameter in AbstractFeature.Builder">TopLevelType</a> extends <a href="../bio/singa/features/model/Feature.html" title="interface in bio.singa.features.model">Feature</a>&lt;<a href="../bio/singa/features/model/AbstractFeature.Builder.html" title="type parameter in AbstractFeature.Builder">FeatureContent</a>&gt;,&#8203;<a href="../bio/singa/features/model/AbstractFeature.Builder.html" title="type parameter in AbstractFeature.Builder">BuilderType</a> extends <a href="../bio/singa/features/model/AbstractFeature.Builder.html" title="class in bio.singa.features.model">AbstractFeature.Builder</a>&gt; - Class in <a href="../bio/singa/features/model/package-summary.html">bio.singa.features.model</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/reactions/reactors/AbstractGraphComplexReactor.html" title="class in bio.singa.simulation.reactions.reactors"><span class="typeNameLink">AbstractGraphComplexReactor</span></a> - Class in <a href="../bio/singa/simulation/reactions/reactors/package-summary.html">bio.singa.simulation.reactions.reactors</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/AbstractGraphComplexReactor.html#%3Cinit%3E()">AbstractGraphComplexReactor()</a></span> - Constructor for class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/AbstractGraphComplexReactor.html" title="class in bio.singa.simulation.reactions.reactors">AbstractGraphComplexReactor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="class in bio.singa.mathematics.graphs.grid"><span class="typeNameLink">AbstractGridGraph</span></a>&lt;<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="type parameter in AbstractGridGraph">NodeType</a> extends <a href="../bio/singa/mathematics/graphs/model/Node.html" title="interface in bio.singa.mathematics.graphs.model">Node</a>&lt;<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="type parameter in AbstractGridGraph">NodeType</a>,&#8203;<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="type parameter in AbstractGridGraph">VectorType</a>,&#8203;<a href="../bio/singa/mathematics/topology/grids/rectangular/RectangularCoordinate.html" title="class in bio.singa.mathematics.topology.grids.rectangular">RectangularCoordinate</a>&gt;,&#8203;<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="type parameter in AbstractGridGraph">EdgeType</a> extends <a href="../bio/singa/mathematics/graphs/model/Edge.html" title="interface in bio.singa.mathematics.graphs.model">Edge</a>&lt;<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="type parameter in AbstractGridGraph">NodeType</a>&gt;,&#8203;<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="type parameter in AbstractGridGraph">VectorType</a> extends <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a>&gt; - Class in <a href="../bio/singa/mathematics/graphs/grid/package-summary.html">bio.singa.mathematics.graphs.grid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html#%3Cinit%3E(int,int)">AbstractGridGraph(int, int)</a></span> - Constructor for class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="class in bio.singa.mathematics.graphs.grid">AbstractGridGraph</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/core/parser/AbstractHTMLParser.html" title="class in bio.singa.core.parser"><span class="typeNameLink">AbstractHTMLParser</span></a>&lt;<a href="../bio/singa/core/parser/AbstractHTMLParser.html" title="type parameter in AbstractHTMLParser">ResultType</a>&gt; - Class in <a href="../bio/singa/core/parser/package-summary.html">bio.singa.core.parser</a></dt>
<dd>
<div class="block">This class allows handling and parameter definition for simple HTML requests.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/parser/AbstractHTMLParser.html#%3Cinit%3E()">AbstractHTMLParser()</a></span> - Constructor for class bio.singa.core.parser.<a href="../bio/singa/core/parser/AbstractHTMLParser.html" title="class in bio.singa.core.parser">AbstractHTMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/identifiers/model/AbstractIdentifier.html" title="class in bio.singa.features.identifiers.model"><span class="typeNameLink">AbstractIdentifier</span></a> - Class in <a href="../bio/singa/features/identifiers/model/package-summary.html">bio.singa.features.identifiers.model</a></dt>
<dd>
<div class="block">The most basic, abstract implementation of <a href="../bio/singa/features/identifiers/model/Identifier.html" title="interface in bio.singa.features.identifiers.model"><code>Identifier</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/model/AbstractIdentifier.html#%3Cinit%3E(java.lang.String,java.util.regex.Pattern)">AbstractIdentifier(String, Pattern)</a></span> - Constructor for class bio.singa.features.identifiers.model.<a href="../bio/singa/features/identifiers/model/AbstractIdentifier.html" title="class in bio.singa.features.identifiers.model">AbstractIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/model/AbstractIdentifier.html#%3Cinit%3E(java.lang.String,java.util.regex.Pattern,bio.singa.features.model.Evidence)">AbstractIdentifier(String, Pattern, Evidence)</a></span> - Constructor for class bio.singa.features.identifiers.model.<a href="../bio/singa/features/identifiers/model/AbstractIdentifier.html" title="class in bio.singa.features.identifiers.model">AbstractIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/model/AbstractIdentifier.html#%3Cinit%3E(java.lang.String,java.util.regex.Pattern,bio.singa.features.model.Evidence...)">AbstractIdentifier(String, Pattern, Evidence...)</a></span> - Constructor for class bio.singa.features.identifiers.model.<a href="../bio/singa/features/identifiers/model/AbstractIdentifier.html" title="class in bio.singa.features.identifiers.model">AbstractIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/kineticlaws/AbstractKineticLaw.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.kineticlaws"><span class="typeNameLink">AbstractKineticLaw</span></a> - Class in <a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/kineticlaws/package-summary.html">bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.kineticlaws</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/kineticlaws/AbstractKineticLaw.html#%3Cinit%3E(bio.singa.simulation.model.modules.concentration.imlementations.reactions.Reaction)">AbstractKineticLaw(Reaction)</a></span> - Constructor for class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.kineticlaws.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/kineticlaws/AbstractKineticLaw.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.kineticlaws">AbstractKineticLaw</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model"><span class="typeNameLink">AbstractMapGraph</span></a>&lt;<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="type parameter in AbstractMapGraph">NodeType</a> extends <a href="../bio/singa/mathematics/graphs/model/Node.html" title="interface in bio.singa.mathematics.graphs.model">Node</a>&lt;<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="type parameter in AbstractMapGraph">NodeType</a>,&#8203;<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="type parameter in AbstractMapGraph">VectorType</a>,&#8203;<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="type parameter in AbstractMapGraph">IdentifierType</a>&gt;,&#8203;<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="type parameter in AbstractMapGraph">EdgeType</a> extends <a href="../bio/singa/mathematics/graphs/model/Edge.html" title="interface in bio.singa.mathematics.graphs.model">Edge</a>&lt;<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="type parameter in AbstractMapGraph">NodeType</a>&gt;,&#8203;<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="type parameter in AbstractMapGraph">VectorType</a> extends <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a>,&#8203;<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="type parameter in AbstractMapGraph">IdentifierType</a>&gt; - Class in <a href="../bio/singa/mathematics/graphs/model/package-summary.html">bio.singa.mathematics.graphs.model</a></dt>
<dd>
<div class="block">This is a simple implementation of the graph interface, that handles the most common operations defined for adding
 and removing edges as well as nodes.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#%3Cinit%3E()">AbstractMapGraph()</a></span> - Constructor for class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>
<div class="block">Creates a new graph object.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#%3Cinit%3E(int,int)">AbstractMapGraph(int, int)</a></span> - Constructor for class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>
<div class="block">Creates a new Graph object with an initial load capacity for the node and
 edge list.</div>
</dd>
<dt><a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="class in bio.singa.mathematics.graphs.model"><span class="typeNameLink">AbstractNode</span></a>&lt;<a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="type parameter in AbstractNode">NodeType</a> extends <a href="../bio/singa/mathematics/graphs/model/Node.html" title="interface in bio.singa.mathematics.graphs.model">Node</a>&lt;<a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="type parameter in AbstractNode">NodeType</a>,&#8203;<a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="type parameter in AbstractNode">VectorType</a>,&#8203;<a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="type parameter in AbstractNode">IdentifierType</a>&gt;,&#8203;<a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="type parameter in AbstractNode">VectorType</a> extends <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a>,&#8203;<a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="type parameter in AbstractNode">IdentifierType</a>&gt; - Class in <a href="../bio/singa/mathematics/graphs/model/package-summary.html">bio.singa.mathematics.graphs.model</a></dt>
<dd>
<div class="block">A simple implementation of the node interface.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractNode.html#%3Cinit%3E(IdentifierType)">AbstractNode(IdentifierType)</a></span> - Constructor for class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="class in bio.singa.mathematics.graphs.model">AbstractNode</a></dt>
<dd>
<div class="block">Creates a new node with the given identifier.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractNode.html#%3Cinit%3E(IdentifierType,VectorType)">AbstractNode(IdentifierType, VectorType)</a></span> - Constructor for class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="class in bio.singa.mathematics.graphs.model">AbstractNode</a></dt>
<dd>
<div class="block">Creates a new node with the given position.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractNode.html#%3Cinit%3E(NodeType)">AbstractNode(NodeType)</a></span> - Constructor for class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="class in bio.singa.mathematics.graphs.model">AbstractNode</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/core/parser/AbstractParser.html" title="class in bio.singa.core.parser"><span class="typeNameLink">AbstractParser</span></a>&lt;<a href="../bio/singa/core/parser/AbstractParser.html" title="type parameter in AbstractParser">ResultType</a>&gt; - Class in <a href="../bio/singa/core/parser/package-summary.html">bio.singa.core.parser</a></dt>
<dd>
<div class="block">The class allows definition of resource path and storage for the fetched result.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/parser/AbstractParser.html#%3Cinit%3E()">AbstractParser()</a></span> - Constructor for class bio.singa.core.parser.<a href="../bio/singa/core/parser/AbstractParser.html" title="class in bio.singa.core.parser">AbstractParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/model/AbstractQuantitativeFeature.html" title="class in bio.singa.features.model"><span class="typeNameLink">AbstractQuantitativeFeature</span></a>&lt;<a href="../bio/singa/features/model/AbstractQuantitativeFeature.html" title="type parameter in AbstractQuantitativeFeature">FeatureContent</a> extends javax.measure.Quantity&lt;<a href="../bio/singa/features/model/AbstractQuantitativeFeature.html" title="type parameter in AbstractQuantitativeFeature">FeatureContent</a>&gt;&gt; - Class in <a href="../bio/singa/features/model/package-summary.html">bio.singa.features.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractQuantitativeFeature.html#%3Cinit%3E()">AbstractQuantitativeFeature()</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractQuantitativeFeature.html" title="class in bio.singa.features.model">AbstractQuantitativeFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractQuantitativeFeature.html#%3Cinit%3E(javax.measure.Quantity)">AbstractQuantitativeFeature(Quantity&lt;FeatureContent&gt;)</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractQuantitativeFeature.html" title="class in bio.singa.features.model">AbstractQuantitativeFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractQuantitativeFeature.html#%3Cinit%3E(javax.measure.Quantity,bio.singa.features.model.Evidence)">AbstractQuantitativeFeature(Quantity&lt;FeatureContent&gt;, Evidence)</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractQuantitativeFeature.html" title="class in bio.singa.features.model">AbstractQuantitativeFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractQuantitativeFeature.html#%3Cinit%3E(javax.measure.Quantity,java.util.List)">AbstractQuantitativeFeature(Quantity&lt;FeatureContent&gt;, List&lt;Evidence&gt;)</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractQuantitativeFeature.html" title="class in bio.singa.features.model">AbstractQuantitativeFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html#%3Cinit%3E()">AbstractReactorBuilder()</a></span> - Constructor for class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AbstractReactorBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/AbstractRepresentationScheme.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.representations"><span class="typeNameLink">AbstractRepresentationScheme</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/package-summary.html">bio.singa.structure.algorithms.superimposition.fit3d.representations</a></dt>
<dd>
<div class="block">This abstract implementation provide a fallback solution to compute the centroid of a <a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructure</code></a>, iff
 the specified representation fails.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/AbstractRepresentationScheme.html#%3Cinit%3E()">AbstractRepresentationScheme()</a></span> - Constructor for class bio.singa.structure.algorithms.superimposition.fit3d.representations.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/AbstractRepresentationScheme.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.representations">AbstractRepresentationScheme</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html" title="class in bio.singa.features.model"><span class="typeNameLink">AbstractScalableQuantitativeFeature</span></a>&lt;<a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html" title="type parameter in AbstractScalableQuantitativeFeature">FeatureContent</a> extends javax.measure.Quantity&lt;<a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html" title="type parameter in AbstractScalableQuantitativeFeature">FeatureContent</a>&gt;&gt; - Class in <a href="../bio/singa/features/model/package-summary.html">bio.singa.features.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html#%3Cinit%3E(javax.measure.Quantity)">AbstractScalableQuantitativeFeature(Quantity&lt;FeatureContent&gt;)</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html" title="class in bio.singa.features.model">AbstractScalableQuantitativeFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html#%3Cinit%3E(javax.measure.Quantity,bio.singa.features.model.Evidence)">AbstractScalableQuantitativeFeature(Quantity&lt;FeatureContent&gt;, Evidence)</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html" title="class in bio.singa.features.model">AbstractScalableQuantitativeFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html#%3Cinit%3E(javax.measure.Quantity,java.util.List)">AbstractScalableQuantitativeFeature(Quantity&lt;FeatureContent&gt;, List&lt;Evidence&gt;)</a></span> - Constructor for class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html" title="class in bio.singa.features.model">AbstractScalableQuantitativeFeature</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/sequence/model/AbstractSequence.html" title="class in bio.singa.sequence.model"><span class="typeNameLink">AbstractSequence</span></a>&lt;<a href="../bio/singa/sequence/model/AbstractSequence.html" title="type parameter in AbstractSequence">FamilyType</a> extends <a href="../bio/singa/structure/model/families/StructuralFamily.html" title="interface in bio.singa.structure.model.families">StructuralFamily</a>&gt; - Class in <a href="../bio/singa/sequence/model/package-summary.html">bio.singa.sequence.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/model/AbstractSequence.html#%3Cinit%3E(java.util.List)">AbstractSequence(List&lt;FamilyType&gt;)</a></span> - Constructor for class bio.singa.sequence.model.<a href="../bio/singa/sequence/model/AbstractSequence.html" title="class in bio.singa.sequence.model">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/parser/pdb/structures/iterators/sources/AbstractSourceIterator.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.sources"><span class="typeNameLink">AbstractSourceIterator</span></a>&lt;<a href="../bio/singa/structure/parser/pdb/structures/iterators/sources/AbstractSourceIterator.html" title="type parameter in AbstractSourceIterator">SourceContent</a>,&#8203;<a href="../bio/singa/structure/parser/pdb/structures/iterators/sources/AbstractSourceIterator.html" title="type parameter in AbstractSourceIterator">ContentType</a>&gt; - Class in <a href="../bio/singa/structure/parser/pdb/structures/iterators/sources/package-summary.html">bio.singa.structure.parser.pdb.structures.iterators.sources</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/sources/AbstractSourceIterator.html#%3Cinit%3E()">AbstractSourceIterator()</a></span> - Constructor for class bio.singa.structure.parser.pdb.structures.iterators.sources.<a href="../bio/singa/structure/parser/pdb/structures/iterators/sources/AbstractSourceIterator.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.sources">AbstractSourceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/sources/AbstractSourceIterator.html#%3Cinit%3E(java.util.List)">AbstractSourceIterator(List&lt;SourceContent&gt;)</a></span> - Constructor for class bio.singa.structure.parser.pdb.structures.iterators.sources.<a href="../bio/singa/structure/parser/pdb/structures/iterators/sources/AbstractSourceIterator.html" title="class in bio.singa.structure.parser.pdb.structures.iterators.sources">AbstractSourceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/parser/pdb/structures/iterators/AbstractStructureIterator.html" title="class in bio.singa.structure.parser.pdb.structures.iterators"><span class="typeNameLink">AbstractStructureIterator</span></a>&lt;<a href="../bio/singa/structure/parser/pdb/structures/iterators/AbstractStructureIterator.html" title="type parameter in AbstractStructureIterator">SourceType</a>,&#8203;<a href="../bio/singa/structure/parser/pdb/structures/iterators/AbstractStructureIterator.html" title="type parameter in AbstractStructureIterator">TargetType</a>&gt; - Class in <a href="../bio/singa/structure/parser/pdb/structures/iterators/package-summary.html">bio.singa.structure.parser.pdb.structures.iterators</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/iterators/AbstractStructureIterator.html#%3Cinit%3E(bio.singa.structure.parser.pdb.structures.iterators.sources.SourceIterator)">AbstractStructureIterator(SourceIterator&lt;SourceType, TargetType&gt;)</a></span> - Constructor for class bio.singa.structure.parser.pdb.structures.iterators.<a href="../bio/singa/structure/parser/pdb/structures/iterators/AbstractStructureIterator.html" title="class in bio.singa.structure.parser.pdb.structures.iterators">AbstractStructureIterator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/core/parser/AbstractUniProtIdentifierMappingParser.html" title="class in bio.singa.core.parser"><span class="typeNameLink">AbstractUniProtIdentifierMappingParser</span></a>&lt;<a href="../bio/singa/core/parser/AbstractUniProtIdentifierMappingParser.html" title="type parameter in AbstractUniProtIdentifierMappingParser">ResultType</a>&gt; - Class in <a href="../bio/singa/core/parser/package-summary.html">bio.singa.core.parser</a></dt>
<dd>
<div class="block">This is a clumsy parser for the response provided by the UniProt ID mapping service.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/parser/AbstractUniProtIdentifierMappingParser.html#%3Cinit%3E()">AbstractUniProtIdentifierMappingParser()</a></span> - Constructor for class bio.singa.core.parser.<a href="../bio/singa/core/parser/AbstractUniProtIdentifierMappingParser.html" title="class in bio.singa.core.parser">AbstractUniProtIdentifierMappingParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html" title="class in bio.singa.simulation.model.modules"><span class="typeNameLink">AbstractUpdateModule</span></a> - Class in <a href="../bio/singa/simulation/model/modules/package-summary.html">bio.singa.simulation.model.modules</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html#%3Cinit%3E()">AbstractUpdateModule()</a></span> - Constructor for class bio.singa.simulation.model.modules.<a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html" title="class in bio.singa.simulation.model.modules">AbstractUpdateModule</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/core/parser/AbstractXMLParser.html" title="class in bio.singa.core.parser"><span class="typeNameLink">AbstractXMLParser</span></a>&lt;<a href="../bio/singa/core/parser/AbstractXMLParser.html" title="type parameter in AbstractXMLParser">ResultType</a>&gt; - Class in <a href="../bio/singa/core/parser/package-summary.html">bio.singa.core.parser</a></dt>
<dd>
<div class="block">The foundation to parse everything xml related.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/parser/AbstractXMLParser.html#%3Cinit%3E()">AbstractXMLParser()</a></span> - Constructor for class bio.singa.core.parser.<a href="../bio/singa/core/parser/AbstractXMLParser.html" title="class in bio.singa.core.parser">AbstractXMLParser</a></dt>
<dd>
<div class="block">Creates a new XMLParser and initializes the XMLReader.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.AccelerationStep.html#acceleratingEntity(bio.singa.simulation.entities.ChemicalEntity)">acceleratingEntity(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.AccelerationStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.AccelerationStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html#acceleratingEntity(bio.singa.simulation.entities.ChemicalEntity)">acceleratingEntity(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.transport.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/transport/EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.transport">EndocytoticPitAbsorption.EndocytoticPitAbsorptionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/AdditionExpressionNode.html#accept(uk.co.cogitolearning.cogpar.ExpressionNodeVisitor)">accept(ExpressionNodeVisitor)</a></span> - Method in class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/AdditionExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">AdditionExpressionNode</a></dt>
<dd>
<div class="block">Implementation of the visitor design pattern.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html#accept(uk.co.cogitolearning.cogpar.ExpressionNodeVisitor)">accept(ExpressionNodeVisitor)</a></span> - Method in class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/ConstantExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">ConstantExpressionNode</a></dt>
<dd>
<div class="block">Implementation of the visitor design pattern.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/ExponentiationExpressionNode.html#accept(uk.co.cogitolearning.cogpar.ExpressionNodeVisitor)">accept(ExpressionNodeVisitor)</a></span> - Method in class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/ExponentiationExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">ExponentiationExpressionNode</a></dt>
<dd>
<div class="block">Implementation of the visitor design pattern.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/ExpressionNode.html#accept(uk.co.cogitolearning.cogpar.ExpressionNodeVisitor)">accept(ExpressionNodeVisitor)</a></span> - Method in interface uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/ExpressionNode.html" title="interface in uk.co.cogitolearning.cogpar">ExpressionNode</a></dt>
<dd>
<div class="block">Method needed for the visitor design pattern</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/FunctionExpressionNode.html#accept(uk.co.cogitolearning.cogpar.ExpressionNodeVisitor)">accept(ExpressionNodeVisitor)</a></span> - Method in class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/FunctionExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">FunctionExpressionNode</a></dt>
<dd>
<div class="block">Implementation of the visitor design pattern.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/MultiplicationExpressionNode.html#accept(uk.co.cogitolearning.cogpar.ExpressionNodeVisitor)">accept(ExpressionNodeVisitor)</a></span> - Method in class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/MultiplicationExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">MultiplicationExpressionNode</a></dt>
<dd>
<div class="block">Implementation of the visitor design pattern.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/VariableExpressionNode.html#accept(uk.co.cogitolearning.cogpar.ExpressionNodeVisitor)">accept(ExpressionNodeVisitor)</a></span> - Method in class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/VariableExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">VariableExpressionNode</a></dt>
<dd>
<div class="block">Implementation of the visitor design pattern.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vectors.html#accumulateGramSchmidtProjection(bio.singa.mathematics.vectors.Vector,java.util.List)">accumulateGramSchmidtProjection(Vector, List&lt;Vector&gt;)</a></span> - Static method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vectors.html" title="class in bio.singa.mathematics.vectors">Vectors</a></dt>
<dd>
<div class="block">Accumulates the <a href="../bio/singa/mathematics/vectors/Vectors.html#gramSchmidtProjection(bio.singa.mathematics.vectors.Vector,bio.singa.mathematics.vectors.Vector)"><code>Graham-Schmidt projection</code></a> for each vector
 in the given list of Vectors.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgent.html#ACTIN">ACTIN</a></span> - Static variable in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgent.html" title="class in bio.singa.simulation.model.agents.linelike">LineLikeAgent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/VesicleStateRegistry.html#ACTIN_ATTACHED">ACTIN_ATTACHED</a></span> - Static variable in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/VesicleStateRegistry.html" title="class in bio.singa.simulation.model.agents.pointlike">VesicleStateRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/VesicleStateRegistry.html#ACTIN_PROPELLED">ACTIN_PROPELLED</a></span> - Static variable in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/VesicleStateRegistry.html" title="class in bio.singa.simulation.model.agents.pointlike">VesicleStateRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/ActinBoostVelocity.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">ActinBoostVelocity</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/ActinBoostVelocity.html#%3Cinit%3E(javax.measure.Quantity)">ActinBoostVelocity(Quantity&lt;Speed&gt;)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/ActinBoostVelocity.html" title="class in bio.singa.simulation.features">ActinBoostVelocity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/ActinBoostVelocity.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">ActinBoostVelocity.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#ACTINIUM">ACTINIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDelta.html#add(double)">add(double)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDelta.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDelta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/intervals/DoubleInterval.html#add(bio.singa.mathematics.intervals.DoubleInterval)">add(DoubleInterval)</a></span> - Method in class bio.singa.mathematics.intervals.<a href="../bio/singa/mathematics/intervals/DoubleInterval.html" title="class in bio.singa.mathematics.intervals">DoubleInterval</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/intervals/IntegerInterval.html#add(bio.singa.mathematics.intervals.IntegerInterval)">add(IntegerInterval)</a></span> - Method in class bio.singa.mathematics.intervals.<a href="../bio/singa/mathematics/intervals/IntegerInterval.html" title="class in bio.singa.mathematics.intervals">IntegerInterval</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/RegularMatrix.html#add(bio.singa.mathematics.matrices.Matrix)">add(Matrix)</a></span> - Method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/RegularMatrix.html" title="class in bio.singa.mathematics.matrices">RegularMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/quaternions/RegularQuaternion.html#add(bio.singa.mathematics.quaternions.Quaternion)">add(Quaternion)</a></span> - Method in class bio.singa.mathematics.quaternions.<a href="../bio/singa/mathematics/quaternions/RegularQuaternion.html" title="class in bio.singa.mathematics.quaternions">RegularQuaternion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/RegularVector.html#add(bio.singa.mathematics.vectors.Vector)">add(Vector)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/RegularVector.html" title="class in bio.singa.mathematics.vectors">RegularVector</a></dt>
<dd>
<div class="block">The addition is an algebraic operation that returns a new vector where each element of this vector is added to
 the corresponding element in the given vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Scalar.html#add(bio.singa.mathematics.vectors.Vector)">add(Vector)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Scalar.html" title="class in bio.singa.mathematics.vectors">Scalar</a></dt>
<dd>
<div class="block">The addition is an algebraic operation that returns a new vector with
 their values added.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#add(bio.singa.mathematics.vectors.Vector)">add(Vector)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#add(bio.singa.mathematics.vectors.Vector2D)">add(Vector2D)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>
<div class="block">The addition is an algebraic operation that returns a new vector where
 each element of this vector is added to the corresponding element in the
 given vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#add(bio.singa.mathematics.vectors.Vector3D)">add(Vector3D)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>
<div class="block">The addition is an algebraic operation that returns a new vector where
 each element of this vector is added to the corresponding element in the
 given vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModificationBuilder.html#add(bio.singa.simulation.entities.BindingSite,bio.singa.simulation.entities.ChemicalEntity)">add(BindingSite, ChemicalEntity)</a></span> - Static method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModificationBuilder.html" title="class in bio.singa.simulation.reactions.modifications">ComplexEntityModificationBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html#add(bio.singa.simulation.entities.BindingSite,bio.singa.simulation.entities.ChemicalEntity)">add(BindingSite, ChemicalEntity)</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AbstractReactorBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.html#add(bio.singa.simulation.entities.BindingSite,bio.singa.simulation.entities.ChemicalEntity)">add(BindingSite, ChemicalEntity)</a></span> - Static method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ReactorChoiceStep.html#add(bio.singa.simulation.entities.BindingSite,bio.singa.simulation.entities.ChemicalEntity)">add(BindingSite, ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ReactorChoiceStep.html" title="interface in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.ReactorChoiceStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html#add(bio.singa.simulation.entities.ChemicalEntity)">add(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AbstractReactorBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.html#add(bio.singa.simulation.entities.ChemicalEntity)">add(ChemicalEntity)</a></span> - Static method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ReactorChoiceStep.html#add(bio.singa.simulation.entities.ChemicalEntity)">add(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ReactorChoiceStep.html" title="interface in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.ReactorChoiceStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#add(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.entities.BindingSite)">add(ChemicalEntity, BindingSite)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/ReactionNetworkGenerator.html#add(bio.singa.simulation.reactions.reactors.ReactionChain)">add(ReactionChain)</a></span> - Method in class bio.singa.simulation.reactions.<a href="../bio/singa/simulation/reactions/ReactionNetworkGenerator.html" title="class in bio.singa.simulation.reactions">ReactionNetworkGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/Tokenizer.html#add(java.lang.String,int)">add(String, int)</a></span> - Method in class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/Tokenizer.html" title="class in uk.co.cogitolearning.cogpar">Tokenizer</a></dt>
<dd>
<div class="block">Add a regular expression and a token id to the internal list of recognized tokens</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/parameters/ParameterStorage.html#add(java.lang.String,bio.singa.simulation.model.parameters.Parameter)">add(String, Parameter)</a></span> - Static method in class bio.singa.simulation.model.parameters.<a href="../bio/singa/simulation/model/parameters/ParameterStorage.html" title="class in bio.singa.simulation.model.parameters">ParameterStorage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html#add(javax.measure.Quantity)">add(Quantity&lt;MembranePermeability&gt;)</a></span> - Method in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html" title="class in bio.singa.chemistry.features.permeability">MembranePermeability</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/OsmoticPermeability.html#add(javax.measure.Quantity)">add(Quantity&lt;OsmoticPermeability&gt;)</a></span> - Method in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/OsmoticPermeability.html" title="class in bio.singa.chemistry.features.permeability">OsmoticPermeability</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarConcentration.html#add(javax.measure.Quantity)">add(Quantity&lt;MolarConcentration&gt;)</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarConcentration.html" title="class in bio.singa.features.quantities">MolarConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarMass.html#add(javax.measure.Quantity)">add(Quantity&lt;MolarMass&gt;)</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarMass.html" title="class in bio.singa.features.quantities">MolarMass</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarVolume.html#add(javax.measure.Quantity)">add(Quantity&lt;MolarVolume&gt;)</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarVolume.html" title="class in bio.singa.features.quantities">MolarVolume</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/concepts/Addable.html#add(NumberConcept)">add(NumberConcept)</a></span> - Method in interface bio.singa.mathematics.concepts.<a href="../bio/singa/mathematics/concepts/Addable.html" title="interface in bio.singa.mathematics.concepts">Addable</a></dt>
<dd>
<div class="block">Adds the given Object (called summand) to this Object (also called summand) and returns a new Object (called
 sum).</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/SequenceExpressionNode.html#add(uk.co.cogitolearning.cogpar.ExpressionNode,boolean)">add(ExpressionNode, boolean)</a></span> - Method in class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/SequenceExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">SequenceExpressionNode</a></dt>
<dd>
<div class="block">Add another term to the sequence</div>
</dd>
<dt><a href="../bio/singa/mathematics/concepts/Addable.html" title="interface in bio.singa.mathematics.concepts"><span class="typeNameLink">Addable</span></a>&lt;<a href="../bio/singa/mathematics/concepts/Addable.html" title="type parameter in Addable">NumberConcept</a> extends <a href="../bio/singa/mathematics/concepts/Addable.html" title="interface in bio.singa.mathematics.concepts">Addable</a>&lt;<a href="../bio/singa/mathematics/concepts/Addable.html" title="type parameter in Addable">NumberConcept</a>&gt;&gt; - Interface in <a href="../bio/singa/mathematics/concepts/package-summary.html">bio.singa.mathematics.concepts</a></dt>
<dd>
<div class="block">The "addition" operation can be any commutative and associative binary operation on a number concept.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html#addActin(bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D)">addActin(Vector2D, Vector2D)</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html" title="class in bio.singa.simulation.model.agents.linelike">LineLikeAgentLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html#addAdditionalIdentifier(bio.singa.features.identifiers.model.Identifier)">addAdditionalIdentifier(Identifier)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/SimpleEntity.html#addAdditionalIdentifier(bio.singa.features.identifiers.model.Identifier)">addAdditionalIdentifier(Identifier)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/SimpleEntity.html" title="class in bio.singa.simulation.entities">SimpleEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html#addAdditionalIdentifiers(java.util.Collection)">addAdditionalIdentifiers(Collection&lt;Identifier&gt;)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/SimpleEntity.html#addAdditionalIdentifiers(java.util.Collection)">addAdditionalIdentifiers(Collection&lt;Identifier&gt;)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/SimpleEntity.html" title="class in bio.singa.simulation.entities">SimpleEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/volumelike/VolumeLayer.html#addAgent(bio.singa.simulation.model.agents.volumelike.VolumeLikeAgent)">addAgent(VolumeLikeAgent)</a></span> - Method in class bio.singa.simulation.model.agents.volumelike.<a href="../bio/singa/simulation/model/agents/volumelike/VolumeLayer.html" title="class in bio.singa.simulation.model.agents.volumelike">VolumeLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/volumelike/VolumeLayer.html#addAgents(java.util.Collection)">addAgents(Collection&lt;VolumeLikeAgent&gt;)</a></span> - Method in class bio.singa.simulation.model.agents.volumelike.<a href="../bio/singa/simulation/model/agents/volumelike/VolumeLayer.html" title="class in bio.singa.simulation.model.agents.volumelike">VolumeLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractFeature.html#addAlternativeContent(FeatureContent)">addAlternativeContent(FeatureContent)</a></span> - Method in class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractFeature.html" title="class in bio.singa.features.model">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/Feature.html#addAlternativeContent(FeatureContent)">addAlternativeContent(FeatureContent)</a></span> - Method in interface bio.singa.features.model.<a href="../bio/singa/features/model/Feature.html" title="interface in bio.singa.features.model">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractQuantitativeFeature.html#addAlternativeValue(java.lang.Double)">addAlternativeValue(Double)</a></span> - Method in class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractQuantitativeFeature.html" title="class in bio.singa.features.model">AbstractQuantitativeFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html#addAlternativeValue(java.lang.Double)">addAlternativeValue(Double)</a></span> - Method in class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractScalableQuantitativeFeature.html" title="class in bio.singa.features.model">AbstractScalableQuantitativeFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/Protein.html#addAminoAcidSequence(java.lang.String)">addAminoAcidSequence(String)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/Protein.html" title="class in bio.singa.chemistry.model">Protein</a></dt>
<dd>
<div class="block">Adds an amino acid sequence as an annotation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/annotations/Annotatable.html#addAnnotation(bio.singa.chemistry.annotations.Annotation)">addAnnotation(Annotation)</a></span> - Method in interface bio.singa.chemistry.annotations.<a href="../bio/singa/chemistry/annotations/Annotatable.html" title="interface in bio.singa.chemistry.annotations">Annotatable</a></dt>
<dd>
<div class="block">Adds an <a href="../bio/singa/chemistry/annotations/Annotation.html" title="class in bio.singa.chemistry.annotations"><code>Annotation</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#addAreaMapping(bio.singa.simulation.model.sections.CellSubsection,bio.singa.simulation.model.graphs.AutomatonNode.AreaMapping)">addAreaMapping(CellSubsection, AutomatonNode.AreaMapping)</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html#addAssociatedNode(bio.singa.simulation.model.graphs.AutomatonNode,double)">addAssociatedNode(AutomatonNode, double)</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html" title="class in bio.singa.simulation.model.agents.pointlike">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html#addAtom(bio.singa.structure.model.oak.OakAtom)">addAtom(OakAtom)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html" title="class in bio.singa.structure.model.oak">OakLeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakStructure.html#addAtom(java.lang.String,java.lang.String,bio.singa.mathematics.vectors.Vector3D)">addAtom(String, String, Vector3D)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakStructure.html" title="class in bio.singa.structure.model.oak">OakStructure</a></dt>
<dd>
<div class="block">Adds an <a href="../bio/singa/structure/model/interfaces/Atom.html" title="interface in bio.singa.structure.model.interfaces"><code>Atom</code></a> to the <a href="../bio/singa/structure/model/interfaces/Structure.html" title="interface in bio.singa.structure.model.interfaces"><code>Structure</code></a>
 FIXME: atom serial overflow may happen (if exceeds 9999)</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/voronoi/model/BeachLine.html#addBeachSection(bio.singa.mathematics.algorithms.voronoi.model.SiteEvent)">addBeachSection(SiteEvent)</a></span> - Method in class bio.singa.mathematics.algorithms.voronoi.model.<a href="../bio/singa/mathematics/algorithms/voronoi/model/BeachLine.html" title="class in bio.singa.mathematics.algorithms.voronoi.model">BeachLine</a></dt>
<dd>
<div class="block">For a given site event, add a beach section to the beach line.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/GraphComplexNode.html#addBindingSite(bio.singa.simulation.entities.BindingSite)">addBindingSite(BindingSite)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/GraphComplexNode.html" title="class in bio.singa.simulation.entities">GraphComplexNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#addBindingSite(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.entities.BindingSite)">addBindingSite(ChemicalEntity, BindingSite)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html#addBondBetween(bio.singa.structure.model.oak.OakAtom,bio.singa.structure.model.oak.OakAtom)">addBondBetween(OakAtom, OakAtom)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html" title="class in bio.singa.structure.model.oak">OakLeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html#addBondBetween(bio.singa.structure.model.oak.OakAtom,bio.singa.structure.model.oak.OakAtom,bio.singa.chemistry.model.CovalentBondType)">addBondBetween(OakAtom, OakAtom, CovalentBondType)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html" title="class in bio.singa.structure.model.oak">OakLeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html#addBondBetween(bio.singa.structure.model.oak.OakBond,bio.singa.structure.model.oak.OakAtom,bio.singa.structure.model.oak.OakAtom)">addBondBetween(OakBond, OakAtom, OakAtom)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html" title="class in bio.singa.structure.model.oak">OakLeafSubstructure</a></dt>
<dd>
<div class="block">Adds a bond connecting the the given atoms.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/AbstractComplexEntityModification.html#addCandidate(bio.singa.simulation.entities.ComplexEntity)">addCandidate(ComplexEntity)</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/AbstractComplexEntityModification.html" title="class in bio.singa.simulation.reactions.modifications">AbstractComplexEntityModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModification.html#addCandidate(bio.singa.simulation.entities.ComplexEntity)">addCandidate(ComplexEntity)</a></span> - Method in interface bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModification.html" title="interface in bio.singa.simulation.reactions.modifications">ComplexEntityModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#addCatalyst(bio.singa.simulation.entities.ChemicalEntity)">addCatalyst(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html#addCatalyst(bio.singa.simulation.entities.ChemicalEntity)">addCatalyst(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticReactantStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#addCatalyst(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.model.sections.CellTopology)">addCatalyst(ChemicalEntity, CellTopology)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html#addCatalyst(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.model.sections.CellTopology)">addCatalyst(ChemicalEntity, CellTopology)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticReactantStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/StaticReactantBehavior.html#addCatalyst(bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.Reactant)">addCatalyst(Reactant)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/StaticReactantBehavior.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">StaticReactantBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonGraph.html#addCellRegion(bio.singa.simulation.model.sections.CellRegion)">addCellRegion(CellRegion)</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonGraph.html" title="class in bio.singa.simulation.model.graphs">AutomatonGraph</a></dt>
<dd>
<div class="block">Adds a cell section to this graph but does not associate any node to it.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakModel.html#addChain(bio.singa.structure.model.oak.OakChain)">addChain(OakChain)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakModel.html" title="class in bio.singa.structure.model.oak">OakModel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/MultiTreeNode.html#addChild(bio.singa.mathematics.graphs.trees.MultiTreeNode)">addChild(MultiTreeNode&lt;T&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/MultiTreeNode.html" title="class in bio.singa.mathematics.graphs.trees">MultiTreeNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/MultiTree.html#addChildToRoot(bio.singa.mathematics.graphs.trees.MultiTreeNode)">addChildToRoot(MultiTreeNode&lt;T&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/MultiTree.html" title="class in bio.singa.mathematics.graphs.trees">MultiTree</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/nested/TrajectoryDataPoint.SubsectionDataPoint.html#addConcentration(bio.singa.simulation.entities.ChemicalEntity,double)">addConcentration(ChemicalEntity, double)</a></span> - Method in class bio.singa.simulation.trajectories.nested.<a href="../bio/singa/simulation/trajectories/nested/TrajectoryDataPoint.SubsectionDataPoint.html" title="class in bio.singa.simulation.trajectories.nested">TrajectoryDataPoint.SubsectionDataPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Simulation.html#addConcentration(bio.singa.simulation.model.concentrations.InitialConcentration)">addConcentration(InitialConcentration)</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Simulation.html" title="class in bio.singa.simulation.model.simulation">Simulation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html#addCondition(bio.singa.simulation.model.concentrations.ConcentrationCondition)">addCondition(ConcentrationCondition)</a></span> - Method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html" title="class in bio.singa.simulation.model.concentrations">InitialConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/Reaction.html#addCondition(bio.singa.simulation.model.concentrations.ConcentrationCondition)">addCondition(ConcentrationCondition)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/Reaction.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">Reaction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/specifity/EntitySpecific.html#addDeltaFunction(bio.singa.simulation.model.modules.concentration.functions.EntityDeltaFunction)">addDeltaFunction(EntityDeltaFunction)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.specifity.<a href="../bio/singa/simulation/model/modules/concentration/specifity/EntitySpecific.html" title="class in bio.singa.simulation.model.modules.concentration.specifity">EntitySpecific</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/specifity/SectionSpecific.html#addDeltaFunction(bio.singa.simulation.model.modules.concentration.functions.SectionDeltaFunction)">addDeltaFunction(SectionDeltaFunction)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.specifity.<a href="../bio/singa/simulation/model/modules/concentration/specifity/SectionSpecific.html" title="class in bio.singa.simulation.model.modules.concentration.specifity">SectionSpecific</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/specifity/UpdatableSpecific.html#addDeltaFunction(bio.singa.simulation.model.modules.concentration.functions.UpdatableDeltaFunction)">addDeltaFunction(UpdatableDeltaFunction)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.specifity.<a href="../bio/singa/simulation/model/modules/concentration/specifity/UpdatableSpecific.html" title="class in bio.singa.simulation.model.modules.concentration.specifity">UpdatableSpecific</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html#addDeltaFunction(DeltaFunctionType)">addDeltaFunction(DeltaFunctionType)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationBasedModule.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationBasedModule</a></dt>
<dd>
<div class="block">Adds a delta function to the module.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/specifity/UpdateSpecificity.html#addDeltaFunction(DeltaFunctionType)">addDeltaFunction(DeltaFunctionType)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.specifity.<a href="../bio/singa/simulation/model/modules/concentration/specifity/UpdateSpecificity.html" title="interface in bio.singa.simulation.model.modules.concentration.specifity">UpdateSpecificity</a></dt>
<dd>
<div class="block">Adds the implementation of a <a href="../bio/singa/simulation/model/modules/concentration/functions/AbstractDeltaFunction.html" title="class in bio.singa.simulation.model.modules.concentration.functions"><code>AbstractDeltaFunction</code></a> to be calculated during simulation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/DisplacementBasedModule.html#addDeltaFunction(java.util.function.Function,java.util.function.Predicate)">addDeltaFunction(Function&lt;Vesicle, DisplacementDelta&gt;, Predicate&lt;Vesicle&gt;)</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/DisplacementBasedModule.html" title="class in bio.singa.simulation.model.modules.displacement">DisplacementBasedModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/GraphPath.html#addEdge(EdgeType)">addEdge(EdgeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/GraphPath.html" title="class in bio.singa.mathematics.graphs.model">GraphPath</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/algorithms/alignment/DynamicProgrammingGraph.html#addEdgeBetween(int,double,bio.singa.sequence.algorithms.alignment.DynamicProgrammingNode,bio.singa.sequence.algorithms.alignment.DynamicProgrammingNode)">addEdgeBetween(int, double, DynamicProgrammingNode, DynamicProgrammingNode)</a></span> - Method in class bio.singa.sequence.algorithms.alignment.<a href="../bio/singa/sequence/algorithms/alignment/DynamicProgrammingGraph.html" title="class in bio.singa.sequence.algorithms.alignment">DynamicProgrammingGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/UndirectedGraph.html#addEdgeBetween(int,int)">addEdgeBetween(int, int)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/UndirectedGraph.html" title="class in bio.singa.mathematics.graphs.model">UndirectedGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/MoleculeGraph.html#addEdgeBetween(int,int,int,bio.singa.chemistry.model.CovalentBondType)">addEdgeBetween(int, int, int, CovalentBondType)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/MoleculeGraph.html" title="class in bio.singa.chemistry.model">MoleculeGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/MoleculeGraph.html#addEdgeBetween(int,bio.singa.chemistry.model.MoleculeAtom,bio.singa.chemistry.model.MoleculeAtom)">addEdgeBetween(int, MoleculeAtom, MoleculeAtom)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/MoleculeGraph.html" title="class in bio.singa.chemistry.model">MoleculeGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/GridGraph.html#addEdgeBetween(int,bio.singa.mathematics.graphs.grid.GridNode,bio.singa.mathematics.graphs.grid.GridNode)">addEdgeBetween(int, GridNode, GridNode)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/GridGraph.html" title="class in bio.singa.mathematics.graphs.grid">GridGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/GenericGraph.html#addEdgeBetween(int,bio.singa.mathematics.graphs.model.GenericNode,bio.singa.mathematics.graphs.model.GenericNode)">addEdgeBetween(int, GenericNode&lt;ContentType&gt;, GenericNode&lt;ContentType&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/GenericGraph.html" title="class in bio.singa.mathematics.graphs.model">GenericGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/UndirectedGraph.html#addEdgeBetween(int,bio.singa.mathematics.graphs.model.RegularNode,bio.singa.mathematics.graphs.model.RegularNode)">addEdgeBetween(int, RegularNode, RegularNode)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/UndirectedGraph.html" title="class in bio.singa.mathematics.graphs.model">UndirectedGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/algorithms/alignment/DynamicProgrammingGraph.html#addEdgeBetween(int,bio.singa.sequence.algorithms.alignment.DynamicProgrammingNode,bio.singa.sequence.algorithms.alignment.DynamicProgrammingNode)">addEdgeBetween(int, DynamicProgrammingNode, DynamicProgrammingNode)</a></span> - Method in class bio.singa.sequence.algorithms.alignment.<a href="../bio/singa/sequence/algorithms/alignment/DynamicProgrammingGraph.html" title="class in bio.singa.sequence.algorithms.alignment">DynamicProgrammingGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#addEdgeBetween(int,bio.singa.simulation.entities.GraphComplexNode,bio.singa.simulation.entities.GraphComplexNode)">addEdgeBetween(int, GraphComplexNode, GraphComplexNode)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/reactiongraph/ReactionGraph.html#addEdgeBetween(int,bio.singa.simulation.export.reactiongraph.ReactionGraphNode,bio.singa.simulation.export.reactiongraph.ReactionGraphNode)">addEdgeBetween(int, ReactionGraphNode, ReactionGraphNode)</a></span> - Method in class bio.singa.simulation.export.reactiongraph.<a href="../bio/singa/simulation/export/reactiongraph/ReactionGraph.html" title="class in bio.singa.simulation.export.reactiongraph">ReactionGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonGraph.html#addEdgeBetween(int,bio.singa.simulation.model.graphs.AutomatonNode,bio.singa.simulation.model.graphs.AutomatonNode)">addEdgeBetween(int, AutomatonNode, AutomatonNode)</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonGraph.html" title="class in bio.singa.simulation.model.graphs">AutomatonGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#addEdgeBetween(int,NodeType,NodeType)">addEdgeBetween(int, NodeType, NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>
<div class="block">Adds a new edge with the given identifier to the graph, connecting source and target nodes.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/DirectedGraph.html#addEdgeBetween(int,NodeType,NodeType)">addEdgeBetween(int, NodeType, NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/DirectedGraph.html" title="class in bio.singa.mathematics.graphs.model">DirectedGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graph.html#addEdgeBetween(int,NodeType,NodeType)">addEdgeBetween(int, NodeType, NodeType)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graph.html" title="interface in bio.singa.mathematics.graphs.model">Graph</a></dt>
<dd>
<div class="block">Adds an edge with the given identifier between the two given nodes.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/MoleculeGraph.html#addEdgeBetween(bio.singa.chemistry.model.MoleculeAtom,bio.singa.chemistry.model.MoleculeAtom)">addEdgeBetween(MoleculeAtom, MoleculeAtom)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/MoleculeGraph.html" title="class in bio.singa.chemistry.model">MoleculeGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/MoleculeGraph.html#addEdgeBetween(bio.singa.chemistry.model.MoleculeAtom,bio.singa.chemistry.model.MoleculeAtom,bio.singa.chemistry.model.CovalentBondType)">addEdgeBetween(MoleculeAtom, MoleculeAtom, CovalentBondType)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/MoleculeGraph.html" title="class in bio.singa.chemistry.model">MoleculeGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/GridGraph.html#addEdgeBetween(bio.singa.mathematics.graphs.grid.GridNode,bio.singa.mathematics.graphs.grid.GridNode)">addEdgeBetween(GridNode, GridNode)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/GridGraph.html" title="class in bio.singa.mathematics.graphs.grid">GridGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/DirectedGraph.html#addEdgeBetween(bio.singa.mathematics.graphs.model.DirectedEdge,NodeType,NodeType)">addEdgeBetween(DirectedEdge&lt;NodeType&gt;, NodeType, NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/DirectedGraph.html" title="class in bio.singa.mathematics.graphs.model">DirectedGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/GenericGraph.html#addEdgeBetween(bio.singa.mathematics.graphs.model.GenericNode,bio.singa.mathematics.graphs.model.GenericNode)">addEdgeBetween(GenericNode&lt;ContentType&gt;, GenericNode&lt;ContentType&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/GenericGraph.html" title="class in bio.singa.mathematics.graphs.model">GenericGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/UndirectedGraph.html#addEdgeBetween(bio.singa.mathematics.graphs.model.RegularNode,bio.singa.mathematics.graphs.model.RegularNode)">addEdgeBetween(RegularNode, RegularNode)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/UndirectedGraph.html" title="class in bio.singa.mathematics.graphs.model">UndirectedGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/algorithms/alignment/DynamicProgrammingGraph.html#addEdgeBetween(bio.singa.sequence.algorithms.alignment.DynamicProgrammingNode,bio.singa.sequence.algorithms.alignment.DynamicProgrammingNode)">addEdgeBetween(DynamicProgrammingNode, DynamicProgrammingNode)</a></span> - Method in class bio.singa.sequence.algorithms.alignment.<a href="../bio/singa/sequence/algorithms/alignment/DynamicProgrammingGraph.html" title="class in bio.singa.sequence.algorithms.alignment">DynamicProgrammingGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#addEdgeBetween(bio.singa.simulation.entities.GraphComplexNode,bio.singa.simulation.entities.GraphComplexNode)">addEdgeBetween(GraphComplexNode, GraphComplexNode)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#addEdgeBetween(bio.singa.simulation.entities.GraphComplexNode,bio.singa.simulation.entities.GraphComplexNode,bio.singa.simulation.entities.BindingSite)">addEdgeBetween(GraphComplexNode, GraphComplexNode, BindingSite)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/reactiongraph/ReactionGraph.html#addEdgeBetween(bio.singa.simulation.export.reactiongraph.ReactionGraphNode,bio.singa.simulation.export.reactiongraph.ReactionGraphNode)">addEdgeBetween(ReactionGraphNode, ReactionGraphNode)</a></span> - Method in class bio.singa.simulation.export.reactiongraph.<a href="../bio/singa/simulation/export/reactiongraph/ReactionGraph.html" title="class in bio.singa.simulation.export.reactiongraph">ReactionGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonGraph.html#addEdgeBetween(bio.singa.simulation.model.graphs.AutomatonNode,bio.singa.simulation.model.graphs.AutomatonNode)">addEdgeBetween(AutomatonNode, AutomatonNode)</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonGraph.html" title="class in bio.singa.simulation.model.graphs">AutomatonGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/GenericGraph.html#addEdgeBetween(ContentType,ContentType)">addEdgeBetween(ContentType, ContentType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/GenericGraph.html" title="class in bio.singa.mathematics.graphs.model">GenericGraph</a></dt>
<dd>
<div class="block">Checks whether nodes containing the source or target content are already contained in the graphs.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html#addEdgeBetween(EdgeType,NodeType,NodeType)">addEdgeBetween(EdgeType, NodeType, NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="class in bio.singa.mathematics.graphs.grid">AbstractGridGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#addEdgeBetween(EdgeType,NodeType,NodeType)">addEdgeBetween(EdgeType, NodeType, NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>
<div class="block">Adds a new edge to the graph, connecting source and target nodes.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graph.html#addEdgeBetween(EdgeType,NodeType,NodeType)">addEdgeBetween(EdgeType, NodeType, NodeType)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graph.html" title="interface in bio.singa.mathematics.graphs.model">Graph</a></dt>
<dd>
<div class="block">Adds the given edge to the graph setting the given source an target nodes.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#addEdgeBetween(NodeType,NodeType)">addEdgeBetween(NodeType, NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>
<div class="block">Adds a new edge with the next free identifier to the graph, connecting source and target nodes.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/DirectedGraph.html#addEdgeBetween(NodeType,NodeType)">addEdgeBetween(NodeType, NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/DirectedGraph.html" title="class in bio.singa.mathematics.graphs.model">DirectedGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graph.html#addEdgeBetween(NodeType,NodeType)">addEdgeBetween(NodeType, NodeType)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graph.html" title="interface in bio.singa.mathematics.graphs.model">Graph</a></dt>
<dd>
<div class="block">Adds a new edge between the source and target nodes to the graph using the next free edge identifier.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/events/UpdateEventEmitter.html#addEventListener(bio.singa.core.events.UpdateEventListener)">addEventListener(UpdateEventListener&lt;EventType&gt;)</a></span> - Method in interface bio.singa.core.events.<a href="../bio/singa/core/events/UpdateEventEmitter.html" title="interface in bio.singa.core.events">UpdateEventEmitter</a></dt>
<dd>
<div class="block">Adds a listener.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/databases/sequencevariants/SequenceVariant.html#addEvidence(bio.singa.features.model.Evidence)">addEvidence(Evidence)</a></span> - Method in class bio.singa.chemistry.features.databases.sequencevariants.<a href="../bio/singa/chemistry/features/databases/sequencevariants/SequenceVariant.html" title="class in bio.singa.chemistry.features.databases.sequencevariants">SequenceVariant</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/AbstractFeature.html#addEvidence(bio.singa.features.model.Evidence)">addEvidence(Evidence)</a></span> - Method in class bio.singa.features.model.<a href="../bio/singa/features/model/AbstractFeature.html" title="class in bio.singa.features.model">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/Feature.html#addEvidence(bio.singa.features.model.Evidence)">addEvidence(Evidence)</a></span> - Method in interface bio.singa.features.model.<a href="../bio/singa/features/model/Feature.html" title="interface in bio.singa.features.model">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html#addEvidence(bio.singa.features.model.Evidence)">addEvidence(Evidence)</a></span> - Method in class bio.singa.simulation.model.modules.<a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html" title="class in bio.singa.simulation.model.modules">AbstractUpdateModule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotif.html#addExchangeableFamily(bio.singa.features.identifiers.LeafIdentifier,bio.singa.structure.model.families.AminoAcidFamily)">addExchangeableFamily(LeafIdentifier, AminoAcidFamily)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak">StructuralMotif</a></dt>
<dd>
<div class="block">For a given <a href="../bio/singa/features/identifiers/LeafIdentifier.html" title="class in bio.singa.features.identifiers"><code>LeafIdentifier</code></a> a possible exchange is added.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotif.html#addExchangeableFamily(bio.singa.features.identifiers.LeafIdentifier,bio.singa.structure.model.families.MatcherFamily)">addExchangeableFamily(LeafIdentifier, MatcherFamily)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak">StructuralMotif</a></dt>
<dd>
<div class="block">For a given <a href="../bio/singa/features/identifiers/LeafIdentifier.html" title="class in bio.singa.features.identifiers"><code>LeafIdentifier</code></a> a possible exchange is added.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotif.html#addExchangeableFamily(bio.singa.features.identifiers.LeafIdentifier,bio.singa.structure.model.families.NucleotideFamily)">addExchangeableFamily(LeafIdentifier, NucleotideFamily)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak">StructuralMotif</a></dt>
<dd>
<div class="block">For a given <a href="../bio/singa/features/identifiers/LeafIdentifier.html" title="class in bio.singa.features.identifiers"><code>LeafIdentifier</code></a> a possible exchange is added.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Exchangeable.html#addExchangeableFamily(FamilyType)">addExchangeableFamily(FamilyType)</a></span> - Method in interface bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Exchangeable.html" title="interface in bio.singa.structure.model.oak">Exchangeable</a></dt>
<dd>
<div class="block">Adds an exchangeable <a href="../bio/singa/structure/model/families/StructuralFamily.html" title="interface in bio.singa.structure.model.families"><code>StructuralFamily</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html#addExchangeableFamily(FamilyType)">addExchangeableFamily(FamilyType)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakLeafSubstructure.html" title="class in bio.singa.structure.model.oak">OakLeafSubstructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotif.html#addExchangeableFamilyToAll(bio.singa.structure.model.families.AminoAcidFamily)">addExchangeableFamilyToAll(AminoAcidFamily)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak">StructuralMotif</a></dt>
<dd>
<div class="block">Assigns an <a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families"><code>AminoAcidFamily</code></a> to all members of this motif as exchange.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotif.html#addExchangeableFamilyToAll(bio.singa.structure.model.families.MatcherFamily)">addExchangeableFamilyToAll(MatcherFamily)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak">StructuralMotif</a></dt>
<dd>
<div class="block">Assigns each member of a <a href="../bio/singa/structure/model/families/MatcherFamily.html" title="enum in bio.singa.structure.model.families"><code>MatcherFamily</code></a> to all members of this motif as exchange.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotif.html#addExchangeableFamilyToAll(bio.singa.structure.model.families.NucleotideFamily)">addExchangeableFamilyToAll(NucleotideFamily)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak">StructuralMotif</a></dt>
<dd>
<div class="block">Assigns an <a href="../bio/singa/structure/model/families/NucleotideFamily.html" title="enum in bio.singa.structure.model.families"><code>NucleotideFamily</code></a> to all members of this motif as exchange.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/FeatureProvider.html#addFallbackRequirement(java.lang.Integer,java.lang.Class)">addFallbackRequirement(Integer, Class&lt;RequirementType&gt;)</a></span> - Method in class bio.singa.features.model.<a href="../bio/singa/features/model/FeatureProvider.html" title="class in bio.singa.features.model">FeatureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html#addFilament(bio.singa.simulation.model.agents.linelike.LineLikeAgent)">addFilament(LineLikeAgent)</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html" title="class in bio.singa.simulation.model.agents.linelike">LineLikeAgentLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html#addFilaments(java.util.Collection)">addFilaments(Collection&lt;LineLikeAgent&gt;)</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html" title="class in bio.singa.simulation.model.agents.linelike">LineLikeAgentLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/layouts/force/ForceDirectedGraphLayout.html#addForce(bio.singa.javafx.renderer.layouts.force.Force)">addForce(Force&lt;NodeType, EdgeType, IdentifierType, GraphType&gt;)</a></span> - Method in class bio.singa.javafx.renderer.layouts.force.<a href="../bio/singa/javafx/renderer/layouts/force/ForceDirectedGraphLayout.html" title="class in bio.singa.javafx.renderer.layouts.force">ForceDirectedGraphLayout</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/FieldSupplier.html#addFullDelta(bio.singa.simulation.model.modules.concentration.ConcentrationDeltaIdentifier,bio.singa.simulation.model.modules.concentration.ConcentrationDelta)">addFullDelta(ConcentrationDeltaIdentifier, ConcentrationDelta)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/FieldSupplier.html" title="class in bio.singa.simulation.model.modules.concentration">FieldSupplier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/Protein.html#addGoTerm(bio.singa.features.identifiers.GoTerm)">addGoTerm(GoTerm)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/Protein.html" title="class in bio.singa.chemistry.model">Protein</a></dt>
<dd>
<div class="block">Adds a new <a href="../bio/singa/features/identifiers/GoTerm.html" title="class in bio.singa.features.identifiers"><code>GoTerm</code></a> to the protein.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationManager.html#addGraphUpdateListener(bio.singa.core.events.UpdateEventListener)">addGraphUpdateListener(UpdateEventListener&lt;GraphUpdatedEvent&gt;)</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationManager.html" title="class in bio.singa.simulation.model.simulation">SimulationManager</a></dt>
<dd>
<div class="block">Adds a new listener for graph based events.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/FieldSupplier.html#addHalfDelta(bio.singa.simulation.model.modules.concentration.ConcentrationDeltaIdentifier,bio.singa.simulation.model.modules.concentration.ConcentrationDelta)">addHalfDelta(ConcentrationDeltaIdentifier, ConcentrationDelta)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/FieldSupplier.html" title="class in bio.singa.simulation.model.modules.concentration">FieldSupplier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/errors/DebugRecorder.html#addInformation(long,java.lang.String)">addInformation(long, String)</a></span> - Method in class bio.singa.simulation.trajectories.errors.<a href="../bio/singa/simulation/trajectories/errors/DebugRecorder.html" title="class in bio.singa.simulation.trajectories.errors">DebugRecorder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/molecules/SwingMoleculeGraphRenderer.html#addInteraction(int,bio.singa.structure.parser.plip.Interaction)">addInteraction(int, Interaction)</a></span> - Method in class bio.singa.javafx.renderer.molecules.<a href="../bio/singa/javafx/renderer/molecules/SwingMoleculeGraphRenderer.html" title="class in bio.singa.javafx.renderer.molecules">SwingMoleculeGraphRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/plip/InteractionContainer.html#addInteraction(bio.singa.structure.parser.plip.Interaction)">addInteraction(Interaction)</a></span> - Method in class bio.singa.structure.parser.plip.<a href="../bio/singa/structure/parser/plip/InteractionContainer.html" title="class in bio.singa.structure.parser.plip">InteractionContainer</a></dt>
<dd>
<div class="block">Adds an interaction to this container.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/ExpressionNode.html#ADDITION_NODE">ADDITION_NODE</a></span> - Static variable in interface uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/ExpressionNode.html" title="interface in uk.co.cogitolearning.cogpar">ExpressionNode</a></dt>
<dd>
<div class="block">Node id for addition nodes</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/annotations/AnnotationType.html#ADDITIONAL_IDENTIFIER">ADDITIONAL_IDENTIFIER</a></span> - bio.singa.chemistry.annotations.<a href="../bio/singa/chemistry/annotations/AnnotationType.html" title="enum in bio.singa.chemistry.annotations">AnnotationType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html#additionalIdentifier(bio.singa.features.identifiers.model.Identifier)">additionalIdentifier(Identifier)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/SimpleEntity.SimpleEntityBuilder.html#additionalIdentifier(bio.singa.features.identifiers.model.Identifier)">additionalIdentifier(Identifier)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/SimpleEntity.SimpleEntityBuilder.html" title="class in bio.singa.simulation.entities">SimpleEntity.SimpleEntityBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html#additionalIdentifiers(java.util.Collection)">additionalIdentifiers(Collection&lt;Identifier&gt;)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/SimpleEntity.SimpleEntityBuilder.html#additionalIdentifiers(java.util.Collection)">additionalIdentifiers(Collection&lt;Identifier&gt;)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/SimpleEntity.SimpleEntityBuilder.html" title="class in bio.singa.simulation.entities">SimpleEntity.SimpleEntityBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../uk/co/cogitolearning/cogpar/AdditionExpressionNode.html" title="class in uk.co.cogitolearning.cogpar"><span class="typeNameLink">AdditionExpressionNode</span></a> - Class in <a href="../uk/co/cogitolearning/cogpar/package-summary.html">uk.co.cogitolearning.cogpar</a></dt>
<dd>
<div class="block">An ExpressionNode that handles additions and subtractions.</div>
</dd>
<dt><span class="memberNameLink"><a href="../uk/co/cogitolearning/cogpar/AdditionExpressionNode.html#%3Cinit%3E(uk.co.cogitolearning.cogpar.ExpressionNode,boolean)">AdditionExpressionNode(ExpressionNode, boolean)</a></span> - Constructor for class uk.co.cogitolearning.cogpar.<a href="../uk/co/cogitolearning/cogpar/AdditionExpressionNode.html" title="class in uk.co.cogitolearning.cogpar">AdditionExpressionNode</a></dt>
<dd>
<div class="block">Constructor to create an addition with the first term already added.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/RegularVector.html#additiveleyInvertElement(int)">additiveleyInvertElement(int)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/RegularVector.html" title="class in bio.singa.mathematics.vectors">RegularVector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Scalar.html#additiveleyInvertElement(int)">additiveleyInvertElement(int)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Scalar.html" title="class in bio.singa.mathematics.vectors">Scalar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector.html#additiveleyInvertElement(int)">additiveleyInvertElement(int)</a></span> - Method in interface bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a></dt>
<dd>
<div class="block">Additively inverts (negates) the element at the given index.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#additiveleyInvertElement(int)">additiveleyInvertElement(int)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/concepts/AdditivelyInvertible.html#additivelyInvert()">additivelyInvert()</a></span> - Method in interface bio.singa.mathematics.concepts.<a href="../bio/singa/mathematics/concepts/AdditivelyInvertible.html" title="interface in bio.singa.mathematics.concepts">AdditivelyInvertible</a></dt>
<dd>
<div class="block">Returns the corresponding additive inverse element.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/intervals/DoubleInterval.html#additivelyInvert()">additivelyInvert()</a></span> - Method in class bio.singa.mathematics.intervals.<a href="../bio/singa/mathematics/intervals/DoubleInterval.html" title="class in bio.singa.mathematics.intervals">DoubleInterval</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/intervals/IntegerInterval.html#additivelyInvert()">additivelyInvert()</a></span> - Method in class bio.singa.mathematics.intervals.<a href="../bio/singa/mathematics/intervals/IntegerInterval.html" title="class in bio.singa.mathematics.intervals">IntegerInterval</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/RegularMatrix.html#additivelyInvert()">additivelyInvert()</a></span> - Method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/RegularMatrix.html" title="class in bio.singa.mathematics.matrices">RegularMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/quaternions/RegularQuaternion.html#additivelyInvert()">additivelyInvert()</a></span> - Method in class bio.singa.mathematics.quaternions.<a href="../bio/singa/mathematics/quaternions/RegularQuaternion.html" title="class in bio.singa.mathematics.quaternions">RegularQuaternion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/RegularVector.html#additivelyInvert()">additivelyInvert()</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/RegularVector.html" title="class in bio.singa.mathematics.vectors">RegularVector</a></dt>
<dd>
<div class="block">Additively inverts (negates) the whole vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Scalar.html#additivelyInvert()">additivelyInvert()</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Scalar.html" title="class in bio.singa.mathematics.vectors">Scalar</a></dt>
<dd>
<div class="block">Additively inverts (negates) the value.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#additivelyInvert()">additivelyInvert()</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>
<div class="block">Additively inverts (negates) the whole vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#additivelyInvert()">additivelyInvert()</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/concepts/AdditivelyInvertible.html" title="interface in bio.singa.mathematics.concepts"><span class="typeNameLink">AdditivelyInvertible</span></a>&lt;<a href="../bio/singa/mathematics/concepts/AdditivelyInvertible.html" title="type parameter in AdditivelyInvertible">NumberConcept</a>&gt; - Interface in <a href="../bio/singa/mathematics/concepts/package-summary.html">bio.singa.mathematics.concepts</a></dt>
<dd>
<div class="block">The "additive inversion" operation can be any unary operation that returns
 the "opposite" of this Object.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotif.html#addLeafSubstructure(bio.singa.structure.model.interfaces.LeafSubstructure)">addLeafSubstructure(LeafSubstructure)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak">StructuralMotif</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakChain.html#addLeafSubstructure(bio.singa.structure.model.oak.OakLeafSubstructure)">addLeafSubstructure(OakLeafSubstructure)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakChain.html" title="class in bio.singa.structure.model.oak">OakChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakChain.html#addLeafSubstructure(bio.singa.structure.model.oak.OakLeafSubstructure,boolean)">addLeafSubstructure(OakLeafSubstructure, boolean)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakChain.html" title="class in bio.singa.structure.model.oak">OakChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html#addLeft(T)">addLeft(T)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTreeNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#addLineLikeAgentSegment(bio.singa.simulation.model.agents.linelike.LineLikeAgent,bio.singa.mathematics.vectors.Vector2D)">addLineLikeAgentSegment(LineLikeAgent, Vector2D)</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/error/TimeStepManager.html#addListener(bio.singa.core.events.UpdateEventListener)">addListener(UpdateEventListener&lt;ErrorManager.Reason&gt;)</a></span> - Static method in class bio.singa.simulation.model.simulation.error.<a href="../bio/singa/simulation/model/simulation/error/TimeStepManager.html" title="class in bio.singa.simulation.model.simulation.error">TimeStepManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/surfacelike/MembraneLayer.html#addMembrane(bio.singa.simulation.model.agents.surfacelike.Membrane)">addMembrane(Membrane)</a></span> - Method in class bio.singa.simulation.model.agents.surfacelike.<a href="../bio/singa/simulation/model/agents/surfacelike/MembraneLayer.html" title="class in bio.singa.simulation.model.agents.surfacelike">MembraneLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/surfacelike/MembraneLayer.html#addMembranes(java.util.Collection)">addMembranes(Collection&lt;Membrane&gt;)</a></span> - Method in class bio.singa.simulation.model.agents.surfacelike.<a href="../bio/singa/simulation/model/agents/surfacelike/MembraneLayer.html" title="class in bio.singa.simulation.model.agents.surfacelike">MembraneLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#addMembraneSegment(bio.singa.simulation.model.agents.surfacelike.MembraneSegment)">addMembraneSegment(MembraneSegment)</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html#addMicrotubule(bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D)">addMicrotubule(Vector2D, Vector2D)</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentLayer.html" title="class in bio.singa.simulation.model.agents.linelike">LineLikeAgentLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakStructure.html#addModel(bio.singa.structure.model.oak.OakModel)">addModel(OakModel)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakStructure.html" title="class in bio.singa.structure.model.oak">OakStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/reactions/modifications/AddModification.html" title="class in bio.singa.simulation.reactions.modifications"><span class="typeNameLink">AddModification</span></a> - Class in <a href="../bio/singa/simulation/reactions/modifications/package-summary.html">bio.singa.simulation.reactions.modifications</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/AddModification.html#%3Cinit%3E(bio.singa.simulation.entities.BindingSite,bio.singa.simulation.entities.ChemicalEntity)">AddModification(BindingSite, ChemicalEntity)</a></span> - Constructor for class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/AddModification.html" title="class in bio.singa.simulation.reactions.modifications">AddModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Simulation.html#addModule(bio.singa.simulation.model.modules.UpdateModule)">addModule(UpdateModule)</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Simulation.html" title="class in bio.singa.simulation.model.simulation">Simulation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/UpdateScheduler.html#addModule(bio.singa.simulation.model.modules.UpdateModule)">addModule(UpdateModule)</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/UpdateScheduler.html" title="class in bio.singa.simulation.model.simulation">UpdateScheduler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html#addName(java.lang.String)">addName(String)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity</a></dt>
<dd>
<div class="block">Adds an name as an annotation to this chemical entity.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/OakAtom.html#addNeighbour(bio.singa.structure.model.oak.OakAtom)">addNeighbour(OakAtom)</a></span> - Method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/OakAtom.html" title="class in bio.singa.structure.model.oak">OakAtom</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractNode.html#addNeighbour(NodeType)">addNeighbour(NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractNode.html" title="class in bio.singa.mathematics.graphs.model">AbstractNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Node.html#addNeighbour(NodeType)">addNeighbour(NodeType)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Node.html" title="interface in bio.singa.mathematics.graphs.model">Node</a></dt>
<dd>
<div class="block">Adds a neighbour to this node.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/MoleculeGraph.html#addNextAtom(char)">addNextAtom(char)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/MoleculeGraph.html" title="class in bio.singa.chemistry.model">MoleculeGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/MoleculeGraph.html#addNextAtom(bio.singa.chemistry.model.elements.Element)">addNextAtom(Element)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/MoleculeGraph.html" title="class in bio.singa.chemistry.model">MoleculeGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/MoleculeGraph.html#addNextAtom(bio.singa.chemistry.model.elements.Element,int)">addNextAtom(Element, int)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/MoleculeGraph.html" title="class in bio.singa.chemistry.model">MoleculeGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/MoleculeGraph.html#addNextAtom(bio.singa.chemistry.model.elements.Element,int,int)">addNextAtom(Element, int, int)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/MoleculeGraph.html" title="class in bio.singa.chemistry.model">MoleculeGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/MoleculeGraph.html#addNextAtom(java.lang.String)">addNextAtom(String)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/MoleculeGraph.html" title="class in bio.singa.chemistry.model">MoleculeGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/UndirectedGraph.html#addNode(bio.singa.mathematics.vectors.Vector2D)">addNode(Vector2D)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/UndirectedGraph.html" title="class in bio.singa.mathematics.graphs.model">UndirectedGraph</a></dt>
<dd>
<div class="block">Adds a node with the given position to the graph.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/algorithms/alignment/DynamicProgrammingGraph.html#addNode(bio.singa.sequence.algorithms.alignment.DynamicProgrammingNode,int,int)">addNode(DynamicProgrammingNode, int, int)</a></span> - Method in class bio.singa.sequence.algorithms.alignment.<a href="../bio/singa/sequence/algorithms/alignment/DynamicProgrammingGraph.html" title="class in bio.singa.sequence.algorithms.alignment">DynamicProgrammingGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/reactiongraph/ReactionGraph.html#addNode(bio.singa.simulation.entities.ChemicalEntity)">addNode(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.export.reactiongraph.<a href="../bio/singa/simulation/export/reactiongraph/ReactionGraph.html" title="class in bio.singa.simulation.export.reactiongraph">ReactionGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/GenericGraph.html#addNode(ContentType)">addNode(ContentType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/GenericGraph.html" title="class in bio.singa.mathematics.graphs.model">GenericGraph</a></dt>
<dd>
<div class="block">Adds a new node with the given content to the graph.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html#addNode(NodeType)">addNode(NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.grid.<a href="../bio/singa/mathematics/graphs/grid/AbstractGridGraph.html" title="class in bio.singa.mathematics.graphs.grid">AbstractGridGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#addNode(NodeType)">addNode(NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/Graph.html#addNode(NodeType)">addNode(NodeType)</a></span> - Method in interface bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/Graph.html" title="interface in bio.singa.mathematics.graphs.model">Graph</a></dt>
<dd>
<div class="block">Adds a node to the graph.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/GraphPath.html#addNode(NodeType)">addNode(NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/GraphPath.html" title="class in bio.singa.mathematics.graphs.model">GraphPath</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#addNodeIf(java.util.function.Predicate,NodeType)">addNodeIf(Predicate&lt;NodeType&gt;, NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>
<div class="block">Evaluates the predicate for every node in the graph, if no node matches the predicate the given node is added.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/SimulationManager.html#addNodeUpdateListener(bio.singa.core.events.UpdateEventListener)">addNodeUpdateListener(UpdateEventListener&lt;UpdatableUpdatedEvent&gt;)</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/SimulationManager.html" title="class in bio.singa.simulation.model.simulation">SimulationManager</a></dt>
<dd>
<div class="block">Adds a new listener for node based events.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/Protein.html#addOrganism(bio.singa.chemistry.annotations.taxonomy.Organism)">addOrganism(Organism)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/Protein.html" title="class in bio.singa.chemistry.model">Protein</a></dt>
<dd>
<div class="block">Adds an organism as an annotation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/Protein.html#addOrganism(bio.singa.chemistry.annotations.taxonomy.Organism,java.lang.String)">addOrganism(Organism, String)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/Protein.html" title="class in bio.singa.chemistry.model">Protein</a></dt>
<dd>
<div class="block">Adds an organism with a description as an annotation to</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/identifiers/model/IdentifierPatternRegistry.html#addPattern(java.lang.Class,java.util.regex.Pattern)">addPattern(Class&lt;IdentifierType&gt;, Pattern)</a></span> - Static method in class bio.singa.features.identifiers.model.<a href="../bio/singa/features/identifiers/model/IdentifierPatternRegistry.html" title="class in bio.singa.features.identifiers.model">IdentifierPatternRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/Protein.html#addPdbIdentifier(bio.singa.features.identifiers.PDBIdentifier)">addPdbIdentifier(PDBIdentifier)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/Protein.html" title="class in bio.singa.chemistry.model">Protein</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/Protein.html#addPdbRange(bio.singa.features.identifiers.PDBIdentifier,java.lang.String)">addPdbRange(PDBIdentifier, String)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/Protein.html" title="class in bio.singa.chemistry.model">Protein</a></dt>
<dd>
<div class="block">Adds a new PDB structure range to the protein.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/nested/TrajectoryDataPoint.SubsectionDataPoint.html#addPosition(bio.singa.mathematics.vectors.Vector2D)">addPosition(Vector2D)</a></span> - Method in class bio.singa.simulation.trajectories.nested.<a href="../bio/singa/simulation/trajectories/nested/TrajectoryDataPoint.SubsectionDataPoint.html" title="class in bio.singa.simulation.trajectories.nested">TrajectoryDataPoint.SubsectionDataPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html#addPotentialDelta(bio.singa.simulation.model.modules.concentration.ConcentrationDelta)">addPotentialDelta(ConcentrationDelta)</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html" title="class in bio.singa.simulation.model.agents.pointlike">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#addPotentialDelta(bio.singa.simulation.model.modules.concentration.ConcentrationDelta)">addPotentialDelta(ConcentrationDelta)</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>
<div class="block">Adds a potential delta to this node.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html#addPotentialDelta(bio.singa.simulation.model.modules.concentration.ConcentrationDelta)">addPotentialDelta(ConcentrationDelta)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaManager</a></dt>
<dd>
<div class="block">Adds a potential delta to this updatable.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/qualitative/implementations/EndocytoticPit.html#addPotentialDelta(bio.singa.simulation.model.modules.concentration.ConcentrationDelta)">addPotentialDelta(ConcentrationDelta)</a></span> - Method in class bio.singa.simulation.model.modules.qualitative.implementations.<a href="../bio/singa/simulation/model/modules/qualitative/implementations/EndocytoticPit.html" title="class in bio.singa.simulation.model.modules.qualitative.implementations">EndocytoticPit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Updatable.html#addPotentialDelta(bio.singa.simulation.model.modules.concentration.ConcentrationDelta)">addPotentialDelta(ConcentrationDelta)</a></span> - Method in interface bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Updatable.html" title="interface in bio.singa.simulation.model.simulation">Updatable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/displacement/DisplacementDeltaManager.html#addPotentialDisplacementDelta(bio.singa.simulation.model.modules.displacement.DisplacementDelta)">addPotentialDisplacementDelta(DisplacementDelta)</a></span> - Method in class bio.singa.simulation.model.modules.displacement.<a href="../bio/singa/simulation/model/modules/displacement/DisplacementDeltaManager.html" title="class in bio.singa.simulation.model.modules.displacement">DisplacementDeltaManager</a></dt>
<dd>
<div class="block">Adds a displacement delta.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html#addPotentialSpatialDelta(bio.singa.simulation.model.modules.displacement.DisplacementDelta)">addPotentialSpatialDelta(DisplacementDelta)</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html" title="class in bio.singa.simulation.model.agents.pointlike">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/ReactionNetworkGenerator.html#addPreReaction(bio.singa.simulation.reactions.reactors.ReactionChain)">addPreReaction(ReactionChain)</a></span> - Method in class bio.singa.simulation.reactions.<a href="../bio/singa/simulation/reactions/ReactionNetworkGenerator.html" title="class in bio.singa.simulation.reactions">ReactionNetworkGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#addProduct(bio.singa.simulation.entities.ChemicalEntity)">addProduct(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html#addProduct(bio.singa.simulation.entities.ChemicalEntity)">addProduct(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticReactantStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#addProduct(bio.singa.simulation.entities.ChemicalEntity,double)">addProduct(ChemicalEntity, double)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html#addProduct(bio.singa.simulation.entities.ChemicalEntity,double)">addProduct(ChemicalEntity, double)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticReactantStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#addProduct(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.model.sections.CellTopology)">addProduct(ChemicalEntity, CellTopology)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html#addProduct(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.model.sections.CellTopology)">addProduct(ChemicalEntity, CellTopology)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticReactantStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#addProduct(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.model.sections.CellTopology,double)">addProduct(ChemicalEntity, CellTopology, double)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html#addProduct(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.model.sections.CellTopology,double)">addProduct(ChemicalEntity, CellTopology, double)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticReactantStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/StaticReactantBehavior.html#addProduct(bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.Reactant)">addProduct(Reactant)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/StaticReactantBehavior.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">StaticReactantBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/FeatureProviderRegistry.html#addProviderForFeature(java.lang.Class,java.lang.Class)">addProviderForFeature(Class&lt;FeatureType&gt;, Class&lt;ProviderType&gt;)</a></span> - Static method in class bio.singa.chemistry.features.<a href="../bio/singa/chemistry/features/FeatureProviderRegistry.html" title="class in bio.singa.chemistry.features">FeatureProviderRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/FeatureRegistry.html#addQualitativeFeature(bio.singa.features.model.QualitativeFeature)">addQualitativeFeature(QualitativeFeature&lt;?&gt;)</a></span> - Static method in class bio.singa.features.model.<a href="../bio/singa/features/model/FeatureRegistry.html" title="class in bio.singa.features.model">FeatureRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/FeatureRegistry.html#addQuantitativeFeature(bio.singa.features.model.AbstractQuantitativeFeature)">addQuantitativeFeature(AbstractQuantitativeFeature&lt;?&gt;)</a></span> - Static method in class bio.singa.features.model.<a href="../bio/singa/features/model/FeatureRegistry.html" title="class in bio.singa.features.model">FeatureRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/ReactantBehavior.html#addReactant(bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.Reactant)">addReactant(Reactant)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/ReactantBehavior.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">ReactantBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/RuleBasedReactantBehavior.html#addReactant(bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.Reactant)">addReactant(Reactant)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/RuleBasedReactantBehavior.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">RuleBasedReactantBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/StaticReactantBehavior.html#addReactant(bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.Reactant)">addReactant(Reactant)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/StaticReactantBehavior.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">StaticReactantBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/RuleBasedReactantBehavior.html#addReactantSet(bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.ReactantSet)">addReactantSet(ReactantSet)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/RuleBasedReactantBehavior.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">RuleBasedReactantBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AddReactionChainBuilder.html#%3Cinit%3E(bio.singa.simulation.entities.BindingSite,bio.singa.simulation.entities.ChemicalEntity)">AddReactionChainBuilder(BindingSite, ChemicalEntity)</a></span> - Constructor for class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AddReactionChainBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AddReactionChainBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AddReactionChainBuilder.html#%3Cinit%3E(bio.singa.simulation.entities.ChemicalEntity)">AddReactionChainBuilder(ChemicalEntity)</a></span> - Constructor for class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AddReactionChainBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AddReactionChainBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html#addReferencedEntities(java.util.Collection)">addReferencedEntities(Collection&lt;? extends ChemicalEntity&gt;)</a></span> - Method in class bio.singa.simulation.model.modules.<a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html" title="class in bio.singa.simulation.model.modules">AbstractUpdateModule</a></dt>
<dd>
<div class="block">Adds multiple referenced chemical entities.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html#addReferencedEntity(bio.singa.simulation.entities.ChemicalEntity)">addReferencedEntity(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.modules.<a href="../bio/singa/simulation/model/modules/AbstractUpdateModule.html" title="class in bio.singa.simulation.model.modules">AbstractUpdateModule</a></dt>
<dd>
<div class="block">Adds a referenced chemical entity.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Simulation.html#addReferencedEntity(bio.singa.simulation.entities.ChemicalEntity)">addReferencedEntity(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Simulation.html" title="class in bio.singa.simulation.model.simulation">Simulation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/FeatureProvider.html#addRequirement(java.lang.Class)">addRequirement(Class&lt;RequirementType&gt;)</a></span> - Method in class bio.singa.features.model.<a href="../bio/singa/features/model/FeatureProvider.html" title="class in bio.singa.features.model">FeatureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html#addRight(T)">addRight(T)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTreeNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/modules/ModuleTable.html#addRow(bio.singa.simulation.export.modules.ModuleTableRow)">addRow(ModuleTableRow)</a></span> - Method in class bio.singa.simulation.export.modules.<a href="../bio/singa/simulation/export/modules/ModuleTable.html" title="class in bio.singa.simulation.export.modules">ModuleTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/FeatureRegistry.html#addScalableQuantitativeFeatures(bio.singa.features.model.AbstractScalableQuantitativeFeature)">addScalableQuantitativeFeatures(AbstractScalableQuantitativeFeature&lt;?&gt;)</a></span> - Static method in class bio.singa.features.model.<a href="../bio/singa/features/model/FeatureRegistry.html" title="class in bio.singa.features.model">FeatureRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/surfacelike/Membrane.html#addSegment(bio.singa.simulation.model.agents.surfacelike.MembraneSegment)">addSegment(MembraneSegment)</a></span> - Method in class bio.singa.simulation.model.agents.surfacelike.<a href="../bio/singa/simulation/model/agents/surfacelike/Membrane.html" title="class in bio.singa.simulation.model.agents.surfacelike">Membrane</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/surfacelike/Membrane.html#addSegment(bio.singa.simulation.model.graphs.AutomatonNode,bio.singa.mathematics.geometry.edges.LineSegment)">addSegment(AutomatonNode, LineSegment)</a></span> - Method in class bio.singa.simulation.model.agents.surfacelike.<a href="../bio/singa/simulation/model/agents/surfacelike/Membrane.html" title="class in bio.singa.simulation.model.agents.surfacelike">Membrane</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/model/SequenceContainer.html#addSequence(java.lang.String,bio.singa.sequence.model.interfaces.Sequence)">addSequence(String, Sequence&lt;?&gt;)</a></span> - Method in class bio.singa.sequence.model.<a href="../bio/singa/sequence/model/SequenceContainer.html" title="class in bio.singa.sequence.model">SequenceContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/VesicleStateRegistry.html#addState(java.lang.String)">addState(String)</a></span> - Static method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/VesicleStateRegistry.html" title="class in bio.singa.simulation.model.agents.pointlike">VesicleStateRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/sections/CellRegion.html#addSubsection(bio.singa.simulation.model.sections.CellTopology,bio.singa.simulation.model.sections.CellSubsection)">addSubsection(CellTopology, CellSubsection)</a></span> - Method in class bio.singa.simulation.model.sections.<a href="../bio/singa/simulation/model/sections/CellRegion.html" title="class in bio.singa.simulation.model.sections">CellRegion</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#addSubsectionRepresentation(bio.singa.simulation.model.sections.CellSubsection,bio.singa.mathematics.geometry.model.Polygon)">addSubsectionRepresentation(CellSubsection, Polygon)</a></span> - Method in class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#addSubstrate(bio.singa.simulation.entities.ChemicalEntity)">addSubstrate(ChemicalEntity)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html#addSubstrate(bio.singa.simulation.entities.ChemicalEntity)">addSubstrate(ChemicalEntity)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticReactantStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#addSubstrate(bio.singa.simulation.entities.ChemicalEntity,double)">addSubstrate(ChemicalEntity, double)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html#addSubstrate(bio.singa.simulation.entities.ChemicalEntity,double)">addSubstrate(ChemicalEntity, double)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticReactantStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#addSubstrate(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.model.sections.CellTopology)">addSubstrate(ChemicalEntity, CellTopology)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html#addSubstrate(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.model.sections.CellTopology)">addSubstrate(ChemicalEntity, CellTopology)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticReactantStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html#addSubstrate(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.model.sections.CellTopology,double)">addSubstrate(ChemicalEntity, CellTopology, double)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html#addSubstrate(bio.singa.simulation.entities.ChemicalEntity,bio.singa.simulation.model.sections.CellTopology,double)">addSubstrate(ChemicalEntity, CellTopology, double)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.StaticReactantStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.StaticReactantStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/StaticReactantBehavior.html#addSubstrate(bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.Reactant)">addSubstrate(Reactant)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/behaviors/reactants/StaticReactantBehavior.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions.behaviors.reactants">StaticReactantBehavior</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusContainer.html#addToConsensusDistance(double)">addToConsensusDistance(double)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusContainer.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/VectorPath.html#addToHead(bio.singa.mathematics.vectors.Vector2D)">addToHead(Vector2D)</a></span> - Method in class bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/VectorPath.html" title="class in bio.singa.mathematics.geometry.edges">VectorPath</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/edges/VectorPath.html#addToTail(bio.singa.mathematics.vectors.Vector2D)">addToTail(Vector2D)</a></span> - Method in class bio.singa.mathematics.geometry.edges.<a href="../bio/singa/mathematics/geometry/edges/VectorPath.html" title="class in bio.singa.mathematics.geometry.edges">VectorPath</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/nested/Trajectories.html#addTrajectoryData(java.lang.Double,bio.singa.simulation.trajectories.nested.TrajectoryData)">addTrajectoryData(Double, TrajectoryData)</a></span> - Method in class bio.singa.simulation.trajectories.nested.<a href="../bio/singa/simulation/trajectories/nested/Trajectories.html" title="class in bio.singa.simulation.trajectories.nested">Trajectories</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.html#addUpdatableToObserve(bio.singa.simulation.model.simulation.Updatable)">addUpdatableToObserve(Updatable)</a></span> - Method in class bio.singa.simulation.trajectories.flat.<a href="../bio/singa/simulation/trajectories/flat/FlatUpdateRecorder.html" title="class in bio.singa.simulation.trajectories.flat">FlatUpdateRecorder</a></dt>
<dd>
<div class="block">Initializes a updatable that will be observed during simulation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/VertexPolygon.html#addVertex(bio.singa.mathematics.vectors.Vector2D)">addVertex(Vector2D)</a></span> - Method in class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/VertexPolygon.html" title="class in bio.singa.mathematics.geometry.faces">VertexPolygon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/VesicleLayer.html#addVesicle(bio.singa.simulation.model.agents.pointlike.Vesicle)">addVesicle(Vesicle)</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/VesicleLayer.html" title="class in bio.singa.simulation.model.agents.pointlike">VesicleLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/VesicleLayer.html#addVesicles(java.util.Collection)">addVesicles(Collection&lt;Vesicle&gt;)</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/VesicleLayer.html" title="class in bio.singa.simulation.model.agents.pointlike">VesicleLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/NucleotideFamily.html#ADENOSINE">ADENOSINE</a></span> - bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/NucleotideFamily.html" title="enum in bio.singa.structure.model.families">NucleotideFamily</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AffectedRegion.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AffectedRegion</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AffectedRegion.html#%3Cinit%3E(bio.singa.simulation.model.sections.CellRegion)">AffectedRegion(CellRegion)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AffectedRegion.html" title="class in bio.singa.simulation.features">AffectedRegion</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AffectedRegion.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AffectedRegion.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AffectedSection.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AffectedSection</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AffectedSection.html#%3Cinit%3E(bio.singa.simulation.model.sections.CellSubsection)">AffectedSection(CellSubsection)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AffectedSection.html" title="class in bio.singa.simulation.features">AffectedSection</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AffectedSection.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AffectedSection.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.html" title="class in bio.singa.structure.algorithms.superimposition.affinity"><span class="typeNameLink">AffinityAlignment</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/affinity/package-summary.html">bio.singa.structure.algorithms.superimposition.affinity</a></dt>
<dd>
<div class="block">An affinity alignment of same-sized <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a>s can be used to cluster them according their geometric
 similarity in a multi-structure alignment manner.</div>
</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.AtomStep.html" title="interface in bio.singa.structure.algorithms.superimposition.affinity"><span class="typeNameLink">AffinityAlignment.AtomStep</span></a> - Interface in <a href="../bio/singa/structure/algorithms/superimposition/affinity/package-summary.html">bio.singa.structure.algorithms.superimposition.affinity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.affinity"><span class="typeNameLink">AffinityAlignment.Builder</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/affinity/package-summary.html">bio.singa.structure.algorithms.superimposition.affinity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.InputStep.html" title="interface in bio.singa.structure.algorithms.superimposition.affinity"><span class="typeNameLink">AffinityAlignment.InputStep</span></a> - Interface in <a href="../bio/singa/structure/algorithms/superimposition/affinity/package-summary.html">bio.singa.structure.algorithms.superimposition.affinity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.ParameterStep.html" title="interface in bio.singa.structure.algorithms.superimposition.affinity"><span class="typeNameLink">AffinityAlignment.ParameterStep</span></a> - Interface in <a href="../bio/singa/structure/algorithms/superimposition/affinity/package-summary.html">bio.singa.structure.algorithms.superimposition.affinity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.html" title="class in bio.singa.mathematics.algorithms.clustering"><span class="typeNameLink">AffinityPropagation</span></a>&lt;<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.html" title="type parameter in AffinityPropagation">DataType</a>&gt; - Class in <a href="../bio/singa/mathematics/algorithms/clustering/package-summary.html">bio.singa.mathematics.algorithms.clustering</a></dt>
<dd>
<div class="block">Implementation of the affinity propagation algorithm, according to:</div>
</dd>
<dt><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.Builder.html" title="class in bio.singa.mathematics.algorithms.clustering"><span class="typeNameLink">AffinityPropagation.Builder</span></a>&lt;<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.Builder.html" title="type parameter in AffinityPropagation.Builder">DataType</a>&gt; - Class in <a href="../bio/singa/mathematics/algorithms/clustering/package-summary.html">bio.singa.mathematics.algorithms.clustering</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.DataStep.html" title="interface in bio.singa.mathematics.algorithms.clustering"><span class="typeNameLink">AffinityPropagation.DataStep</span></a>&lt;<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.DataStep.html" title="type parameter in AffinityPropagation.DataStep">DataType</a>&gt; - Interface in <a href="../bio/singa/mathematics/algorithms/clustering/package-summary.html">bio.singa.mathematics.algorithms.clustering</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.DistanceStep.html" title="interface in bio.singa.mathematics.algorithms.clustering"><span class="typeNameLink">AffinityPropagation.DistanceStep</span></a>&lt;<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.DistanceStep.html" title="type parameter in AffinityPropagation.DistanceStep">DataType</a>&gt; - Interface in <a href="../bio/singa/mathematics/algorithms/clustering/package-summary.html">bio.singa.mathematics.algorithms.clustering</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.MatrixStep.html" title="interface in bio.singa.mathematics.algorithms.clustering"><span class="typeNameLink">AffinityPropagation.MatrixStep</span></a>&lt;<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.MatrixStep.html" title="type parameter in AffinityPropagation.MatrixStep">DataType</a>&gt; - Interface in <a href="../bio/singa/mathematics/algorithms/clustering/package-summary.html">bio.singa.mathematics.algorithms.clustering</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.ParameterStep.html" title="interface in bio.singa.mathematics.algorithms.clustering"><span class="typeNameLink">AffinityPropagation.ParameterStep</span></a>&lt;<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagation.ParameterStep.html" title="type parameter in AffinityPropagation.ParameterStep">DataType</a>&gt; - Interface in <a href="../bio/singa/mathematics/algorithms/clustering/package-summary.html">bio.singa.mathematics.algorithms.clustering</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagationSampler.html" title="class in bio.singa.mathematics.algorithms.clustering"><span class="typeNameLink">AffinityPropagationSampler</span></a>&lt;<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagationSampler.html" title="type parameter in AffinityPropagationSampler">DataType</a>&gt; - Class in <a href="../bio/singa/mathematics/algorithms/clustering/package-summary.html">bio.singa.mathematics.algorithms.clustering</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagationSampler.html#%3Cinit%3E(java.util.List,bio.singa.mathematics.matrices.LabeledMatrix)">AffinityPropagationSampler(List&lt;DataType&gt;, LabeledMatrix&lt;DataType&gt;)</a></span> - Constructor for class bio.singa.mathematics.algorithms.clustering.<a href="../bio/singa/mathematics/algorithms/clustering/AffinityPropagationSampler.html" title="class in bio.singa.mathematics.algorithms.clustering">AffinityPropagationSampler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/AminoAcidFamily.html#ALANINE">ALANINE</a></span> - bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families">AminoAcidFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#ALANINE_ATOM_NAMES">ALANINE_ATOM_NAMES</a></span> - Static variable in enum bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/AlignmentMethod.html" title="class in bio.singa.structure.algorithms.superimposition"><span class="typeNameLink">AlignmentMethod</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/package-summary.html">bio.singa.structure.algorithms.superimposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/AlignmentMethod.html#%3Cinit%3E()">AlignmentMethod()</a></span> - Constructor for class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/AlignmentMethod.html" title="class in bio.singa.structure.algorithms.superimposition">AlignmentMethod</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/MirrorRenderer.html#alignPyramid(java.util.List,java.util.List,java.lang.String,java.lang.String,java.lang.String)">alignPyramid(List&lt;Atom&gt;, List&lt;Atom&gt;, String, String, String)</a></span> - Static method in class bio.singa.javafx.renderer.<a href="../bio/singa/javafx/renderer/MirrorRenderer.html" title="class in bio.singa.javafx.renderer">MirrorRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.Builder.html#alignWithinClusters(boolean)">alignWithinClusters(boolean)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.affinity.<a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.affinity">AffinityAlignment.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.ParameterStep.html#alignWithinClusters(boolean)">alignWithinClusters(boolean)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.affinity.<a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.ParameterStep.html" title="interface in bio.singa.structure.algorithms.superimposition.affinity">AffinityAlignment.ParameterStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html#alignWithinClusters(boolean)">alignWithinClusters(boolean)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.ParameterStep.html#alignWithinClusters(boolean)">alignWithinClusters(boolean)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.ParameterStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.ParameterStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.LocalSourceStep.html#all()">all()</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.LocalSourceStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.LocalSourceStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html#all()">all()</a></span> - Method in class bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.Builder.html" title="class in bio.singa.structure.parser.pfam">PfamParser.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pfam/PfamParser.ChainStep.html#all()">all()</a></span> - Method in interface bio.singa.structure.parser.pfam.<a href="../bio/singa/structure/parser/pfam/PfamParser.ChainStep.html" title="interface in bio.singa.structure.parser.pfam">PfamParser.ChainStep</a></dt>
<dd>
<div class="block">Parse all chains annotated in Pfam.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.LocalSourceStep.html#all(int)">all(int)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.LocalSourceStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.LocalSourceStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/MatcherFamily.html#ALL">ALL</a></span> - bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/MatcherFamily.html" title="enum in bio.singa.structure.model.families">MatcherFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/MatcherFamily.html#ALL_AMINO_ACIDS">ALL_AMINO_ACIDS</a></span> - Static variable in enum bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/MatcherFamily.html" title="enum in bio.singa.structure.model.families">MatcherFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.MultiChainStep.html#allChains()">allChains()</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.MultiChainStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.MultiChainStep</a></dt>
<dd>
<div class="block">Parses all chains.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.SingleChainStep.html#allChains()">allChains()</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.SingleChainStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.SingleChainStep</a></dt>
<dd>
<div class="block">Parses all chains.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/error/ErrorManager.html#allErrorsAreSafe()">allErrorsAreSafe()</a></span> - Method in class bio.singa.simulation.model.simulation.error.<a href="../bio/singa/simulation/model/simulation/error/ErrorManager.html" title="class in bio.singa.simulation.model.simulation.error">ErrorManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.MultiModelStep.html#allModels()">allModels()</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.MultiModelStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.MultiModelStep</a></dt>
<dd>
<div class="block">Short cut to parse all models.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParser.SingleModelStep.html#allModels()">allModels()</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParser.SingleModelStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureParser.SingleModelStep</a></dt>
<dd>
<div class="block">Short cut to parse all models.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/EntityRegistry.html#allWith(java.lang.String...)">allWith(String...)</a></span> - Static method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/EntityRegistry.html" title="class in bio.singa.simulation.entities">EntityRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#almostEqual(bio.singa.mathematics.vectors.Vector3D,double)">almostEqual(Vector3D, double)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationSchemeType.html#ALPHA_CARBON">ALPHA_CARBON</a></span> - bio.singa.structure.algorithms.superimposition.fit3d.representations.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/RepresentationSchemeType.html" title="enum in bio.singa.structure.algorithms.superimposition.fit3d.representations">RepresentationSchemeType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilterType.html#ALPHA_CARBON">ALPHA_CARBON</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilterType.html" title="enum in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilterType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/mmtf/MmtfSecondaryStructure.html#ALPHA_HELIX">ALPHA_HELIX</a></span> - bio.singa.structure.model.mmtf.<a href="../bio/singa/structure/model/mmtf/MmtfSecondaryStructure.html" title="enum in bio.singa.structure.model.mmtf">MmtfSecondaryStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/AlphaCarbonRepresentationScheme.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.representations"><span class="typeNameLink">AlphaCarbonRepresentationScheme</span></a> - Class in <a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/package-summary.html">bio.singa.structure.algorithms.superimposition.fit3d.representations</a></dt>
<dd>
<div class="block">An implementation to represent a given <a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructure</code></a> by its alpha carbon.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/AlphaCarbonRepresentationScheme.html#%3Cinit%3E()">AlphaCarbonRepresentationScheme()</a></span> - Constructor for class bio.singa.structure.algorithms.superimposition.fit3d.representations.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/representations/AlphaCarbonRepresentationScheme.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d.representations">AlphaCarbonRepresentationScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html#ALTERNATE_LOCATION_INDICATOR">ALTERNATE_LOCATION_INDICATOR</a></span> - bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">AtomToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#ALUMINIUM">ALUMINIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#AMERICIUM">AMERICIUM</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilterType.html#AMINO_ACID">AMINO_ACID</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilterType.html" title="enum in bio.singa.structure.model.oak">StructuralEntityFilter.LeafFilterType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/LeafSkeleton.AssignedFamily.html#AMINO_ACID">AMINO_ACID</a></span> - bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/LeafSkeleton.AssignedFamily.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">LeafSkeleton.AssignedFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/annotations/AnnotationType.html#AMINO_ACID_SEQUENCE">AMINO_ACID_SEQUENCE</a></span> - bio.singa.chemistry.annotations.<a href="../bio/singa/chemistry/annotations/AnnotationType.html" title="enum in bio.singa.chemistry.annotations">AnnotationType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.ResidueStep.html#aminoAcid(int)">aminoAcid(int)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.ResidueStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureSelector.ResidueStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.Selector.html#aminoAcid(int)">aminoAcid(int)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.Selector.html" title="class in bio.singa.structure.parser.pdb.structures">StructureSelector.Selector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/model/interfaces/AminoAcid.html" title="interface in bio.singa.structure.model.interfaces"><span class="typeNameLink">AminoAcid</span></a> - Interface in <a href="../bio/singa/structure/model/interfaces/package-summary.html">bio.singa.structure.model.interfaces</a></dt>
<dd>
<div class="block">A specific type of <a href="../bio/singa/structure/model/interfaces/LeafSubstructure.html" title="interface in bio.singa.structure.model.interfaces"><code>LeafSubstructure</code></a>, representing amino acids.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/viewer/MaterialProvider.html#AMINOACID">AMINOACID</a></span> - Static variable in class bio.singa.javafx.viewer.<a href="../bio/singa/javafx/viewer/MaterialProvider.html" title="class in bio.singa.javafx.viewer">MaterialProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families"><span class="typeNameLink">AminoAcidFamily</span></a> - Enum in <a href="../bio/singa/structure/model/families/package-summary.html">bio.singa.structure.model.families</a></dt>
<dd>
<div class="block">A <a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families"><code>AminoAcidFamily</code></a> contains the general information of a type of amino acid.</div>
</dd>
<dt><a href="../bio/singa/structure/model/oak/AminoAcids.html" title="class in bio.singa.structure.model.oak"><span class="typeNameLink">AminoAcids</span></a> - Class in <a href="../bio/singa/structure/model/oak/package-summary.html">bio.singa.structure.model.oak</a></dt>
<dd>
<div class="block">Utility methods for <a href="../bio/singa/structure/model/interfaces/AminoAcid.html" title="interface in bio.singa.structure.model.interfaces"><code>AminoAcid</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html#and()">and()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.AbstractReactorBuilder.html" title="class in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.AbstractReactorBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ConnectorStep.html#and()">and()</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionChainBuilder.ConnectorStep.html" title="interface in bio.singa.simulation.reactions.reactors">ReactionChainBuilder.ConnectorStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/BitVector.html#and(bio.singa.mathematics.vectors.BitVector)">and(BitVector)</a></span> - Method in interface bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/BitVector.html" title="interface in bio.singa.mathematics.vectors">BitVector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector.html#angleTo(bio.singa.mathematics.vectors.Vector)">angleTo(Vector)</a></span> - Method in interface bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a></dt>
<dd>
<div class="block">Returns the angle between this vector and the given vector in radians.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#angleTo(bio.singa.mathematics.vectors.Vector2D)">angleTo(Vector2D)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>
<div class="block">Returns the angle between this vector and the given vector, in relation to the origin (0,0) in radians.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector2D.html#angleTo(bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D)">angleTo(Vector2D, Vector2D)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector2D.html" title="class in bio.singa.mathematics.vectors">Vector2D</a></dt>
<dd>
<div class="block">Returns the angle between this vector and the given target vector, in relation to the given origin in radians.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector.html#angleToInDegrees(bio.singa.mathematics.vectors.Vector)">angleToInDegrees(Vector)</a></span> - Method in interface bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a></dt>
<dd>
<div class="block">Returns the angle between this vector and the given vector in degrees.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/units/UnitProvider.html#ANGSTROEM">ANGSTROEM</a></span> - Static variable in class bio.singa.features.units.<a href="../bio/singa/features/units/UnitProvider.html" title="class in bio.singa.features.units">UnitProvider</a></dt>
<dd>
<div class="block">The Angstroem is a unit of length equal to 10e10 m (one ten-billionth of a metre) or 0.1 nanometre.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/model/VectorMetricProvider.html#ANGULAR_DISTANCE">ANGULAR_DISTANCE</a></span> - Static variable in class bio.singa.mathematics.metrics.model.<a href="../bio/singa/mathematics/metrics/model/VectorMetricProvider.html" title="class in bio.singa.mathematics.metrics.model">VectorMetricProvider</a></dt>
<dd>
<div class="block">Using the same calculation of similarity as the <a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html" title="class in bio.singa.mathematics.metrics.implementations"><code>CosineSimilarity</code></a>, the normalised angle between the
 vectors can be used as a bounded similarity function within [0,1].</div>
</dd>
<dt><a href="../bio/singa/mathematics/metrics/implementations/AngularDistance.html" title="class in bio.singa.mathematics.metrics.implementations"><span class="typeNameLink">AngularDistance</span></a>&lt;<a href="../bio/singa/mathematics/metrics/implementations/AngularDistance.html" title="type parameter in AngularDistance">VectorDimension</a> extends <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a>&gt; - Class in <a href="../bio/singa/mathematics/metrics/implementations/package-summary.html">bio.singa.mathematics.metrics.implementations</a></dt>
<dd>
<div class="block">Using the same calculation of similarity as the <a href="../bio/singa/mathematics/metrics/implementations/CosineSimilarity.html" title="class in bio.singa.mathematics.metrics.implementations"><code>CosineSimilarity</code></a>, the normalised angle between the vectors
 can be used as a bounded similarity function within [0,1].</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/metrics/implementations/AngularDistance.html#%3Cinit%3E()">AngularDistance()</a></span> - Constructor for class bio.singa.mathematics.metrics.implementations.<a href="../bio/singa/mathematics/metrics/implementations/AngularDistance.html" title="class in bio.singa.mathematics.metrics.implementations">AngularDistance</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/annotations/Annotatable.html" title="interface in bio.singa.chemistry.annotations"><span class="typeNameLink">Annotatable</span></a> - Interface in <a href="../bio/singa/chemistry/annotations/package-summary.html">bio.singa.chemistry.annotations</a></dt>
<dd>
<div class="block">Annotations in a chemical/biological environment are relevant information about the Object implementing this
 interface, albeit not directly required for any application or method of this object.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html#annotation(bio.singa.chemistry.annotations.Annotation)">annotation(Annotation)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/chemistry/annotations/Annotation.html" title="class in bio.singa.chemistry.annotations"><span class="typeNameLink">Annotation</span></a>&lt;<a href="../bio/singa/chemistry/annotations/Annotation.html" title="type parameter in Annotation">ContentType</a>&gt; - Class in <a href="../bio/singa/chemistry/annotations/package-summary.html">bio.singa.chemistry.annotations</a></dt>
<dd>
<div class="block">An Annotation in a chemical/biological environment is a relevant information about the Object that holds it, albeit
 not directly required for any application or method of this object.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/annotations/Annotation.html#%3Cinit%3E(bio.singa.chemistry.annotations.AnnotationType)">Annotation(AnnotationType)</a></span> - Constructor for class bio.singa.chemistry.annotations.<a href="../bio/singa/chemistry/annotations/Annotation.html" title="class in bio.singa.chemistry.annotations">Annotation</a></dt>
<dd>
<div class="block">Creates a new Annotation with the given <a href="../bio/singa/chemistry/annotations/AnnotationType.html" title="enum in bio.singa.chemistry.annotations"><code>AnnotationType</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/annotations/Annotation.html#%3Cinit%3E(bio.singa.chemistry.annotations.AnnotationType,ContentType)">Annotation(AnnotationType, ContentType)</a></span> - Constructor for class bio.singa.chemistry.annotations.<a href="../bio/singa/chemistry/annotations/Annotation.html" title="class in bio.singa.chemistry.annotations">Annotation</a></dt>
<dd>
<div class="block">Creates a new Annotation with the given <a href="../bio/singa/chemistry/annotations/AnnotationType.html" title="enum in bio.singa.chemistry.annotations"><code>AnnotationType</code></a> and content.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/annotations/Annotation.html#%3Cinit%3E(bio.singa.chemistry.annotations.AnnotationType,java.lang.String,ContentType)">Annotation(AnnotationType, String, ContentType)</a></span> - Constructor for class bio.singa.chemistry.annotations.<a href="../bio/singa/chemistry/annotations/Annotation.html" title="class in bio.singa.chemistry.annotations">Annotation</a></dt>
<dd>
<div class="block">Creates a new Annotation with the given <a href="../bio/singa/chemistry/annotations/AnnotationType.html" title="enum in bio.singa.chemistry.annotations"><code>AnnotationType</code></a>, description, and content.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html#annotations">annotations</a></span> - Variable in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity</a></dt>
<dd>
<div class="block">All annotations of this entity.</div>
</dd>
<dt><a href="../bio/singa/chemistry/annotations/AnnotationType.html" title="enum in bio.singa.chemistry.annotations"><span class="typeNameLink">AnnotationType</span></a> - Enum in <a href="../bio/singa/chemistry/annotations/package-summary.html">bio.singa.chemistry.annotations</a></dt>
<dd>
<div class="block">The <a href="../bio/singa/chemistry/annotations/AnnotationType.html" title="enum in bio.singa.chemistry.annotations"><code>AnnotationType</code></a> provides a primary identification criteria for <a href="../bio/singa/chemistry/annotations/Annotation.html" title="class in bio.singa.chemistry.annotations"><code>Annotation</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_BLACK">ANSI_BLACK</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_BLUE">ANSI_BLUE</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_BRIGHT_BLACK">ANSI_BRIGHT_BLACK</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_BRIGHT_BLUE">ANSI_BRIGHT_BLUE</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_BRIGHT_CYAN">ANSI_BRIGHT_CYAN</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_BRIGHT_GREEN">ANSI_BRIGHT_GREEN</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_BRIGHT_PURPLE">ANSI_BRIGHT_PURPLE</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_BRIGHT_RED">ANSI_BRIGHT_RED</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_BRIGHT_WHITE">ANSI_BRIGHT_WHITE</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_BRIGHT_YELLOW">ANSI_BRIGHT_YELLOW</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_CYAN">ANSI_CYAN</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_GREEN">ANSI_GREEN</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_PURPLE">ANSI_PURPLE</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_RED">ANSI_RED</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_RESET">ANSI_RESET</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_WHITE">ANSI_WHITE</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/AnsiColor.html#ANSI_YELLOW">ANSI_YELLOW</a></span> - bio.singa.core.utility.<a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility">AnsiColor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/core/utility/AnsiColor.html" title="enum in bio.singa.core.utility"><span class="typeNameLink">AnsiColor</span></a> - Enum in <a href="../bio/singa/core/utility/package-summary.html">bio.singa.core.utility</a></dt>
<dd>
<div class="block">A class that allows printing to terminals in ANSI colors.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#ANTIMONY">ANTIMONY</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html#append(bio.singa.mathematics.graphs.trees.BinaryTreeNode)">append(BinaryTreeNode&lt;T&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTreeNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ReactionTrack.html#append(java.util.List)">append(List&lt;ComplexEntity&gt;)</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ReactionTrack.html" title="class in bio.singa.simulation.reactions.reactors">ReactionTrack</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html#append(T)">append(T)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTreeNode.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTreeNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#appendGraph(NodeType,NodeType,bio.singa.mathematics.graphs.model.Graph)">appendGraph(NodeType, NodeType, Graph&lt;NodeType, EdgeType, IdentifierType&gt;)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html#appendNode(java.util.function.Predicate,NodeType)">appendNode(Predicate&lt;NodeType&gt;, NodeType)</a></span> - Method in class bio.singa.mathematics.graphs.model.<a href="../bio/singa/mathematics/graphs/model/AbstractMapGraph.html" title="class in bio.singa.mathematics.graphs.model">AbstractMapGraph</a></dt>
<dd>
<div class="block">Evaluates the predicate for every node in the graph.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html#appendNodeTo(bio.singa.mathematics.graphs.trees.BinaryTreeNode,T)">appendNodeTo(BinaryTreeNode&lt;T&gt;, T)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTree</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html#appendNodeTo(T,T)">appendNodeTo(T, T)</a></span> - Method in class bio.singa.mathematics.graphs.trees.<a href="../bio/singa/mathematics/graphs/trees/BinaryTree.html" title="class in bio.singa.mathematics.graphs.trees">BinaryTree</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/trajectories/Recorders.html#appendTimestampedFolder(java.nio.file.Path)">appendTimestampedFolder(Path)</a></span> - Static method in class bio.singa.simulation.trajectories.<a href="../bio/singa/simulation/trajectories/Recorders.html" title="class in bio.singa.simulation.trajectories">Recorders</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AppliedVesicleState.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AppliedVesicleState</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AppliedVesicleState.html#%3Cinit%3E(java.lang.String)">AppliedVesicleState(String)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AppliedVesicleState.html" title="class in bio.singa.simulation.features">AppliedVesicleState</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AppliedVesicleState.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AppliedVesicleState.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/AddModification.html#apply()">apply()</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/AddModification.html" title="class in bio.singa.simulation.reactions.modifications">AddModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/BindModification.html#apply()">apply()</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/BindModification.html" title="class in bio.singa.simulation.reactions.modifications">BindModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModification.html#apply()">apply()</a></span> - Method in interface bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/ComplexEntityModification.html" title="interface in bio.singa.simulation.reactions.modifications">ComplexEntityModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/ReleaseModification.html#apply()">apply()</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/ReleaseModification.html" title="class in bio.singa.simulation.reactions.modifications">ReleaseModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/RemoveModification.html#apply()">apply()</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/RemoveModification.html" title="class in bio.singa.simulation.reactions.modifications">RemoveModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/ComplexReactor.html#apply()">apply()</a></span> - Method in interface bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/ComplexReactor.html" title="interface in bio.singa.simulation.reactions.reactors">ComplexReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/OneToOneReactor.html#apply()">apply()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/OneToOneReactor.html" title="class in bio.singa.simulation.reactions.reactors">OneToOneReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/OneToTwoReactor.html#apply()">apply()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/OneToTwoReactor.html" title="class in bio.singa.simulation.reactions.reactors">OneToTwoReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/reactors/TwoToOneReactor.html#apply()">apply()</a></span> - Method in class bio.singa.simulation.reactions.reactors.<a href="../bio/singa/simulation/reactions/reactors/TwoToOneReactor.html" title="class in bio.singa.simulation.reactions.reactors">TwoToOneReactor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/AddModification.html#apply(bio.singa.simulation.entities.ComplexEntity)">apply(ComplexEntity)</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/AddModification.html" title="class in bio.singa.simulation.reactions.modifications">AddModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/ReleaseModification.html#apply(bio.singa.simulation.entities.ComplexEntity)">apply(ComplexEntity)</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/ReleaseModification.html" title="class in bio.singa.simulation.reactions.modifications">ReleaseModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/RemoveModification.html#apply(bio.singa.simulation.entities.ComplexEntity)">apply(ComplexEntity)</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/RemoveModification.html" title="class in bio.singa.simulation.reactions.modifications">RemoveModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/reactions/modifications/BindModification.html#apply(bio.singa.simulation.entities.ComplexEntity,bio.singa.simulation.entities.ComplexEntity)">apply(ComplexEntity, ComplexEntity)</a></span> - Method in class bio.singa.simulation.reactions.modifications.<a href="../bio/singa/simulation/reactions/modifications/BindModification.html" title="class in bio.singa.simulation.reactions.modifications">BindModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html#apply(bio.singa.simulation.model.simulation.Simulation)">apply(Simulation)</a></span> - Method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html" title="class in bio.singa.simulation.model.concentrations">InitialConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html#apply(bio.singa.simulation.model.simulation.Updatable)">apply(Updatable)</a></span> - Method in class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/InitialConcentration.html" title="class in bio.singa.simulation.model.concentrations">InitialConcentration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/layouts/force/BinaryForce.html#apply(GraphType)">apply(GraphType)</a></span> - Method in class bio.singa.javafx.renderer.layouts.force.<a href="../bio/singa/javafx/renderer/layouts/force/BinaryForce.html" title="class in bio.singa.javafx.renderer.layouts.force">BinaryForce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/layouts/force/Force.html#apply(GraphType)">apply(GraphType)</a></span> - Method in class bio.singa.javafx.renderer.layouts.force.<a href="../bio/singa/javafx/renderer/layouts/force/Force.html" title="class in bio.singa.javafx.renderer.layouts.force">Force</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/layouts/force/UnaryForce.html#apply(GraphType)">apply(GraphType)</a></span> - Method in class bio.singa.javafx.renderer.layouts.force.<a href="../bio/singa/javafx/renderer/layouts/force/UnaryForce.html" title="class in bio.singa.javafx.renderer.layouts.force">UnaryForce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/simulation/Simulation.html#applyAssignmentRules()">applyAssignmentRules()</a></span> - Method in class bio.singa.simulation.model.simulation.<a href="../bio/singa/simulation/model/simulation/Simulation.html" title="class in bio.singa.simulation.model.simulation">Simulation</a></dt>
<dd>
<div class="block">Apply all referenced assignment rules.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/VesicleLayer.html#applyDeltas()">applyDeltas()</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/VesicleLayer.html" title="class in bio.singa.simulation.model.agents.pointlike">VesicleLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html#applyDeltas()">applyDeltas()</a></span> - Method in class bio.singa.simulation.model.modules.concentration.<a href="../bio/singa/simulation/model/modules/concentration/ConcentrationDeltaManager.html" title="class in bio.singa.simulation.model.modules.concentration">ConcentrationDeltaManager</a></dt>
<dd>
<div class="block">Applies all final deltas and clears the delta list.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html#applyMapping()">applyMapping()</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorQuaternionSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorQuaternionSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html#applyMapping()">applyMapping()</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposer.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/rules/AssignmentRule.html#applyRule(bio.singa.simulation.model.graphs.AutomatonNode)">applyRule(AutomatonNode)</a></span> - Method in class bio.singa.simulation.model.rules.<a href="../bio/singa/simulation/model/rules/AssignmentRule.html" title="class in bio.singa.simulation.model.rules">AssignmentRule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html#applySettings(bio.singa.structure.parser.pdb.structures.StructureParserOptions.Setting...)">applySettings(StructureParserOptions.Setting...)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureParserOptions.html" title="class in bio.singa.structure.parser.pdb.structures">StructureParserOptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/transformations/model/MatrixTransformation.html#applyTo(bio.singa.mathematics.matrices.Matrix)">applyTo(Matrix)</a></span> - Method in interface bio.singa.mathematics.transformations.model.<a href="../bio/singa/mathematics/transformations/model/MatrixTransformation.html" title="interface in bio.singa.mathematics.transformations.model">MatrixTransformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/transformations/model/BitvectorTransformation.html#applyTo(bio.singa.mathematics.vectors.BitVector)">applyTo(BitVector)</a></span> - Method in interface bio.singa.mathematics.transformations.model.<a href="../bio/singa/mathematics/transformations/model/BitvectorTransformation.html" title="interface in bio.singa.mathematics.transformations.model">BitvectorTransformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/transformations/model/VectorTransformation.html#applyTo(bio.singa.mathematics.vectors.Vector)">applyTo(Vector)</a></span> - Method in interface bio.singa.mathematics.transformations.model.<a href="../bio/singa/mathematics/transformations/model/VectorTransformation.html" title="interface in bio.singa.mathematics.transformations.model">VectorTransformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposition.html#applyTo(java.util.List)">applyTo(List&lt;LeafSubstructure&lt;?&gt;&gt;)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposition.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/Superimposition.html#applyTo(java.util.List)">applyTo(List&lt;T&gt;)</a></span> - Method in interface bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/Superimposition.html" title="interface in bio.singa.mathematics.algorithms.superimposition">Superimposition</a></dt>
<dd>
<div class="block">applies this superimposition to a list of candidate</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposition.html#applyTo(java.util.List)">applyTo(List&lt;VectorType&gt;)</a></span> - Method in class bio.singa.mathematics.algorithms.superimposition.<a href="../bio/singa/mathematics/algorithms/superimposition/VectorSuperimposition.html" title="class in bio.singa.mathematics.algorithms.superimposition">VectorSuperimposition</a></dt>
<dd>
<div class="block">Applies the superimposition to the given list of <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors"><code>Vector</code></a>s and returns new instances.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposition.html#applyTo(T)">applyTo(T)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.<a href="../bio/singa/structure/algorithms/superimposition/SubstructureSuperimposition.html" title="class in bio.singa.structure.algorithms.superimposition">SubstructureSuperimposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/transformations/model/Transformation.html#applyTo(TransformedType)">applyTo(TransformedType)</a></span> - Method in interface bio.singa.mathematics.transformations.model.<a href="../bio/singa/mathematics/transformations/model/Transformation.html" title="interface in bio.singa.mathematics.transformations.model">Transformation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/Matrix.html#applyTransformation(bio.singa.mathematics.transformations.model.Transformation)">applyTransformation(Transformation&lt;Matrix&gt;)</a></span> - Method in interface bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/Matrix.html" title="interface in bio.singa.mathematics.matrices">Matrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector.html#applyTransformation(bio.singa.mathematics.transformations.model.Transformation)">applyTransformation(Transformation&lt;Vector&gt;)</a></span> - Method in interface bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/transformations/model/Transformable.html#applyTransformation(bio.singa.mathematics.transformations.model.Transformation)">applyTransformation(Transformation&lt;TransformationType&gt;)</a></span> - Method in interface bio.singa.mathematics.transformations.model.<a href="../bio/singa/mathematics/transformations/model/Transformable.html" title="interface in bio.singa.mathematics.transformations.model">Transformable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/MirrorRenderer.html#applyTransformation(bio.singa.mathematics.vectors.Vector3D,bio.singa.mathematics.algorithms.superimposition.VectorSuperimposition)">applyTransformation(Vector3D, VectorSuperimposition&lt;Vector3D&gt;)</a></span> - Static method in class bio.singa.javafx.renderer.<a href="../bio/singa/javafx/renderer/MirrorRenderer.html" title="class in bio.singa.javafx.renderer">MirrorRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilterType.html#ARBITRARY">ARBITRARY</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilterType.html" title="enum in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilterType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Circles.html#arcLength(bio.singa.mathematics.geometry.faces.Circle,bio.singa.mathematics.vectors.Vector2D,bio.singa.mathematics.vectors.Vector2D)">arcLength(Circle, Vector2D, Vector2D)</a></span> - Static method in class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Circles.html" title="class in bio.singa.mathematics.geometry.faces">Circles</a></dt>
<dd>
<div class="block">Returns the length of the arc created by projecting the two vectors on the circumference of the circle.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/geometry/faces/Circles.html#area(double)">area(double)</a></span> - Static method in class bio.singa.mathematics.geometry.faces.<a href="../bio/singa/mathematics/geometry/faces/Circles.html" title="class in bio.singa.mathematics.geometry.faces">Circles</a></dt>
<dd>
<div class="block">Returns the area of a circle with the given radius.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/concentrations/AreaCondition.html" title="class in bio.singa.simulation.model.concentrations"><span class="typeNameLink">AreaCondition</span></a> - Class in <a href="../bio/singa/simulation/model/concentrations/package-summary.html">bio.singa.simulation.model.concentrations</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/concentrations/AreaCondition.html#%3Cinit%3E(bio.singa.mathematics.geometry.model.Polygon)">AreaCondition(Polygon)</a></span> - Constructor for class bio.singa.simulation.model.concentrations.<a href="../bio/singa/simulation/model/concentrations/AreaCondition.html" title="class in bio.singa.simulation.model.concentrations">AreaCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.AreaMapping.html#%3Cinit%3E(bio.singa.simulation.model.graphs.AutomatonNode,bio.singa.simulation.model.graphs.AutomatonNode,bio.singa.simulation.model.sections.CellSubsection,double)">AreaMapping(AutomatonNode, AutomatonNode, CellSubsection, double)</a></span> - Constructor for class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.AreaMapping.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode.AreaMapping</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/AminoAcidFamily.html#ARGININE">ARGININE</a></span> - bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families">AminoAcidFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#ARGININE_ATOM_NAMES">ARGININE_ATOM_NAMES</a></span> - Static variable in enum bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#ARGON">ARGON</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/CovalentBondType.html#AROMATIC_BOND">AROMATIC_BOND</a></span> - bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/CovalentBondType.html" title="enum in bio.singa.chemistry.model">CovalentBondType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/layouts/force/ForceDirectedGraphLayout.html#arrangeGraph(int)">arrangeGraph(int)</a></span> - Method in class bio.singa.javafx.renderer.layouts.force.<a href="../bio/singa/javafx/renderer/layouts/force/ForceDirectedGraphLayout.html" title="class in bio.singa.javafx.renderer.layouts.force">ForceDirectedGraphLayout</a></dt>
<dd>
<div class="block">Calculates one iteration of the optimization process and returns the resulting graph.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html#arrangeGraph(GraphType)">arrangeGraph(GraphType)</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html" title="class in bio.singa.javafx.renderer.graphs">GraphRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html#arrangeGraph(GraphType,int)">arrangeGraph(GraphType, int)</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html" title="class in bio.singa.javafx.renderer.graphs">GraphRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html#arrangeOnce(GraphType)">arrangeOnce(GraphType)</a></span> - Method in class bio.singa.javafx.renderer.graphs.<a href="../bio/singa/javafx/renderer/graphs/GraphRenderer.html" title="class in bio.singa.javafx.renderer.graphs">GraphRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#ARSENIC">ARSENIC</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/Matrix.html#as(java.lang.Class)">as(Class&lt;M&gt;)</a></span> - Method in interface bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/Matrix.html" title="interface in bio.singa.mathematics.matrices">Matrix</a></dt>
<dd>
<div class="block">Converts this matrix into a specific implementation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/RegularMatrix.html#as(java.lang.Class)">as(Class&lt;MatrixClass&gt;)</a></span> - Method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/RegularMatrix.html" title="class in bio.singa.mathematics.matrices">RegularMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Scalar.html#as(java.lang.Class)">as(Class&lt;V&gt;)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Scalar.html" title="class in bio.singa.mathematics.vectors">Scalar</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/RegularVector.html#as(java.lang.Class)">as(Class&lt;VectorClass&gt;)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/RegularVector.html" title="class in bio.singa.mathematics.vectors">RegularVector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector.html#as(java.lang.Class)">as(Class&lt;VectorType&gt;)</a></span> - Method in interface bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors">Vector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/vectors/Vector3D.html#as(java.lang.Class)">as(Class&lt;VectorType&gt;)</a></span> - Method in class bio.singa.mathematics.vectors.<a href="../bio/singa/mathematics/vectors/Vector3D.html" title="class in bio.singa.mathematics.vectors">Vector3D</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/Element.html#asAnion(int)">asAnion(int)</a></span> - Method in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/Element.html" title="class in bio.singa.chemistry.model.elements">Element</a></dt>
<dd>
<div class="block">Converts this element into an anion by increasing its electron count.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/Element.html#asCation(int)">asCation(int)</a></span> - Method in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/Element.html" title="class in bio.singa.chemistry.model.elements">Element</a></dt>
<dd>
<div class="block">Converts this element into an cation by decreasing its electron count.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/Element.html#asIon(int)">asIon(int)</a></span> - Method in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/Element.html" title="class in bio.singa.chemistry.model.elements">Element</a></dt>
<dd>
<div class="block">Converts this element into an ion (cation) by decreasing its electron count.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/Element.html#asIsotope(int)">asIsotope(int)</a></span> - Method in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/Element.html" title="class in bio.singa.chemistry.model.elements">Element</a></dt>
<dd>
<div class="block">Converts this element into an isotope by adjusting its neutron count.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/AminoAcidFamily.html#ASPARAGINE">ASPARAGINE</a></span> - bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families">AminoAcidFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#ASPARAGINE_ATOM_NAMES">ASPARAGINE_ATOM_NAMES</a></span> - Static variable in enum bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/families/AminoAcidFamily.html#ASPARTIC_ACID">ASPARTIC_ACID</a></span> - bio.singa.structure.model.families.<a href="../bio/singa/structure/model/families/AminoAcidFamily.html" title="enum in bio.singa.structure.model.families">AminoAcidFamily</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/AtomName.html#ASPARTIC_ACID_ATOM_NAMES">ASPARTIC_ACID_ATOM_NAMES</a></span> - Static variable in enum bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak">AtomName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html#assembleAtom(java.lang.String)">assembleAtom(String)</a></span> - Static method in enum bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">AtomToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/Matrices.html#assembleMatrixFromColumns(java.util.List)">assembleMatrixFromColumns(List&lt;Vector&gt;)</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/Matrices.html" title="class in bio.singa.mathematics.matrices">Matrices</a></dt>
<dd>
<div class="block">Creates a new matrix from vectors in a list.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/FastMatrices.html#assembleMatrixFromRows(java.util.List)">assembleMatrixFromRows(List&lt;Vector&gt;)</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/FastMatrices.html" title="class in bio.singa.mathematics.matrices">FastMatrices</a></dt>
<dd>
<div class="block">Creates a <a href="../bio/singa/mathematics/matrices/Matrix.html" title="interface in bio.singa.mathematics.matrices"><code>Matrix</code></a> from the given row <a href="../bio/singa/mathematics/vectors/Vector.html" title="interface in bio.singa.mathematics.vectors"><code>Vector</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/Matrices.html#assembleMatrixFromRows(java.util.List)">assembleMatrixFromRows(List&lt;Vector&gt;)</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/Matrices.html" title="class in bio.singa.mathematics.matrices">Matrices</a></dt>
<dd>
<div class="block">Creates a new matrix from vectors in a list.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html#assemblePDBLine(bio.singa.structure.model.interfaces.LeafSubstructure)">assemblePDBLine(LeafSubstructure)</a></span> - Static method in enum bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">AtomToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/ChainTerminatorToken.html#assemblePDBLine(bio.singa.structure.model.interfaces.LeafSubstructure)">assemblePDBLine(LeafSubstructure)</a></span> - Static method in enum bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/ChainTerminatorToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">ChainTerminatorToken</a></dt>
<dd>
<div class="block">Creates a terminate record from the last leaf in the consecutive part of a chain.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/HeaderToken.html#assemblePDBLine(java.lang.String)">assemblePDBLine(String)</a></span> - Static method in enum bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/HeaderToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">HeaderToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/TitleToken.html#assemblePDBLines(java.lang.String)">assemblePDBLines(String)</a></span> - Static method in enum bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/TitleToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">TitleToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/export/modules/ReactionTableRow.html#assembleRowsSet(int)">assembleRowsSet(int)</a></span> - Method in class bio.singa.simulation.export.modules.<a href="../bio/singa/simulation/export/modules/ReactionTableRow.html" title="class in bio.singa.simulation.export.modules">ReactionTableRow</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/core/utility/Bounded.html#assertIfValueInRange(BoundType)">assertIfValueInRange(BoundType)</a></span> - Method in interface bio.singa.core.utility.<a href="../bio/singa/core/utility/Bounded.html" title="interface in bio.singa.core.utility">Bounded</a></dt>
<dd>
<div class="block">Asserts whether the given value is larger than the lower bound and smaller than the upper bound specified for
 this object.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/concepts/MultiDimensional.html#assertThatDimensionsMatch(NumberConcept)">assertThatDimensionsMatch(NumberConcept)</a></span> - Method in interface bio.singa.mathematics.concepts.<a href="../bio/singa/mathematics/concepts/MultiDimensional.html" title="interface in bio.singa.mathematics.concepts">MultiDimensional</a></dt>
<dd>
<div class="block">Asserts that the given number concept has the same dimension as this
 vector.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/Matrix.html#assertThatInnerDimensionsMatch(bio.singa.mathematics.matrices.Matrix)">assertThatInnerDimensionsMatch(Matrix)</a></span> - Method in interface bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/Matrix.html" title="interface in bio.singa.mathematics.matrices">Matrix</a></dt>
<dd>
<div class="block">Asserts that the inner dimensions of both matrices match and throws an <a href="../bio/singa/mathematics/exceptions/IncompatibleDimensionsException.html" title="class in bio.singa.mathematics.exceptions"><code>IncompatibleDimensionsException</code></a>
 otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/Matrix.html#assertThatInnerDimensionsMatch(bio.singa.mathematics.vectors.Vector)">assertThatInnerDimensionsMatch(Vector)</a></span> - Method in interface bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/Matrix.html" title="interface in bio.singa.mathematics.matrices">Matrix</a></dt>
<dd>
<div class="block">Asserts that the inner dimensions of the matrix and the dimension of the vector match and throws an
 <a href="../bio/singa/mathematics/exceptions/IncompatibleDimensionsException.html" title="class in bio.singa.mathematics.exceptions"><code>IncompatibleDimensionsException</code></a> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html#assertThatValuesAreSymmetric(double%5B%5D%5B%5D)">assertThatValuesAreSymmetric(double[][])</a></span> - Static method in class bio.singa.mathematics.matrices.<a href="../bio/singa/mathematics/matrices/SymmetricMatrix.html" title="class in bio.singa.mathematics.matrices">SymmetricMatrix</a></dt>
<dd>
<div class="block">Asserts that the given potential values are square and symmetric and throws an <a href="../bio/singa/mathematics/exceptions/IncompatibleDimensionsException.html" title="class in bio.singa.mathematics.exceptions"><code>IncompatibleDimensionsException</code></a> otherwise.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/model/FeatureProvider.html#assign(FeatureableType)">assign(FeatureableType)</a></span> - Method in class bio.singa.features.model.<a href="../bio/singa/features/model/FeatureProvider.html" title="class in bio.singa.features.model">FeatureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/Structures.html#assignBFactors(bio.singa.structure.model.interfaces.Structure,java.util.Map)">assignBFactors(Structure, Map&lt;UniqueAtomIdentifer, Double&gt;)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/Structures.html" title="class in bio.singa.structure.model.oak">Structures</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralMotifs.html#assignComplexExchanges(bio.singa.structure.model.oak.StructuralMotif,java.util.EnumSet)">assignComplexExchanges(StructuralMotif, EnumSet&lt;MatcherFamily&gt;)</a></span> - Static method in class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralMotifs.html" title="class in bio.singa.structure.model.oak">StructuralMotifs</a></dt>
<dd>
<div class="block">Assigns the given <code>EnumSet</code> of exchanges to the specified <a href="../bio/singa/structure/model/oak/StructuralMotif.html" title="class in bio.singa.structure.model.oak"><code>StructuralMotif</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html#assignFeature(bio.singa.features.model.Feature)">assignFeature(Feature)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/SimpleEntity.SimpleEntityBuilder.html#assignFeature(bio.singa.features.model.Feature)">assignFeature(Feature)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/SimpleEntity.SimpleEntityBuilder.html" title="class in bio.singa.simulation.entities">SimpleEntity.SimpleEntityBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html#assignFeature(java.lang.Class)">assignFeature(Class&lt;? extends Feature&gt;)</a></span> - Method in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.Builder.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/SimpleEntity.SimpleEntityBuilder.html#assignFeature(java.lang.Class)">assignFeature(Class&lt;? extends Feature&gt;)</a></span> - Method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/SimpleEntity.SimpleEntityBuilder.html" title="class in bio.singa.simulation.entities">SimpleEntity.SimpleEntityBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/rules/AssignmentRule.html" title="class in bio.singa.simulation.model.rules"><span class="typeNameLink">AssignmentRule</span></a> - Class in <a href="../bio/singa/simulation/model/rules/package-summary.html">bio.singa.simulation.model.rules</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/rules/AssignmentRule.html#%3Cinit%3E(bio.singa.simulation.entities.ChemicalEntity)">AssignmentRule(ChemicalEntity)</a></span> - Constructor for class bio.singa.simulation.model.rules.<a href="../bio/singa/simulation/model/rules/AssignmentRule.html" title="class in bio.singa.simulation.model.rules">AssignmentRule</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/rules/AssignmentRules.html" title="class in bio.singa.simulation.model.rules"><span class="typeNameLink">AssignmentRules</span></a> - Class in <a href="../bio/singa/simulation/model/rules/package-summary.html">bio.singa.simulation.model.rules</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgent.html#associateInGraph(bio.singa.simulation.model.graphs.AutomatonGraph)">associateInGraph(AutomatonGraph)</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgent.html" title="class in bio.singa.simulation.model.agents.linelike">LineLikeAgent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentTemplate.html#associateInGraph(bio.singa.simulation.model.graphs.AutomatonGraph)">associateInGraph(AutomatonGraph)</a></span> - Method in class bio.singa.simulation.model.agents.linelike.<a href="../bio/singa/simulation/model/agents/linelike/LineLikeAgentTemplate.html" title="class in bio.singa.simulation.model.agents.linelike">LineLikeAgentTemplate</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/VesicleLayer.html#associateVesicles()">associateVesicles()</a></span> - Method in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/VesicleLayer.html" title="class in bio.singa.simulation.model.agents.pointlike">VesicleLayer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.ComplexBuildingReactionStep.html#associationRate(bio.singa.chemistry.features.reactions.RateConstant)">associationRate(RateConstant)</a></span> - Method in interface bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.ComplexBuildingReactionStep.html" title="interface in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.ComplexBuildingReactionStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.GeneralReactionBuilder.html#associationRate(bio.singa.chemistry.features.reactions.RateConstant)">associationRate(RateConstant)</a></span> - Method in class bio.singa.simulation.model.modules.concentration.imlementations.reactions.<a href="../bio/singa/simulation/model/modules/concentration/imlementations/reactions/ReactionBuilder.GeneralReactionBuilder.html" title="class in bio.singa.simulation.model.modules.concentration.imlementations.reactions">ReactionBuilder.GeneralReactionBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/elements/ElementProvider.html#ASTATINE">ASTATINE</a></span> - Static variable in class bio.singa.chemistry.model.elements.<a href="../bio/singa/chemistry/model/elements/ElementProvider.html" title="class in bio.singa.chemistry.model.elements">ElementProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html#asType(java.lang.Class)">asType(Class&lt;T&gt;)</a></span> - Method in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/MembranePermeability.html" title="class in bio.singa.chemistry.features.permeability">MembranePermeability</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/features/permeability/OsmoticPermeability.html#asType(java.lang.Class)">asType(Class&lt;T&gt;)</a></span> - Method in class bio.singa.chemistry.features.permeability.<a href="../bio/singa/chemistry/features/permeability/OsmoticPermeability.html" title="class in bio.singa.chemistry.features.permeability">OsmoticPermeability</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarMass.html#asType(java.lang.Class)">asType(Class&lt;T&gt;)</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarMass.html" title="class in bio.singa.features.quantities">MolarMass</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/MolarVolume.html#asType(java.lang.Class)">asType(Class&lt;T&gt;)</a></span> - Method in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/MolarVolume.html" title="class in bio.singa.features.quantities">MolarVolume</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.AtomStep.html#atom(int)">atom(int)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.AtomStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureSelector.AtomStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.Selector.html#atom(int)">atom(int)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.Selector.html" title="class in bio.singa.structure.parser.pdb.structures">StructureSelector.Selector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/model/interfaces/Atom.html" title="interface in bio.singa.structure.model.interfaces"><span class="typeNameLink">Atom</span></a> - Interface in <a href="../bio/singa/structure/model/interfaces/package-summary.html">bio.singa.structure.model.interfaces</a></dt>
<dd>
<div class="block">An Atom is the physical instance of an atom in three dimensional space.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilterType.html#ATOM_CONTAINER">ATOM_CONTAINER</a></span> - bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.LeafFilterType.html" title="enum in bio.singa.structure.model.oak">StructuralEntityFilter.LeafFilterType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html#ATOM_NAME">ATOM_NAME</a></span> - bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">AtomToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html#ATOM_SERIAL">ATOM_SERIAL</a></span> - bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">AtomToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/tokens/ChainTerminatorToken.html#ATOM_SERIAL">ATOM_SERIAL</a></span> - bio.singa.structure.parser.pdb.structures.tokens.<a href="../bio/singa/structure/parser/pdb/structures/tokens/ChainTerminatorToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens">ChainTerminatorToken</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/molecules/MoleculeIsomorphismFinder.AtomConditions.html#%3Cinit%3E()">AtomConditions()</a></span> - Constructor for class bio.singa.structure.algorithms.molecules.<a href="../bio/singa/structure/algorithms/molecules/MoleculeIsomorphismFinder.AtomConditions.html" title="class in bio.singa.structure.algorithms.molecules">MoleculeIsomorphismFinder.AtomConditions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.ResidueStep.html#atomContainer(int)">atomContainer(int)</a></span> - Method in interface bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.ResidueStep.html" title="interface in bio.singa.structure.parser.pdb.structures">StructureSelector.ResidueStep</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.Selector.html#atomContainer(int)">atomContainer(int)</a></span> - Method in class bio.singa.structure.parser.pdb.structures.<a href="../bio/singa/structure/parser/pdb/structures/StructureSelector.Selector.html" title="class in bio.singa.structure.parser.pdb.structures">StructureSelector.Selector</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/model/interfaces/AtomContainer.html" title="interface in bio.singa.structure.model.interfaces"><span class="typeNameLink">AtomContainer</span></a> - Interface in <a href="../bio/singa/structure/model/interfaces/package-summary.html">bio.singa.structure.model.interfaces</a></dt>
<dd>
<div class="block">Everything that contains <a href="../bio/singa/structure/model/interfaces/Atom.html" title="interface in bio.singa.structure.model.interfaces"><code>Atom</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.AtomStep.html#atomFilter(java.util.function.Predicate)">atomFilter(Predicate&lt;Atom&gt;)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.affinity.<a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.AtomStep.html" title="interface in bio.singa.structure.algorithms.superimposition.affinity">AffinityAlignment.AtomStep</a></dt>
<dd>
<div class="block">Defines the <a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak"><code>StructuralEntityFilter.AtomFilter</code></a> to be used for the computed alignments.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.Builder.html#atomFilter(java.util.function.Predicate)">atomFilter(Predicate&lt;Atom&gt;)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.affinity.<a href="../bio/singa/structure/algorithms/superimposition/affinity/AffinityAlignment.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.affinity">AffinityAlignment.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.AtomStep.html#atomFilter(java.util.function.Predicate)">atomFilter(Predicate&lt;Atom&gt;)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.AtomStep.html" title="interface in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.AtomStep</a></dt>
<dd>
<div class="block">Defines the <a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak"><code>StructuralEntityFilter.AtomFilter</code></a> to be used during consensus calculation.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html#atomFilter(java.util.function.Predicate)">atomFilter(Predicate&lt;Atom&gt;)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.consensus.<a href="../bio/singa/structure/algorithms/superimposition/consensus/ConsensusBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.consensus">ConsensusBuilder.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.AtomStep.html#atomFilter(java.util.function.Predicate)">atomFilter(Predicate&lt;Atom&gt;)</a></span> - Method in interface bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.AtomStep.html" title="interface in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DBuilder.AtomStep</a></dt>
<dd>
<div class="block">Defines a <a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak"><code>StructuralEntityFilter.AtomFilter</code></a> filter to be used for the <a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3D.html" title="interface in bio.singa.structure.algorithms.superimposition.fit3d"><code>Fit3D</code></a> alignment (e.g.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.Builder.html#atomFilter(java.util.function.Predicate)">atomFilter(Predicate&lt;Atom&gt;)</a></span> - Method in class bio.singa.structure.algorithms.superimposition.fit3d.<a href="../bio/singa/structure/algorithms/superimposition/fit3d/Fit3DBuilder.Builder.html" title="class in bio.singa.structure.algorithms.superimposition.fit3d">Fit3DBuilder.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html#%3Cinit%3E()">AtomFilter()</a></span> - Constructor for class bio.singa.structure.model.oak.<a href="../bio/singa/structure/model/oak/StructuralEntityFilter.AtomFilter.html" title="class in bio.singa.structure.model.oak">StructuralEntityFilter.AtomFilter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/structure/model/oak/AtomName.html" title="enum in bio.singa.structure.model.oak"><span class="typeNameLink">AtomName</span></a> - Enum in <a href="../bio/singa/structure/model/oak/package-summary.html">bio.singa.structure.model.oak</a></dt>
<dd>
<div class="block">Most (some) of the AtomNames in the PDB.</div>
</dd>
<dt><a href="../bio/singa/structure/parser/pdb/structures/tokens/AtomToken.html" title="enum in bio.singa.structure.parser.pdb.structures.tokens"><span class="typeNameLink">AtomToken</span></a> - Enum in <a href="../bio/singa/structure/parser/pdb/structures/tokens/package-summary.html">bio.singa.structure.parser.pdb.structures.tokens</a></dt>
<dd>
<div class="block">The atom and hetatm token.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntityBuilder.html#attachBindingSites(java.util.Map,java.util.Map.Entry)">attachBindingSites(Map&lt;ChemicalEntity, Set&lt;BindingSite&gt;&gt;, Map.Entry&lt;BindingSite, Pair&lt;ChemicalEntity&gt;&gt;)</a></span> - Static method in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntityBuilder.html" title="class in bio.singa.simulation.entities">ComplexEntityBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AttachedFilament.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AttachedFilament</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AttachedFilament.html#%3Cinit%3E(java.lang.String)">AttachedFilament(String)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AttachedFilament.html" title="class in bio.singa.simulation.features">AttachedFilament</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AttachedFilament.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AttachedFilament.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AttachedMotor.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AttachedMotor</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AttachedMotor.html#%3Cinit%3E(bio.singa.simulation.entities.ChemicalEntity)">AttachedMotor(ChemicalEntity)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AttachedMotor.html" title="class in bio.singa.simulation.features">AttachedMotor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AttachedMotor.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AttachedMotor.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AttachmentDistance.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AttachmentDistance</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/features/AttachmentDistance.html#%3Cinit%3E(javax.measure.Quantity)">AttachmentDistance(Quantity&lt;Length&gt;)</a></span> - Constructor for class bio.singa.simulation.features.<a href="../bio/singa/simulation/features/AttachmentDistance.html" title="class in bio.singa.simulation.features">AttachmentDistance</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/features/AttachmentDistance.Builder.html" title="class in bio.singa.simulation.features"><span class="typeNameLink">AttachmentDistance.Builder</span></a> - Class in <a href="../bio/singa/simulation/features/package-summary.html">bio.singa.simulation.features</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/graphs/AutomatonEdge.html" title="class in bio.singa.simulation.model.graphs"><span class="typeNameLink">AutomatonEdge</span></a> - Class in <a href="../bio/singa/simulation/model/graphs/package-summary.html">bio.singa.simulation.model.graphs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonEdge.html#%3Cinit%3E(int)">AutomatonEdge(int)</a></span> - Constructor for class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonEdge.html" title="class in bio.singa.simulation.model.graphs">AutomatonEdge</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/graphs/AutomatonGraph.html" title="class in bio.singa.simulation.model.graphs"><span class="typeNameLink">AutomatonGraph</span></a> - Class in <a href="../bio/singa/simulation/model/graphs/package-summary.html">bio.singa.simulation.model.graphs</a></dt>
<dd>
<div class="block">The Automaton graph class is the underlying graph of cellular graph automaton <a href="../bio/singa/simulation/model/simulation/Simulation.html" title="class in bio.singa.simulation.model.simulation"><code>Simulation</code></a>s.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonGraph.html#%3Cinit%3E(int,int)">AutomatonGraph(int, int)</a></span> - Constructor for class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonGraph.html" title="class in bio.singa.simulation.model.graphs">AutomatonGraph</a></dt>
<dd>
<div class="block">Creates a new empty graph, initialized with node and edge capacity.</div>
</dd>
<dt><a href="../bio/singa/simulation/model/graphs/AutomatonGraphs.html" title="class in bio.singa.simulation.model.graphs"><span class="typeNameLink">AutomatonGraphs</span></a> - Class in <a href="../bio/singa/simulation/model/graphs/package-summary.html">bio.singa.simulation.model.graphs</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs"><span class="typeNameLink">AutomatonNode</span></a> - Class in <a href="../bio/singa/simulation/model/graphs/package-summary.html">bio.singa.simulation.model.graphs</a></dt>
<dd>
<div class="block">A node of the <a href="../bio/singa/simulation/model/graphs/AutomatonGraph.html" title="class in bio.singa.simulation.model.graphs"><code>AutomatonGraph</code></a>.</div>
</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#%3Cinit%3E(int,int)">AutomatonNode(int, int)</a></span> - Constructor for class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/graphs/AutomatonNode.html#%3Cinit%3E(bio.singa.mathematics.topology.grids.rectangular.RectangularCoordinate)">AutomatonNode(RectangularCoordinate)</a></span> - Constructor for class bio.singa.simulation.model.graphs.<a href="../bio/singa/simulation/model/graphs/AutomatonNode.html" title="class in bio.singa.simulation.model.graphs">AutomatonNode</a></dt>
<dd>&nbsp;</dd>
<dt><a href="../bio/singa/simulation/model/graphs/AutomatonNode.AreaMapping.html" title="class in bio.singa.simulation.model.graphs"><span class="typeNameLink">AutomatonNode.AreaMapping</span></a> - Class in <a href="../bio/singa/simulation/model/graphs/package-summary.html">bio.singa.simulation.model.graphs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html#availableFeatures">availableFeatures</a></span> - Static variable in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/AbstractChemicalEntity.html" title="class in bio.singa.chemistry.model">AbstractChemicalEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/chemistry/model/SmallMolecule.html#availableFeatures">availableFeatures</a></span> - Static variable in class bio.singa.chemistry.model.<a href="../bio/singa/chemistry/model/SmallMolecule.html" title="class in bio.singa.chemistry.model">SmallMolecule</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/model/AbstractSequence.html#availableFeatures">availableFeatures</a></span> - Static variable in class bio.singa.sequence.model.<a href="../bio/singa/sequence/model/AbstractSequence.html" title="class in bio.singa.sequence.model">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/model/NucleotideSequence.html#availableFeatures">availableFeatures</a></span> - Static variable in class bio.singa.sequence.model.<a href="../bio/singa/sequence/model/NucleotideSequence.html" title="class in bio.singa.sequence.model">NucleotideSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/sequence/model/ProteinSequence.html#availableFeatures">availableFeatures</a></span> - Static variable in class bio.singa.sequence.model.<a href="../bio/singa/sequence/model/ProteinSequence.html" title="class in bio.singa.sequence.model">ProteinSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/ComplexEntity.html#availableFeatures">availableFeatures</a></span> - Static variable in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/ComplexEntity.html" title="class in bio.singa.simulation.entities">ComplexEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/entities/SimpleEntity.html#availableFeatures">availableFeatures</a></span> - Static variable in class bio.singa.simulation.entities.<a href="../bio/singa/simulation/entities/SimpleEntity.html" title="class in bio.singa.simulation.entities">SimpleEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html#availableFeatures">availableFeatures</a></span> - Static variable in class bio.singa.simulation.model.agents.pointlike.<a href="../bio/singa/simulation/model/agents/pointlike/Vesicle.html" title="class in bio.singa.simulation.model.agents.pointlike">Vesicle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="memberNameLink"><a href="../bio/singa/features/quantities/NaturalConstants.html#AVOGADRO_CONSTANT">AVOGADRO_CONSTANT</a></span> - Static variable in class bio.singa.features.quantities.<a href="../bio/singa/features/quantities/NaturalConstants.html" title="class in bio.singa.features.quantities">NaturalConstants</a></dt>
<dd>
<div class="block">The avogadro constant is the number of constituent particles, usually atoms or molecules, that are contained in
 the amount of substance given by one mole.</div>
</dd>
</dl>
<a href="index-1.html">A</a>&nbsp;<a href="index-2.html">B</a>&nbsp;<a href="index-3.html">C</a>&nbsp;<a href="index-4.html">D</a>&nbsp;<a href="index-5.html">E</a>&nbsp;<a href="index-6.html">F</a>&nbsp;<a href="index-7.html">G</a>&nbsp;<a href="index-8.html">H</a>&nbsp;<a href="index-9.html">I</a>&nbsp;<a href="index-10.html">J</a>&nbsp;<a href="index-11.html">K</a>&nbsp;<a href="index-12.html">L</a>&nbsp;<a href="index-13.html">M</a>&nbsp;<a href="index-14.html">N</a>&nbsp;<a href="index-15.html">O</a>&nbsp;<a href="index-16.html">P</a>&nbsp;<a href="index-17.html">Q</a>&nbsp;<a href="index-18.html">R</a>&nbsp;<a href="index-19.html">S</a>&nbsp;<a href="index-20.html">T</a>&nbsp;<a href="index-21.html">U</a>&nbsp;<a href="index-22.html">V</a>&nbsp;<a href="index-23.html">W</a>&nbsp;<a href="index-24.html">X</a>&nbsp;<a href="index-25.html">Y</a>&nbsp;<a href="index-26.html">Z</a>&nbsp;<br><a href="../allclasses-index.html">All&nbsp;Classes</a>&nbsp;<a href="../allpackages-index.html">All&nbsp;Packages</a></div>
</main>
<footer role="contentinfo">
<nav role="navigation">
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a id="navbar.bottom">
<!--   -->
</a>
<div class="skipNav"><a href="#skip.navbar.bottom" title="Skip navigation links">Skip navigation links</a></div>
<a id="navbar.bottom.firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../index.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li><a href="../overview-tree.html">Tree</a></li>
<li><a href="../deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../allclasses.html">All&nbsp;Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
</div>
<a id="skip.navbar.bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</nav>
</footer>
</body>
</html>
